<!DOCTYPE html><html lang="en" class="bg-surface-base antialiased"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1, interactive-widget=resizes-content"/><link rel="stylesheet" href="/_next/static/css/62c4caba71dfda84.css" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/eb3d87f98fe1565f.css" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/1b5e561215938d4d.css" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/0227d069a630d414.css" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/f87fff2ab93d05a7.css" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" data-precedence="next"/><link rel="preload" as="script" fetchPriority="low" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" href="/_next/static/chunks/webpack-e121ed42680f327e.js"/><script src="/_next/static/chunks/78a669d9-e7993486b22c4915.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/40c4a5a7-abdabb07419d1cfc.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/2230-f7c87dc9fa57c408.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/main-app-536d4b8fca62396a.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/63f0ec43-8ee6a76d70472249.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/7670-7326298c9856172b.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/282-691fffb366e24ca5.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/7515-336759ef5dad2b52.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/927-34e2a6da3e15e26b.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/5855-243eeca8f0e4cabd.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/4393-1c8dff25ec904469.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/2225-454cf9f44775e7ce.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/7618-43536a8fb0dd3bfe.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/7720-3a1e7e411adba2d3.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/7086-1f1c4891d766e04f.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/global-error-4d07d20223cd4b4c.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/8208b75a-a48e5a4507a0de91.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/5497-ab85ef3ea59dd353.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/6281-c6e84786dade3614.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/2660-d983a7f287e89f18.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/5002-9735c25beda556ba.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/3655-6717081f7512305a.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/8945-e4e3cf487f5e27c7.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/9214-a10614844a67ba31.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/4789-2080f6497f78b5b6.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/layout-226f2bd71acf9f9e.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/24cf1b50-159cca90b09285e0.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/9ffa21ba-c069766d809bd4c7.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/600-9a9fcafaaa6c1c4c.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/5448-2a3cfd853d899c9a.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/8-8ca70ca619d8e099.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/error-4a29e9399afba038.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><script src="/_next/static/chunks/app/not-found-dd95690acf732f18.js" async="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script><meta name="next-size-adjust" content=""/><title>Grokipedia</title><meta name="description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><link rel="icon" type="image/x-icon" href="/favicon.ico" sizes="48x48"/><link rel="icon" href="/images/icon-dark.png" media="(prefers-color-scheme: light)" type="image/png"/><link rel="icon" href="/images/icon-light.png" media="(prefers-color-scheme: dark)" type="image/png"/><link rel="apple-touch-icon" href="/icon-192x192.png"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><meta property="og:title" content="Grokipedia"/><meta property="og:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta property="og:url" content="https://grokipedia.com"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en_US"/><meta property="og:type" content="website"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Grokipedia"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Grokipedia"/><meta name="twitter:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><title>Phred quality score</title><meta name="description" content="A Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the computer program Phred to help in the automation of DNA sequencing in the Human Genome Project. Phred quality scores are assigned to each nucleotide base call in automated sequencer traces.[1][2] The FASTQ format encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely..."/><meta name="author" content="system"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="keywords" content="Phred score, phred score"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><link rel="canonical" href="https://grokipedia.com/page/Phred_quality_score"/><meta property="og:title" content="Phred quality score"/><meta property="og:description" content="A Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the computer program Phred to help in the automation of DNA sequencing in the Human Genome Project. Phred quality scores are assigned to each nucleotide base call in automated sequencer traces.[1][2] The FASTQ format encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely..."/><meta property="og:url" content="https://grokipedia.com/page/Phred_quality_score"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Phred quality score"/><meta property="og:type" content="article"/><meta property="article:modified_time" content="1970-01-21T09:19:13.076Z"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Phred quality score"/><meta name="twitter:description" content="A Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the computer program Phred to help in the automation of DNA sequencing in the Human Genome Project. Phred quality scores are assigned to each nucleotide base call in automated sequencer traces.[1][2] The FASTQ format encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely..."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><link rel="icon" href="/favicon.ico" type="image/x-icon" sizes="16x16"/><meta name="sentry-trace" content="841ed3de2cec589eaa0efa059d284c37-c5690aad799f3e8d-1"/><meta name="baggage" content="sentry-environment=production,sentry-public_key=5f2258f71198ee26a355127af230c3a6,sentry-trace_id=841ed3de2cec589eaa0efa059d284c37,sentry-org_id=4508179396558848,sentry-transaction=GET%20%2Fpage%2F%5Bslug%5D,sentry-sampled=true,sentry-sample_rand=0.599537692203518,sentry-sample_rate=1"/><script src="/_next/static/chunks/polyfills-42372ed130431b0a.js" noModule="" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0"></script></head><body class="flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b"><div hidden=""><!--$--><!--/$--></div><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">((a,b,c,d,e,f,g,h)=>{let i=document.documentElement,j=["light","dark"];function k(b){var c;(Array.isArray(a)?a:[a]).forEach(a=>{let c="class"===a,d=c&&f?e.map(a=>f[a]||a):e;c?(i.classList.remove(...d),i.classList.add(f&&f[b]?f[b]:b)):i.setAttribute(a,b)}),c=b,h&&j.includes(c)&&(i.style.colorScheme=c)}if(d)k(d);else try{let a=localStorage.getItem(b)||c,d=g&&"system"===a?window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light":a;k(d)}catch(a){}})("class","theme","dark",null,["light","dark"],null,true,true)</script><header class="h-16 bg-surface-base fixed top-0 z-50 w-full"><div class="hidden h-full grid-cols-[1fr_3fr_1fr] items-center gap-4 !py-0 md:grid max-w-full py-6 px-4 w-full mx-auto"><div class="-ml-2 justify-self-start"><a class="cursor-pointer" href="/"><svg width="200" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="mx-auto flex w-full"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full px-3 shadow-none sm:w-80 focus-visible:z-1 mx-auto h-full w-full rounded-full bg-surface-l1" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>âŒ˜</span><span class="text-xs">K</span></div></span></div></button></div><div class="flex items-center gap-2 justify-self-end"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0" type="button" id="radix-_R_7apfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button></div></div><div class="flex h-full items-center md:hidden justify-between max-w-full py-6 px-4 w-full mx-auto pt-8"><div class="flex items-center"><div class="-ml-2"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button" aria-label="Toggle table of contents"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-list h-6 w-6"><path d="M3 12h.01"></path><path d="M3 18h.01"></path><path d="M3 6h.01"></path><path d="M8 12h13"></path><path d="M8 18h13"></path><path d="M8 6h13"></path></svg></button></div><a class="cursor-pointer" href="/"><svg width="160" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="flex items-center gap-1"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0 [&amp;_svg]:!w-5" type="button" id="radix-_R_aipfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search h-5 w-5"><circle cx="11" cy="11" r="8"></circle><path d="m21 21-4.3-4.3"></path></svg></button></div></div></header><div class="bg-surface-base fixed right-0 top-16 z-50 h-[calc(100vh-4rem)] w-full transform border-l transition-transform duration-300 ease-in-out sm:w-1/2 md:hidden translate-x-full"><div class="flex flex-col gap-4 p-4"><div class="flex flex-row gap-2"><button class="focus-visible:ring-ring inline-flex items-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full shadow-none sm:w-80 h-10 w-full justify-start rounded-full bg-surface-l1 px-4" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>âŒ˜</span><span class="text-xs">K</span></div></span></div></button></div></div></div><div><div class="min-[1350px]:grid min-[1350px]:grid-cols-[1fr_3fr_1fr]"><nav class="bg-surface-base hidden h-[calc(100vh-4rem)] overflow-y-auto px-6 pb-32 min-[1350px]:sticky min-[1350px]:top-16 min-[1350px]:block scrollbar-none [-ms-overflow-style:none] [scrollbar-width:none] [&amp;::-webkit-scrollbar]:hidden [overscroll-behavior:contain] pt-8"><ul class="space-y-2 text-sm"><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#phred-quality-score" class="transition-opacity hover:opacity-100 opacity-50">Phred quality score</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#definition" class="transition-opacity hover:opacity-100 opacity-50">Definition</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#history" class="transition-opacity hover:opacity-100 opacity-50">History</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#methods" class="transition-opacity hover:opacity-100 opacity-50">Methods</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#applications" class="transition-opacity hover:opacity-100 opacity-50">Applications</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#compression" class="transition-opacity hover:opacity-100 opacity-50">Compression</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#symbols" class="transition-opacity hover:opacity-100 opacity-50">Symbols</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#references" class="transition-opacity hover:opacity-100 opacity-50">References</a></li></ul></nav><div class="relative top-16 pb-32 pt-8 px-4 md:px-8"><div class="mx-auto max-w-[850px]"><button data-state="closed" type="button" class="flex items-center"><div class="text-fg-tertiary mb-2 flex cursor-help items-center gap-2 text-sm"><span class="inline-flex items-center justify-center p-0 m-0" data-namespace="@xai/icons" data-slot="icon" style="height:16px;width:16px"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 33 33" aria-hidden="true" class="" focusable="false" style="fill:currentColor;height:16px;width:16px"><path fill="currentColor" d="m13.237 21.04 11.082-8.19c.543-.4 1.32-.244 1.578.38 1.363 3.288.754 7.241-1.957 9.955-2.71 2.714-6.482 3.31-9.93 1.954l-3.765 1.745c5.401 3.697 11.96 2.782 16.059-1.324 3.251-3.255 4.258-7.692 3.317-11.693l.008.009c-1.365-5.878.336-8.227 3.82-13.031q.123-.17.247-.345l-4.585 4.59v-.014L13.234 21.044M10.95 23.031c-3.877-3.707-3.208-9.446.1-12.755 2.446-2.449 6.454-3.448 9.952-1.979L24.76 6.56c-.677-.49-1.545-1.017-2.54-1.387A12.465 12.465 0 0 0 8.675 7.901c-3.519 3.523-4.625 8.94-2.725 13.561 1.42 3.454-.907 5.898-3.251 8.364-.83.874-1.664 1.749-2.335 2.674l10.583-9.466"></path></svg></span><span>Fact-checked by Grok<!-- --> <!-- -->4 days ago</span></div></button><article class="text-[16px]"><h1 id="phred-quality-score" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Phred quality score<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">A <strong>Phred quality score</strong> is a measure of the quality of the identification of the <span class="inline text-[1em] leading-7">nucleobases</span> generated by automated <span class="inline text-[1em] leading-7">DNA sequencing</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_40qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup> It was originally developed for the computer program <span class="inline text-[1em] leading-7">Phred</span> to help in the automation of DNA sequencing in the <span class="inline text-[1em] leading-7">Human Genome Project</span>. Phred quality scores are assigned to each <span class="inline text-[1em] leading-7">nucleotide</span> base call in automated sequencer traces.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_8gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_90qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup> The <span class="inline text-[1em] leading-7">FASTQ format</span> encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely accepted to characterize the quality of DNA sequences, and can be used to compare the efficacy of different sequencing methods. Perhaps the most important use of Phred quality scores is the automatic determination of accurate, quality-based <span class="inline text-[1em] leading-7">consensus sequences</span>.</span>
<h2 id="definition" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Definition<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">Phred quality scores <span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>Q</mi></mrow><annotation encoding="application/x-tex">Q</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8778em;vertical-align:-0.1944em"></span><span class="mord mathnormal">Q</span></span></span></span> are logarithmically related to the base-calling error probabilities <span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>P</mi></mrow><annotation encoding="application/x-tex">P</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.13889em">P</span></span></span></span> and defined as <sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_31qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup></span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>Q</mi><mo>=</mo><mo>âˆ’</mo><mn>10</mn><msub><mrow><mi>log</mi><mo>â¡</mo></mrow><mn>10</mn></msub><mi>P</mi></mrow><annotation encoding="application/x-tex">Q = -10 \log_{10} P</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8778em;vertical-align:-0.1944em"></span><span class="mord mathnormal">Q</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:0.9386em;vertical-align:-0.2441em"></span><span class="mord">âˆ’</span><span class="mord">10</span><span class="mspace" style="margin-right:0.1667em"></span><span class="mop"><span class="mop">lo<span style="margin-right:0.01389em">g</span></span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.207em"><span style="top:-2.4559em;margin-right:0.05em"><span class="pstrut" style="height:2.7em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">10</span></span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.2441em"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.1667em"></span><span class="mord mathnormal" style="margin-right:0.13889em">P</span></span></span></span></span>
<span class="mb-4 block break-words text-[1em] leading-7">This relation can also be written as</span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>P</mi><mo>=</mo><mn>1</mn><msup><mn>0</mn><mrow><mo>âˆ’</mo><mfrac><mi>Q</mi><mn>10</mn></mfrac></mrow></msup></mrow><annotation encoding="application/x-tex">P = 10^{-\frac{Q}{10}}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.13889em">P</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:1.0783em"></span><span class="mord">1</span><span class="mord"><span class="mord">0</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:1.0783em"><span style="top:-3.413em;margin-right:0.05em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">âˆ’</span><span class="mord mtight"><span class="mopen nulldelimiter sizing reset-size3 size6"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.9505em"><span style="top:-2.656em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">10</span></span></span></span><span style="top:-3.2255em"><span class="pstrut" style="height:3em"></span><span class="frac-line mtight" style="border-bottom-width:0.049em"></span></span><span style="top:-3.4624em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mathnormal mtight">Q</span></span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.344em"><span></span></span></span></span></span><span class="mclose nulldelimiter sizing reset-size3 size6"></span></span></span></span></span></span></span></span></span></span></span></span></span></span>
<span class="mb-4 block break-words text-[1em] leading-7">For example, if Phred assigns a quality score of 30 to a base, the chances that this base is called incorrectly are 1 in 1000.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Phred quality scores are logarithmically linked to error probabilities.</span>








































<div class="border-border-l1 overflow-x-auto rounded-lg border my-6 w-full"><table node="[object Object]" class="w-full border-collapse text-sm"><thead node="[object Object]" class="border-border-l1 border-b bg-surface-l1"><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Phred Quality Score</span></th><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Probability of incorrect base call</span></th><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Base call accuracy</span></th></tr></thead><tbody node="[object Object]" class="divide-border-l1 divide-y"><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">10</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1 in 10</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">90%</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">20</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1 in 100</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">99%</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">30</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1 in 1000</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">99.9%</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">40</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1 in 10,000</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">99.99%</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">50</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1 in 100,000</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">99.999%</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">60</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1 in 1,000,000</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">99.9999%</span></td></tr></tbody></table></div>
<h2 id="history" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">History<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">The idea of sequence quality scores can be traced back to the original description of the SCF file format by Rodger Staden&#x27;s group in 1992.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_15qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup> In 1995, Bonfield and Staden proposed a method to use base-specific quality scores to improve the accuracy of consensus sequences in DNA sequencing projects.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_25qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">However, early attempts to develop base-specific quality scores<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_16abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[5]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_1mabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[6]</sup> had only limited success.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The first program to develop accurate and powerful base-specific quality scores was the program <span class="inline text-[1em] leading-7">Phred</span>. Phred was able to calculate highly accurate quality scores that were logarithmically linked to the error probabilities. Phred was quickly adopted by all the major genome sequencing centers as well as many other laboratories; the vast majority of the DNA sequences produced during the <span class="inline text-[1em] leading-7">Human Genome Project</span> were processed with Phred.</span>
<span class="mb-4 block break-words text-[1em] leading-7">After Phred quality scores became the required standard in DNA sequencing, other manufacturers of DNA sequencing instruments, including <span class="inline text-[1em] leading-7">Li-Cor</span> and <span class="inline text-[1em] leading-7">ABI</span>, developed similar quality scoring metrics for their base calling software.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_37abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup></span>
<h2 id="methods" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Methods<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">Phred&#x27;s approach to base calling and calculating quality scores was outlined by Ewing <em>et al.</em>. To determine quality scores, Phred first calculates several parameters related to peak shape and peak resolution at each base. Phred then uses these parameters to look up a corresponding quality score in huge lookup tables. These lookup tables were generated from sequence traces where the correct sequence was known, and are hard coded in Phred; different lookup tables are used for different sequencing chemistries and machines. An evaluation of the accuracy of Phred quality scores for a number of variations in sequencing chemistry and instrumentation showed that Phred quality scores are highly accurate.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_28abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[8]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Phred was originally developed for &quot;slab gel&quot; sequencing machines like the ABI373. When originally developed, Phred had a lower base calling error rate than the manufacturer&#x27;s base calling software, which also did not provide quality scores. However, Phred was only partially adapted to the capillary DNA sequencers that became popular later. In contrast, instrument manufacturers like ABI continued to adapt their base calling software changes in sequencing chemistry, and have included the ability to create Phred-like quality scores. Therefore, the need to use Phred for base calling of DNA sequencing traces has diminished, and using the manufacturer&#x27;s current software versions can often give more accurate results.</span>
<h2 id="applications" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Applications<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">Phred quality scores are used for assessment of sequence quality, recognition and removal of low-quality sequence (end clipping), and determination of accurate consensus sequences.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Originally, Phred quality scores were primarily used by the sequence assembly program <span class="inline text-[1em] leading-7">Phrap</span>. Phrap was routinely used in some of the largest sequencing projects in the <span class="inline text-[1em] leading-7">Human Genome Project</span> and continues to be used in the biotech industry for certain assembly tasks, particularly with Sanger sequencing data. Phrap uses Phred quality scores to determine highly accurate consensus sequences and to estimate the quality of the consensus sequences. Phrap also uses Phred quality scores to estimate whether discrepancies between two overlapping sequences are more likely to arise from random errors, or from different copies of a repeated sequence.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Within the <span class="inline text-[1em] leading-7">Human Genome Project</span>, the most important use of Phred quality scores was for automatic determination of consensus sequences. Before Phred and Phrap, scientists had to carefully look at discrepancies between overlapping DNA fragments; often, this involved manual determination of the highest-quality sequence, and manual editing of any errors. Phrap&#x27;s use of Phred quality scores effectively automated finding the highest-quality consensus sequence; in most cases, this completely circumvents the need for any manual editing. As a result, the estimated error rate in assemblies that were created automatically with Phred and Phrap is typically substantially lower than the error rate of manually edited sequence.</span>
<span class="mb-4 block break-words text-[1em] leading-7">As of 2025, Phred quality scores continue to be widely utilized in bioinformatics software for various tasks. Programs like <span class="inline text-[1em] leading-7">Sequencher</span> use quality scores for display, end clipping, and consensus determination; other programs like <span class="inline text-[1em] leading-7">CodonCode Aligner</span> also implement quality-based consensus methods. In next-generation sequencing (NGS) workflows, Phred scoresâ€”encoded in the FASTQ formatâ€”are essential for quality control, including read trimming and filtering in tools such as FastQC, and setting quality thresholds in alignment (e.g., BWA) and variant calling pipelines (e.g., GATK).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3babav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[9]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3rabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[10]</sup></span>
<h2 id="compression" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Compression<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">Quality scores are normally stored together with the nucleotide sequence in the widely accepted <span class="inline text-[1em] leading-7">FASTQ format</span>. They account for about half of the required disk space in the FASTQ format (before compression), and therefore the compression of the quality values can significantly reduce storage requirements and speed up analysis and transmission of sequencing data. Both <span class="inline text-[1em] leading-7">lossless</span> and <span class="inline text-[1em] leading-7">lossy compression</span> have been explored in the literature. For example, the algorithm QualComp<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4cabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[11]</sup> performs lossy compression with a rate (number of bits per quality value) specified by the user. Based on rate-distortion theory results, it allocates the number of bits so as to minimize the MSE (mean squared error) between the original (uncompressed) and the reconstructed (after compression) quality values. Other algorithms for compression of quality values include the lossless SCALCE,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5cabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[12]</sup> Fastqz,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6cabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup> the lossy QVZ,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_7cabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[14]</sup> the adaptive AQUa,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_8cabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[15]</sup> and MPEG-G (ISO/IEC 23092), an international standard published starting in 2020 with ongoing amendments that supports both lossless and lossy modes. For example, SCALCE reduces the alphabet size based on the observation that â€œneighboringâ€ quality values are similar in general. More recent developments include PQSDC (2024), a parallel lossless compressor for quality scores,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_9cabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup> and GeneSqueeze (2025), a lossless reference-free compressor for FASTQ files.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_acabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[17]</sup></span>
<h1 id="symbols" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Symbols<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">In the standard FASTQ format, Phred quality scores are encoded using the Phred+33 scheme, where scores from 0 to 93 correspond to ASCII characters with codes 33 (!) to 126 (~). Higher scores beyond 40 are common in next-generation sequencing and follow the pattern: symbol = character with ASCII code = score + 33, and error probability P â‰ˆ 10^{-Q/10}. The following table shows the mapping for scores 0 to 40, including the approximate probability of incorrect base call.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_1dabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[18]</sup></span>























































































































































































































<div class="border-border-l1 overflow-x-auto rounded-lg border my-6 w-full"><table node="[object Object]" class="w-full border-collapse text-sm"><thead node="[object Object]" class="border-border-l1 border-b bg-surface-l1"><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Symbol</span></th><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Phred Quality Score</span></th><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Probability of Incorrect Base Call</span></th></tr></thead><tbody node="[object Object]" class="divide-border-l1 divide-y"><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">!</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1.000</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">&quot;</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.794</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">#</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">2</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.631</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">$</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">3</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.501</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">%</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">4</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.398</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">&amp;</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">5</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.316</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">&#x27;</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">6</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.251</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">(</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">7</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.199</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">)</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">8</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.158</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">*</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">9</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.126</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">+</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">10</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.100</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">,</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">11</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.079</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">-</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">12</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.063</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">.</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">13</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.050</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">/</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">14</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.040</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">15</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.032</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">1</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">16</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.025</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">2</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">17</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.020</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">3</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">18</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.016</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">4</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">19</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.013</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">5</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">20</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.010</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">6</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">21</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.008</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">7</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">22</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.006</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">8</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">23</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.005</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">9</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">24</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.004</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">:</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">25</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.003</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">;</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">26</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.002</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">&lt;</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">27</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.002</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">=</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">28</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.001</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">&gt;</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">29</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.001</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">?</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">30</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.001</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">@</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">31</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0008</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">A</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">32</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0006</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">B</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">33</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0005</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">C</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">34</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0004</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">D</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">35</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0003</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">E</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">36</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0002</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">F</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">37</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0002</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">G</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">38</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0002</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">H</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">39</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0001</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">I</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">40</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">0.0001</span></td></tr></tbody></table></div></article><div id="references" class="min-w-0 scroll-mt-8 overflow-hidden"><div class="text-[16px]"><h2 id="references" node="[object Object]" class="mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden">References</h2></div><ol class="columns-1 gap-x-12 [counter-reset:item] md:columns-2"><li id="1" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Ewing B; <span class="inline text-[1em] leading-7">Hillier L</span>; <span class="inline text-[1em] leading-7">Wendl MC</span>; <span class="inline text-[1em] leading-7">Green P.</span> (1998). <a href="https://doi.org/10.1101%2Fgr.8.3.175" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Base-calling of automated sequencer traces using phred. I. Accuracy assessment&quot;</a>. <em>Genome Research</em>. <strong>8</strong> (3): 175â€“185. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1101%2Fgr.8.3.175" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1101/gr.8.3.175</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/9521921" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">9521921</a>.</span></div></li><li id="2" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Ewing B, Green P (1998). <a href="https://doi.org/10.1101%2Fgr.8.3.186" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Base-calling of automated sequencer traces using phred. II. Error probabilities&quot;</a>. <em>Genome Research</em>. <strong>8</strong> (3): 186â€“194. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1101%2Fgr.8.3.186" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1101/gr.8.3.186</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/9521922" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">9521922</a>.</span></div></li><li id="3" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Dear S, Staden R (1992). &quot;A standard file format for data from DNA sequencing instruments&quot;. <em>DNA Sequence</em>. <strong>3</strong> (2): 107â€“110. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.3109%2F10425179209034003" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.3109/10425179209034003</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/1457811" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1457811</a>.</span></div></li><li id="4" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Bonfield JK, Staden R (25 Apr 1995). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The application of numerical estimates of base calling accuracy to DNA sequencing projects&quot;</a>. <em>Nucleic Acids Research</em>. <strong>23</strong> (8): 1406â€“1410. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fnar%2F23.8.1406" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/nar/23.8.1406</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">306869</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/7753633" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">7753633</a>.</span></div></li><li id="5" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Churchill GA, <span class="inline text-[1em] leading-7">Waterman MS</span> (Sep 1992). &quot;The accuracy of DNA sequences: estimating sequence quality&quot;. <em>Genomics</em>. <strong>14</strong> (1): 89â€“98. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2FS0888-7543%2805%2980288-5" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/S0888-7543(05)80288-5</a>. <span class="inline text-[1em] leading-7">hdl</span>:<a href="https://hdl.handle.net/1813%2F31678" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1813/31678</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/1358801" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1358801</a>.</span></div></li><li id="6" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Lawrence CB, Solovyev VV (1994). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Assignment of position-specific error probability to primary DNA sequence data&quot;</a>. <em>Nucleic Acids Research</em>. <strong>22</strong> (7): 1272â€“1280. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fnar%2F22.7.1272" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/nar/22.7.1272</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">523653</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/8165143" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8165143</a>.</span></div></li><li id="7" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="http://www3.appliedbiosystems.com/cms/groups/mcb_marketing/documents/generaldocuments/cms_040383.pdf" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Life Technologies - US&quot;</a> (PDF).</span></div></li><li id="8" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Richterich P (1998). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Estimation of errors in &quot;raw&quot; DNA sequences: a validation study&quot;</a>. <em>Genome Research</em>. <strong>8</strong> (3): 251â€“259. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1101%2Fgr.8.3.251" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1101/gr.8.3.251</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">310698</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/9521928" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">9521928</a>.</span></div></li><li id="https://www.illumina.com/science/genomics-research/articles/data-quality-q-scores.html" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.illumina.com/science/genomics-research/articles/data-quality-q-scores.html" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.illumina.com/science/genomics-research/articles/data-quality-q-scores.html</a></span></div></li><li id="https://medium.com/@xieshaojun0526/understanding-phred-scores-for-fastq-format-24fed667dd11" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://medium.com/@xieshaojun0526/understanding-phred-scores-for-fastq-format-24fed667dd11" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://medium.com/@xieshaojun0526/understanding-phred-scores-for-fastq-format-24fed667dd11</a></span></div></li><li id="9" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Ochoa, Idoia; Asnani, Himanshu; Bharadia, Dinesh; Chowdhury, Mainak; Weissman, Tsachy; Yona, Golan (2013). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Qual <em>Comp</em> : A new lossy compressor for quality scores based on rate distortion theory&quot;</a>. <em>BMC Bioinformatics</em>. <strong>14</strong> 187. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1186%2F1471-2105-14-187" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1186/1471-2105-14-187</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3698011</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/23758828" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">23758828</a>.</span></div></li><li id="10" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Hach, F; Numanagic, I; Alkan, C; Sahinalp, S. C. (2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;SCALCE: Boosting sequence compression algorithms using locally consistent encoding&quot;</a>. <em>Bioinformatics</em>. <strong>28</strong> (23): 3051â€“3057. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbts593" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/bioinformatics/bts593</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3509486</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/23047557" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">23047557</a>.</span></div></li><li id="11" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="http://mattmahoney.net/dc/fastqz" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;fastqz - FASTQ compressor&quot;</a>.</span></div></li><li id="12" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Malysa, Greg; Hernaez, Mikel; Ochoa, Idoia; Rao, Milind; Ganesan, Karthik; Weissman, Tsachy (2015-10-01). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;QVZ: lossy compression of quality values&quot;</a>. <em>Bioinformatics</em>. <strong>31</strong> (19): 3122â€“3129. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv330" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/bioinformatics/btv330</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1367-4803" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1367-4803</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">5856090</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/26026138" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">26026138</a>.</span></div></li><li id="13" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Paridaens, Tom; Van Wallendael, Glenn; De Neve, Wesley; Lambert, Peter (2018). <a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtx607" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;AQUa: an adaptive framework for compression of sequencing quality scores with random access functionality&quot;</a>. <em>Bioinformatics</em>. <strong>34</strong> (3): 425â€“433. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtx607" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/bioinformatics/btx607</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/29028894" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">29028894</a>.</span></div></li><li id="https://academic.oup.com/bioinformatics/article/40/5/btae323/7676123" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://academic.oup.com/bioinformatics/article/40/5/btae323/7676123" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://academic.oup.com/bioinformatics/article/40/5/btae323/7676123</a></span></div></li><li id="https://www.nature.com/articles/s41598-024-79258-6" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.nature.com/articles/s41598-024-79258-6" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.nature.com/articles/s41598-024-79258-6</a></span></div></li><li id="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/</a></span></div></li></ol></div><div class="text-fg-secondary text-xs [&amp;_*]:mb-2"><hr class="border-border-l1 my-8 rounded-sm border-t"/>
<span class="mb-4 block break-words text-[1em] leading-7">The content is adapted from <a href="https://en.wikipedia.org/wiki/Phred_quality_score" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">Wikipedia</a>, licensed under Creative Commons Attribution-ShareAlike 4.0 License.</span></div></div></div><div class="hidden min-[1400px]:block"></div></div></div><!--$--><!--/$--><div role="region" aria-label="Notifications (F8)" tabindex="-1" style="pointer-events:none"><ol tabindex="-1" class="pointer-events-none fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 max-sm:left-0 sm:bottom-0 sm:right-0 sm:top-auto sm:max-w-[420px] sm:flex-col pl-16 pr-2 pt-16 sm:px-4"></ol></div><script type="application/json" id="server-client-data-experimentation">{"status":"uninitialized"}</script><script src="/_next/static/chunks/webpack-e121ed42680f327e.js" nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0" id="_R_" async=""></script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">(self.__next_f=self.__next_f||[]).push([0])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"1:\"$Sreact.fragment\"\n3:I[91363,[],\"\"]\n4:I[23775,[],\"\"]\n5:I[57654,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"PageHeaderProvider\"]\n6:I[17618,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"HeaderContent\"]\n7:I[25529,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"6751\",\"static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js\"],\"default\"]\n9:I[6666,[],\"OutletBoundary\"]\nb:I[80415,[],\"AsyncMetadataOutlet\"]\nd:I[6666,[],\"ViewportBoundary\"]\nf:I[6666,[],\"MetadataBoundary\"]\n10:\"$Sreact.suspense\"\n12:I[95909,[\"4219\",\"static/chunks/app/global-error-4d07d20223cd4b4c.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"13:I[51498,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"ConstantsProvider\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"14:I[91073,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"15:I[38642,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"16:I[99648,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"17:I[78825,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"18:I[8550,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MixpanelProvider\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"19:I[12290,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MobileTocProvider\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"1a:I[53947,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"NuqsAdapter\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"1b:I[91873,[\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8039\",\"static/chunks/app/error-4a29e9399afba038.js\"],\"default\"]\n1c:I[5091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"4345\",\"static/chunks/app/not-found-dd95690acf732f18.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"1d:I[16091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,":HL[\"/_next/static/media/1f2316909698f815.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/3d4419af2cf8609b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/4dec29efcaeb336c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/74452ea3ef0f9101.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/904ef0a86fe32a00.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/d886a03bcda7ad8f.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/e1447589d6f59c4b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/f5a90156f8995c8c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/css/62c4caba71dfda84.css\",\"style\",{\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]\n:HL[\"/_next/static/css/eb3d87f98fe1565f.css\",\"style\",{\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]\n:HL[\"/_next/static/css/1b5e561215938d4d.css\",\"style\",{\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]\n:HL[\"/_next/static/css/0227d069a630d414.css\",\"style\",{\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]\n:HL[\"/_next/static/css/f87fff2ab93d05a7.css\",\"style\",{\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"0:{\"P\":null,\"b\":\"BpM29AX4fmgcUxZ_6t0le\",\"p\":\"\",\"c\":[\"\",\"page\",\"Phred_quality_score\"],\"i\":false,\"f\":[[[\"\",{\"children\":[\"page\",{\"children\":[[\"slug\",\"Phred_quality_score\",\"d\"],{\"children\":[\"__PAGE__\",{}]}]}]},\"$undefined\",\"$undefined\",true],[\"\",[\"$\",\"$1\",\"c\",{\"children\":[[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/62c4caba71dfda84.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}],[\"$\",\"link\",\"1\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/eb3d87f98fe1565f.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}],[\"$\",\"link\",\"2\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/1b5e561215938d4d.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}],[\"$\",\"link\",\"3\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/0227d069a630d414.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]],\"$L2\"]}],{\"children\":[\"page\",[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[[\"slug\",\"Phred_quality_score\",\"d\"],[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L5\",null,{\"children\":[[\"$\",\"$L6\",null,{\"maxWidth\":\"full\",\"mobileOptions\":{\"right\":{\"showFixedIssues\":true,\"showThemeSwitcher\":true,\"showSearch\":true,\"showTableOfContents\":true}}}],[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L7\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}]]}],{\"children\":[\"__PAGE__\",[\"$\",\"$1\",\"c\",{\"children\":[\"$L8\",[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/f87fff2ab93d05a7.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\"}]],[\"$\",\"$L9\",null,{\"children\":[\"$La\",[\"$\",\"$Lb\",null,{\"promise\":\"$@c\"}]]}]]}],{},null,false]},null,false]},null,false]},null,false],[\"$\",\"$1\",\"h\",{\"children\":[null,[[\"$\",\"$Ld\",null,{\"children\":\"$Le\"}],[\"$\",\"meta\",null,{\"name\":\"next-size-adjust\",\"content\":\"\"}]],[\"$\",\"$Lf\",null,{\"children\":[\"$\",\"div\",null,{\"hidden\":true,\"children\":[\"$\",\"$10\",null,{\"fallback\":null,\"children\":\"$L11\"}]}]}]]}],false]],\"m\":\"$undefined\",\"G\":[\"$12\",[]],\"s\":false,\"S\":false}\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"2:[\"$\",\"html\",null,{\"lang\":\"en\",\"className\":\"bg-surface-base antialiased\",\"suppressHydrationWarning\":true,\"children\":[[\"$\",\"head\",null,{\"children\":[[\"$\",\"title\",null,{\"children\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"type\":\"image/x-icon\",\"href\":\"/favicon.ico\",\"sizes\":\"48x48\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-dark.png\",\"media\":\"(prefers-color-scheme: light)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-light.png\",\"media\":\"(prefers-color-scheme: dark)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"apple-touch-icon\",\"href\":\"/icon-192x192.png\"}],[\"$\",\"link\",null,{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\"}],[\"$\",\"meta\",null,{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",null,{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"meta\",null,{\"property\":\"og:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"property\":\"og:url\",\"content\":\"https://grokipedia.com\"}],[\"$\",\"meta\",null,{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:locale\",\"content\":\"en_US\"}],[\"$\",\"meta\",null,{\"property\":\"og:type\",\"content\":\"website\"}],[\"$\",\"meta\",null,{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:alt\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}]]}],[\"$\",\"body\",null,{\"className\":\"flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b\",\"children\":[[\"$\",\"$L13\",null,{\"children\":[\"$\",\"$L14\",null,{\"children\":[\"$\",\"$L15\",null,{\"defaultTheme\":\"dark\",\"nonce\":\"NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0\",\"children\":[\"$\",\"$L16\",null,{\"children\":[\"$\",\"$L17\",null,{\"children\":[\"$\",\"$L18\",null,{\"children\":[\"$\",\"$L19\",null,{\"children\":[\"$\",\"$L1a\",null,{\"children\":[[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$1b\",\"errorStyles\":[],\"errorScripts\":[],\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L1c\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}],[\"$\",\"$L1d\",null,{}]]}]}]}]}]}]}]}]}],[\"$\",\"script\",null,{\"type\":\"application/json\",\"id\":\"server-client-data-experimentation\",\"children\":\"{\\\"status\\\":\\\"uninitialized\\\"}\"}]]}]]}]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"e:[[\"$\",\"meta\",\"0\",{\"charSet\":\"utf-8\"}],[\"$\",\"meta\",\"1\",{\"name\":\"viewport\",\"content\":\"width=device-width, initial-scale=1, interactive-widget=resizes-content\"}]]\na:null\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"1e:I[42712,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"HydrationBoundary\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"21:I[6367,[],\"IconMark\"]\n1f:T3196,"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"# Phred quality score\n\nA **Phred quality score** is a measure of the quality of the identification of the [nucleobases](Nucleobase \"Nucleobase\") generated by automated [DNA sequencing](DNA_sequencing \"DNA sequencing\").[1][2] It was originally developed for the computer program [Phred](Phred_\\(software\\) \"Phred \\(software\\)\") to help in the automation of DNA sequencing in the [Human Genome Project](Human_Genome_Project \"Human Genome Project\"). Phred quality scores are assigned to each [nucleotide](Nucleotide \"Nucleotide\") base call in automated sequencer traces.[1][2] The [FASTQ format](FASTQ_format \"FASTQ format\") encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely accepted to characterize the quality of DNA sequences, and can be used to compare the efficacy of different sequencing methods. Perhaps the most important use of Phred quality scores is the automatic determination of accurate, quality-based [consensus sequences](Consensus_sequence \"Consensus sequence\").\n# Phred quality score\n\n## Definition\n\nPhred quality scores \\( Q \\) are logarithmically related to the base-calling error probabilities \\( P \\) and defined as [2]\n\n\\[\nQ = -10 \\log_{10} P\n\\]\n\nThis relation can also be written as \n\n\\[\nP = 10^{-\\frac{Q}{10}}\n\\]\n\nFor example, if Phred assigns a quality score of 30 to a base, the chances that this base is called incorrectly are 1 in 1000. \n\nPhred quality scores are logarithmically linked to error probabilities.\n\n| Phred Quality Score | Probability of incorrect base call | Base call accuracy |\n|---------------------|------------------------------------|--------------------|\n| 10                  | 1 in 10                            | 90%                |\n| 20                  | 1 in 100                           | 99%                |\n| 30                  | 1 in 1000                          | 99.9%              |\n| 40                  | 1 in 10,000                        | 99.99%             |\n| 50                  | 1 in 100,000                       | 99.999%            |\n| 60                  | 1 in 1,000,000                     | 99.9999%           |\n## History\n\nThe idea of sequence quality scores can be traced back to the original description of the SCF file format by Rodger Staden's group in 1992.[3] In 1995, Bonfield and Staden proposed a method to use base-specific quality scores to improve the accuracy of consensus sequences in DNA sequencing projects.[4]\n\nHowever, early attempts to develop base-specific quality scores[5][6] had only limited success. \n\nThe first program to develop accurate and powerful base-specific quality scores was the program [Phred](Phred_base_calling \"Phred base calling\"). Phred was able to calculate highly accurate quality scores that were logarithmically linked to the error probabilities. Phred was quickly adopted by all the major genome sequencing centers as well as many other laboratories; the vast majority of the DNA sequences produced during the [Human Genome Project](Human_Genome_Project \"Human Genome Project\") were processed with Phred. \n\nAfter Phred quality scores became the required standard in DNA sequencing, other manufacturers of DNA sequencing instruments, including [Li-Cor](LI-COR_Biosciences \"LI-COR Biosciences\") and [ABI](Applied_Biosystems \"Applied Biosystems\"), developed similar quality scoring metrics for their base calling software.[7]\n## Methods\n\nPhred's approach to base calling and calculating quality scores was outlined by Ewing *et al.*. To determine quality scores, Phred first calculates several parameters related to peak shape and peak resolution at each base. Phred then uses these parameters to look up a corresponding quality score in huge lookup tables. These lookup tables were generated from sequence traces where the correct sequence was known, and are hard coded in Phred; different lookup tables are used for different sequencing chemistries and machines. An evaluation of the accuracy of Phred quality scores for a number of variations in sequencing chemistry and instrumentation showed that Phred quality scores are highly accurate.[8]\n\nPhred was originally developed for \"slab gel\" sequencing machines like the ABI373. When originally developed, Phred had a lower base calling error rate than the manufacturer's base calling software, which also did not provide quality scores. However, Phred was only partially adapted to the capillary DNA sequencers that became popular later. In contrast, instrument manufacturers like ABI continued to adapt their base calling software changes in sequencing chemistry, and have included the ability to create Phred-like quality scores. Therefore, the need to use Phred for base calling of DNA sequencing traces has diminished, and using the manufacturer's current software versions can often give more accurate results.\n## Applications\n\nPhred quality scores are used for assessment of sequence quality, recognition and removal of low-quality sequence (end clipping), and determination of accurate consensus sequences. \n\nOriginally, Phred quality scores were primarily used by the sequence assembly program [Phrap](https://en.wikipedia.org/wiki/Phrap). Phrap was routinely used in some of the largest sequencing projects in the [Human Genome Project](https://en.wikipedia.org/wiki/Human_Genome_Project) and continues to be used in the biotech industry for certain assembly tasks, particularly with Sanger sequencing data. Phrap uses Phred quality scores to determine highly accurate consensus sequences and to estimate the quality of the consensus sequences. Phrap also uses Phred quality scores to estimate whether discrepancies between two overlapping sequences are more likely to arise from random errors, or from different copies of a repeated sequence. \n\nWithin the [Human Genome Project](https://en.wikipedia.org/wiki/Human_Genome_Project), the most important use of Phred quality scores was for automatic determination of consensus sequences. Before Phred and Phrap, scientists had to carefully look at discrepancies between overlapping DNA fragments; often, this involved manual determination of the highest-quality sequence, and manual editing of any errors. Phrap's use of Phred quality scores effectively automated finding the highest-quality consensus sequence; in most cases, this completely circumvents the need for any manual editing. As a result, the estimated error rate in assemblies that were created automatically with Phred and Phrap is typically substantially lower than the error rate of manually edited sequence. \n\nAs of 2025, Phred quality scores continue to be widely utilized in bioinformatics software for various tasks. Programs like [Sequencher](https://en.wikipedia.org/wiki/Gene_Codes_Corporation#Sequencher) use quality scores for display, end clipping, and consensus determination; other programs like [CodonCode Aligner](https://en.wikipedia.org/wiki/CodonCode_Aligner) also implement quality-based consensus methods. In next-generation sequencing (NGS) workflows, Phred scoresâ€”encoded in the FASTQ formatâ€”are essential for quality control, including read trimming and filtering in tools such as FastQC, and setting quality thresholds in alignment (e.g., BWA) and variant calling pipelines (e.g., GATK).[](https://www.illumina.com/science/genomics-research/articles/data-quality-q-scores.html)[](https://medium.com/@xieshaojun0526/understanding-phred-scores-for-fastq-format-24fed667dd11)\n## Compression\n\nQuality scores are normally stored together with the nucleotide sequence in the widely accepted [FASTQ format](https://en.wikipedia.org/wiki/FASTQ_format). They account for about half of the required disk space in the FASTQ format (before compression), and therefore the compression of the quality values can significantly reduce storage requirements and speed up analysis and transmission of sequencing data. Both [lossless](https://en.wikipedia.org/wiki/Lossless_compression) and [lossy compression](https://en.wikipedia.org/wiki/Lossy_compression) have been explored in the literature. For example, the algorithm QualComp[9] performs lossy compression with a rate (number of bits per quality value) specified by the user. Based on rate-distortion theory results, it allocates the number of bits so as to minimize the MSE (mean squared error) between the original (uncompressed) and the reconstructed (after compression) quality values. Other algorithms for compression of quality values include the lossless SCALCE,[10] Fastqz,[11] the lossy QVZ,[12] the adaptive AQUa,[13] and MPEG-G (ISO/IEC 23092), an international standard published starting in 2020 with ongoing amendments that supports both lossless and lossy modes. For example, SCALCE reduces the alphabet size based on the observation that â€œneighboringâ€ quality values are similar in general. More recent developments include PQSDC (2024), a parallel lossless compressor for quality scores,[](https://academic.oup.com/bioinformatics/article/40/5/btae323/7676123) and GeneSqueeze (2025), a lossless reference-free compressor for FASTQ files.[](https://www.nature.com/articles/s41598-024-79258-6)\n# Symbols\n\nIn the standard FASTQ format, Phred quality scores are encoded using the Phred+33 scheme, where scores from 0 to 93 correspond to ASCII characters with codes 33 (!) to 126 (~). Higher scores beyond 40 are common in next-generation sequencing and follow the pattern: symbol = character with ASCII code = score + 33, and error probability P â‰ˆ 10^{-Q/10}. The following table shows the mapping for scores 0 to 40, including the approximate probability of incorrect base call.[](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/)\n\n| Symbol | Phred Quality Score | Probability of Incorrect Base Call |\n|--------|---------------------|------------------------------------|\n| !      | 0                   | 1.000                              |\n| \"      | 1                   | 0.794                              |\n| #      | 2                   | 0.631                              |\n| $      | 3                   | 0.501                              |\n| %      | 4                   | 0.398                              |\n| \u0026      | 5                   | 0.316                              |\n| '      | 6                   | 0.251                              |\n| (      | 7                   | 0.199                              |\n| )      | 8                   | 0.158                              |\n| *      | 9                   | 0.126                              |\n| +      | 10                  | 0.100                              |\n| ,      | 11                  | 0.079                              |\n| -      | 12                  | 0.063                              |\n| .      | 13                  | 0.050                              |\n| /      | 14                  | 0.040                              |\n| 0      | 15                  | 0.032                              |\n| 1      | 16                  | 0.025                              |\n| 2      | 17                  | 0.020                              |\n| 3      | 18                  | 0.016                              |\n| 4      | 19                  | 0.013                              |\n| 5      | 20                  | 0.010                              |\n| 6      | 21                  | 0.008                              |\n| 7      | 22                  | 0.006                              |\n| 8      | 23                  | 0.005                              |\n| 9      | 24                  | 0.004                              |\n| :      | 25                  | 0.003                              |\n| ;      | 26                  | 0.002                              |\n| \u003c      | 27                  | 0.002                              |\n| =      | 28                  | 0.001                              |\n| \u003e      | 29                  | 0.001                              |\n| ?      | 30                  | 0.001                              |\n| @      | 31                  | 0.0008                             |\n| A      | 32                  | 0.0006                             |\n| B      | 33                  | 0.0005                             |\n| C      | 34                  | 0.0004                             |\n| D      | 35                  | 0.0003                             |\n| E      | 36                  | 0.0002                             |\n| F      | 37                  | 0.0002                             |\n| G      | 38                  | 0.0002                             |\n| H      | 39                  | 0.0001                             |\n| I      | 40                  | 0.0001                             |"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"8:[\"$\",\"$L1e\",null,{\"state\":{\"mutations\":[],\"queries\":[{\"dehydratedAt\":1761883342213,\"state\":{\"data\":{\"page\":{\"citations\":[{\"id\":\"1\",\"title\":\"_**a**_ _**b**_ Ewing B; [Hillier L](Ladeana_Hillier \\\"Ladeana Hillier\\\"); [Wendl MC](Michael_Christopher_Wendl \\\"Michael Christopher Wendl\\\"); [Green P.](Philip_Palmer_Green \\\"Philip Palmer Green\\\") (1998). [\\\"Base-calling of automated sequencer traces using phred. I. Accuracy assessment\\\"](https://doi.org/10.1101%2Fgr.8.3.175). _Genome Research_. **8** (3): 175â€“185\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1101/gr.8.3.175](https://doi.org/10.1101%2Fgr.8.3.175). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9521921](https://pubmed.ncbi.nlm.nih.gov/9521921).\",\"description\":\"1. ^ _**a**_ _**b**_ Ewing B; [Hillier L](Ladeana_Hillier \\\"Ladeana Hillier\\\"); [Wendl MC](Michael_Christopher_Wendl \\\"Michael Christopher Wendl\\\"); [Green P.](Philip_Palmer_Green \\\"Philip Palmer Green\\\") (1998). [\\\"Base-calling of automated sequencer traces using phred. I. Accuracy assessment\\\"](https://doi.org/10.1101%2Fgr.8.3.175). _Genome Research_. **8** (3): 175â€“185\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1101/gr.8.3.175](https://doi.org/10.1101%2Fgr.8.3.175). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9521921](https://pubmed.ncbi.nlm.nih.gov/9521921).\",\"url\":\"Ladeana_Hillier \\\"Ladeana Hillier\\\"\",\"favicon\":\"\"},{\"id\":\"2\",\"title\":\"_**a**_ _**b**_ _**c**_ Ewing B, Green P (1998). [\\\"Base-calling of automated sequencer traces using phred. II. Error probabilities\\\"](https://doi.org/10.1101%2Fgr.8.3.186). _Genome Research_. **8** (3): 186â€“194\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1101/gr.8.3.186](https://doi.org/10.1101%2Fgr.8.3.186). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9521922](https://pubmed.ncbi.nlm.nih.gov/9521922).\",\"description\":\"2. ^ _**a**_ _**b**_ _**c**_ Ewing B, Green P (1998). [\\\"Base-calling of automated sequencer traces using phred. II. Error probabilities\\\"](https://doi.org/10.1101%2Fgr.8.3.186). _Genome Research_. **8** (3): 186â€“194\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1101/gr.8.3.186](https://doi.org/10.1101%2Fgr.8.3.186). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9521922](https://pubmed.ncbi.nlm.nih.gov/9521922).\",\"url\":\"https://doi.org/10.1101%2Fgr.8.3.186\",\"favicon\":\"\"},{\"id\":\"3\",\"title\":\"** Dear S, Staden R (1992). \\\"A standard file format for data from DNA sequencing instruments\\\". _DNA Sequence_. **3** (2): 107â€“110\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.3109/10425179209034003](https://doi.org/10.3109%2F10425179209034003). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1457811](https://pubmed.ncbi.nlm.nih.gov/1457811).\",\"description\":\"3. **^** Dear S, Staden R (1992). \\\"A standard file format for data from DNA sequencing instruments\\\". _DNA Sequence_. **3** (2): 107â€“110\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.3109/10425179209034003](https://doi.org/10.3109%2F10425179209034003). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1457811](https://pubmed.ncbi.nlm.nih.gov/1457811).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"4\",\"title\":\"** Bonfield JK, Staden R (25 Apr 1995). [\\\"The application of numerical estimates of base calling accuracy to DNA sequencing projects\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869). _Nucleic Acids Research_. **23** (8): 1406â€“1410\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/nar/23.8.1406](https://doi.org/10.1093%2Fnar%2F23.8.1406). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [306869](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [7753633](https://pubmed.ncbi.nlm.nih.gov/7753633).\",\"description\":\"4. **^** Bonfield JK, Staden R (25 Apr 1995). [\\\"The application of numerical estimates of base calling accuracy to DNA sequencing projects\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869). _Nucleic Acids Research_. **23** (8): 1406â€“1410\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/nar/23.8.1406](https://doi.org/10.1093%2Fnar%2F23.8.1406). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [306869](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [7753633](https://pubmed.ncbi.nlm.nih.gov/7753633).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC306869\",\"favicon\":\"\"},{\"id\":\"5\",\"title\":\"** Churchill GA, [Waterman MS](Michael_Waterman \\\"Michael Waterman\\\") (Sep 1992). \\\"The accuracy of DNA sequences: estimating sequence quality\\\". _Genomics_. **14** (1): 89â€“98\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0888-7543(05)80288-5](https://doi.org/10.1016%2FS0888-7543%2805%2980288-5). [hdl](Hdl_\\\\(identifier\\\\) \\\"Hdl \\\\(identifier\\\\)\\\"):[1813/31678](https://hdl.handle.net/1813%2F31678). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1358801](https://pubmed.ncbi.nlm.nih.gov/1358801).\",\"description\":\"5. **^** Churchill GA, [Waterman MS](Michael_Waterman \\\"Michael Waterman\\\") (Sep 1992). \\\"The accuracy of DNA sequences: estimating sequence quality\\\". _Genomics_. **14** (1): 89â€“98\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0888-7543(05)80288-5](https://doi.org/10.1016%2FS0888-7543%2805%2980288-5). [hdl](Hdl_\\\\(identifier\\\\) \\\"Hdl \\\\(identifier\\\\)\\\"):[1813/31678](https://hdl.handle.net/1813%2F31678). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1358801](https://pubmed.ncbi.nlm.nih.gov/1358801).\",\"url\":\"Michael_Waterman \\\"Michael Waterman\\\"\",\"favicon\":\"\"},{\"id\":\"6\",\"title\":\"** Lawrence CB, Solovyev VV (1994). [\\\"Assignment of position-specific error probability to primary DNA sequence data\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653). _Nucleic Acids Research_. **22** (7): 1272â€“1280\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/nar/22.7.1272](https://doi.org/10.1093%2Fnar%2F22.7.1272). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [523653](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8165143](https://pubmed.ncbi.nlm.nih.gov/8165143).\",\"description\":\"6. **^** Lawrence CB, Solovyev VV (1994). [\\\"Assignment of position-specific error probability to primary DNA sequence data\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653). _Nucleic Acids Research_. **22** (7): 1272â€“1280\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/nar/22.7.1272](https://doi.org/10.1093%2Fnar%2F22.7.1272). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [523653](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8165143](https://pubmed.ncbi.nlm.nih.gov/8165143).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523653\",\"favicon\":\"\"},{\"id\":\"7\",\"title\":\"** [\\\"Life Technologies - US\\\"](http://www3.appliedbiosystems.com/cms/groups/mcb_marketing/documents/generaldocuments/cms_040383.pdf) (PDF).\",\"description\":\"7. **^** [\\\"Life Technologies - US\\\"](http://www3.appliedbiosystems.com/cms/groups/mcb_marketing/documents/generaldocuments/cms_040383.pdf) (PDF).\",\"url\":\"http://www3.appliedbiosystems.com/cms/groups/mcb_marketing/documents/generaldocuments/cms_040383.pdf\",\"favicon\":\"\"},{\"id\":\"8\",\"title\":\"** Richterich P (1998). [\\\"Estimation of errors in \\\"raw\\\" DNA sequences: a validation study\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698). _Genome Research_. **8** (3): 251â€“259\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1101/gr.8.3.251](https://doi.org/10.1101%2Fgr.8.3.251). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [310698](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9521928](https://pubmed.ncbi.nlm.nih.gov/9521928).\",\"description\":\"8. **^** Richterich P (1998). [\\\"Estimation of errors in \\\"raw\\\" DNA sequences: a validation study\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698). _Genome Research_. **8** (3): 251â€“259\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1101/gr.8.3.251](https://doi.org/10.1101%2Fgr.8.3.251). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [310698](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9521928](https://pubmed.ncbi.nlm.nih.gov/9521928).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC310698\",\"favicon\":\"\"},{\"id\":\"9\",\"title\":\"** Ochoa, Idoia; Asnani, Himanshu; Bharadia, Dinesh; Chowdhury, Mainak; Weissman, Tsachy; Yona, Golan (2013). [\\\"Qual _Comp_ : A new lossy compressor for quality scores based on rate distortion theory\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011). _BMC Bioinformatics_. **14** 187\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/1471-2105-14-187](https://doi.org/10.1186%2F1471-2105-14-187). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3698011](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23758828](https://pubmed.ncbi.nlm.nih.gov/23758828).\",\"description\":\"9. **^** Ochoa, Idoia; Asnani, Himanshu; Bharadia, Dinesh; Chowdhury, Mainak; Weissman, Tsachy; Yona, Golan (2013). [\\\"Qual _Comp_ : A new lossy compressor for quality scores based on rate distortion theory\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011). _BMC Bioinformatics_. **14** 187\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/1471-2105-14-187](https://doi.org/10.1186%2F1471-2105-14-187). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3698011](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23758828](https://pubmed.ncbi.nlm.nih.gov/23758828).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698011\",\"favicon\":\"\"},{\"id\":\"10\",\"title\":\"** Hach, F; Numanagic, I; Alkan, C; Sahinalp, S. C. (2012). [\\\"SCALCE: Boosting sequence compression algorithms using locally consistent encoding\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486). _Bioinformatics_. **28** (23): 3051â€“3057\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/bioinformatics/bts593](https://doi.org/10.1093%2Fbioinformatics%2Fbts593). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3509486](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23047557](https://pubmed.ncbi.nlm.nih.gov/23047557).\",\"description\":\"10. **^** Hach, F; Numanagic, I; Alkan, C; Sahinalp, S. C. (2012). [\\\"SCALCE: Boosting sequence compression algorithms using locally consistent encoding\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486). _Bioinformatics_. **28** (23): 3051â€“3057\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/bioinformatics/bts593](https://doi.org/10.1093%2Fbioinformatics%2Fbts593). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3509486](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23047557](https://pubmed.ncbi.nlm.nih.gov/23047557).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509486\",\"favicon\":\"\"},{\"id\":\"11\",\"title\":\"** [\\\"fastqz - FASTQ compressor\\\"](http://mattmahoney.net/dc/fastqz).\",\"description\":\"11. **^** [\\\"fastqz - FASTQ compressor\\\"](http://mattmahoney.net/dc/fastqz).\",\"url\":\"http://mattmahoney.net/dc/fastqz\",\"favicon\":\"\"},{\"id\":\"12\",\"title\":\"** Malysa, Greg; Hernaez, Mikel; Ochoa, Idoia; Rao, Milind; Ganesan, Karthik; Weissman, Tsachy (2015-10-01). [\\\"QVZ: lossy compression of quality values\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090). _Bioinformatics_. **31** (19): 3122â€“3129\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/bioinformatics/btv330](https://doi.org/10.1093%2Fbioinformatics%2Fbtv330). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1367-4803](https://search.worldcat.org/issn/1367-4803). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [5856090](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [26026138](https://pubmed.ncbi.nlm.nih.gov/26026138).\",\"description\":\"12. **^** Malysa, Greg; Hernaez, Mikel; Ochoa, Idoia; Rao, Milind; Ganesan, Karthik; Weissman, Tsachy (2015-10-01). [\\\"QVZ: lossy compression of quality values\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090). _Bioinformatics_. **31** (19): 3122â€“3129\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/bioinformatics/btv330](https://doi.org/10.1093%2Fbioinformatics%2Fbtv330). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1367-4803](https://search.worldcat.org/issn/1367-4803). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [5856090](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [26026138](https://pubmed.ncbi.nlm.nih.gov/26026138).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856090\",\"favicon\":\"\"},{\"id\":\"13\",\"title\":\"** Paridaens, Tom; Van Wallendael, Glenn; De Neve, Wesley; Lambert, Peter (2018). [\\\"AQUa: an adaptive framework for compression of sequencing quality scores with random access functionality\\\"](https://doi.org/10.1093%2Fbioinformatics%2Fbtx607). _Bioinformatics_. **34** (3): 425â€“433\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/bioinformatics/btx607](https://doi.org/10.1093%2Fbioinformatics%2Fbtx607). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [29028894](https://pubmed.ncbi.nlm.nih.gov/29028894).\",\"description\":\"13. **^** Paridaens, Tom; Van Wallendael, Glenn; De Neve, Wesley; Lambert, Peter (2018). [\\\"AQUa: an adaptive framework for compression of sequencing quality scores with random access functionality\\\"](https://doi.org/10.1093%2Fbioinformatics%2Fbtx607). _Bioinformatics_. **34** (3): 425â€“433\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/bioinformatics/btx607](https://doi.org/10.1093%2Fbioinformatics%2Fbtx607). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [29028894](https://pubmed.ncbi.nlm.nih.gov/29028894).\",\"url\":\"https://doi.org/10.1093%2Fbioinformatics%2Fbtx607\",\"favicon\":\"\"}],\"images\":[],\"fixedIssues\":[{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"is currently one of the most widely used DNA sequence assembly programs in the biotech industry.\\\"\",\"explanation\":\"The claim uses \\\"currently\\\" without a date and is outdated as of 2025. While Phrap continues to be used in specific contexts like Sanger sequencing assemblies, it is not among the most widely used programs; modern de novo assembly tools such as SPAdes and Unicycler are more prevalent in current biotech practices.\",\"fixedText\":\"Revise to: \\\"and continues to be used in the biotech industry for certain assembly tasks, particularly with Sanger sequencing data.\\\"\",\"evidenceSource\":\"[](https://www.cd-genomics.com/resource-genome-assembly-overview-tools.html)\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"In 2009, many commonly used software packages make use of Phred quality scores, albeit to a different extent.\\\"\",\"explanation\":\"The paragraph is dated to 2009 and presents information as relevant without updates, creating a knowledge gap for post-2009 developments. Phred scores remain central to modern workflows, but the examples are outdated, and broader applications in next-generation sequencing (NGS) are not covered, affecting the section's completeness and currency as of 2025.\",\"fixedText\":\"Update the paragraph to reflect ongoing and modern usage, including examples from NGS pipelines, and correct the grammatical tense error.\",\"evidenceSource\":\"[](https://medium.com/@xieshaojun0526/understanding-phred-scores-for-fastq-format-24fed667dd11)\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"No specific text; entire section lacks coverage of modern applications.\",\"explanation\":\"The section focuses heavily on historical uses in the Human Genome Project and early software but omits key modern applications of Phred quality scores, such as in NGS data processing (e.g., read filtering in tools like FastQC, quality thresholds in variant calling with GATK), which are essential for understanding current significance and fill a knowledge gap up to 2025.\",\"fixedText\":\"Add a paragraph or expand the final one to include: Phred scores are integral to NGS workflows, encoded in FASTQ files, and used for quality filtering, trimming, and error estimation in alignment and variant calling pipelines.\",\"evidenceSource\":\"[](https://www.illumina.com/science/genomics-research/articles/data-quality-q-scores.html)\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"the MPEG-G standard, that is currently under development by the [MPEG](Moving_Picture_Experts_Group \\\"Moving Picture Experts Group\\\") standardisation working group.\\\"\",\"explanation\":\"MPEG-G (ISO/IEC 23092) was first published in 2020, with subsequent parts in 2022 and updates in 2025. It is an established international standard, not under development as of October 2025.\",\"fixedText\":\"Update to reflect that MPEG-G is an ISO/IEC standard published starting in 2020, with ongoing amendments.\",\"evidenceSource\":\"[](https://www.iso.org/standard/79882.html)\"},{\"criticality\":\"Critical\",\"issueType\":\"Internal Contradiction\",\"problematicText\":\"\\\"Both are lossless compression algorithms that provide an optional controlled lossy transformation approach.\\\"\",\"explanation\":\"The pronoun \\\"both\\\" is ambiguous and factually incorrect; the preceding list includes a mix of lossless (e.g., SCALCE) and lossy (e.g., QVZ) algorithms, causing confusion about their compression types.\",\"fixedText\":\"Reword the sentence to accurately describe the algorithms' properties without ambiguity, specifying types where appropriate.\",\"evidenceSource\":\"[10] [12]\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"Both [lossless](Lossless_compression \\\"Lossless compression\\\") and [lossy compression](Lossy_compression \\\"Lossy compression\\\") are recently being considered in the literature.\\\"\",\"explanation\":\"Compression methods for quality scores have been actively researched since at least 2012, making \\\"recently\\\" outdated in the context of 2025.\",\"fixedText\":\"Remove \\\"recently\\\" and rephrase to \\\"have been explored in the literature\\\" to reflect historical and ongoing research.\",\"evidenceSource\":\"[10]\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"\",\"explanation\":\"The section lacks mention of more recent advancements in quality score compression post-2018, such as PQSDC (2024), a parallel lossless compressor, and GeneSqueeze (2025), a lossless reference-free compressor for FASTQ files, which represent ongoing developments up to 2025.\",\"fixedText\":\"Add brief mentions of recent algorithms like PQSDC and GeneSqueeze to update the list of examples, with citations.\",\"evidenceSource\":\"[](https://academic.oup.com/bioinformatics/article/40/5/btae323/7676123) [](https://www.nature.com/articles/s41598-024-79258-6)\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"The table lists mappings only up to Phred quality score 40 (symbol 'I').\",\"explanation\":\"The standard FASTQ format uses Phred+33 encoding, supporting quality scores from 0 to 93 encoded as ASCII characters from '!' (code 33) to '~' (code 126). The table is incomplete, omitting mappings for scores 41-93 commonly used in next-generation sequencing technologies like Illumina, which can produce higher quality scores.\",\"fixedText\":\"Add an introductory sentence explaining the full Phred+33 encoding range (0-93) and note that the table shows mappings up to 40, with higher scores following the same pattern: symbol = chr(Q + 33), P â‰ˆ 10^{-Q/10}.\",\"evidenceSource\":\"[](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847217/)\"}],\"slug\":\"Phred_quality_score\",\"title\":\"Phred quality score\",\"content\":\"$1f\",\"description\":\"Phred quality score\\n\\nA Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the...\",\"metadata\":{\"categories\":[\"Phred score\",\"phred score\"],\"lastModified\":\"1761553076\",\"contentLength\":\"12694\",\"version\":\"1.0\",\"lastEditor\":\"system\",\"language\":\"en\",\"isRedirect\":false,\"redirectTarget\":\"\",\"isWithheld\":false},\"stats\":{\"totalViews\":\"53967\",\"recentViews\":\"53967\",\"dailyAvgViews\":1798.9000244140625,\"qualityScore\":1,\"lastViewed\":\"1761883342\"},\"linkedPages\":null},\"found\":true},\"dataUpdateCount\":1,\"dataUpdatedAt\":1761883342212,\"error\":null,\"errorUpdateCount\":0,\"errorUpdatedAt\":0,\"fetchFailureCount\":0,\"fetchFailureReason\":null,\"fetchMeta\":null,\"isInvalidated\":false,\"status\":\"success\",\"fetchStatus\":\"idle\"},\"queryKey\":[\"page\",\"Phred_quality_score\"],\"queryHash\":\"[\\\"page\\\",\\\"Phred_quality_score\\\"]\"}]},\"children\":\"$L20\"}]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"c:{\"metadata\":[[\"$\",\"title\",\"0\",{\"children\":\"Phred quality score\"}],[\"$\",\"meta\",\"1\",{\"name\":\"description\",\"content\":\"A Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the computer program Phred to help in the automation of DNA sequencing in the Human Genome Project. Phred quality scores are assigned to each nucleotide base call in automated sequencer traces.[1][2] The FASTQ format encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely...\"}],[\"$\",\"meta\",\"2\",{\"name\":\"author\",\"content\":\"system\"}],[\"$\",\"link\",\"3\",{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\",\"crossOrigin\":\"$undefined\"}],[\"$\",\"meta\",\"4\",{\"name\":\"keywords\",\"content\":\"Phred score, phred score\"}],[\"$\",\"meta\",\"5\",{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",\"6\",{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"link\",\"7\",{\"rel\":\"canonical\",\"href\":\"https://grokipedia.com/page/Phred_quality_score\"}],[\"$\",\"meta\",\"8\",{\"property\":\"og:title\",\"content\":\"Phred quality score\"}],[\"$\",\"meta\",\"9\",{\"property\":\"og:description\",\"content\":\"A Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the computer program Phred to help in the automation of DNA sequencing in the Human Genome Project. Phred quality scores are assigned to each nucleotide base call in automated sequencer traces.[1][2] The FASTQ format encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely...\"}],[\"$\",\"meta\",\"10\",{\"property\":\"og:url\",\"content\":\"https://grokipedia.com/page/Phred_quality_score\"}],[\"$\",\"meta\",\"11\",{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",\"12\",{\"property\":\"og:locale\",\"content\":\"en\"}],[\"$\",\"meta\",\"13\",{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",\"14\",{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",\"15\",{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",\"16\",{\"property\":\"og:image:alt\",\"content\":\"Phred quality score\"}],[\"$\",\"meta\",\"17\",{\"property\":\"og:type\",\"content\":\"article\"}],[\"$\",\"meta\",\"18\",{\"property\":\"article:modified_time\",\"content\":\"1970-01-21T09:19:13.076Z\"}],[\"$\",\"meta\",\"19\",{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",\"20\",{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",\"21\",{\"name\":\"twitter:title\",\"content\":\"Phred quality score\"}],[\"$\",\"meta\",\"22\",{\"name\":\"twitter:description\",\"content\":\"A Phred quality score is a measure of the quality of the identification of the nucleobases generated by automated DNA sequencing.[1][2] It was originally developed for the computer program Phred to help in the automation of DNA sequencing in the Human Genome Project. Phred quality scores are assigned to each nucleotide base call in automated sequencer traces.[1][2] The FASTQ format encodes phred scores as ASCII characters alongside the read sequences. Phred quality scores have become widely...\"}],[\"$\",\"meta\",\"23\",{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"link\",\"24\",{\"rel\":\"icon\",\"href\":\"/favicon.ico\",\"type\":\"image/x-icon\",\"sizes\":\"16x16\"}],[\"$\",\"$L21\",\"25\",{}]],\"error\":null,\"digest\":\"$undefined\"}\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"11:\"$c:metadata\"\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"22:I[61172,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"PageEditorProvider\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"23:I[63493,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="NzIxODAzYjktZmQwZS00OThmLThiY2UtMWIzYWE5OGMzMWY0">self.__next_f.push([1,"20:[\"$\",\"$L22\",null,{\"children\":[\"$\",\"$L23\",null,{\"slug\":\"Phred_quality_score\"}]}]\n"])</script></body></html>