<!DOCTYPE html><html lang="en" class="bg-surface-base antialiased"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1, interactive-widget=resizes-content"/><link rel="stylesheet" href="/_next/static/css/62c4caba71dfda84.css" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/eb3d87f98fe1565f.css" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/1b5e561215938d4d.css" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/0227d069a630d414.css" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/f87fff2ab93d05a7.css" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" data-precedence="next"/><link rel="preload" as="script" fetchPriority="low" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" href="/_next/static/chunks/webpack-e121ed42680f327e.js"/><script src="/_next/static/chunks/78a669d9-e7993486b22c4915.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/40c4a5a7-abdabb07419d1cfc.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/2230-f7c87dc9fa57c408.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/main-app-536d4b8fca62396a.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/63f0ec43-8ee6a76d70472249.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/7670-7326298c9856172b.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/282-691fffb366e24ca5.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/7515-336759ef5dad2b52.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/927-34e2a6da3e15e26b.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/5855-243eeca8f0e4cabd.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/4393-1c8dff25ec904469.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/2225-454cf9f44775e7ce.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/7618-43536a8fb0dd3bfe.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/7720-3a1e7e411adba2d3.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/7086-1f1c4891d766e04f.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/global-error-4d07d20223cd4b4c.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/8208b75a-a48e5a4507a0de91.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/5497-ab85ef3ea59dd353.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/6281-c6e84786dade3614.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/2660-d983a7f287e89f18.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/5002-9735c25beda556ba.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/3655-6717081f7512305a.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/8945-e4e3cf487f5e27c7.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/9214-a10614844a67ba31.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/4789-2080f6497f78b5b6.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/layout-226f2bd71acf9f9e.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/24cf1b50-159cca90b09285e0.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/9ffa21ba-c069766d809bd4c7.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/600-9a9fcafaaa6c1c4c.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/5448-2a3cfd853d899c9a.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/8-8ca70ca619d8e099.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/error-4a29e9399afba038.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><script src="/_next/static/chunks/app/not-found-dd95690acf732f18.js" async="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script><meta name="next-size-adjust" content=""/><title>Grokipedia</title><meta name="description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><link rel="icon" type="image/x-icon" href="/favicon.ico" sizes="48x48"/><link rel="icon" href="/images/icon-dark.png" media="(prefers-color-scheme: light)" type="image/png"/><link rel="icon" href="/images/icon-light.png" media="(prefers-color-scheme: dark)" type="image/png"/><link rel="apple-touch-icon" href="/icon-192x192.png"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><meta property="og:title" content="Grokipedia"/><meta property="og:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta property="og:url" content="https://grokipedia.com"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en_US"/><meta property="og:type" content="website"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Grokipedia"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Grokipedia"/><meta name="twitter:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><title>Genomic library</title><meta name="description" content="A genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism&#x27;s DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA..."/><meta name="author" content="system"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="keywords" content="genomic DNA library, gDNA library"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><link rel="canonical" href="https://grokipedia.com/page/Genomic_library"/><meta property="og:title" content="Genomic library"/><meta property="og:description" content="A genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism&#x27;s DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA..."/><meta property="og:url" content="https://grokipedia.com/page/Genomic_library"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Genomic library"/><meta property="og:type" content="article"/><meta property="article:modified_time" content="1970-01-21T09:19:14.060Z"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Genomic library"/><meta name="twitter:description" content="A genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism&#x27;s DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA..."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><link rel="icon" href="/favicon.ico" type="image/x-icon" sizes="16x16"/><meta name="sentry-trace" content="fb62486894142cbfaa60e18a541198d4-cddcdfcd3a240f41-1"/><meta name="baggage" content="sentry-environment=production,sentry-public_key=5f2258f71198ee26a355127af230c3a6,sentry-trace_id=fb62486894142cbfaa60e18a541198d4,sentry-org_id=4508179396558848,sentry-transaction=GET%20%2Fpage%2F%5Bslug%5D,sentry-sampled=true,sentry-sample_rand=0.2487345644551051,sentry-sample_rate=1"/><script src="/_next/static/chunks/polyfills-42372ed130431b0a.js" noModule="" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl"></script></head><body class="flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b"><div hidden=""><!--$--><!--/$--></div><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">((a,b,c,d,e,f,g,h)=>{let i=document.documentElement,j=["light","dark"];function k(b){var c;(Array.isArray(a)?a:[a]).forEach(a=>{let c="class"===a,d=c&&f?e.map(a=>f[a]||a):e;c?(i.classList.remove(...d),i.classList.add(f&&f[b]?f[b]:b)):i.setAttribute(a,b)}),c=b,h&&j.includes(c)&&(i.style.colorScheme=c)}if(d)k(d);else try{let a=localStorage.getItem(b)||c,d=g&&"system"===a?window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light":a;k(d)}catch(a){}})("class","theme","dark",null,["light","dark"],null,true,true)</script><header class="h-16 bg-surface-base fixed top-0 z-50 w-full"><div class="hidden h-full grid-cols-[1fr_3fr_1fr] items-center gap-4 !py-0 md:grid max-w-full py-6 px-4 w-full mx-auto"><div class="-ml-2 justify-self-start"><a class="cursor-pointer" href="/"><svg width="200" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="mx-auto flex w-full"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full px-3 shadow-none sm:w-80 focus-visible:z-1 mx-auto h-full w-full rounded-full bg-surface-l1" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>âŒ˜</span><span class="text-xs">K</span></div></span></div></button></div><div class="flex items-center gap-2 justify-self-end"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0" type="button" id="radix-_R_7apfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button></div></div><div class="flex h-full items-center md:hidden justify-between max-w-full py-6 px-4 w-full mx-auto pt-8"><div class="flex items-center"><div class="-ml-2"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button" aria-label="Toggle table of contents"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-list h-6 w-6"><path d="M3 12h.01"></path><path d="M3 18h.01"></path><path d="M3 6h.01"></path><path d="M8 12h13"></path><path d="M8 18h13"></path><path d="M8 6h13"></path></svg></button></div><a class="cursor-pointer" href="/"><svg width="160" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="flex items-center gap-1"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0 [&amp;_svg]:!w-5" type="button" id="radix-_R_aipfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search h-5 w-5"><circle cx="11" cy="11" r="8"></circle><path d="m21 21-4.3-4.3"></path></svg></button></div></div></header><div class="bg-surface-base fixed right-0 top-16 z-50 h-[calc(100vh-4rem)] w-full transform border-l transition-transform duration-300 ease-in-out sm:w-1/2 md:hidden translate-x-full"><div class="flex flex-col gap-4 p-4"><div class="flex flex-row gap-2"><button class="focus-visible:ring-ring inline-flex items-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full shadow-none sm:w-80 h-10 w-full justify-start rounded-full bg-surface-l1 px-4" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>âŒ˜</span><span class="text-xs">K</span></div></span></div></button></div></div></div><div><div class="min-[1350px]:grid min-[1350px]:grid-cols-[1fr_3fr_1fr]"><nav class="bg-surface-base hidden h-[calc(100vh-4rem)] overflow-y-auto px-6 pb-32 min-[1350px]:sticky min-[1350px]:top-16 min-[1350px]:block scrollbar-none [-ms-overflow-style:none] [scrollbar-width:none] [&amp;::-webkit-scrollbar]:hidden [overscroll-behavior:contain] pt-8"><ul class="space-y-2 text-sm"><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#genomic-library" class="transition-opacity hover:opacity-100 opacity-50">Genomic library</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#history" class="transition-opacity hover:opacity-100 opacity-50">History</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#genomic-library-construction" class="transition-opacity hover:opacity-100 opacity-50">Genomic library construction</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#determining-titer-of-library" class="transition-opacity hover:opacity-100 opacity-50">Determining titer of library</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#screening-library" class="transition-opacity hover:opacity-100 opacity-50">Screening library</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#types-of-vectors" class="transition-opacity hover:opacity-100 opacity-50">Types of vectors</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#plasmids" class="transition-opacity hover:opacity-100 opacity-50">Plasmids</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#phage-lambda-Î»" class="transition-opacity hover:opacity-100 opacity-50">Phage lambda (Î»)</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#cosmids" class="transition-opacity hover:opacity-100 opacity-50">Cosmids</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#bacteriophage-p1-vectors" class="transition-opacity hover:opacity-100 opacity-50">Bacteriophage P1 vectors</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#p1-artificial-chromosomes" class="transition-opacity hover:opacity-100 opacity-50">P1 artificial chromosomes</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#bacterial-artificial-chromosomes" class="transition-opacity hover:opacity-100 opacity-50">Bacterial artificial chromosomes</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#yeast-artificial-chromosomes" class="transition-opacity hover:opacity-100 opacity-50">Yeast artificial chromosomes</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#how-to-select-a-vector" class="transition-opacity hover:opacity-100 opacity-50">How to select a vector</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#vector-selection-example" class="transition-opacity hover:opacity-100 opacity-50">Vector selection example</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#applications" class="transition-opacity hover:opacity-100 opacity-50">Applications</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#hierarchical-sequencing" class="transition-opacity hover:opacity-100 opacity-50">Hierarchical sequencing</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#genome-wide-association-studies" class="transition-opacity hover:opacity-100 opacity-50">Genome-wide association studies</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#references" class="transition-opacity hover:opacity-100 opacity-50">References</a></li></ul></nav><div class="relative top-16 pb-32 pt-8 px-4 md:px-8"><div class="mx-auto max-w-[850px]"><button data-state="closed" type="button" class="flex items-center"><div class="text-fg-tertiary mb-2 flex cursor-help items-center gap-2 text-sm"><span class="inline-flex items-center justify-center p-0 m-0" data-namespace="@xai/icons" data-slot="icon" style="height:16px;width:16px"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 33 33" aria-hidden="true" class="" focusable="false" style="fill:currentColor;height:16px;width:16px"><path fill="currentColor" d="m13.237 21.04 11.082-8.19c.543-.4 1.32-.244 1.578.38 1.363 3.288.754 7.241-1.957 9.955-2.71 2.714-6.482 3.31-9.93 1.954l-3.765 1.745c5.401 3.697 11.96 2.782 16.059-1.324 3.251-3.255 4.258-7.692 3.317-11.693l.008.009c-1.365-5.878.336-8.227 3.82-13.031q.123-.17.247-.345l-4.585 4.59v-.014L13.234 21.044M10.95 23.031c-3.877-3.707-3.208-9.446.1-12.755 2.446-2.449 6.454-3.448 9.952-1.979L24.76 6.56c-.677-.49-1.545-1.017-2.54-1.387A12.465 12.465 0 0 0 8.675 7.901c-3.519 3.523-4.625 8.94-2.725 13.561 1.42 3.454-.907 5.898-3.251 8.364-.83.874-1.664 1.749-2.335 2.674l10.583-9.466"></path></svg></span><span>Fact-checked by Grok<!-- --> <!-- -->4 days ago</span></div></button><article class="text-[16px]"><h1 id="genomic-library" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Genomic library<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">A <strong>genomic library</strong> is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism&#x27;s DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA ligase.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup> Next, the vector DNA can be taken up by a host organism - commonly a population of Escherichia coli or yeast - with each cell containing only one vector molecule. Using a host cell to carry the vector allows for easy amplification and retrieval of specific clones from the library for analysis.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">There are several kinds of vectors available with various insert capacities. Generally, libraries made from organisms with larger genomes require vectors featuring larger inserts, thereby fewer vector molecules are needed to make the library. Researchers can choose a vector also considering the ideal insert size to find the desired number of clones necessary for full genome coverage.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_21abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Genomic libraries are commonly used for sequencing applications. They have played an important role in the whole genome sequencing of several organisms, including the human genome and several model organisms.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_21qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_31qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[5]</sup></span>
<h2 id="history" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">History<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">The first DNA-based genome ever fully sequenced was that of the bacteriophage phi X 174, achieved by Frederick Sanger and his team in 1977 using the plus and minus sequencing method.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_22qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[6]</sup> This milestone highlighted the potential of DNA sequencing technologies.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Genomic libraries emerged in the mid-to-late 1970s as part of the development of recombinant DNA technology. Early libraries were constructed using bacterial plasmids and phage lambda vectors to clone overlapping DNA fragments, enabling systematic study and sequencing of larger genomes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_23abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_33abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[8]</sup> These advances facilitated subsequent whole-genome sequencing projects, including those for model organisms and the human genome.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_53abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_63abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[9]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Prior to genomic libraries, gene isolation relied on more limited techniques such as partial purification or small-scale cloning.</span>
<h1 id="genomic-library-construction" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Genomic library construction<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">Construction of a genomic library involves creating many recombinant DNA molecules. An organism&#x27;s genomic DNA is extracted and then partially digested with a restriction enzyme. For very small genomes, such as those of some viruses (~5-10 kb), the digested fragments can be separated by gel electrophoresis. The separated fragments can then be excised and cloned into the vector separately. However, when a large genome is partially digested with a restriction enzyme, there are far too many fragments to excise individually. The entire set of fragments must be cloned together with the vector, and separation of clones can occur after. In either case, the fragments are ligated into a vector that has been digested with a compatible restriction enzyme. The vector containing the inserted fragments of genomic DNA can then be introduced into a host organism.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_24qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Below are the steps for creating a genomic library from a large genome.</span>
<ol class="marker:text-primary my-4 ml-6 list-decimal space-y-2 text-[1em]">
<li class="text-[1em]">Extract and purify DNA.</li>
<li class="text-[1em]">Partially digest the DNA with a restriction enzyme. This creates overlapping fragments that are similar in size, each containing one or more genes.</li>
<li class="text-[1em]">Size-select the fragments (e.g., via gel electrophoresis or sucrose density gradient centrifugation) to obtain the desired length range for the vector.</li>
<li class="text-[1em]">Insert the fragments of DNA into vectors that were cut with a compatible restriction enzyme. Use the enzyme DNA ligase to seal the DNA fragments into the vector. This creates a large pool of recombinant molecules.</li>
<li class="text-[1em]">These recombinant molecules are taken up by a host bacterium by transformation, creating a DNA library.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_1a5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[10]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_1q5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[11]</sup></li>
</ol>
<h3 id="determining-titer-of-library" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Determining titer of library<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">After a genomic library is constructed with a viral vector, such as lambda phage, the titer of the library can be determined. Calculating the titer allows researchers to approximate how many infectious viral particles were successfully created in the library. To do this, dilutions of the library are used to infect cultures of E. coli of known concentrations. The cultures are then plated on agar plates and incubated overnight. The number of viral plaques are counted and can be used to calculate the total number of infectious viral particles in the library. Most viral vectors also carry a marker that allows clones containing an insert to be distinguished from those that do not have an insert. This allows researchers to also determine the percentage of infectious viral particles actually carrying a fragment of the library.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_26qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[12]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">A similar method can be used to titer genomic libraries made with non-viral vectors, such as plasmids and BACs. A test ligation of the library can be used to transform E. coli. The transformation is then spread on agar plates and incubated overnight. The titer of the transformation is determined by counting the number of colonies present on the plates. These vectors generally have a selectable marker allowing the differentiation of clones containing an insert from those that do not. By doing this test, researchers can also determine the efficiency of the ligation and make adjustments as needed to ensure they get the desired number of clones for the library.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_27abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup></span>
<h3 id="screening-library" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Screening library<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">In order to isolate clones that contain regions of interest from a library, the library must first be screened. One method of screening is hybridization. Each transformed host cell of a library will contain only one vector with one insert of DNA. The whole library can be plated onto a filter over media. The filter and colonies are prepared for hybridization and then hybridized with a labeled probe. The target DNA-insert of interest-can be identified by detection such as autoradiography because of the hybridization with the probe as seen below.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Another method of screening is with polymerase chain reaction (PCR). Some libraries are stored as pools of clones and screening by PCR is an efficient way to identify pools containing specific clones.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_28qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup></span>
<h1 id="types-of-vectors" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Types of vectors<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">Genome size varies among different organisms and the cloning vector must be selected accordingly. For a large genome, a vector with a large capacity should be chosen so that a relatively small number of clones are sufficient for coverage of the entire genome. However, it is often more difficult to characterize an insert contained in a higher capacity vector.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_29qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Below is a table of several kinds of vectors commonly used for genomic libraries and the insert size that each generally holds.</span>





































<div class="border-border-l1 overflow-x-auto rounded-lg border my-6 w-full"><table node="[object Object]" class="w-full border-collapse text-sm"><thead node="[object Object]" class="border-border-l1 border-b bg-surface-l1"><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Vector type</span></th><th node="[object Object]" class="dark:text-fg-primary bg-gray-100 px-2 py-1 text-left align-top font-bold dark:bg-black"><span class="text-[1em] leading-7">Insert size (thousands of bases)</span></th></tr></thead><tbody node="[object Object]" class="divide-border-l1 divide-y"><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">Plasmids</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">up to 15</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">Phage lambda (Î»)</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">up to 25</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">Cosmids</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">up to 45</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">Bacteriophage P1</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">70 to 100</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">P1 artificial chromosomes (PACs)</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">130 to 150</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">Bacterial artificial chromosomes (BACs)</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">120 to 300</span></td></tr><tr node="[object Object]" class="divide-border-l1 divide-x bg-surface-l1"><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">Yeast artificial chromosomes (YACs)</span></td><td node="[object Object]" class="text-fg-primary px-2 py-1 align-top"><span class="text-[1em] leading-7">250 to 2000</span></td></tr></tbody></table></div>
<h3 id="plasmids" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Plasmids<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">A plasmid is a double stranded circular DNA molecule commonly used for molecular cloning. Plasmids are generally 2 to 4 kilobase-pairs (kb) in length and are capable of carrying inserts up to 15kb. Plasmids contain an origin of replication allowing them to replicate inside a bacterium independently of the host chromosome. Plasmids commonly carry a gene for antibiotic resistance that allows for the selection of bacterial cells containing the plasmid. Many plasmids also carry a reporter gene that allows researchers to distinguish clones containing an insert from those that do not.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2bqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup></span>
<h3 id="phage-lambda-Î»" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Phage lambda (Î»)<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Phage Î» is a double-stranded DNA virus that infects E. coli. The Î» chromosome is 48.5kb long and can carry inserts up to 25kb. These inserts replace non-essential viral sequences in the Î» chromosome, while the genes required for formation of viral particles and infection remain intact. The insert DNA is replicated with the viral DNA; thus, together they are packaged into viral particles. These particles are very efficient at infection and multiplication leading to a higher production of the recombinant Î» chromosomes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2cqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup> However, due to the smaller insert size, libraries made with Î» phage may require many clones for full genome coverage.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4cqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup></span>
<h3 id="cosmids" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Cosmids<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Cosmid vectors are plasmids that contain a small region of bacteriophage Î» DNA called the cos sequence. This sequence allows the cosmid to be packaged into bacteriophage Î» particles. These particles- containing a linearized cosmid- are introduced into the host cell by transduction. Once inside the host, the cosmids circularize with the aid of the host&#x27;s DNA ligase and then function as plasmids. Cosmids are capable of carrying inserts up to 45kb in size.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2dqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup></span>
<h3 id="bacteriophage-p1-vectors" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Bacteriophage P1 vectors<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Bacteriophage P1 vectors can hold inserts 70 â€“ 100kb in size. They begin as linear DNA molecules packaged into bacteriophage P1 particles. These particles are injected into an E. coli strain expressing Cre recombinase. The linear P1 vector becomes circularized by recombination between two loxP sites in the vector. P1 vectors generally contain a gene for antibiotic resistance and a positive selection marker to distinguish clones containing an insert from those that do not. P1 vectors also contain a P1 plasmid replicon, which ensures only one copy of the vector is present in a cell. However, there is a second P1 replicon- called the P1 lytic replicon- that is controlled by an inducible promoter. This promoter allows the amplification of more than one copy of the vector per cell prior to DNA extraction.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2eqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup></span>
<h3 id="p1-artificial-chromosomes" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">P1 artificial chromosomes<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">P1 artificial chromosomes (PACs) have features of both P1 vectors and Bacterial Artificial Chromosomes (BACs). Similar to P1 vectors, they contain a plasmid and a lytic replicon as described above. Unlike P1 vectors, they do not need to be packaged into bacteriophage particles for transduction. Instead they are introduced into E. coli as circular DNA molecules through electroporation just as BACs are.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2fqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup> Also similar to BACs, these are relatively harder to prepare due to a single origin of replication.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4fqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup></span>
<h3 id="bacterial-artificial-chromosomes" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Bacterial artificial chromosomes<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Bacterial artificial chromosomes (BACs) are circular DNA molecules, usually about 7kb in length, that are capable of holding inserts up to 300kb in size. BAC vectors contain a replicon derived from E. coli F factor, which ensures they are maintained at one copy per cell.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup> Once an insert is ligated into a BAC, the BAC is introduced into recombination deficient strains of E. coli by electroporation. Most BAC vectors contain a gene for antibiotic resistance and also a positive selection marker.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup> The figure to the right depicts a BAC vector being cut with a restriction enzyme, followed by the insertion of foreign DNA that is re-annealed by a ligase. Overall, this is a very stable vector, but they may be hard to prepare due to a single origin of replication just like PACs.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup></span>
<h3 id="yeast-artificial-chromosomes" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Yeast artificial chromosomes<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Yeast artificial chromosomes (YACs) are linear DNA molecules containing the necessary features of an authentic yeast chromosome, including telomeres, a centromere, and an origin of replication. Large inserts of DNA can be ligated into the middle of the YAC so that there is an â€œarmâ€ of the YAC on either side of the insert. The recombinant YAC is introduced into yeast by transformation; selectable markers present in the YAC allow for the identification of successful transformants. YACs can hold inserts up to 2000kb, but most YAC libraries contain inserts 250-400kb in size. Theoretically there is no upper limit on the size of insert a YAC can hold. It is the quality in the preparation of DNA used for inserts that determines the size limit.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2hqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup> The most challenging aspect of using YAC is the fact they are prone to rearrangement.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4hqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup></span>
<h2 id="how-to-select-a-vector" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">How to select a vector<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">Vector selection requires one to ensure the library made is representative of the entire genome. Any insert of the genome derived from a restriction enzyme should have an equal chance of being in the library compared to any other insert. Furthermore, recombinant molecules should contain large enough inserts ensuring the library size is able to be handled conveniently.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2iqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup> This is particularly determined by the number of clones needed to have in a library. The number of clones to get a sampling of all the genes is determined by the size of the organism&#x27;s genome as well as the average insert size. This is represented by the formula (also known as the Carbon and Clarke formula):<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4iqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[8]</sup></span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>N</mi><mo>=</mo><mfrac><mrow><mi>ln</mi><mo>â¡</mo><mo stretchy="false">(</mo><mn>1</mn><mo>âˆ’</mo><mi>P</mi><mo stretchy="false">)</mo></mrow><mrow><mi>ln</mi><mo>â¡</mo><mo stretchy="false">(</mo><mn>1</mn><mo>âˆ’</mo><mi>f</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow><annotation encoding="application/x-tex">N = \frac{\ln(1 - P)}{\ln(1 - f)}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.10903em">N</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:2.363em;vertical-align:-0.936em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.427em"><span style="top:-2.314em"><span class="pstrut" style="height:3em"></span><span class="mord"><span class="mop">ln</span><span class="mopen">(</span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">âˆ’</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord mathnormal" style="margin-right:0.10764em">f</span><span class="mclose">)</span></span></span><span style="top:-3.23em"><span class="pstrut" style="height:3em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.677em"><span class="pstrut" style="height:3em"></span><span class="mord"><span class="mop">ln</span><span class="mopen">(</span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">âˆ’</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord mathnormal" style="margin-right:0.13889em">P</span><span class="mclose">)</span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.936em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span>
<span class="mb-4 block break-words text-[1em] leading-7">where,</span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>N</mi></mrow><annotation encoding="application/x-tex">N</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.10903em">N</span></span></span></span> is the necessary number of recombinants<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3kabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[14]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>P</mi></mrow><annotation encoding="application/x-tex">P</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.13889em">P</span></span></span></span> is the desired probability that any fragment in the genome will occur at least once in the library created</span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>f</mi></mrow><annotation encoding="application/x-tex">f</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8889em;vertical-align:-0.1944em"></span><span class="mord mathnormal" style="margin-right:0.10764em">f</span></span></span></span> is the fractional proportion of the genome in a single recombinant</span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>f</mi></mrow><annotation encoding="application/x-tex">f</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8889em;vertical-align:-0.1944em"></span><span class="mord mathnormal" style="margin-right:0.10764em">f</span></span></span></span> can be further shown to be:</span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>f</mi><mo>=</mo><mfrac><mi>i</mi><mi>g</mi></mfrac></mrow><annotation encoding="application/x-tex">f = \frac{i}{g}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8889em;vertical-align:-0.1944em"></span><span class="mord mathnormal" style="margin-right:0.10764em">f</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:2.217em;vertical-align:-0.8804em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.3365em"><span style="top:-2.314em"><span class="pstrut" style="height:3em"></span><span class="mord"><span class="mord mathnormal" style="margin-right:0.03588em">g</span></span></span><span style="top:-3.23em"><span class="pstrut" style="height:3em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.677em"><span class="pstrut" style="height:3em"></span><span class="mord"><span class="mord mathnormal">i</span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.8804em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span>
<span class="mb-4 block break-words text-[1em] leading-7">where,</span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>i</mi></mrow><annotation encoding="application/x-tex">i</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6595em"></span><span class="mord mathnormal">i</span></span></span></span> is the insert size</span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>g</mi></mrow><annotation encoding="application/x-tex">g</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.625em;vertical-align:-0.1944em"></span><span class="mord mathnormal" style="margin-right:0.03588em">g</span></span></span></span> is the genome size</span>
<span class="mb-4 block break-words text-[1em] leading-7">Thus, increasing the insert size (by choice of vector) would allow for fewer clones needed to represent a genome. The proportion of the insert size versus the genome size represents the proportion of the respective genome in a single clone.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2oabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup> Here is the equation with all parts considered:</span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>N</mi><mo>=</mo><mfrac><mrow><mi>ln</mi><mo>â¡</mo><mo stretchy="false">(</mo><mn>1</mn><mo>âˆ’</mo><mi>P</mi><mo stretchy="false">)</mo></mrow><mrow><mi>ln</mi><mo>â¡</mo><mrow><mo fence="true">(</mo><mn>1</mn><mo>âˆ’</mo><mfrac><mi>i</mi><mi>g</mi></mfrac><mo fence="true">)</mo></mrow></mrow></mfrac></mrow><annotation encoding="application/x-tex">N = \frac{\ln(1 - P)}{\ln\left(1 - \frac{i}{g}\right)}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.10903em">N</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:3.117em;vertical-align:-1.69em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.427em"><span style="top:-2.11em"><span class="pstrut" style="height:3.15em"></span><span class="mord"><span class="mop">ln</span><span class="mspace" style="margin-right:0.1667em"></span><span class="minner"><span class="mopen delimcenter" style="top:0em"><span class="delimsizing size2">(</span></span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">âˆ’</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8557em"><span style="top:-2.655em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathnormal mtight" style="margin-right:0.03588em">g</span></span></span></span><span style="top:-3.23em"><span class="pstrut" style="height:3em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.394em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathnormal mtight">i</span></span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.4811em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span><span class="mclose delimcenter" style="top:0em"><span class="delimsizing size2">)</span></span></span></span></span><span style="top:-3.38em"><span class="pstrut" style="height:3.15em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.827em"><span class="pstrut" style="height:3.15em"></span><span class="mord"><span class="mop">ln</span><span class="mopen">(</span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">âˆ’</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord mathnormal" style="margin-right:0.13889em">P</span><span class="mclose">)</span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:1.69em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span>
<h3 id="vector-selection-example" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Vector selection example<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">The above formula can be used to determine the 99% confidence level that all sequences in a genome are represented by using a vector with an insert size of twenty thousand basepairs (such as the phage lambda vector). The genome size of the organism is three billion basepairs in this example.</span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>N</mi><mo>=</mo><mfrac><mrow><mi>ln</mi><mo>â¡</mo><mo stretchy="false">(</mo><mn>1</mn><mo>âˆ’</mo><mn>0.99</mn><mo stretchy="false">)</mo></mrow><mrow><mi>ln</mi><mo>â¡</mo><mrow><mo fence="true">[</mo><mn>1</mn><mo>âˆ’</mo><mfrac><mrow><mn>2.0</mn><mo>Ã—</mo><mn>1</mn><msup><mn>0</mn><mn>4</mn></msup><mtext>Â basepairs</mtext></mrow><mrow><mn>3.0</mn><mo>Ã—</mo><mn>1</mn><msup><mn>0</mn><mn>9</mn></msup><mtext>Â basepairs</mtext></mrow></mfrac><mo fence="true">]</mo></mrow></mrow></mfrac></mrow><annotation encoding="application/x-tex">N = \frac{\ln(1 - 0.99)}{\ln\left[1 - \frac{2.0 \times 10^{4} \text{ basepairs}}{3.0 \times 10^{9} \text{ basepairs}}\right]}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.10903em">N</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:3.117em;vertical-align:-1.69em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.427em"><span style="top:-2.11em"><span class="pstrut" style="height:3.15em"></span><span class="mord"><span class="mop">ln</span><span class="mspace" style="margin-right:0.1667em"></span><span class="minner"><span class="mopen delimcenter" style="top:0em"><span class="delimsizing size2">[</span></span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">âˆ’</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.9685em"><span style="top:-2.655em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">3.0</span><span class="mbin mtight">Ã—</span><span class="mord mtight">1</span><span class="mord mtight"><span class="mord mtight">0</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.7463em"><span style="top:-2.786em;margin-right:0.0714em"><span class="pstrut" style="height:2.5em"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">9</span></span></span></span></span></span></span></span></span><span class="mord text mtight"><span class="mord mtight">Â basepairs</span></span></span></span></span><span style="top:-3.23em"><span class="pstrut" style="height:3em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.4461em"><span class="pstrut" style="height:3em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">2.0</span><span class="mbin mtight">Ã—</span><span class="mord mtight">1</span><span class="mord mtight"><span class="mord mtight">0</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.7463em"><span style="top:-2.786em;margin-right:0.0714em"><span class="pstrut" style="height:2.5em"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">4</span></span></span></span></span></span></span></span></span><span class="mord text mtight"><span class="mord mtight">Â basepairs</span></span></span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.4811em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span><span class="mclose delimcenter" style="top:0em"><span class="delimsizing size2">]</span></span></span></span></span><span style="top:-3.38em"><span class="pstrut" style="height:3.15em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.827em"><span class="pstrut" style="height:3.15em"></span><span class="mord"><span class="mop">ln</span><span class="mopen">(</span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">âˆ’</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord">0.99</span><span class="mclose">)</span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:1.69em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>N</mi><mo>=</mo><mfrac><mrow><mo>âˆ’</mo><mn>4.61</mn></mrow><mrow><mo>âˆ’</mo><mn>6.7</mn><mo>Ã—</mo><mn>1</mn><msup><mn>0</mn><mrow><mo>âˆ’</mo><mn>6</mn></mrow></msup></mrow></mfrac></mrow><annotation encoding="application/x-tex">N = \frac{-4.61}{-6.7 \times 10^{-6}}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.10903em">N</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:2.0908em;vertical-align:-0.7693em"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.3214em"><span style="top:-2.314em"><span class="pstrut" style="height:3em"></span><span class="mord"><span class="mord">âˆ’</span><span class="mord">6.7</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">Ã—</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mord">1</span><span class="mord"><span class="mord">0</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.7401em"><span style="top:-2.989em;margin-right:0.05em"><span class="pstrut" style="height:2.7em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">âˆ’</span><span class="mord mtight">6</span></span></span></span></span></span></span></span></span></span></span><span style="top:-3.23em"><span class="pstrut" style="height:3em"></span><span class="frac-line" style="border-bottom-width:0.04em"></span></span><span style="top:-3.677em"><span class="pstrut" style="height:3em"></span><span class="mord"><span class="mord">âˆ’</span><span class="mord">4.61</span></span></span></span><span class="vlist-s">â€‹</span></span><span class="vlist-r"><span class="vlist" style="height:0.7693em"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>N</mi><mo>=</mo><mn>688</mn><mo separator="true">,</mo><mn>060</mn></mrow><annotation encoding="application/x-tex">N = 688,060</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.6833em"></span><span class="mord mathnormal" style="margin-right:0.10903em">N</span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:0.8389em;vertical-align:-0.1944em"></span><span class="mord">688</span><span class="mpunct">,</span><span class="mspace" style="margin-right:0.1667em"></span><span class="mord">060</span></span></span></span></span>
<span class="mb-4 block break-words text-[1em] leading-7">clones</span>
<span class="mb-4 block break-words text-[1em] leading-7">Thus, approximately 688,060 clones are required to ensure a 99% probability that a given DNA sequence from this three billion basepair genome will be present in a library using a vector with an insert size of twenty thousand basepairs.</span>
<h2 id="applications" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Applications<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">After a library is created, the genome of an organism can be <span class="inline text-[1em] leading-7">sequenced</span> to elucidate how genes affect an organism or to compare similar organisms at the genome-level. The aforementioned <span class="inline text-[1em] leading-7">genome-wide association studies</span> can identify candidate genes stemming from many functional traits. Genes can be isolated through genomic libraries and used on human cell lines or animal models to further research.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6tabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[15]</sup> Furthermore, creating high-fidelity clones with accurate genome representation and no stability issues would contribute well as intermediates for <span class="inline text-[1em] leading-7">shotgun sequencing</span> or the study of complete genes in functional analysis.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_atabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[11]</sup></span>
<h3 id="hierarchical-sequencing" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Hierarchical sequencing<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">One major use of genomic libraries is <span class="inline text-[1em] leading-7">hierarchichal shotgun sequencing</span>, which is also called top-down, map-based or clone-by-clone sequencing. This strategy was developed in the 1980s for sequencing whole genomes before high throughput techniques for sequencing were available. Individual clones from genomic libraries can be sheared into smaller fragments, usually 500bp to 1000bp, which are more manageable for sequencing.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4uabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup> Once a clone from a genomic library is sequenced, the sequence can be used to screen the library for other clones containing inserts which overlap with the sequenced clone. Any new overlapping clones can then be sequenced forming a <span class="inline text-[1em] leading-7">contig</span>. This technique, called <span class="inline text-[1em] leading-7">chromosome walking</span>, can be exploited to sequence entire chromosomes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_auabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Whole genome shotgun sequencing</span> is another method of genome sequencing that does not require a library of high-capacity vectors. Rather, it uses computer algorithms to assemble short sequence reads to cover the entire genome. Genomic libraries are often used in combination with whole genome shotgun sequencing for this reason. A high resolution map can be created by sequencing both ends of inserts from several clones in a genomic library. This map provides sequences of known distances apart, which can be used to help with the assembly of sequence reads acquired through shotgun sequencing.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3uqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup> The human genome sequence, which was declared complete in 2003, was assembled using both a BAC library and shotgun sequencing.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5uqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6uqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[9]</sup></span>
<h3 id="genome-wide-association-studies" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Genome-wide association studies<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Genome-wide association studies</span> are general applications to find specific gene targets and polymorphisms within the human race. In fact, the International HapMap project was created through a partnership of scientists and agencies from several countries to catalog and utilize this data.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3vqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[17]</sup> The goal of this project is to compare genetic sequences of different individuals to elucidate similarities and differences within chromosomal regions.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5vqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[17]</sup> The project, which completed its final phase in 2010 and paved the way for larger initiatives like the 1000 Genomes Project, cataloged these attributes with data from populations of African, Asian, and European ancestry. Such genome-wide assessments may lead to further diagnostic and drug therapies while also helping future teams focus on orchestrating therapeutics with genetic features in mind. These concepts are already being exploited in <span class="inline text-[1em] leading-7">genetic engineering</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_9vqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[17]</sup> For example, a research team has actually constructed a PAC shuttle vector that creates a library representing two-fold coverage of the human genome.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_bvqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[15]</sup> This could serve as an incredible resource to identify genes, or sets of genes, causing disease. Moreover, these studies can serve as a powerful way to investigate transcriptional regulation as it has been seen in the study of baculoviruses.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_dvqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[18]</sup> Overall, advances in genome library construction and <span class="inline text-[1em] leading-7">DNA sequencing</span> has allowed for efficient discovery of different molecular targets.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_hvqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[5]</sup> Assimilation of these features through such efficient methods can hasten the employment of novel drug candidates.</span></article><div id="references" class="min-w-0 scroll-mt-8 overflow-hidden"><div class="text-[16px]"><h2 id="references" node="[object Object]" class="mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden">References</h2></div><ol class="columns-1 gap-x-12 [counter-reset:item] md:columns-2"><li id="1" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Losick, Richard; Watson, James D.; Tania A. Baker; Bell, Stephen; Gann, Alexander; Levine, Michael W. (2008). <em>Molecular biology of the gene</em>. San Francisco: Pearson/Benjamin Cummings. <span class="inline text-[1em] leading-7">ISBN</span> 978-0-8053-9592-1.</span></div></li><li id="2" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Russell, David W.; Sambrook, Joseph (2001). <em>Molecular cloning: a laboratory manual</em>. Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory. <span class="inline text-[1em] leading-7">ISBN</span> 978-0-87969-577-4.</span></div></li><li id="3" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Hartwell, Leland (2008). <a href="https://archive.org/details/genetics00lela_0" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300"><em>Genetics: from genes to genomes</em></a>. Boston: McGraw-Hill Higher Education. <span class="inline text-[1em] leading-7">ISBN</span> 978-0-07-284846-5.</span></div></li><li id="4" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Muse, Spencer V.; Gibson, Greg (2004). <a href="https://archive.org/details/primerofgenomesc00greg_0" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300"><em>A primer of genome science</em></a>. Sunderland, Mass: Sinauer Associates. <span class="inline text-[1em] leading-7">ISBN</span> 978-0-87893-232-0.</span></div></li><li id="5" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Henry RJ, Edwards M, Waters DL, et al. (November 2012). &quot;Application of large-scale sequencing to marker discovery in plants&quot;. <em>J. Biosci</em>. <strong>37</strong> (5): 829â€“41. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1007%2Fs12038-012-9253-z" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1007/s12038-012-9253-z</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/23107919" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">23107919</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:14457634" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">14457634</a>.</span></div></li><li id="6" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Sanger F, Air GM, Barrell BG, et al. (February 1977). &quot;Nucleotide sequence of bacteriophage phi X174 DNA&quot;. <em>Nature</em>. <strong>265</strong> (5596): 687â€“95. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1977Natur.265..687S" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1977Natur.265..687S</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2F265687a0" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/265687a0</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/870828" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">870828</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:4206886" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4206886</a>.</span></div></li><li id="14" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="http://www.ucl.ac.uk/~ucbhjow/b200/Cloning_genomic_DNA.doc" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Cloning Genomic DNA&quot;</a>. University College London. Retrieved 13 March 2013.</span></div></li><li id="15" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://web.archive.org/web/20130331122119/http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Gene libraries&quot;</a>. Archived from <a href="http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on 2013-03-31. Retrieved 2013-06-05.</span></div></li><li id="19" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Pennisi E</span> (April 2003). <a href="https://doi.org/10.1126%2Fscience.300.5618.409" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Human genome. Reaching their goal early, sequencing labs celebrate&quot;</a>. <em>Science</em>. <strong>300</strong> (5618): 409. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1126%2Fscience.300.5618.409" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1126/science.300.5618.409</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12702850" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12702850</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:206577479" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">206577479</a>.</span></div></li><li id="9" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Yoo EY, Kim S, Kim JY, Kim BD (August 2001). <a href="https://doi.org/10.1016%2FS1016-8478%2823%2917070-1" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Construction and characterization of a bacterial artificial chromosome library from chili pepper&quot;</a>. <em>Mol. Cells</em>. <strong>12</strong> (1): 117â€“20. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2FS1016-8478%2823%2917070-1" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/S1016-8478(23)17070-1</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11561720" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11561720</a>.</span></div></li><li id="10" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Osoegawa K, de Jong PJ, Frengen E, Ioannou PA (May 2001). &quot;Construction of bacterial artificial chromosome (BAC/PAC) libraries&quot;. <em>Curr Protoc Hum Genet</em>. Chapter 5: 5.15.1â€“5.15.33. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1002%2F0471142905.hg0515s21" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1002/0471142905.hg0515s21</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/18428289" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">18428289</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:8208834" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8208834</a>.</span></div></li><li id="11" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">John R. McCarrey; Williams, Steven J.; Barton E. Slatko (2006). <em>Laboratory investigations in molecular biology</em>. Boston: Jones and Bartlett Publishers. <span class="inline text-[1em] leading-7">ISBN</span> 978-0-7637-3329-2.</span></div></li><li id="12" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Peterson, Daniel; Jeffrey Tomkins; David Frisch (2000). <a href="https://web.archive.org/web/20090115122947/http://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Construction of Plant Bacterial Artificial Chromosome (BAC) Libraries: An Illustrated Guide&quot;</a>. <em>Journal of Agricultural Genomics</em>. <strong>5</strong>. Archived from <a href="https://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on January 15, 2009.</span></div></li><li id="16" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Blaber, Michael. <a href="https://web.archive.org/web/20150513050011/http://www.mikeblaber.org/oldwine/bch5425/lect28/lect28.htm" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Genomic Libraries&quot;</a>. Archived from the original on 13 May 2015. Retrieved 1 April 2013.</span></div></li><li id="17" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Fuesler J, Nagahama Y, Szulewski J, Mundorff J, Bireley S, Coren JS (April 2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;An arrayed human genomic library constructed in the PAC shuttle vector pJCPAC-Mam2 for genome-wide association studies and gene therapy&quot;</a>. <em>Gene</em>. <strong>496</strong> (2): 103â€“9. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2Fj.gene.2012.01.011" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/j.gene.2012.01.011</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3488463</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/22285925" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">22285925</a>.</span></div></li><li id="18" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Pareek CS, Smoczynski R, Tretyn A (November 2011). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Sequencing technologies and genome sequencing&quot;</a>. <em>J. Appl. Genet</em>. <strong>52</strong> (4): 413â€“35. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1007%2Fs13353-011-0057-x" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1007/s13353-011-0057-x</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3189340</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/21698376" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">21698376</a>.</span></div></li><li id="20" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://web.archive.org/web/20090925032755/http://hapmap.ncbi.nlm.nih.gov/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;HapMap Homepage&quot;</a>. Archived from <a href="http://hapmap.ncbi.nlm.nih.gov/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on September 25, 2009.</span></div></li><li id="21" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Chen Y, Lin X, Yi Y, Lu Y, Zhang Z (2009). <a href="https://doi.org/10.1515%2Fznc-2009-7-817" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Construction and application of a baculovirus genomic library&quot;</a>. <em>Z. Naturforsch. C</em>. <strong>64</strong> (7â€“8): 574â€“80. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1515%2Fznc-2009-7-817" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1515/znc-2009-7-817</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/19791511" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">19791511</a>.</span></div></li></ol></div><div class="text-fg-secondary text-xs [&amp;_*]:mb-2"><hr class="border-border-l1 my-8 rounded-sm border-t"/>
<span class="mb-4 block break-words text-[1em] leading-7">The content is adapted from <a href="https://en.wikipedia.org/wiki/Genomic_library" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">Wikipedia</a>, licensed under Creative Commons Attribution-ShareAlike 4.0 License.</span></div></div></div><div class="hidden min-[1400px]:block"></div></div></div><!--$--><!--/$--><div role="region" aria-label="Notifications (F8)" tabindex="-1" style="pointer-events:none"><ol tabindex="-1" class="pointer-events-none fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 max-sm:left-0 sm:bottom-0 sm:right-0 sm:top-auto sm:max-w-[420px] sm:flex-col pl-16 pr-2 pt-16 sm:px-4"></ol></div><script type="application/json" id="server-client-data-experimentation">{"status":"uninitialized"}</script><script src="/_next/static/chunks/webpack-e121ed42680f327e.js" nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl" id="_R_" async=""></script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">(self.__next_f=self.__next_f||[]).push([0])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"1:\"$Sreact.fragment\"\n3:I[91363,[],\"\"]\n4:I[23775,[],\"\"]\n5:I[57654,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"PageHeaderProvider\"]\n6:I[17618,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"HeaderContent\"]\n7:I[25529,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"6751\",\"static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js\"],\"default\"]\n9:I[6666,[],\"OutletBoundary\"]\nb:I[80415,[],\"AsyncMetadataOutlet\"]\nd:I[6666,[],\"ViewportBoundary\"]\nf:I[6666,[],\"MetadataBoundary\"]\n10:\"$Sreact.suspense\"\n12:I[95909,[\"4219\",\"static/chunks/app/global-error-4d07d20223cd4b4c.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"13:I[51498,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"ConstantsProvider\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"14:I[91073,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"15:I[38642,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"16:I[99648,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"17:I[78825,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"18:I[8550,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MixpanelProvider\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"19:I[12290,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MobileTocProvider\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"1a:I[53947,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"NuqsAdapter\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"1b:I[91873,[\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8039\",\"static/chunks/app/error-4a29e9399afba038.js\"],\"default\"]\n1c:I[5091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"4345\",\"static/chunks/app/not-found-dd95690acf732f18.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"1d:I[16091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,":HL[\"/_next/static/media/1f2316909698f815.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/3d4419af2cf8609b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/4dec29efcaeb336c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/74452ea3ef0f9101.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/904ef0a86fe32a00.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/d886a03bcda7ad8f.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/e1447589d6f59c4b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/f5a90156f8995c8c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/css/62c4caba71dfda84.css\",\"style\",{\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]\n:HL[\"/_next/static/css/eb3d87f98fe1565f.css\",\"style\",{\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]\n:HL[\"/_next/static/css/1b5e561215938d4d.css\",\"style\",{\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]\n:HL[\"/_next/static/css/0227d069a630d414.css\",\"style\",{\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]\n:HL[\"/_next/static/css/f87fff2ab93d05a7.css\",\"style\",{\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"0:{\"P\":null,\"b\":\"BpM29AX4fmgcUxZ_6t0le\",\"p\":\"\",\"c\":[\"\",\"page\",\"Genomic_library\"],\"i\":false,\"f\":[[[\"\",{\"children\":[\"page\",{\"children\":[[\"slug\",\"Genomic_library\",\"d\"],{\"children\":[\"__PAGE__\",{}]}]}]},\"$undefined\",\"$undefined\",true],[\"\",[\"$\",\"$1\",\"c\",{\"children\":[[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/62c4caba71dfda84.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}],[\"$\",\"link\",\"1\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/eb3d87f98fe1565f.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}],[\"$\",\"link\",\"2\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/1b5e561215938d4d.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}],[\"$\",\"link\",\"3\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/0227d069a630d414.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]],\"$L2\"]}],{\"children\":[\"page\",[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[[\"slug\",\"Genomic_library\",\"d\"],[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L5\",null,{\"children\":[[\"$\",\"$L6\",null,{\"maxWidth\":\"full\",\"mobileOptions\":{\"right\":{\"showFixedIssues\":true,\"showThemeSwitcher\":true,\"showSearch\":true,\"showTableOfContents\":true}}}],[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L7\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}]]}],{\"children\":[\"__PAGE__\",[\"$\",\"$1\",\"c\",{\"children\":[\"$L8\",[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/f87fff2ab93d05a7.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\"}]],[\"$\",\"$L9\",null,{\"children\":[\"$La\",[\"$\",\"$Lb\",null,{\"promise\":\"$@c\"}]]}]]}],{},null,false]},null,false]},null,false]},null,false],[\"$\",\"$1\",\"h\",{\"children\":[null,[[\"$\",\"$Ld\",null,{\"children\":\"$Le\"}],[\"$\",\"meta\",null,{\"name\":\"next-size-adjust\",\"content\":\"\"}]],[\"$\",\"$Lf\",null,{\"children\":[\"$\",\"div\",null,{\"hidden\":true,\"children\":[\"$\",\"$10\",null,{\"fallback\":null,\"children\":\"$L11\"}]}]}]]}],false]],\"m\":\"$undefined\",\"G\":[\"$12\",[]],\"s\":false,\"S\":false}\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"2:[\"$\",\"html\",null,{\"lang\":\"en\",\"className\":\"bg-surface-base antialiased\",\"suppressHydrationWarning\":true,\"children\":[[\"$\",\"head\",null,{\"children\":[[\"$\",\"title\",null,{\"children\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"type\":\"image/x-icon\",\"href\":\"/favicon.ico\",\"sizes\":\"48x48\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-dark.png\",\"media\":\"(prefers-color-scheme: light)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-light.png\",\"media\":\"(prefers-color-scheme: dark)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"apple-touch-icon\",\"href\":\"/icon-192x192.png\"}],[\"$\",\"link\",null,{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\"}],[\"$\",\"meta\",null,{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",null,{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"meta\",null,{\"property\":\"og:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"property\":\"og:url\",\"content\":\"https://grokipedia.com\"}],[\"$\",\"meta\",null,{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:locale\",\"content\":\"en_US\"}],[\"$\",\"meta\",null,{\"property\":\"og:type\",\"content\":\"website\"}],[\"$\",\"meta\",null,{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:alt\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}]]}],[\"$\",\"body\",null,{\"className\":\"flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b\",\"children\":[[\"$\",\"$L13\",null,{\"children\":[\"$\",\"$L14\",null,{\"children\":[\"$\",\"$L15\",null,{\"defaultTheme\":\"dark\",\"nonce\":\"OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl\",\"children\":[\"$\",\"$L16\",null,{\"children\":[\"$\",\"$L17\",null,{\"children\":[\"$\",\"$L18\",null,{\"children\":[\"$\",\"$L19\",null,{\"children\":[\"$\",\"$L1a\",null,{\"children\":[[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$1b\",\"errorStyles\":[],\"errorScripts\":[],\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L1c\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}],[\"$\",\"$L1d\",null,{}]]}]}]}]}]}]}]}]}],[\"$\",\"script\",null,{\"type\":\"application/json\",\"id\":\"server-client-data-experimentation\",\"children\":\"{\\\"status\\\":\\\"uninitialized\\\"}\"}]]}]]}]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"e:[[\"$\",\"meta\",\"0\",{\"charSet\":\"utf-8\"}],[\"$\",\"meta\",\"1\",{\"name\":\"viewport\",\"content\":\"width=device-width, initial-scale=1, interactive-widget=resizes-content\"}]]\na:null\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"1e:I[42712,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"HydrationBoundary\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"21:I[6367,[],\"IconMark\"]\n1f:T4987,"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"# Genomic library\n\nA **genomic library** is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism's DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA ligase.[1] Next, the vector DNA can be taken up by a host organism - commonly a population of Escherichia coli or yeast - with each cell containing only one vector molecule. Using a host cell to carry the vector allows for easy amplification and retrieval of specific clones from the library for analysis.[2]\n\nThere are several kinds of vectors available with various insert capacities. Generally, libraries made from organisms with larger genomes require vectors featuring larger inserts, thereby fewer vector molecules are needed to make the library. Researchers can choose a vector also considering the ideal insert size to find the desired number of clones necessary for full genome coverage.[3]\n\nGenomic libraries are commonly used for sequencing applications. They have played an important role in the whole genome sequencing of several organisms, including the human genome and several model organisms.[4][5]\n## History\n\nThe first DNA-based genome ever fully sequenced was that of the bacteriophage phi X 174, achieved by Frederick Sanger and his team in 1977 using the plus and minus sequencing method.[6] This milestone highlighted the potential of DNA sequencing technologies.\n\nGenomic libraries emerged in the mid-to-late 1970s as part of the development of recombinant DNA technology. Early libraries were constructed using bacterial plasmids and phage lambda vectors to clone overlapping DNA fragments, enabling systematic study and sequencing of larger genomes.[14][15] These advances facilitated subsequent whole-genome sequencing projects, including those for model organisms and the human genome.[4][19]\n\nPrior to genomic libraries, gene isolation relied on more limited techniques such as partial purification or small-scale cloning.\n# Genomic library construction\n\nConstruction of a genomic library involves creating many recombinant DNA molecules. An organism's genomic DNA is extracted and then partially digested with a restriction enzyme. For very small genomes, such as those of some viruses (~5-10 kb), the digested fragments can be separated by gel electrophoresis. The separated fragments can then be excised and cloned into the vector separately. However, when a large genome is partially digested with a restriction enzyme, there are far too many fragments to excise individually. The entire set of fragments must be cloned together with the vector, and separation of clones can occur after. In either case, the fragments are ligated into a vector that has been digested with a compatible restriction enzyme. The vector containing the inserted fragments of genomic DNA can then be introduced into a host organism.[1]\n\nBelow are the steps for creating a genomic library from a large genome. \n\n1. Extract and purify DNA.\n2. Partially digest the DNA with a restriction enzyme. This creates overlapping fragments that are similar in size, each containing one or more genes.\n3. Size-select the fragments (e.g., via gel electrophoresis or sucrose density gradient centrifugation) to obtain the desired length range for the vector.\n4. Insert the fragments of DNA into vectors that were cut with a compatible restriction enzyme. Use the enzyme DNA ligase to seal the DNA fragments into the vector. This creates a large pool of recombinant molecules.\n5. These recombinant molecules are taken up by a host bacterium by transformation, creating a DNA library.[9][10]\n\n### Determining titer of library\n\nAfter a genomic library is constructed with a viral vector, such as lambda phage, the titer of the library can be determined. Calculating the titer allows researchers to approximate how many infectious viral particles were successfully created in the library. To do this, dilutions of the library are used to infect cultures of E. coli of known concentrations. The cultures are then plated on agar plates and incubated overnight. The number of viral plaques are counted and can be used to calculate the total number of infectious viral particles in the library. Most viral vectors also carry a marker that allows clones containing an insert to be distinguished from those that do not have an insert. This allows researchers to also determine the percentage of infectious viral particles actually carrying a fragment of the library.[11]\n\nA similar method can be used to titer genomic libraries made with non-viral vectors, such as plasmids and BACs. A test ligation of the library can be used to transform E. coli. The transformation is then spread on agar plates and incubated overnight. The titer of the transformation is determined by counting the number of colonies present on the plates. These vectors generally have a selectable marker allowing the differentiation of clones containing an insert from those that do not. By doing this test, researchers can also determine the efficiency of the ligation and make adjustments as needed to ensure they get the desired number of clones for the library.[12]\n\n### Screening library\n\nIn order to isolate clones that contain regions of interest from a library, the library must first be screened. One method of screening is hybridization. Each transformed host cell of a library will contain only one vector with one insert of DNA. The whole library can be plated onto a filter over media. The filter and colonies are prepared for hybridization and then hybridized with a labeled probe. The target DNA-insert of interest-can be identified by detection such as autoradiography because of the hybridization with the probe as seen below. \n\nAnother method of screening is with polymerase chain reaction (PCR). Some libraries are stored as pools of clones and screening by PCR is an efficient way to identify pools containing specific clones.[2]\n# Types of vectors\n\nGenome size varies among different organisms and the cloning vector must be selected accordingly. For a large genome, a vector with a large capacity should be chosen so that a relatively small number of clones are sufficient for coverage of the entire genome. However, it is often more difficult to characterize an insert contained in a higher capacity vector.[3]\n\nBelow is a table of several kinds of vectors commonly used for genomic libraries and the insert size that each generally holds. \n\n| Vector type                          | Insert size (thousands of bases) |\n|--------------------------------------|----------------------------------|\n| Plasmids                             | up to 15                         |\n| Phage lambda (Î»)                     | up to 25                         |\n| Cosmids                              | up to 45                         |\n| Bacteriophage P1                     | 70 to 100                        |\n| P1 artificial chromosomes (PACs)     | 130 to 150                       |\n| Bacterial artificial chromosomes (BACs) | 120 to 300                    |\n| Yeast artificial chromosomes (YACs)  | 250 to 2000                      |\n  \n### Plasmids\n\nA plasmid is a double stranded circular DNA molecule commonly used for molecular cloning. Plasmids are generally 2 to 4 kilobase-pairs (kb) in length and are capable of carrying inserts up to 15kb. Plasmids contain an origin of replication allowing them to replicate inside a bacterium independently of the host chromosome. Plasmids commonly carry a gene for antibiotic resistance that allows for the selection of bacterial cells containing the plasmid. Many plasmids also carry a reporter gene that allows researchers to distinguish clones containing an insert from those that do not.[3]\n\n### Phage lambda (Î»)\n\nPhage Î» is a double-stranded DNA virus that infects E. coli. The Î» chromosome is 48.5kb long and can carry inserts up to 25kb. These inserts replace non-essential viral sequences in the Î» chromosome, while the genes required for formation of viral particles and infection remain intact. The insert DNA is replicated with the viral DNA; thus, together they are packaged into viral particles. These particles are very efficient at infection and multiplication leading to a higher production of the recombinant Î» chromosomes.[3] However, due to the smaller insert size, libraries made with Î» phage may require many clones for full genome coverage.[14]\n\n### Cosmids\n\nCosmid vectors are plasmids that contain a small region of bacteriophage Î» DNA called the cos sequence. This sequence allows the cosmid to be packaged into bacteriophage Î» particles. These particles- containing a linearized cosmid- are introduced into the host cell by transduction. Once inside the host, the cosmids circularize with the aid of the host's DNA ligase and then function as plasmids. Cosmids are capable of carrying inserts up to 45kb in size.[2]\n\n### Bacteriophage P1 vectors\n\nBacteriophage P1 vectors can hold inserts 70 â€“ 100kb in size. They begin as linear DNA molecules packaged into bacteriophage P1 particles. These particles are injected into an E. coli strain expressing Cre recombinase. The linear P1 vector becomes circularized by recombination between two loxP sites in the vector. P1 vectors generally contain a gene for antibiotic resistance and a positive selection marker to distinguish clones containing an insert from those that do not. P1 vectors also contain a P1 plasmid replicon, which ensures only one copy of the vector is present in a cell. However, there is a second P1 replicon- called the P1 lytic replicon- that is controlled by an inducible promoter. This promoter allows the amplification of more than one copy of the vector per cell prior to DNA extraction.[2]\n\n### P1 artificial chromosomes\n\nP1 artificial chromosomes (PACs) have features of both P1 vectors and Bacterial Artificial Chromosomes (BACs). Similar to P1 vectors, they contain a plasmid and a lytic replicon as described above. Unlike P1 vectors, they do not need to be packaged into bacteriophage particles for transduction. Instead they are introduced into E. coli as circular DNA molecules through electroporation just as BACs are.[2] Also similar to BACs, these are relatively harder to prepare due to a single origin of replication.[14]\n\n### Bacterial artificial chromosomes\n\nBacterial artificial chromosomes (BACs) are circular DNA molecules, usually about 7kb in length, that are capable of holding inserts up to 300kb in size. BAC vectors contain a replicon derived from E. coli F factor, which ensures they are maintained at one copy per cell.[4] Once an insert is ligated into a BAC, the BAC is introduced into recombination deficient strains of E. coli by electroporation. Most BAC vectors contain a gene for antibiotic resistance and also a positive selection marker.[2] The figure to the right depicts a BAC vector being cut with a restriction enzyme, followed by the insertion of foreign DNA that is re-annealed by a ligase. Overall, this is a very stable vector, but they may be hard to prepare due to a single origin of replication just like PACs.[14]\n\n### Yeast artificial chromosomes\n\nYeast artificial chromosomes (YACs) are linear DNA molecules containing the necessary features of an authentic yeast chromosome, including telomeres, a centromere, and an origin of replication. Large inserts of DNA can be ligated into the middle of the YAC so that there is an â€œarmâ€ of the YAC on either side of the insert. The recombinant YAC is introduced into yeast by transformation; selectable markers present in the YAC allow for the identification of successful transformants. YACs can hold inserts up to 2000kb, but most YAC libraries contain inserts 250-400kb in size. Theoretically there is no upper limit on the size of insert a YAC can hold. It is the quality in the preparation of DNA used for inserts that determines the size limit.[2] The most challenging aspect of using YAC is the fact they are prone to rearrangement.[14]\n## How to select a vector\n\nVector selection requires one to ensure the library made is representative of the entire genome. Any insert of the genome derived from a restriction enzyme should have an equal chance of being in the library compared to any other insert. Furthermore, recombinant molecules should contain large enough inserts ensuring the library size is able to be handled conveniently.[14] This is particularly determined by the number of clones needed to have in a library. The number of clones to get a sampling of all the genes is determined by the size of the organism's genome as well as the average insert size. This is represented by the formula (also known as the Carbon and Clarke formula):[15]\n\n\\[\nN = \\frac{\\ln(1 - P)}{\\ln(1 - f)}\n\\]\n\nwhere, \n\n\\( N \\) is the necessary number of recombinants[16]\n\n\\( P \\) is the desired probability that any fragment in the genome will occur at least once in the library created \n\n\\( f \\) is the fractional proportion of the genome in a single recombinant \n\n\\( f \\) can be further shown to be: \n\n\\[\nf = \\frac{i}{g}\n\\]\n\nwhere, \n\n\\( i \\) is the insert size \n\n\\( g \\) is the genome size \n\nThus, increasing the insert size (by choice of vector) would allow for fewer clones needed to represent a genome. The proportion of the insert size versus the genome size represents the proportion of the respective genome in a single clone.[14] Here is the equation with all parts considered: \n\n\\[\nN = \\frac{\\ln(1 - P)}{\\ln\\left(1 - \\frac{i}{g}\\right)}\n\\]\n\n### Vector selection example\n\nThe above formula can be used to determine the 99% confidence level that all sequences in a genome are represented by using a vector with an insert size of twenty thousand basepairs (such as the phage lambda vector). The genome size of the organism is three billion basepairs in this example. \n\n\\[\nN = \\frac{\\ln(1 - 0.99)}{\\ln\\left[1 - \\frac{2.0 \\times 10^{4} \\text{ basepairs}}{3.0 \\times 10^{9} \\text{ basepairs}}\\right]}\n\\]\n\n\\[\nN = \\frac{-4.61}{-6.7 \\times 10^{-6}}\n\\]\n\n\\[\nN = 688,060\n\\]\n\nclones \n\nThus, approximately 688,060 clones are required to ensure a 99% probability that a given DNA sequence from this three billion basepair genome will be present in a library using a vector with an insert size of twenty thousand basepairs.\n## Applications\n\nAfter a library is created, the genome of an organism can be [sequenced](Sequencing#DNA_sequencing \"Sequencing\") to elucidate how genes affect an organism or to compare similar organisms at the genome-level. The aforementioned [genome-wide association studies](Genome-wide_association_study \"Genome-wide association study\") can identify candidate genes stemming from many functional traits. Genes can be isolated through genomic libraries and used on human cell lines or animal models to further research.[17] Furthermore, creating high-fidelity clones with accurate genome representation and no stability issues would contribute well as intermediates for [shotgun sequencing](Shotgun_sequencing \"Shotgun sequencing\") or the study of complete genes in functional analysis.[10]\n\n### Hierarchical sequencing\n\nOne major use of genomic libraries is [hierarchichal shotgun sequencing](Shotgun_sequencing#Hierarchical_Shotgun_sequencing \"Shotgun sequencing\"), which is also called top-down, map-based or clone-by-clone sequencing. This strategy was developed in the 1980s for sequencing whole genomes before high throughput techniques for sequencing were available. Individual clones from genomic libraries can be sheared into smaller fragments, usually 500bp to 1000bp, which are more manageable for sequencing.[4] Once a clone from a genomic library is sequenced, the sequence can be used to screen the library for other clones containing inserts which overlap with the sequenced clone. Any new overlapping clones can then be sequenced forming a [contig](Contig \"Contig\"). This technique, called [chromosome walking](Primer_walking \"Primer walking\"), can be exploited to sequence entire chromosomes.[2]\n\n[Whole genome shotgun sequencing](Shotgun_sequencing#Whole_genome_shotgun_sequencing \"Shotgun sequencing\") is another method of genome sequencing that does not require a library of high-capacity vectors. Rather, it uses computer algorithms to assemble short sequence reads to cover the entire genome. Genomic libraries are often used in combination with whole genome shotgun sequencing for this reason. A high resolution map can be created by sequencing both ends of inserts from several clones in a genomic library. This map provides sequences of known distances apart, which can be used to help with the assembly of sequence reads acquired through shotgun sequencing.[4] The human genome sequence, which was declared complete in 2003, was assembled using both a BAC library and shotgun sequencing.[18][19]\n\n### Genome-wide association studies\n\n[Genome-wide association studies](Genome-wide_association_study \"Genome-wide association study\") are general applications to find specific gene targets and polymorphisms within the human race. In fact, the International HapMap project was created through a partnership of scientists and agencies from several countries to catalog and utilize this data.[20] The goal of this project is to compare genetic sequences of different individuals to elucidate similarities and differences within chromosomal regions.[20] The project, which completed its final phase in 2010 and paved the way for larger initiatives like the 1000 Genomes Project, cataloged these attributes with data from populations of African, Asian, and European ancestry. Such genome-wide assessments may lead to further diagnostic and drug therapies while also helping future teams focus on orchestrating therapeutics with genetic features in mind. These concepts are already being exploited in [genetic engineering](Genetic_engineering \"Genetic engineering\").[20] For example, a research team has actually constructed a PAC shuttle vector that creates a library representing two-fold coverage of the human genome.[17] This could serve as an incredible resource to identify genes, or sets of genes, causing disease. Moreover, these studies can serve as a powerful way to investigate transcriptional regulation as it has been seen in the study of baculoviruses.[21] Overall, advances in genome library construction and [DNA sequencing](DNA_sequencing \"DNA sequencing\") has allowed for efficient discovery of different molecular targets.[5] Assimilation of these features through such efficient methods can hasten the employment of novel drug candidates."])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"8:[\"$\",\"$L1e\",null,{\"state\":{\"mutations\":[],\"queries\":[{\"dehydratedAt\":1761886990518,\"state\":{\"data\":{\"page\":{\"citations\":[{\"id\":\"1\",\"title\":\"_**a**_ _**b**_ Losick, Richard; Watson, James D.; Tania A. Baker; Bell, Stephen; Gann, Alexander; Levine, Michael W. (2008). _Molecular biology of the gene_. San Francisco: Pearson/Benjamin Cummings. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-8053-9592-1.\",\"description\":\"1. ^ _**a**_ _**b**_ Losick, Richard; Watson, James D.; Tania A. Baker; Bell, Stephen; Gann, Alexander; Levine, Michael W. (2008). _Molecular biology of the gene_. San Francisco: Pearson/Benjamin Cummings. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-8053-9592-1.\",\"url\":\"ISBN_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"2\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ _**g**_ _**h**_ Russell, David W.; Sambrook, Joseph (2001). _Molecular cloning: a laboratory manual_. Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-87969-577-4.\",\"description\":\"2. ^ _**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ _**g**_ _**h**_ Russell, David W.; Sambrook, Joseph (2001). _Molecular cloning: a laboratory manual_. Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-87969-577-4.\",\"url\":\"ISBN_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"3\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ Hartwell, Leland (2008). [_Genetics: from genes to genomes_](https://archive.org/details/genetics00lela_0). Boston: McGraw-Hill Higher Education. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-07-284846-5.\",\"description\":\"3. ^ _**a**_ _**b**_ _**c**_ _**d**_ Hartwell, Leland (2008). [_Genetics: from genes to genomes_](https://archive.org/details/genetics00lela_0). Boston: McGraw-Hill Higher Education. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-07-284846-5.\",\"url\":\"https://archive.org/details/genetics00lela_0\",\"favicon\":\"\"},{\"id\":\"4\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ Muse, Spencer V.; Gibson, Greg (2004). [_A primer of genome science_](https://archive.org/details/primerofgenomesc00greg_0). Sunderland, Mass: Sinauer Associates. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-87893-232-0.\",\"description\":\"4. ^ _**a**_ _**b**_ _**c**_ _**d**_ Muse, Spencer V.; Gibson, Greg (2004). [_A primer of genome science_](https://archive.org/details/primerofgenomesc00greg_0). Sunderland, Mass: Sinauer Associates. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-87893-232-0.\",\"url\":\"https://archive.org/details/primerofgenomesc00greg_0\",\"favicon\":\"\"},{\"id\":\"5\",\"title\":\"_**a**_ _**b**_ Henry RJ, Edwards M, Waters DL, et al. (November 2012). \\\"Application of large-scale sequencing to marker discovery in plants\\\". _J. Biosci_. **37** (5): 829â€“41\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/s12038-012-9253-z](https://doi.org/10.1007%2Fs12038-012-9253-z). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23107919](https://pubmed.ncbi.nlm.nih.gov/23107919). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [14457634](https://api.semanticscholar.org/CorpusID:14457634).\",\"description\":\"5. ^ _**a**_ _**b**_ Henry RJ, Edwards M, Waters DL, et al. (November 2012). \\\"Application of large-scale sequencing to marker discovery in plants\\\". _J. Biosci_. **37** (5): 829â€“41\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/s12038-012-9253-z](https://doi.org/10.1007%2Fs12038-012-9253-z). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23107919](https://pubmed.ncbi.nlm.nih.gov/23107919). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [14457634](https://api.semanticscholar.org/CorpusID:14457634).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"6\",\"title\":\"** Sanger F, Air GM, Barrell BG, et al. (February 1977). \\\"Nucleotide sequence of bacteriophage phi X174 DNA\\\". _Nature_. **265** (5596): 687â€“95\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1977Natur.265..687S](https://ui.adsabs.harvard.edu/abs/1977Natur.265..687S). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/265687a0](https://doi.org/10.1038%2F265687a0). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [870828](https://pubmed.ncbi.nlm.nih.gov/870828). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4206886](https://api.semanticscholar.org/CorpusID:4206886).\",\"description\":\"6. **^** Sanger F, Air GM, Barrell BG, et al. (February 1977). \\\"Nucleotide sequence of bacteriophage phi X174 DNA\\\". _Nature_. **265** (5596): 687â€“95\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1977Natur.265..687S](https://ui.adsabs.harvard.edu/abs/1977Natur.265..687S). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/265687a0](https://doi.org/10.1038%2F265687a0). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [870828](https://pubmed.ncbi.nlm.nih.gov/870828). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4206886](https://api.semanticscholar.org/CorpusID:4206886).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"7\",\"title\":\"** Menon R, Farina C (2011). [\\\"Shared molecular and functional frameworks among five complex human disorders: a comparative study on interactomes linked to susceptibility genes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080867). _PLOS ONE_. **6** (4): e18660. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2011PLoSO...618660M](https://ui.adsabs.harvard.edu/abs/2011PLoSO...618660M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.pone.0018660](https://doi.org/10.1371%2Fjournal.pone.0018660). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3080867](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080867). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [21533026](https://pubmed.ncbi.nlm.nih.gov/21533026).\",\"description\":\"7. **^** Menon R, Farina C (2011). [\\\"Shared molecular and functional frameworks among five complex human disorders: a comparative study on interactomes linked to susceptibility genes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080867). _PLOS ONE_. **6** (4): e18660. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2011PLoSO...618660M](https://ui.adsabs.harvard.edu/abs/2011PLoSO...618660M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.pone.0018660](https://doi.org/10.1371%2Fjournal.pone.0018660). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3080867](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080867). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [21533026](https://pubmed.ncbi.nlm.nih.gov/21533026).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080867\",\"favicon\":\"\"},{\"id\":\"8\",\"title\":\"** Cichon S, MÃ¼hleisen TW, Degenhardt FA, et al. (March 2011). [\\\"Genome-wide association study identifies genetic variation in neurocan as a susceptibility factor for bipolar disorder\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059436). _Am. J. Hum. Genet_. **88** (3): 372â€“81\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.ajhg.2011.01.017](https://doi.org/10.1016%2Fj.ajhg.2011.01.017). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3059436](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059436). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [21353194](https://pubmed.ncbi.nlm.nih.gov/21353194).\",\"description\":\"8. **^** Cichon S, MÃ¼hleisen TW, Degenhardt FA, et al. (March 2011). [\\\"Genome-wide association study identifies genetic variation in neurocan as a susceptibility factor for bipolar disorder\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059436). _Am. J. Hum. Genet_. **88** (3): 372â€“81\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.ajhg.2011.01.017](https://doi.org/10.1016%2Fj.ajhg.2011.01.017). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3059436](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059436). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [21353194](https://pubmed.ncbi.nlm.nih.gov/21353194).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059436\",\"favicon\":\"\"},{\"id\":\"9\",\"title\":\"** Yoo EY, Kim S, Kim JY, Kim BD (August 2001). [\\\"Construction and characterization of a bacterial artificial chromosome library from chili pepper\\\"](https://doi.org/10.1016%2FS1016-8478%2823%2917070-1). _Mol. Cells_. **12** (1): 117â€“20\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S1016-8478(23)17070-1](https://doi.org/10.1016%2FS1016-8478%2823%2917070-1). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11561720](https://pubmed.ncbi.nlm.nih.gov/11561720).\",\"description\":\"9. **^** Yoo EY, Kim S, Kim JY, Kim BD (August 2001). [\\\"Construction and characterization of a bacterial artificial chromosome library from chili pepper\\\"](https://doi.org/10.1016%2FS1016-8478%2823%2917070-1). _Mol. Cells_. **12** (1): 117â€“20\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S1016-8478(23)17070-1](https://doi.org/10.1016%2FS1016-8478%2823%2917070-1). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11561720](https://pubmed.ncbi.nlm.nih.gov/11561720).\",\"url\":\"https://doi.org/10.1016%2FS1016-8478%2823%2917070-1\",\"favicon\":\"\"},{\"id\":\"10\",\"title\":\"_**a**_ _**b**_ Osoegawa K, de Jong PJ, Frengen E, Ioannou PA (May 2001). \\\"Construction of bacterial artificial chromosome (BAC/PAC) libraries\\\". _Curr Protoc Hum Genet_. Chapter 5: 5.15.1â€“5.15.33. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1002/0471142905.hg0515s21](https://doi.org/10.1002%2F0471142905.hg0515s21). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [18428289](https://pubmed.ncbi.nlm.nih.gov/18428289). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [8208834](https://api.semanticscholar.org/CorpusID:8208834).\",\"description\":\"10. ^ _**a**_ _**b**_ Osoegawa K, de Jong PJ, Frengen E, Ioannou PA (May 2001). \\\"Construction of bacterial artificial chromosome (BAC/PAC) libraries\\\". _Curr Protoc Hum Genet_. Chapter 5: 5.15.1â€“5.15.33. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1002/0471142905.hg0515s21](https://doi.org/10.1002%2F0471142905.hg0515s21). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [18428289](https://pubmed.ncbi.nlm.nih.gov/18428289). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [8208834](https://api.semanticscholar.org/CorpusID:8208834).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"11\",\"title\":\"** John R. McCarrey; Williams, Steven J.; Barton E. Slatko (2006). _Laboratory investigations in molecular biology_. Boston: Jones and Bartlett Publishers. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-7637-3329-2.\",\"description\":\"11. **^** John R. McCarrey; Williams, Steven J.; Barton E. Slatko (2006). _Laboratory investigations in molecular biology_. Boston: Jones and Bartlett Publishers. [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-0-7637-3329-2.\",\"url\":\"ISBN_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"12\",\"title\":\"** Peterson, Daniel; Jeffrey Tomkins; David Frisch (2000). [\\\"Construction of Plant Bacterial Artificial Chromosome (BAC) Libraries: An Illustrated Guide\\\"](https://web.archive.org/web/20090115122947/http://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html). _Journal of Agricultural Genomics_. **5**. Archived from [the original](https://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html) on January 15, 2009.\",\"description\":\"12. **^** Peterson, Daniel; Jeffrey Tomkins; David Frisch (2000). [\\\"Construction of Plant Bacterial Artificial Chromosome (BAC) Libraries: An Illustrated Guide\\\"](https://web.archive.org/web/20090115122947/http://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html). _Journal of Agricultural Genomics_. **5**. Archived from [the original](https://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html) on January 15, 2009.\",\"url\":\"https://web.archive.org/web/20090115122947/http://wheat.pw.usda.gov/jag/papers00/paper300/indexpage300.html\",\"favicon\":\"\"},{\"id\":\"13\",\"title\":\"** Kim UJ, Birren BW, Slepak T, et al. (June 1996). \\\"Construction and characterization of a human bacterial artificial chromosome library\\\". _Genomics_. **34** (2): 213â€“8\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1006/geno.1996.0268](https://doi.org/10.1006%2Fgeno.1996.0268). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8661051](https://pubmed.ncbi.nlm.nih.gov/8661051).\",\"description\":\"13. **^** Kim UJ, Birren BW, Slepak T, et al. (June 1996). \\\"Construction and characterization of a human bacterial artificial chromosome library\\\". _Genomics_. **34** (2): 213â€“8\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1006/geno.1996.0268](https://doi.org/10.1006%2Fgeno.1996.0268). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8661051](https://pubmed.ncbi.nlm.nih.gov/8661051).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"14\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ [\\\"Cloning Genomic DNA\\\"](http://www.ucl.ac.uk/~ucbhjow/b200/Cloning_genomic_DNA.doc). University College London. Retrieved 13 March 2013.\",\"description\":\"14. ^ _**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ [\\\"Cloning Genomic DNA\\\"](http://www.ucl.ac.uk/~ucbhjow/b200/Cloning_genomic_DNA.doc). University College London. Retrieved 13 March 2013.\",\"url\":\"http://www.ucl.ac.uk/~ucbhjow/b200/Cloning_genomic_DNA.doc\",\"favicon\":\"\"},{\"id\":\"15\",\"title\":\"** [\\\"Gene libraries\\\"](https://web.archive.org/web/20130331122119/http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm). Archived from [the original](http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm) on 2013-03-31. Retrieved 2013-06-05.\",\"description\":\"15. **^** [\\\"Gene libraries\\\"](https://web.archive.org/web/20130331122119/http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm). Archived from [the original](http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm) on 2013-03-31. Retrieved 2013-06-05.\",\"url\":\"https://web.archive.org/web/20130331122119/http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm\",\"favicon\":\"\"},{\"id\":\"16\",\"title\":\"** Blaber, Michael. [\\\"Genomic Libraries\\\"](https://web.archive.org/web/20150513050011/http://www.mikeblaber.org/oldwine/bch5425/lect28/lect28.htm). Archived from the original on 13 May 2015. Retrieved 1 April 2013.\",\"description\":\"16. **^** Blaber, Michael. [\\\"Genomic Libraries\\\"](https://web.archive.org/web/20150513050011/http://www.mikeblaber.org/oldwine/bch5425/lect28/lect28.htm). Archived from the original on 13 May 2015. Retrieved 1 April 2013.\",\"url\":\"https://web.archive.org/web/20150513050011/http://www.mikeblaber.org/oldwine/bch5425/lect28/lect28.htm\",\"favicon\":\"\"},{\"id\":\"17\",\"title\":\"_**a**_ _**b**_ Fuesler J, Nagahama Y, Szulewski J, Mundorff J, Bireley S, Coren JS (April 2012). [\\\"An arrayed human genomic library constructed in the PAC shuttle vector pJCPAC-Mam2 for genome-wide association studies and gene therapy\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463). _Gene_. **496** (2): 103â€“9\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.gene.2012.01.011](https://doi.org/10.1016%2Fj.gene.2012.01.011). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3488463](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [22285925](https://pubmed.ncbi.nlm.nih.gov/22285925).\",\"description\":\"17. ^ _**a**_ _**b**_ Fuesler J, Nagahama Y, Szulewski J, Mundorff J, Bireley S, Coren JS (April 2012). [\\\"An arrayed human genomic library constructed in the PAC shuttle vector pJCPAC-Mam2 for genome-wide association studies and gene therapy\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463). _Gene_. **496** (2): 103â€“9\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.gene.2012.01.011](https://doi.org/10.1016%2Fj.gene.2012.01.011). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3488463](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [22285925](https://pubmed.ncbi.nlm.nih.gov/22285925).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488463\",\"favicon\":\"\"},{\"id\":\"18\",\"title\":\"** Pareek CS, Smoczynski R, Tretyn A (November 2011). [\\\"Sequencing technologies and genome sequencing\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340). _J. Appl. Genet_. **52** (4): 413â€“35\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/s13353-011-0057-x](https://doi.org/10.1007%2Fs13353-011-0057-x). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3189340](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [21698376](https://pubmed.ncbi.nlm.nih.gov/21698376).\",\"description\":\"18. **^** Pareek CS, Smoczynski R, Tretyn A (November 2011). [\\\"Sequencing technologies and genome sequencing\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340). _J. Appl. Genet_. **52** (4): 413â€“35\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/s13353-011-0057-x](https://doi.org/10.1007%2Fs13353-011-0057-x). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3189340](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [21698376](https://pubmed.ncbi.nlm.nih.gov/21698376).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340\",\"favicon\":\"\"},{\"id\":\"19\",\"title\":\"** [Pennisi E](Elizabeth_Pennisi \\\"Elizabeth Pennisi\\\") (April 2003). [\\\"Human genome. Reaching their goal early, sequencing labs celebrate\\\"](https://doi.org/10.1126%2Fscience.300.5618.409). _Science_. **300** (5618): 409. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.300.5618.409](https://doi.org/10.1126%2Fscience.300.5618.409). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12702850](https://pubmed.ncbi.nlm.nih.gov/12702850). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [206577479](https://api.semanticscholar.org/CorpusID:206577479).\",\"description\":\"19. **^** [Pennisi E](Elizabeth_Pennisi \\\"Elizabeth Pennisi\\\") (April 2003). [\\\"Human genome. Reaching their goal early, sequencing labs celebrate\\\"](https://doi.org/10.1126%2Fscience.300.5618.409). _Science_. **300** (5618): 409. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.300.5618.409](https://doi.org/10.1126%2Fscience.300.5618.409). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12702850](https://pubmed.ncbi.nlm.nih.gov/12702850). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [206577479](https://api.semanticscholar.org/CorpusID:206577479).\",\"url\":\"Elizabeth_Pennisi \\\"Elizabeth Pennisi\\\"\",\"favicon\":\"\"},{\"id\":\"20\",\"title\":\"_**a**_ _**b**_ _**c**_ [\\\"HapMap Homepage\\\"](https://web.archive.org/web/20090925032755/http://hapmap.ncbi.nlm.nih.gov/). Archived from [the original](http://hapmap.ncbi.nlm.nih.gov/) on September 25, 2009.\",\"description\":\"20. ^ _**a**_ _**b**_ _**c**_ [\\\"HapMap Homepage\\\"](https://web.archive.org/web/20090925032755/http://hapmap.ncbi.nlm.nih.gov/). Archived from [the original](http://hapmap.ncbi.nlm.nih.gov/) on September 25, 2009.\",\"url\":\"https://web.archive.org/web/20090925032755/http://hapmap.ncbi.nlm.nih.gov/\",\"favicon\":\"\"},{\"id\":\"21\",\"title\":\"** Chen Y, Lin X, Yi Y, Lu Y, Zhang Z (2009). [\\\"Construction and application of a baculovirus genomic library\\\"](https://doi.org/10.1515%2Fznc-2009-7-817). _Z. Naturforsch. C_. **64** (7â€“8): 574â€“80\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1515/znc-2009-7-817](https://doi.org/10.1515%2Fznc-2009-7-817). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [19791511](https://pubmed.ncbi.nlm.nih.gov/19791511).\",\"description\":\"21. **^** Chen Y, Lin X, Yi Y, Lu Y, Zhang Z (2009). [\\\"Construction and application of a baculovirus genomic library\\\"](https://doi.org/10.1515%2Fznc-2009-7-817). _Z. Naturforsch. C_. **64** (7â€“8): 574â€“80\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1515/znc-2009-7-817](https://doi.org/10.1515%2Fznc-2009-7-817). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [19791511](https://pubmed.ncbi.nlm.nih.gov/19791511).\",\"url\":\"https://doi.org/10.1515%2Fznc-2009-7-817\",\"favicon\":\"\"}],\"images\":[],\"fixedIssues\":[{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"The first DNA-based genome ever fully sequenced was achieved by two-time Nobel Prize winner, Frederick Sanger, in 1977. Sanger and his team of scientists created a library of the bacteriophage, phi X 174, for use in DNA sequencing.[6]\",\"explanation\":\"The claim that Sanger created a genomic library for phi X 174 sequencing is not supported by reference [6], the original 1977 Nature paper, which describes using the 'plus and minus' method directly on the DNA without mentioning cloning or libraries. This misrepresents the sequencing method and attributes an anachronistic technique to early work.\",\"fixedText\":\"Remove the claim about creating a library; retain the fact of sequencing phi X 174 as the first DNA genome sequenced, but clarify no library was used.\",\"evidenceSource\":\"[6] Sanger et al. (1977) Nature paper; confirmed via direct source review.\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"Teams are now able to catalog polymorphisms in genomes and investigate those candidate genes contributing to maladies such as Parkinsonâ€™s disease, Alzheimerâ€™s disease, multiple sclerosis, rheumatoid arthritis, and Type 1 diabetes.[7]\",\"explanation\":\"Reference [7] (Menon \u0026 Farina, 2011) discusses interactomes for autism spectrum disorder, bipolar disorder, schizophrenia, non-syndromic cleft lip/palate, and glioblastoma, not the listed diseases (Parkinson's, Alzheimer's, MS, RA, Type 1 diabetes). The citation does not support the specific examples given.\",\"fixedText\":\"Remove or correct the specific disease examples; remove citation [7] as it does not apply. If retaining, use accurate examples or omit.\",\"evidenceSource\":\"[7] Menon \u0026 Farina (2011) PLOS ONE; paper abstract and content review.\"},{\"criticality\":\"Critical\",\"issueType\":\"Scope/Applicability Misstatement\",\"problematicText\":\"These are due to the advance of genome-wide association studies from the ability to create and sequence genomic libraries. Prior, linkage and candidate-gene studies were some of the only approaches.[8]\",\"explanation\":\"Genome-wide association studies (GWAS) primarily rely on SNP genotyping arrays and high-throughput sequencing, not directly on traditional genomic libraries (which are cloning-based for large-insert coverage). Reference [8] is a specific GWAS paper on bipolar disorder and does not discuss historical advances or the role of genomic libraries in enabling GWAS. This overstates the direct role of genomic libraries in GWAS and duplicates content better suited to the \\\"Applications\\\" section.\",\"fixedText\":\"Remove the GWAS attribution to genomic libraries; move or omit GWAS history to avoid duplication with \\\"Applications\\\" section. Retain mention of prior methods if reframed to sequencing/cloning history.\",\"evidenceSource\":\"[8] Cichon et al. (2011) Am J Hum Genet; general knowledge of GWAS methods from authoritative sources like NHGRI.\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"[Entire section lacks proper historical context]\",\"explanation\":\"The section titled \\\"History\\\" focuses incorrectly on general DNA sequencing and GWAS rather than the development of genomic libraries themselves. Key historical milestones, such as the emergence of recombinant DNA technology in the mid-1970s enabling the first genomic libraries using vectors like plasmids and lambda phage, are absent. This creates a knowledge gap on the topic's core subject.\",\"fixedText\":\"Add brief overview of genomic library development in the 1970s, citing appropriate sources; ensure neutrality and verifiability.\",\"evidenceSource\":\"Authoritative reviews on recombinant DNA history, e.g., ScienceDirect topics on genomic libraries (mid-late 1970s origin).\"},{\"criticality\":\"Critical\",\"issueType\":\"Quantitative/Units Error\",\"problematicText\":\"For organisms with very small genomes _(~10 kb)_\",\"explanation\":\"10 kb represents the size of small viral genomes (e.g., bacteriophage Ï†X174 at ~5 kb), not typical cellular organisms. The smallest free-living bacterial genomes are around 500-800 kb (e.g., Mycoplasma genitalium ~580 kb), making this an order-of-magnitude understatement that misleads on applicability to organisms.\",\"fixedText\":\"Revise to \\\"for very small genomes, such as those of some viruses (~5-10 kb)\\\" to accurately reflect scale.\",\"evidenceSource\":\"Standard genome size data; e.g., PNAS on Mycoplasma genitalium as smallest self-replicating organism genome at 580 kb.\"},{\"criticality\":\"Critical\",\"issueType\":\"Mechanism/Specification Error\",\"problematicText\":\"Digest the DNA with a restriction enzyme. This creates fragments that are similar in size, each containing one or more genes.\",\"explanation\":\"Standard genomic library construction for large genomes requires partial restriction digestion (limited enzyme or time) to produce overlapping fragments of desired size (e.g., 10-40 kb depending on vector), followed by size selection (e.g., gel electrophoresis or gradient centrifugation). Complete digestion yields small, non-overlapping fragments unsuitable for comprehensive genome coverage.\",\"fixedText\":\"Specify partial digestion in the description and add a step for size selection of fragments before ligation.\",\"evidenceSource\":\"[2] Sambrook \u0026 Russell, Molecular Cloning manual, which details partial digestion with Sau3AI for lambda libraries and size fractionation.\"},{\"criticality\":\"Critical\",\"issueType\":\"Mechanism/Specification Error\",\"problematicText\":\"dilutions of the library are used to transform cultures of E. coli of known concentrations.\",\"explanation\":\"For viral vectors like lambda phage, the library stock (containing phage particles) is used to infect a lawn of host E. coli cells to form plaques; \\\"transformation\\\" specifically refers to the uptake of naked plasmid DNA by competent bacterial cells, not phage infection.\",\"fixedText\":\"Replace \\\"transform\\\" with \\\"infect\\\" in the context of viral vectors.\",\"evidenceSource\":\"[2] Sambrook \u0026 Russell, Molecular Cloning, which uses \\\"infect\\\" for phage titering via plaque assays.\"},{\"criticality\":\"Critical\",\"issueType\":\"Mechanism/Specification Error\",\"problematicText\":\"The filter and colonies are prepared for hybridization and then labeled with a probe.\",\"explanation\":\"In colony hybridization, the DNA from colonies is denatured and fixed on the filter, then hybridized with a labeled probe (e.g., radioactive or biotinylated); the target DNA is not labeled by the probeâ€”the probe detects the target via complementary binding.\",\"fixedText\":\"Revise to \\\"hybridized with a labeled probe\\\" and clarify the detection process.\",\"evidenceSource\":\"[2] Sambrook \u0026 Russell, Molecular Cloning, standard protocol for colony/plaque hybridization using labeled probes.\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"with a probe.[13]\",\"explanation\":\"Reference [13] describes construction and characterization of a human BAC library, focusing on cloning and stability, not screening methods like hybridization.\",\"fixedText\":\"Remove [13] as it does not support the screening claim; cite [2] if retaining a reference.\",\"evidenceSource\":\"[13] Kim et al. (1996), Genomics, which does not cover library screening protocols.\"},{\"criticality\":\"Critical\",\"issueType\":\"Internal Contradiction\",\"problematicText\":\"Vector type | Insert size (thousands of bases) ... Plasmids | up to 10 ... (table) ... capable of carrying inserts up to 15kb. (Plasmids subsection)\",\"explanation\":\"The reported maximum insert size for plasmids differs between the table (10 kb) and the subsection text (15 kb), creating an internal inconsistency that could mislead readers on vector capacity limits.\",\"fixedText\":\"Standardize the maximum insert size for plasmids to 15 kb in both the table and text.\",\"evidenceSource\":\"[3]; standard cloning references confirm up to 15 kb is feasible for plasmids.\"},{\"criticality\":\"Critical\",\"issueType\":\"Internal Contradiction\",\"problematicText\":\"Vector type | Insert size (thousands of bases) ... Cosmids | up to 45 ... (table) ... Cosmids are capable of carrying inserts up to 40kb in size. (Cosmids subsection)\",\"explanation\":\"The reported maximum insert size for cosmids differs between the table (45 kb) and the subsection text (40 kb), creating an internal inconsistency; standard capacity is up to 45 kb.\",\"fixedText\":\"Update the text to state inserts up to 45 kb.\",\"evidenceSource\":\"[2]; cloning literature confirms 35-45 kb range, typically up to 45 kb.\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"Scientists from all of the participating nations are cataloging these attributes with data from populations of African, Asian, and European ancestry.\\\"\",\"explanation\":\"The International HapMap Project completed its final phase (HapMap 3) in 2010, and the project was retired in 2016 due to a security breach. As of 2025, cataloging efforts under the project are no longer ongoing, rendering the present tense factually incorrect and presenting a historical effort as current.\",\"fixedText\":\"Revise to past tense: \\\"were cataloging\\\". Add brief note on completion for context: \\\"The project, which completed its final phase in 2010, cataloged...\\\" to reflect historical status without introducing new content.\",\"evidenceSource\":\"Web search results on \\\"status of International HapMap Project as of 2025\\\", e.g., embryo.asu.edu (project 2002â€“2016, website retired 2016).\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"\",\"explanation\":\"The subsection discusses the HapMap Project but lacks mention of its completion and successors (e.g., 1000 Genomes Project, completed 2015), which built on its data for modern GWAS. This creates a knowledge gap on the evolution of genomic applications post-2010, relevant as of 2025, potentially misleading readers on current practices.\",\"fixedText\":\"Add a sentence: \\\"The project completed in 2010, paving the way for larger initiatives like the 1000 Genomes Project, which have advanced GWAS using similar principles.\\\"\",\"evidenceSource\":\"Web search on HapMap status and successors.\"}],\"slug\":\"Genomic_library\",\"title\":\"Genomic library\",\"content\":\"$1f\",\"description\":\"Genomic library\\n\\nA genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA.\",\"metadata\":{\"categories\":[\"genomic DNA library\",\"gDNA library\"],\"lastModified\":\"1761554060\",\"contentLength\":\"18823\",\"version\":\"1.0\",\"lastEditor\":\"system\",\"language\":\"en\",\"isRedirect\":false,\"redirectTarget\":\"\",\"isWithheld\":false},\"stats\":{\"totalViews\":\"45204\",\"recentViews\":\"45204\",\"dailyAvgViews\":1506.800048828125,\"qualityScore\":1,\"lastViewed\":\"1761886990\"},\"linkedPages\":null},\"found\":true},\"dataUpdateCount\":1,\"dataUpdatedAt\":1761886990516,\"error\":null,\"errorUpdateCount\":0,\"errorUpdatedAt\":0,\"fetchFailureCount\":0,\"fetchFailureReason\":null,\"fetchMeta\":null,\"isInvalidated\":false,\"status\":\"success\",\"fetchStatus\":\"idle\"},\"queryKey\":[\"page\",\"Genomic_library\"],\"queryHash\":\"[\\\"page\\\",\\\"Genomic_library\\\"]\"}]},\"children\":\"$L20\"}]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"c:{\"metadata\":[[\"$\",\"title\",\"0\",{\"children\":\"Genomic library\"}],[\"$\",\"meta\",\"1\",{\"name\":\"description\",\"content\":\"A genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism's DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA...\"}],[\"$\",\"meta\",\"2\",{\"name\":\"author\",\"content\":\"system\"}],[\"$\",\"link\",\"3\",{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\",\"crossOrigin\":\"$undefined\"}],[\"$\",\"meta\",\"4\",{\"name\":\"keywords\",\"content\":\"genomic DNA library, gDNA library\"}],[\"$\",\"meta\",\"5\",{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",\"6\",{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"link\",\"7\",{\"rel\":\"canonical\",\"href\":\"https://grokipedia.com/page/Genomic_library\"}],[\"$\",\"meta\",\"8\",{\"property\":\"og:title\",\"content\":\"Genomic library\"}],[\"$\",\"meta\",\"9\",{\"property\":\"og:description\",\"content\":\"A genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism's DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA...\"}],[\"$\",\"meta\",\"10\",{\"property\":\"og:url\",\"content\":\"https://grokipedia.com/page/Genomic_library\"}],[\"$\",\"meta\",\"11\",{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",\"12\",{\"property\":\"og:locale\",\"content\":\"en\"}],[\"$\",\"meta\",\"13\",{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",\"14\",{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",\"15\",{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",\"16\",{\"property\":\"og:image:alt\",\"content\":\"Genomic library\"}],[\"$\",\"meta\",\"17\",{\"property\":\"og:type\",\"content\":\"article\"}],[\"$\",\"meta\",\"18\",{\"property\":\"article:modified_time\",\"content\":\"1970-01-21T09:19:14.060Z\"}],[\"$\",\"meta\",\"19\",{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",\"20\",{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",\"21\",{\"name\":\"twitter:title\",\"content\":\"Genomic library\"}],[\"$\",\"meta\",\"22\",{\"name\":\"twitter:description\",\"content\":\"A genomic library is a collection of overlapping DNA fragments that together make up the total genomic DNA of a single organism. The DNA is stored in a population of identical vectors, each containing a different insert of DNA. In order to construct a genomic library, the organism's DNA is extracted from cells and then partially digested with a restriction enzyme to produce fragments, which are size-selected to a specific range. The fragments are then inserted into the vector using DNA...\"}],[\"$\",\"meta\",\"23\",{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"link\",\"24\",{\"rel\":\"icon\",\"href\":\"/favicon.ico\",\"type\":\"image/x-icon\",\"sizes\":\"16x16\"}],[\"$\",\"$L21\",\"25\",{}]],\"error\":null,\"digest\":\"$undefined\"}\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"11:\"$c:metadata\"\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"22:I[61172,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"PageEditorProvider\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"23:I[63493,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="OTc5Y2FhM2MtZGM5OS00NWM0LTliMzAtYWU0MjM5YzdkMzhl">self.__next_f.push([1,"20:[\"$\",\"$L22\",null,{\"children\":[\"$\",\"$L23\",null,{\"slug\":\"Genomic_library\"}]}]\n"])</script></body></html>