<!DOCTYPE html><html lang="en" class="bg-surface-base antialiased"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1, interactive-widget=resizes-content"/><link rel="stylesheet" href="/_next/static/css/62c4caba71dfda84.css" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/eb3d87f98fe1565f.css" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/1b5e561215938d4d.css" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/0227d069a630d414.css" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/f87fff2ab93d05a7.css" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" data-precedence="next"/><link rel="preload" as="script" fetchPriority="low" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" href="/_next/static/chunks/webpack-e121ed42680f327e.js"/><script src="/_next/static/chunks/78a669d9-e7993486b22c4915.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/40c4a5a7-abdabb07419d1cfc.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/2230-f7c87dc9fa57c408.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/main-app-536d4b8fca62396a.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/63f0ec43-8ee6a76d70472249.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/7670-7326298c9856172b.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/282-691fffb366e24ca5.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/7515-336759ef5dad2b52.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/927-34e2a6da3e15e26b.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/5855-243eeca8f0e4cabd.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/4393-1c8dff25ec904469.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/2225-454cf9f44775e7ce.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/7618-43536a8fb0dd3bfe.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/7720-3a1e7e411adba2d3.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/7086-1f1c4891d766e04f.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/global-error-4d07d20223cd4b4c.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/8208b75a-a48e5a4507a0de91.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/5497-ab85ef3ea59dd353.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/6281-c6e84786dade3614.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/2660-d983a7f287e89f18.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/5002-9735c25beda556ba.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/3655-6717081f7512305a.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/8945-e4e3cf487f5e27c7.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/9214-a10614844a67ba31.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/4789-2080f6497f78b5b6.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/layout-226f2bd71acf9f9e.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/24cf1b50-159cca90b09285e0.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/9ffa21ba-c069766d809bd4c7.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/600-9a9fcafaaa6c1c4c.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/5448-2a3cfd853d899c9a.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/8-8ca70ca619d8e099.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/error-4a29e9399afba038.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><script src="/_next/static/chunks/app/not-found-dd95690acf732f18.js" async="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script><meta name="next-size-adjust" content=""/><title>Grokipedia</title><meta name="description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><link rel="icon" type="image/x-icon" href="/favicon.ico" sizes="48x48"/><link rel="icon" href="/images/icon-dark.png" media="(prefers-color-scheme: light)" type="image/png"/><link rel="icon" href="/images/icon-light.png" media="(prefers-color-scheme: dark)" type="image/png"/><link rel="apple-touch-icon" href="/icon-192x192.png"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><meta property="og:title" content="Grokipedia"/><meta property="og:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta property="og:url" content="https://grokipedia.com"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en_US"/><meta property="og:type" content="website"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Grokipedia"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Grokipedia"/><meta name="twitter:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><title>Genome size</title><meta name="description" content="Genome size is the total amount of DNA contained within one copy of a single complete genome. It is typically measured in terms of mass in picograms (trillionths or 10−12 of a gram, abbreviated pg) or less frequently in daltons, or as the total number of nucleotide base pairs, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In diploid organisms, genome size is often used interchangeably with the term C-value. An organism&#x27;s..."/><meta name="author" content="system"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="keywords" content="C-value"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><link rel="canonical" href="https://grokipedia.com/page/Genome_size"/><meta property="og:title" content="Genome size"/><meta property="og:description" content="Genome size is the total amount of DNA contained within one copy of a single complete genome. It is typically measured in terms of mass in picograms (trillionths or 10−12 of a gram, abbreviated pg) or less frequently in daltons, or as the total number of nucleotide base pairs, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In diploid organisms, genome size is often used interchangeably with the term C-value. An organism&#x27;s..."/><meta property="og:url" content="https://grokipedia.com/page/Genome_size"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Genome size"/><meta property="og:type" content="article"/><meta property="article:modified_time" content="1970-01-21T09:19:12.900Z"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Genome size"/><meta name="twitter:description" content="Genome size is the total amount of DNA contained within one copy of a single complete genome. It is typically measured in terms of mass in picograms (trillionths or 10−12 of a gram, abbreviated pg) or less frequently in daltons, or as the total number of nucleotide base pairs, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In diploid organisms, genome size is often used interchangeably with the term C-value. An organism&#x27;s..."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><link rel="icon" href="/favicon.ico" type="image/x-icon" sizes="16x16"/><meta name="sentry-trace" content="2ce1227a96cdb2e7deb9b4c95ec4522d-690533b1a8ef1008-1"/><meta name="baggage" content="sentry-environment=production,sentry-public_key=5f2258f71198ee26a355127af230c3a6,sentry-trace_id=2ce1227a96cdb2e7deb9b4c95ec4522d,sentry-org_id=4508179396558848,sentry-transaction=GET%20%2Fpage%2F%5Bslug%5D,sentry-sampled=true,sentry-sample_rand=0.8353666465976657,sentry-sample_rate=1"/><script src="/_next/static/chunks/polyfills-42372ed130431b0a.js" noModule="" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz"></script></head><body class="flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b"><div hidden=""><!--$--><!--/$--></div><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">((a,b,c,d,e,f,g,h)=>{let i=document.documentElement,j=["light","dark"];function k(b){var c;(Array.isArray(a)?a:[a]).forEach(a=>{let c="class"===a,d=c&&f?e.map(a=>f[a]||a):e;c?(i.classList.remove(...d),i.classList.add(f&&f[b]?f[b]:b)):i.setAttribute(a,b)}),c=b,h&&j.includes(c)&&(i.style.colorScheme=c)}if(d)k(d);else try{let a=localStorage.getItem(b)||c,d=g&&"system"===a?window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light":a;k(d)}catch(a){}})("class","theme","dark",null,["light","dark"],null,true,true)</script><header class="h-16 bg-surface-base fixed top-0 z-50 w-full"><div class="hidden h-full grid-cols-[1fr_3fr_1fr] items-center gap-4 !py-0 md:grid max-w-full py-6 px-4 w-full mx-auto"><div class="-ml-2 justify-self-start"><a class="cursor-pointer" href="/"><svg width="200" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="mx-auto flex w-full"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full px-3 shadow-none sm:w-80 focus-visible:z-1 mx-auto h-full w-full rounded-full bg-surface-l1" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>⌘</span><span class="text-xs">K</span></div></span></div></button></div><div class="flex items-center gap-2 justify-self-end"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0" type="button" id="radix-_R_7apfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button></div></div><div class="flex h-full items-center md:hidden justify-between max-w-full py-6 px-4 w-full mx-auto pt-8"><div class="flex items-center"><div class="-ml-2"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button" aria-label="Toggle table of contents"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-list h-6 w-6"><path d="M3 12h.01"></path><path d="M3 18h.01"></path><path d="M3 6h.01"></path><path d="M8 12h13"></path><path d="M8 18h13"></path><path d="M8 6h13"></path></svg></button></div><a class="cursor-pointer" href="/"><svg width="160" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="flex items-center gap-1"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0 [&amp;_svg]:!w-5" type="button" id="radix-_R_aipfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search h-5 w-5"><circle cx="11" cy="11" r="8"></circle><path d="m21 21-4.3-4.3"></path></svg></button></div></div></header><div class="bg-surface-base fixed right-0 top-16 z-50 h-[calc(100vh-4rem)] w-full transform border-l transition-transform duration-300 ease-in-out sm:w-1/2 md:hidden translate-x-full"><div class="flex flex-col gap-4 p-4"><div class="flex flex-row gap-2"><button class="focus-visible:ring-ring inline-flex items-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full shadow-none sm:w-80 h-10 w-full justify-start rounded-full bg-surface-l1 px-4" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>⌘</span><span class="text-xs">K</span></div></span></div></button></div></div></div><div><div class="min-[1350px]:grid min-[1350px]:grid-cols-[1fr_3fr_1fr]"><nav class="bg-surface-base hidden h-[calc(100vh-4rem)] overflow-y-auto px-6 pb-32 min-[1350px]:sticky min-[1350px]:top-16 min-[1350px]:block scrollbar-none [-ms-overflow-style:none] [scrollbar-width:none] [&amp;::-webkit-scrollbar]:hidden [overscroll-behavior:contain] pt-8"><ul class="space-y-2 text-sm"><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#genome-size" class="transition-opacity hover:opacity-100 opacity-50">Genome size</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#variation-in-genome-size-and-gene-content" class="transition-opacity hover:opacity-100 opacity-50">Variation in genome size and gene content</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#human-genome-size" class="transition-opacity hover:opacity-100 opacity-50">Human genome size</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#genome-reduction" class="transition-opacity hover:opacity-100 opacity-50">Genome reduction</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#in-obligate-endosymbiotic-species" class="transition-opacity hover:opacity-100 opacity-50">In obligate endosymbiotic species</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#conversion-from-picograms-pg-to-base-pairs-bp" class="transition-opacity hover:opacity-100 opacity-50">Conversion from picograms (pg) to base pairs (bp)</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#drakes-rule" class="transition-opacity hover:opacity-100 opacity-50">Drake&#x27;s rule</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#genome-miniaturization-and-optimal-size" class="transition-opacity hover:opacity-100 opacity-50">Genome miniaturization and optimal size</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#mechanisms-of-genome-miniaturization" class="transition-opacity hover:opacity-100 opacity-50">Mechanisms of genome miniaturization</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#evidence-of-genome-miniaturization" class="transition-opacity hover:opacity-100 opacity-50">Evidence of genome miniaturization</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#references" class="transition-opacity hover:opacity-100 opacity-50">References</a></li></ul></nav><div class="relative top-16 pb-32 pt-8 px-4 md:px-8"><div class="mx-auto max-w-[850px]"><button data-state="closed" type="button" class="flex items-center"><div class="text-fg-tertiary mb-2 flex cursor-help items-center gap-2 text-sm"><span class="inline-flex items-center justify-center p-0 m-0" data-namespace="@xai/icons" data-slot="icon" style="height:16px;width:16px"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 33 33" aria-hidden="true" class="" focusable="false" style="fill:currentColor;height:16px;width:16px"><path fill="currentColor" d="m13.237 21.04 11.082-8.19c.543-.4 1.32-.244 1.578.38 1.363 3.288.754 7.241-1.957 9.955-2.71 2.714-6.482 3.31-9.93 1.954l-3.765 1.745c5.401 3.697 11.96 2.782 16.059-1.324 3.251-3.255 4.258-7.692 3.317-11.693l.008.009c-1.365-5.878.336-8.227 3.82-13.031q.123-.17.247-.345l-4.585 4.59v-.014L13.234 21.044M10.95 23.031c-3.877-3.707-3.208-9.446.1-12.755 2.446-2.449 6.454-3.448 9.952-1.979L24.76 6.56c-.677-.49-1.545-1.017-2.54-1.387A12.465 12.465 0 0 0 8.675 7.901c-3.519 3.523-4.625 8.94-2.725 13.561 1.42 3.454-.907 5.898-3.251 8.364-.83.874-1.664 1.749-2.335 2.674l10.583-9.466"></path></svg></span><span>Fact-checked by Grok<!-- --> <!-- -->4 days ago</span></div></button><article class="text-[16px]"><h1 id="genome-size" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Genome size<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7"><strong>Genome size</strong> is the total amount of <span class="inline text-[1em] leading-7">DNA</span> contained within one copy of a single complete <span class="inline text-[1em] leading-7">genome</span>. It is typically measured in terms of <span class="inline text-[1em] leading-7">mass</span> in picograms (trillionths or 10−12 of a <span class="inline text-[1em] leading-7">gram</span>, abbreviated pg) or less frequently in <span class="inline text-[1em] leading-7">daltons</span>, or as the total number of <span class="inline text-[1em] leading-7">nucleotide</span> <span class="inline text-[1em] leading-7">base pairs</span>, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_hqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup> In <span class="inline text-[1em] leading-7">diploid</span> <span class="inline text-[1em] leading-7">organisms</span>, genome size is often used interchangeably with the term <span class="inline text-[1em] leading-7">C-value</span>.</span>
<span class="mb-4 block break-words text-[1em] leading-7">An organism&#x27;s complexity is not directly proportional to its genome size; total DNA content is widely variable between biological taxa. Some single-celled organisms have much more DNA than humans, for reasons that remain unclear (see <span class="inline text-[1em] leading-7">Junk DNA</span> and <span class="inline text-[1em] leading-7">C-value</span>).</span>
<h1 id="variation-in-genome-size-and-gene-content" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Variation in genome size and gene content<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">Since the 1950s, with the emergence of various molecular techniques, the genome sizes of thousands of <span class="inline text-[1em] leading-7">eukaryotes</span> have been analyzed, and these data are available in online databases for animals, plants, and fungi (see external links), such as the Animal Genome Size Database, Plant DNA C-values Database, and the Protist 10,000 Genomes Project portal. Nuclear genome size is typically measured in eukaryotes using either <span class="inline text-[1em] leading-7">densitometric</span> measurements of <span class="inline text-[1em] leading-7">Feulgen</span>-stained nuclei (previously using specialized densitometers, now more commonly using computerized <span class="inline text-[1em] leading-7">image analysis</span><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_92abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup>) or <span class="inline text-[1em] leading-7">flow cytometry</span>. In <span class="inline text-[1em] leading-7">prokaryotes</span>, <span class="inline text-[1em] leading-7">pulsed field gel electrophoresis</span> and complete <span class="inline text-[1em] leading-7">genome sequencing</span> are the predominant methods of genome size determination.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Nuclear genome sizes are well known to vary enormously among eukaryotic species. In animals they range more than 6,600-fold, and in land plants they differ by a factor of over 2,300 (with the largest known now ~160 Gbp in the fork fern <em>Tmesipteris oblanceolata</em> as of 2024).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_42qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_52qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_62qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[5]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_72qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[6]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_82qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup> <span class="inline text-[1em] leading-7">Protist</span> genomes have been reported to vary more than 28,600-fold in size, though some extreme values (e.g., in amoebae) remain debated.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_c2qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[8]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Genome size in eukaryotes was thought to be proportional to the complexity of an organism but by the mid-20th century it became apparent that closely related species could differ substantially in the size of their genomes. This counter-intuitive observation gave rise to what became known as the &quot;<span class="inline text-[1em] leading-7">C-value paradox</span>.&quot;</span>
<span class="mb-4 block break-words text-[1em] leading-7">By the end of 1960s this &quot;paradox&quot; was resolved by the discovery of <span class="inline text-[1em] leading-7">repetitive DNA</span> and the realization that much of the differences in genomes sizes was due to the presence or absence of large amounts of repetitive DNA. Further advances at that time indicated that most of the DNA in large genomes was non-functional <span class="inline text-[1em] leading-7">junk DNA</span> and only a small fraction corresponded to functional DNA, including the functional parts of genes.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Genome size correlates with a range of measurable characteristics at the <span class="inline text-[1em] leading-7">cell</span> and organism levels, including cell size, <span class="inline text-[1em] leading-7">cell division</span> rate, and, depending on the <span class="inline text-[1em] leading-7">taxon</span>, body size, <span class="inline text-[1em] leading-7">metabolic rate</span>, developmental rate, <span class="inline text-[1em] leading-7">organ</span> complexity, geographical distribution, or <span class="inline text-[1em] leading-7">extinction</span> risk.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_e4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_f4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup> Early analyses of sequenced genomes (as of 2009) suggested a linear correlation between log-transformed gene number and log-transformed genome size in prokaryotes, viruses, and organelles, and a nonlinear correlation in eukaryotes, potentially due to increasing junk DNA.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_h4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[9]</sup> However, with extensive additional sequencing by 2024, no clear correlation exists between gene number and genome size in eukaryotes, as variation is largely driven by non-coding elements.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_j4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[10]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_k4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[11]</sup></span>
<h3 id="human-genome-size" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Human genome size<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">In humans, the total female <span class="inline text-[1em] leading-7">diploid</span> <span class="inline text-[1em] leading-7">nuclear genome</span> per cell extends for 6.37 Gigabase pairs (Gbp), is 208.23 cm long and weighs 6.51 picograms (pg).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_65abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[12]</sup> Male values are 6.27 Gbp, 205.00 cm, 6.41 pg.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_85abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[12]</sup> Each DNA polymer can contain hundreds of millions of nucleotides, such as in <span class="inline text-[1em] leading-7">chromosome 1</span>. Chromosome 1 is the largest human <span class="inline text-[1em] leading-7">chromosome</span> with approximately 220 million <span class="inline text-[1em] leading-7">base pairs</span>, and would be 85 mm long if straightened.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_g5abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">In <span class="inline text-[1em] leading-7">eukaryotes</span>, in addition to <span class="inline text-[1em] leading-7">nuclear DNA</span>, there is also <span class="inline text-[1em] leading-7">mitochondrial DNA</span> (mtDNA) which encodes certain proteins used by the mitochondria. The mtDNA is usually relatively small in comparison to the nuclear DNA. For example, the <span class="inline text-[1em] leading-7">human mitochondrial DNA</span> forms closed circular molecules, each of which contains 16,569<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_a5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[14]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_b5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[15]</sup> DNA base pairs,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_d5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup> with each such molecule normally containing a full set of the mitochondrial genes. Each human mitochondrion contains, on average, approximately 5 such mtDNA molecules.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_f5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup> Each human <span class="inline text-[1em] leading-7">cell</span> contains approximately 100 mitochondria, giving a total number of mtDNA molecules per human cell of approximately 500.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_j5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup> However, the amount of mitochondria per cell also varies by cell type, and an <span class="inline text-[1em] leading-7">egg cell</span> can contain 100,000 mitochondria, corresponding to up to 1,500,000 copies of the mitochondrial genome (constituting up to 90% of the DNA of the cell).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_n5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[17]</sup></span>
<h1 id="genome-reduction" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Genome reduction<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7"><strong>Genome reduction</strong>, also known as <strong>genome degradation</strong>, is the process by which an organism&#x27;s genome shrinks relative to that of its ancestors. Genomes fluctuate in size regularly, and genome size reduction is most significant in bacteria.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The most evolutionarily significant cases of genome reduction may be observed in the eukaryotic organelles known to be derived from bacteria: mitochondria and plastids. These organelles are descended from primordial endosymbionts, which were capable of surviving within the host cell and which the host cell likewise needed for survival. Many present-day mitochondria have less than 20 genes in their entire genome, whereas a modern free-living bacterium generally has at least 1,000 genes. Many genes have apparently been transferred to the host nucleus, while others have simply been lost and their function replaced by host processes.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Other bacteria have become endosymbionts or obligate intracellular pathogens and experienced extensive genome reduction as a result. This process seems to be dominated by genetic drift resulting from small population size, low recombination rates, and high mutation rates, as opposed to selection for smaller genomes. Some free-living marine bacterioplanktons also show signs of genome reduction, which are hypothesized to be driven by natural selection.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_27qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[18]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_37qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[19]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_47qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[20]</sup> In contrast, terrestrial prokaryotes appear to have larger genome sizes than both aquatic and host-associated prokaryotes (average of 3.7 Mbp for terrestrial, 3.1 Mbp for aquatic and 3.0 Mbp for host-associated).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_67qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[21]</sup></span>
<h3 id="in-obligate-endosymbiotic-species" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">In obligate endosymbiotic species<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Obligate endosymbiotic species are characterized by a complete inability to survive external to their host environment. While some obligate endosymbionts are pathogenic and pose threats to human health by evading immune systems and manipulating host environments for nutrients, many others form mutualistic relationships, providing essential benefits to their hosts such as nutrient provisioning. Their compact genomes result from massive losses of extraneous DNA, an occurrence strongly associated with the loss of a free-living stage. As much as 90% of the genetic material can be lost when a species makes the evolutionary transition from a free-living to an obligate intracellular lifestyle. During this process, the organism is subjected to a metabolite-rich environment and must persist within the host cell; these factors reduce retention of non-essential genes and increase genetic drift, accelerating the loss of superfluous genes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_28qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[22]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_38qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[23]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_48qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[24]</sup> Common examples of species with reduced genomes include <em>Buchnera aphidicola</em> (a mutualist), <em>Rickettsia prowazekii</em> (a pathogen), and <em>Mycobacterium leprae</em> (a pathogen). One obligate endosymbiont of leafhoppers, <em>Nasuia deltocephalinicola</em>, had one of the smallest known genomes at the time of its discovery (112 kb in 2013).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_e8qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[25]</sup> More recent research has identified even smaller genomes, such as approximately 50-52 kb in symbionts of planthoppers (as of 2025).[<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_g8qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[26]</sup>] While some obligate intracellular species are pathogenic, many provide positive fitness effects to their hosts.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The <strong>reductive evolution model</strong> has been proposed as an effort to define the genomic commonalities seen in all obligate endosymbionts.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_49abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[27]</sup> This model illustrates four general features of reduced genomes and obligate intracellular species:</span>
<ol class="marker:text-primary my-4 ml-6 list-decimal space-y-2 text-[1em]">
<li class="text-[1em]">&quot;genome streamlining&quot; resulting from relaxed selection on genes that are superfluous in the intracellular environment;</li>
<li class="text-[1em]">a bias towards deletions (rather than insertions), which heavily affects genes that have been disrupted by accumulation of mutations (pseudogenes);<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_149qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[28]</sup></li>
<li class="text-[1em]">very little or no capability for acquiring new DNA; and</li>
<li class="text-[1em]">considerable reduction of effective population size in endosymbiotic populations, particularly in species that rely on vertical transmission of genetic material.</li>
</ol>
<span class="mb-4 block break-words text-[1em] leading-7">Recent studies continue to support this model, highlighting deletional bias and genetic drift as key drivers of genome miniaturization, with patterns varying across different symbionts.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2aabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[29]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3aabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[30]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4aabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[31]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5aabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[32]</sup></span>
<h1 id="conversion-from-picograms-pg-to-base-pairs-bp" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Conversion from picograms (pg) to base pairs (bp)<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mtext>number of base pairs</mtext><mo>=</mo><mtext>mass in pg</mtext><mo>×</mo><mn>9.78</mn><mo>×</mo><mn>1</mn><msup><mn>0</mn><mn>8</mn></msup></mrow><annotation encoding="application/x-tex">\text{number of base pairs} = \text{mass in pg} \times 9.78 \times 10^{8}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8889em;vertical-align:-0.1944em"></span><span class="mord text"><span class="mord">number of base pairs</span></span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:0.8623em;vertical-align:-0.1944em"></span><span class="mord text"><span class="mord">mass in pg</span></span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">×</span><span class="mspace" style="margin-right:0.2222em"></span></span><span class="base"><span class="strut" style="height:0.7278em;vertical-align:-0.0833em"></span><span class="mord">9.78</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">×</span><span class="mspace" style="margin-right:0.2222em"></span></span><span class="base"><span class="strut" style="height:0.8641em"></span><span class="mord">1</span><span class="mord"><span class="mord">0</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.8641em"><span style="top:-3.113em;margin-right:0.05em"><span class="pstrut" style="height:2.7em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">8</span></span></span></span></span></span></span></span></span></span></span></span></span>
<span class="mb-4 block break-words text-[1em] leading-7">or simply:</span>
<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mn>1</mn><mtext> </mtext><mtext>pg</mtext><mo>=</mo><mn>978</mn><mtext> </mtext><mtext>Mbp</mtext></mrow><annotation encoding="application/x-tex">1 \, \text{pg} = 978 \, \text{Mbp}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8389em;vertical-align:-0.1944em"></span><span class="mord">1</span><span class="mspace" style="margin-right:0.1667em"></span><span class="mord text"><span class="mord">pg</span></span><span class="mspace" style="margin-right:0.2778em"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em"></span></span><span class="base"><span class="strut" style="height:0.8889em;vertical-align:-0.1944em"></span><span class="mord">978</span><span class="mspace" style="margin-right:0.1667em"></span><span class="mord text"><span class="mord">Mbp</span></span></span></span></span></span>
<h1 id="drakes-rule" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Drake&#x27;s rule<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">In 1991, <span class="inline text-[1em] leading-7">John W. Drake</span> proposed a general rule: that the per-base-pair mutation rate and genome size are inversely correlated, resulting in a roughly constant mutation rate per genome.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4dabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[33]</sup> This rule has been found to be approximately correct for simple genomes such as those in <span class="inline text-[1em] leading-7">DNA viruses</span> and unicellular organisms. Its basis is unknown.</span>
<span class="mb-4 block break-words text-[1em] leading-7">It has been proposed that the small size of <span class="inline text-[1em] leading-7">RNA viruses</span> is locked into a three-part relation between replication fidelity, genome size, and genetic complexity. The majority of RNA viruses lack an RNA proofreading facility, which limits their replication fidelity and hence their genome size. This has also been described as the <span class="inline text-[1em] leading-7">Eigen paradox</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6dqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[34]</sup> An exception to the rule of small genome sizes in RNA viruses is found in the <span class="inline text-[1em] leading-7">Nidoviruses</span>. These viruses appear to have acquired a <span class="inline text-[1em] leading-7">3′-to-5′ exoribonuclease</span> (ExoN) which has allowed for an increase in genome size.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_cdqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[35]</sup></span>
<h1 id="genome-miniaturization-and-optimal-size" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Genome miniaturization and optimal size<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">In 1972, Michael D. Bennett hypothesized a correlation between nuclear DNA content and minimum generation time in herbaceous plants.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2eqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[36]</sup> Van&#x27;t Hof and Sparrow postulated relationships between DNA content, nuclear volume, cell size, and minimum mitotic cycle time.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4eqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[37]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Cavalier-Smith disputed these and other earlier explanations in his 2005 article,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2fabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup> proposing the skeletal theory to explain the relation between genome size and cell volume. According to this theory, nuclear DNA acts as a nucleoskeleton that determines nuclear volume, which scales with cell volume to maintain an optimal karyoplasmic ratio—the ratio of nuclear to cytoplasmic volume that balances nuclear RNA export and cytoplasmic protein synthesis for efficient cell growth.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4fabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[39]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5fabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[40]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6fabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup> Larger genomes are more prone to accumulation of non-coding &quot;skeletal&quot; DNA, including duplicative transposons.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_8fabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup> In response to cell size reduction, selection favors deletions over duplications to economize on scarce resources like phosphorus and energy.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_afabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">From an economic perspective, evolutionary pressures favor DNA reduction unless offset by functional benefits, as random deletions are often deleterious but can occasionally enhance fitness. This trade-off between economy and non-coding DNA accumulation helps maintain the karyoplasmic ratio.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2fqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup></span>
<h2 id="mechanisms-of-genome-miniaturization" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Mechanisms of genome miniaturization<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">The base question behind the process of genome miniaturization is whether it occurs through large steps or due to a constant erosion of the gene content. In order to assess the evolution of this process is necessary to compare an ancestral genome with the one where the shrinkage is supposed to be occurred. Thanks to the similarity among the gene content of <em>Buchnera aphidicola</em> and the enteric bacteria <em>Escherichia coli</em> , 89% identity for the 16S rDNA and 62% for <span class="inline text-[1em] leading-7">orthologous</span> genes was possible to shed light on the mechanism of genome miniaturization.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_8gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[41]</sup> The genome of the <span class="inline text-[1em] leading-7">endosymbiont</span> <em>B. aphidicola</em> is characterized by a genome size that is seven times smaller than <em>E. coli</em> (643 kb compared to 4.6 Mb)<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_ggqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[42]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_hgqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[43]</sup> and can be view as a subset of the enteric bacteria gene inventory.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_jgqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[43]</sup> From the confrontation of the two genomes emerged that some genes persist as partially degraded.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_lgqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[43]</sup> indicating that the function was lost during the process and that consequent events of erosion shortened the length as documented in <em>Rickettsia</em>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_pgqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[44]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_qgqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[45]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_rgqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[32]</sup> This hypothesis is confirmed by the analysis of the <span class="inline text-[1em] leading-7">pseudogenes</span> of <em>Buchnera</em> where the number of deletions was more than ten times higher compared to the insertion.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_11gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[32]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">In <em>Rickettsia prowazekii</em> , as with other small genome bacteria, this parasitic endosymbiont has experienced a vast reduction of functional activity. The common effects of the genome shrinking between this endosymbiont and the other parasites are the reduction of the ability to produce phospholipids, repair and recombination and an overall conversion of the composition of the gene to a richer A-T<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4habav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[46]</sup> content due to mutation and substitutions.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6habav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[22]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_7habav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[47]</sup> Evidence of the deletion of the function of repair and recombination is the loss of the gene <em>rec</em> A, gene involved in the <span class="inline text-[1em] leading-7">recombinase</span> pathway. This event happened during the removal of a larger region containing ten genes for a total of almost 10 kb.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_dhabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[43]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_ehabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[47]</sup> Same faith occurred <em>uvr</em> A, <em>uvr</em> B and <em>uvr</em> C, genes encoding for excision enzymes involved in the repair of damaged DNA due to UV exposure.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_mhabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[41]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">One of the most plausible mechanisms for the explanation of the genome shrinking is the chromosomal rearrangement because insertion/deletion of larger portion of sequence are more easily to be seen in during homologous recombination compared to the illegitimate, therefore the spread of the <span class="inline text-[1em] leading-7">transposable elements</span> will positively affect the rate of deletion.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4hqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup> The loss of those genes in the early stages of miniaturization not only this function but must played a role in the evolution of the consequent deletions. Evidences of the fact that larger event of removal occurred before smaller deletion emerged from the comparison of the genome of <em>Bucknera</em> and a reconstructed ancestor, where the gene that have been lost are in fact not randomly dispersed in the ancestor gene but aggregated and the negative relation between number of lost genes and length of the spacers.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_8hqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[41]</sup> The event of small local indels plays a marginal role on the genome reduction<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_ahqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[48]</sup> especially in the early stages where a larger number of genes became superfluous.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_chqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[49]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_dhqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[41]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Single events instead occurred due to the lack of selection pressure for the retention of genes especially if part of a pathway that lost its function during a previous deletion. An example for this is the deletion of <em>rec</em> F, gene required for the function of <em>rec</em> A, and its flanking genes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6iabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[41]</sup> One of the consequences of the elimination of such amount of sequences affected even the regulation of the remaining genes. The loss of large section of genomes could in fact lead to a loss in promotor sequences. This could in fact pushed the selection for the evolution of <span class="inline text-[1em] leading-7">polycistronic regions</span> with a positive effect for both size reduction<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_aiabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[50]</sup> and transcription efficiency.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_ciabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[51]</sup></span>
<h2 id="evidence-of-genome-miniaturization" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Evidence of genome miniaturization<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">One example of the miniaturization of the genome occurred in the microsporidia, an anaerobic intracellular parasite of animals evolved from aerobic fungi.</span>
<span class="mb-4 block break-words text-[1em] leading-7">During this process the mitosomes<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2jqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[52]</sup> was formed consequent to the reduction of the mitochondria to a relic voided of genomes and metabolic activity except to the production of iron sulfur centers.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4jqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[53]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5jqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[54]</sup> Except for the ribosomes, miniaturized as well, many other organelles have been almost lost during the process of the formation of the smallest known eukaryotic nuclear genome.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_7jqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup> For example, <em>Encephalitozoon intestinalis</em> has a nuclear genome of 2.3 Mb.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_bjqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[55]</sup> From their possible ancestor, a zygomycotine fungi, the microsporidia have shrunk their genome eliminating thousands of genes and reduced even the size of protein and protein-coding genes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_djqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[56]</sup> This extreme process was possible thanks to the advantageous selection for a smaller cell size imposed by the parasitism.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Another example of miniaturization is represented by the presence of nucleomorphs, enslaved nuclei, inside of the cell of two different algae, cryptophytes and chlorarachniophytes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2kabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[57]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Nucleomorphs are characterized by one of the smallest genomes known (551 and 380 kb) and as noticed for microsporidia, some genomes are noticeably reduced in length compared to other eukaryotes due to a virtual lack of non-coding DNA.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2kqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[38]</sup> The most interesting factor is represented by the coexistence of those small nuclei inside of a cell that contains another nucleus that never experienced such genome reduction. Moreover, even if the host cells have different volumes from species to species and a consequent variability in genome size, the nucleomorph remain invariant denoting a double effect of selection within the same cell.</span></article><div id="references" class="min-w-0 scroll-mt-8 overflow-hidden"><div class="text-[16px]"><h2 id="references" node="[object Object]" class="mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden">References</h2></div><ol class="columns-1 gap-x-12 [counter-reset:item] md:columns-2"><li id="1" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Dolezel J, Bartoš J, Voglmayr H, Greilhuber J (2003). &quot;Nuclear DNA content and genome size of trout and human&quot;. <em>Cytometry Part A</em>. <strong>51</strong> (2): 127–128. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1002%2Fcyto.a.10013" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1002/cyto.a.10013</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12541287" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12541287</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:221604791" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">221604791</a>.</span></div></li><li id="2" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Hardie DC, Gregory TR, Hebert PD (2002). &quot;From pixels to picograms: a beginners&#x27; guide to genome quantification by Feulgen image analysis densitometry&quot;. <em>Journal of Histochemistry and Cytochemistry</em>. <strong>50</strong> (6): 735–749. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1177%2F002215540205000601" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1177/002215540205000601</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12019291" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12019291</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:33117040" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">33117040</a>.</span></div></li><li id="3" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Bennett MD, Leitch IJ (2005). &quot;Genome size evolution in plants&quot;. In T.R. Gregory (ed.). <a href="https://archive.org/details/evolutiongenome00greg" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300"><em>The Evolution of the Genome</em></a>. San Diego: Elsevier. pp. <a href="https://archive.org/details/evolutiongenome00greg/page/n116" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">89</a>–162.</span></div></li><li id="4" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Gregory TR (2005). &quot;Genome size evolution in animals&quot;. In T.R. Gregory (ed.). <a href="https://archive.org/details/evolutiongenome00greg" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300"><em>The Evolution of the Genome</em></a>. San Diego: Elsevier. pp. <a href="https://archive.org/details/evolutiongenome00greg/page/n30" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3</a>–87.</span></div></li><li id="https://www.sciencedirect.com/science/article/pii/S0960982219302027" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.sciencedirect.com/science/article/pii/S0960982219302027" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.sciencedirect.com/science/article/pii/S0960982219302027</a></span></div></li><li id="https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.16315" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.16315" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.16315</a></span></div></li><li id="https://www.cell.com/iscience/fulltext/S2589-0042(24)01111-8" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.cell.com/iscience/fulltext/S2589-0042(24)01111-8" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.cell.com/iscience/fulltext/S2589-0042(24)01111-8</a></span></div></li><li id="https://pmc.ncbi.nlm.nih.gov/articles/PMC7596369/" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7596369/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://pmc.ncbi.nlm.nih.gov/articles/PMC7596369/</a></span></div></li><li id="5" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Hou Y, Lin S (2009). <span class="inline text-[1em] leading-7">Redfield RJ</span> (ed.). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2737104" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Distinct Gene Number- Genome Size Relationships for Eukaryotes and Non-Eukaryotes: Gene Content Estimation for Dinoflagellate Genomes&quot;</a>. <em>PLOS ONE</em>. <strong>4</strong> (9): e6978. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2009PLoSO...4.6978H" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2009PLoSO...4.6978H</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1371%2Fjournal.pone.0006978" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1371/journal.pone.0006978</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2737104" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2737104</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/19750009" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">19750009</a>.</span></div></li><li id="https://elifesciences.org/articles/100574" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://elifesciences.org/articles/100574" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://elifesciences.org/articles/100574</a></span></div></li><li id="https://www.pnas.org/doi/10.1073/pnas.2404332122" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.pnas.org/doi/10.1073/pnas.2404332122" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.pnas.org/doi/10.1073/pnas.2404332122</a></span></div></li><li id="6" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Piovesan A, Pelleri MC, Antonaros F, Strippoli P, Caracausi M, Vitale L (2019). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;On the length, weight and GC content of the human genome&quot;</a>. <em>BMC Res Notes</em>. <strong>12</strong> (1): 106. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1186%2Fs13104-019-4137-z" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1186/s13104-019-4137-z</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">6391780</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/30813969" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">30813969</a>.</span></div></li><li id="7" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Gregory SG, Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, et al. (May 2006). <a href="https://doi.org/10.1038%2Fnature04727" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The DNA sequence and biological annotation of human chromosome 1&quot;</a>. <em>Nature</em>. <strong>441</strong> (7091): 315–21. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2006Natur.441..315G" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2006Natur.441..315G</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2Fnature04727" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/nature04727</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/16710414" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">16710414</a>.</span></div></li><li id="8" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Anderson, S.; Bankier, A. T.; Barrell, B. G.; de Bruijn, M. H. L.; Coulson, A. R.; Drouin, J.; Eperon, I. C.; Nierlich, D. P.; Roe, B. A.; Sanger, F.; Schreier, P. H.; Smith, A. J. H.; Staden, R.; Young, I. G. (April 1981). &quot;Sequence and organization of the human mitochondrial genome&quot;. <em>Nature</em>. <strong>290</strong> (5806): 457–465. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1981Natur.290..457A" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1981Natur.290..457A</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2F290457a0" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/290457a0</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/7219534" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">7219534</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:4355527" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4355527</a>.</span></div></li><li id="9" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://web.archive.org/web/20110813123936/http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Untitled&quot;</a>. Archived from <a href="http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on 2011-08-13. Retrieved 2012-06-13.</span></div></li><li id="10" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Satoh, M; Kuroiwa, T (September 1991). &quot;Organization of multiple nucleoids and DNA molecules in mitochondria of a human cell&quot;. <em>Experimental Cell Research</em>. <strong>196</strong> (1): 137–140. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2F0014-4827%2891%2990467-9" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/0014-4827(91)90467-9</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/1715276" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1715276</a>.</span></div></li><li id="11" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Zhang D, Keilty D, Zhang ZF, Chian RC (2017). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Mitochondria in oocyte aging: current understanding&quot;</a>. <em>Facts Views Vis Obgyn</em>. <strong>9</strong> (1): 29–38. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">5506767</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/28721182" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">28721182</a>.</span></div></li><li id="12" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Dufresne A, Garczarek L, Partensky F (2005). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Accelerated evolution associated with genome reduction in a free-living prokaryote&quot;</a>. <em>Genome Biol</em>. <strong>6</strong> (2): R14. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1186%2Fgb-2005-6-2-r14" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1186/gb-2005-6-2-r14</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">551534</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/15693943" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">15693943</a>.</span></div></li><li id="13" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Giovannoni SJ; et al. (2005). &quot;Genome streamlining in a cosmopolitan oceanic bacterium&quot;. <em>Science</em>. <strong>309</strong> (5738): 1242–1245. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2005Sci...309.1242G" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2005Sci...309.1242G</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1126%2Fscience.1114057" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1126/science.1114057</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/16109880" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">16109880</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:16221415" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">16221415</a>.</span></div></li><li id="14" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Giovannoni SJ; et al. (2008). &quot;The small genome of an abundant coastal ocean methylotroph&quot;. <em>Environmental Microbiology</em>. <strong>10</strong> (7): 1771–1782. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2008EnvMi..10.1771G" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2008EnvMi..10.1771G</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1111%2Fj.1462-2920.2008.01598.x" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1111/j.1462-2920.2008.01598.x</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/18393994" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">18393994</a>.</span></div></li><li id="15" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Rodríguez-Gijón A; et al. (2022). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;A Genomic Perspective Across Earth&#x27;s Microbiomes Reveals That Genome Size in Archaea and Bacteria Is Linked to Ecosystem Type and Trophic Strategy&quot;</a>. <em>Frontiers in Microbiology</em>. <strong>12</strong>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.3389%2Ffmicb.2021.761869" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.3389/fmicb.2021.761869</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8767057</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/35069467" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">35069467</a>.</span></div></li><li id="16" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Moran, N. A. (1996-04-02). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Accelerated evolution and Muller&#x27;s rachet in endosymbiotic bacteria&quot;</a>. <em>Proceedings of the National Academy of Sciences</em>. <strong>93</strong> (7): 2873–2878. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1996PNAS...93.2873M" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1996PNAS...93.2873M</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1073%2Fpnas.93.7.2873" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1073/pnas.93.7.2873</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0027-8424" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0027-8424</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">39726</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/8610134" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8610134</a>.</span></div></li><li id="17" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Wernegreen, J. J.; Moran, N. A. (1999-01-01). <a href="https://academic.oup.com/mbe/article/16/1/83/993196" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Evidence for genetic drift in endosymbionts (Buchnera): analyses of protein-coding genes&quot;</a>. <em>Molecular Biology and Evolution</em>. <strong>16</strong> (1): 83–97. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a026040" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/oxfordjournals.molbev.a026040</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0737-4038" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0737-4038</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/10331254" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10331254</a>.</span></div></li><li id="18" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Spaulding, Allen W.; Dohlen, Carol D. von (2001). &quot;Psyllid endosymbionts exhibit patterns of co-speciation with hosts and destabilizing substitutions in ribosomal RNA&quot;. <em>Insect Molecular Biology</em>. <strong>10</strong> (1): 57–67. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1046%2Fj.1365-2583.2001.00231.x" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1046/j.1365-2583.2001.00231.x</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1365-2583" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1365-2583</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11240637" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11240637</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:46186732" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">46186732</a>.</span></div></li><li id="19" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://web.archive.org/web/20130823164405/http://phenomena.nationalgeographic.com/2013/08/23/and-the-genomes-keep-shrinking/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">And the Genomes Keep Shrinking…</a></span></div></li><li id="https://www.biorxiv.org/content/10.1101/2025.05.16.654412v1" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654412v1" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.biorxiv.org/content/10.1101/2025.05.16.654412v1</a></span></div></li><li id="20" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Wernegreen J (2005). <a href="https://web.archive.org/web/20110722013724/http://journals2005.pasteur.ac.ir/COGD/15(6).pdf" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;For better or worse: Genomic consequences of genomic mutualism and parasitism&quot;</a> (PDF). <em>Current Opinion in Genetics &amp; Development</em>. <strong>15</strong> (6): 1–12. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2Fj.gde.2005.09.013" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/j.gde.2005.09.013</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/16230003" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">16230003</a>. Archived from <a href="http://journals2005.pasteur.ac.ir/COGD/15(6).pdf#page=10" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> (PDF) on 2011-07-22.</span></div></li><li id="21" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Moran NA, Plague GR (2004). &quot;Genomic changes following host restriction in bacteria&quot;. <em>Current Opinion in Genetics &amp; Development</em>. <strong>14</strong> (6): 627–633. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2Fj.gde.2004.09.003" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/j.gde.2004.09.003</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/15531157" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">15531157</a>.</span></div></li><li id="22" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Mushegian, A. R.; Koonin, E. V. (1996-09-17). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;A minimal gene set for cellular life derived by comparison of complete bacterial genomes&quot;</a>. <em>Proceedings of the National Academy of Sciences</em>. <strong>93</strong> (19): 10268–10273. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1996PNAS...9310268M" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1996PNAS...9310268M</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1073%2Fpnas.93.19.10268" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1073/pnas.93.19.10268</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0027-8424" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0027-8424</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">38373</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/8816789" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8816789</a>.</span></div></li><li id="23" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Huynen, Martijn A.; Bork, Peer (1998-05-26). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Measuring genome evolution&quot;</a>. <em>Proceedings of the National Academy of Sciences</em>. <strong>95</strong> (11): 5849–5856. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1998PNAS...95.5849H" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1998PNAS...95.5849H</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1073%2Fpnas.95.11.5849" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1073/pnas.95.11.5849</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0027-8424" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0027-8424</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">34486</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/9600883" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">9600883</a>.</span></div></li><li id="24" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Maniloff, J (1996-09-17). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The minimal cell genome: &quot;on being the right size&quot;&quot;</a>. <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <strong>93</strong> (19): 10004–10006. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1996PNAS...9310004M" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1996PNAS...9310004M</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1073%2Fpnas.93.19.10004" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1073/pnas.93.19.10004</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0027-8424" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0027-8424</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">38325</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/8816738" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8816738</a>.</span></div></li><li id="40" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Mira, Alex; Ochman, Howard; Moran, Nancy A. (2001-10-01). &quot;Deletional bias and the evolution of bacterial genomes&quot;. <em>Trends in Genetics</em>. <strong>17</strong> (10): 589–596. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2FS0168-9525%2801%2902447-7" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/S0168-9525(01)02447-7</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0168-9525" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0168-9525</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11585665" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11585665</a>.</span></div></li><li id="25" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Drake, J W (1991). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;A constant rate of spontaneous mutation in DNA-based microbes&quot;</a>. <em>Proc Natl Acad Sci USA</em>. <strong>88</strong> (16): 7160–7164. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1991PNAS...88.7160D" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1991PNAS...88.7160D</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1073%2Fpnas.88.16.7160" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1073/pnas.88.16.7160</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">52253</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/1831267" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1831267</a>.</span></div></li><li id="26" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Kun, A; Santos, M; Szathmary, E (2005). &quot;Real ribozymes suggest a relaxed error threshold&quot;. <em>Nat Genet</em>. <strong>37</strong> (9): 1008–1011. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2Fng1621" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/ng1621</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/16127452" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">16127452</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:30582475" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">30582475</a>.</span></div></li><li id="27" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Lauber, C; Goeman, JJ; Parquet Mdel, C; Thi Nga, P; Snijder, EJ; Morita, K; Gorbalenya, AE (Jul 2013). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The footprint of genome architecture in the largest genome expansion in RNA viruses&quot;</a>. <em>PLOS Pathog</em>. <strong>9</strong> (7): e1003500. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1371%2Fjournal.ppat.1003500" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1371/journal.ppat.1003500</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3715407</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/23874204" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">23874204</a>.</span></div></li><li id="28" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Bennett, Michael David; Riley, Ralph (1972-06-06). &quot;Nuclear DNA content and minimum generation time in herbaceous plants&quot;. <em>Proceedings of the Royal Society of London. Series B. Biological Sciences</em>. <strong>181</strong> (1063): 109–135. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1972RSPSB.181..109B" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1972RSPSB.181..109B</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1098%2Frspb.1972.0042" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1098/rspb.1972.0042</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/4403285" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4403285</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:26642634" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">26642634</a>.</span></div></li><li id="29" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Hof, J. Van&#x27;t; Sparrow, A. H. (June 1963). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;A relationship between DNA content, nuclear volume, and minimum mitotic cycle time&quot;</a>. <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <strong>49</strong> (6): 897–902. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1963PNAS...49..897V" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1963PNAS...49..897V</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1073%2Fpnas.49.6.897" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1073/pnas.49.6.897</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0027-8424" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0027-8424</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">300029</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/13996145" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">13996145</a>.</span></div></li><li id="32" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Cavalier-Smith, Thomas (2005-01-01). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Economy, Speed and Size Matter: Evolutionary Forces Driving Nuclear Genome Miniaturization and Expansion&quot;</a>. <em>Annals of Botany</em>. <strong>95</strong> (1): 147–175. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Faob%2Fmci010" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/aob/mci010</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0305-7364" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0305-7364</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4246715</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/15596464" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">15596464</a>.</span></div></li><li id="33" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Strasburger, Eduard (1893). <em>Ueber die wirkungssphäre der Kerne und die Zellgrösse</em> (in German). <span class="inline text-[1em] leading-7">OCLC</span> <a href="https://search.worldcat.org/oclc/80359142" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">80359142</a>.</span></div></li><li id="34" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Huxley, J. S. (May 1925). &quot;The Cell in Development and Heredity&quot;. <em>Nature</em>. <strong>115</strong> (2897): 669–671. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1925Natur.115..669H" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1925Natur.115..669H</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2F115669a0" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/115669a0</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1476-4687" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1476-4687</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:26264738" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">26264738</a>.</span></div></li><li id="35" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Moran, Nancy A.; Mira, Alex (2001-11-14). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The process of genome shrinkage in the obligate symbiont Buchnera aphidicola&quot;</a>. <em>Genome Biology</em>. <strong>2</strong> (12): research0054.1. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1186%2Fgb-2001-2-12-research0054" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1186/gb-2001-2-12-research0054</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1474-760X" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1474-760X</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">64839</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11790257" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11790257</a>.</span></div></li><li id="36" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Blattner, Frederick R.; Plunkett, Guy; Bloch, Craig A.; Perna, Nicole T.; Burland, Valerie; Riley, Monica; Collado-Vides, Julio; Glasner, Jeremy D.; Rode, Christopher K.; Mayhew, George F.; Gregor, Jason (1997-09-05). <a href="https://doi.org/10.1126%2Fscience.277.5331.1453" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The Complete Genome Sequence of Escherichia coli K-12&quot;</a>. <em>Science</em>. <strong>277</strong> (5331): 1453–1462. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1126%2Fscience.277.5331.1453" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1126/science.277.5331.1453</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0036-8075" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0036-8075</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/9278503" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">9278503</a>.</span></div></li><li id="37" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Shigenobu, Shuji; Watanabe, Hidemi; Hattori, Masahira; Sakaki, Yoshiyuki; Ishikawa, Hajime (September 2000). <a href="https://doi.org/10.1038%2F35024074" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Genome sequence of the endocellular bacterial symbiont of aphids Buchnera sp. APS&quot;</a>. <em>Nature</em>. <strong>407</strong> (6800): 81–86. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2000Natur.407...81S" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2000Natur.407...81S</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2F35024074" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/35024074</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1476-4687" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1476-4687</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/10993077" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10993077</a>.</span></div></li><li id="38" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Andersson, J. O.; <span class="inline text-[1em] leading-7">Andersson, S. G.</span> (1999-09-01). <a href="https://academic.oup.com/mbe/article/16/9/1178/2925514" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Genome degradation is an ongoing process in Rickettsia&quot;</a>. <em>Molecular Biology and Evolution</em>. <strong>16</strong> (9): 1178–1191. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a026208" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/oxfordjournals.molbev.a026208</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0737-4038" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0737-4038</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/10486973" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10486973</a>.</span></div></li><li id="39" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Andersson, Jan O.; Andersson, Siv G. E. (2001-05-01). <a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a003864" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Pseudogenes, Junk DNA, and the Dynamics of Rickettsia Genomes&quot;</a>. <em>Molecular Biology and Evolution</em>. <strong>18</strong> (5): 829–839. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a003864" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/oxfordjournals.molbev.a003864</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0737-4038" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0737-4038</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11319266" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11319266</a>.</span></div></li><li id="43" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Wernegreen, J. J.; Moran, N. A. (2000-07-22). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Decay of mutualistic potential in aphid endosymbionts through silencing of biosynthetic loci: Buchnera of Diuraphis&quot;</a>. <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em>. <strong>267</strong> (1451): 1423–1431. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1098%2Frspb.2000.1159" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1098/rspb.2000.1159</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1690690</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/10983826" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10983826</a>.</span></div></li><li id="41" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Andersson, Siv G. E.</span>; Zomorodipour, Alireza; Andersson, Jan O.; Sicheritz-Pontén, Thomas; Alsmark, U. Cecilia M.; Podowski, Raf M.; Näslund, A. Kristina; Eriksson, Ann-Sofie; Winkler, Herbert H.; <span class="inline text-[1em] leading-7">Kurland, Charles G.</span> (November 1998). <a href="https://doi.org/10.1038%2F24094" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The genome sequence of Rickettsia prowazekii and the origin of mitochondria&quot;</a>. <em>Nature</em>. <strong>396</strong> (6707): 133–140. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1998Natur.396..133A" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1998Natur.396..133A</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2F24094" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/24094</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1476-4687" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1476-4687</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/9823893" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">9823893</a>.</span></div></li><li id="44" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Petrov, Dmitri A. (2002-06-01). &quot;Mutational Equilibrium Model of Genome Size Evolution&quot;. <em>Theoretical Population Biology</em>. <strong>61</strong> (4): 531–544. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2002TPBio..61..531P" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2002TPBio..61..531P</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1006%2Ftpbi.2002.1605" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1006/tpbi.2002.1605</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0040-5809" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0040-5809</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12167373" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12167373</a>.</span></div></li><li id="45" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Gregory, T. Ryan (2003-09-01). &quot;Is small indel bias a determinant of genome size?&quot;. <em>Trends in Genetics</em>. <strong>19</strong> (9): 485–488. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2FS0168-9525%2803%2900192-6" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/S0168-9525(03)00192-6</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0168-9525" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0168-9525</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12957541" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12957541</a>.</span></div></li><li id="47" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Selosse, M.-A.; Albert, B.; Godelle, B. (2001-03-01). &quot;Reducing the genome size of organelles favours gene transfer to the nucleus&quot;. <em>Trends in Ecology &amp; Evolution</em>. <strong>16</strong> (3): 135–141. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2Fs0169-5347%2800%2902084-x" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/s0169-5347(00)02084-x</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1872-8383" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1872-8383</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11179577" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11179577</a>.</span></div></li><li id="48" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Scherbakov, D. V.; Garber, M. B. (2000-07-01). &quot;Overlapping genes in bacterial and phage genomes&quot;. <em>Molecular Biology</em>. <strong>34</strong> (4): 485–495. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1007%2FBF02759558" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1007/BF02759558</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1608-3245" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1608-3245</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:24144602" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">24144602</a>.</span></div></li><li id="49" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Williams, Bryony A. P.; Hirt, Robert P.; Lucocq, John M.; Embley, T. Martin (August 2002). &quot;A mitochondrial remnant in the microsporidian Trachipleistophora hominis&quot;. <em>Nature</em>. <strong>418</strong> (6900): 865–869. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2002Natur.418..865W" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2002Natur.418..865W</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2Fnature00949" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/nature00949</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1476-4687" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1476-4687</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12192407" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12192407</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:4358253" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4358253</a>.</span></div></li><li id="50" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Keeling, Patrick J.; Fast, Naomi M. (2002). &quot;Microsporidia: Biology and Evolution of Highly Reduced Intracellular Parasites&quot;. <em>Annual Review of Microbiology</em>. <strong>56</strong> (1): 93–116. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1146%2Fannurev.micro.56.012302.160854" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1146/annurev.micro.56.012302.160854</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12142484" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12142484</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:22943269" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">22943269</a>.</span></div></li><li id="51" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Cavalier-Smith, T. (2001). &quot;What are Fungi?&quot;. In McLaughlin, David J.; McLaughlin, Esther G.; Lemke, Paul A. (eds.). <em>Systematics and Evolution</em>. The Mycota. Springer Berlin Heidelberg. pp. 3–37. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1007%2F978-3-662-10376-0_1" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1007/978-3-662-10376-0_1</a>. <span class="inline text-[1em] leading-7">ISBN</span> 978-3-662-10376-0. <code class="text-fg-solar bg-fg-solar-opaque break-all rounded px-2 align-middle font-mono text-sm" node="[object Object]">{{cite book}}</code>: <code class="text-fg-solar bg-fg-solar-opaque break-all rounded px-2 align-middle font-mono text-sm" node="[object Object]">|work=</code> ignored (help)</span></div></li><li id="https://www.cell.com/current-biology/fulltext/S0960-9822(23)00862-X" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(23)00862-X" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.cell.com/current-biology/fulltext/S0960-9822(23)00862-X</a></span></div></li><li id="52" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Vivarès, Christian P; Gouy, Manolo; Thomarat, Fabienne; Méténier, Guy (2002-10-01). &quot;Functional and evolutionary analysis of a eukaryotic parasitic genome&quot;. <em>Current Opinion in Microbiology</em>. <strong>5</strong> (5): 499–505. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2FS1369-5274%2802%2900356-9" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/S1369-5274(02)00356-9</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1369-5274" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1369-5274</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/12354558" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">12354558</a>.</span></div></li><li id="53" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Douglas, Susan; Zauner, Stefan; Fraunholz, Martin; Beaton, Margaret; Penny, Susanne; Deng, Lang-Tuo; Wu, Xiaonan; Reith, Michael; Cavalier-Smith, Thomas; Maier, Uwe-G. (April 2001). <a href="https://doi.org/10.1038%2F35074092" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The highly reduced genome of an enslaved algal nucleus&quot;</a>. <em>Nature</em>. <strong>410</strong> (6832): 1091–1096. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2001Natur.410.1091D" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2001Natur.410.1091D</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2F35074092" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/35074092</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1476-4687" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1476-4687</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11323671" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11323671</a>.</span></div></li></ol></div><div class="text-fg-secondary text-xs [&amp;_*]:mb-2"><hr class="border-border-l1 my-8 rounded-sm border-t"/>
<span class="mb-4 block break-words text-[1em] leading-7">The content is adapted from <a href="https://en.wikipedia.org/wiki/Genome_size" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">Wikipedia</a>, licensed under Creative Commons Attribution-ShareAlike 4.0 License.</span></div></div></div><div class="hidden min-[1400px]:block"></div></div></div><!--$--><!--/$--><div role="region" aria-label="Notifications (F8)" tabindex="-1" style="pointer-events:none"><ol tabindex="-1" class="pointer-events-none fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 max-sm:left-0 sm:bottom-0 sm:right-0 sm:top-auto sm:max-w-[420px] sm:flex-col pl-16 pr-2 pt-16 sm:px-4"></ol></div><script type="application/json" id="server-client-data-experimentation">{"status":"uninitialized"}</script><script src="/_next/static/chunks/webpack-e121ed42680f327e.js" nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz" id="_R_" async=""></script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">(self.__next_f=self.__next_f||[]).push([0])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"1:\"$Sreact.fragment\"\n3:I[91363,[],\"\"]\n4:I[23775,[],\"\"]\n5:I[57654,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"PageHeaderProvider\"]\n6:I[17618,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"HeaderContent\"]\n7:I[25529,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"6751\",\"static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js\"],\"default\"]\n9:I[6666,[],\"OutletBoundary\"]\nb:I[80415,[],\"AsyncMetadataOutlet\"]\nd:I[6666,[],\"ViewportBoundary\"]\nf:I[6666,[],\"MetadataBoundary\"]\n10:\"$Sreact.suspense\"\n12:I[95909,[\"4219\",\"static/chunks/app/global-error-4d07d20223cd4b4c.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"13:I[51498,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"ConstantsProvider\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"14:I[91073,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"15:I[38642,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"16:I[99648,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"17:I[78825,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"18:I[8550,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MixpanelProvider\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"19:I[12290,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MobileTocProvider\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"1a:I[53947,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"NuqsAdapter\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"1b:I[91873,[\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8039\",\"static/chunks/app/error-4a29e9399afba038.js\"],\"default\"]\n1c:I[5091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"4345\",\"static/chunks/app/not-found-dd95690acf732f18.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"1d:I[16091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,":HL[\"/_next/static/media/1f2316909698f815.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/3d4419af2cf8609b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/4dec29efcaeb336c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/74452ea3ef0f9101.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/904ef0a86fe32a00.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/d886a03bcda7ad8f.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/e1447589d6f59c4b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/f5a90156f8995c8c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/css/62c4caba71dfda84.css\",\"style\",{\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]\n:HL[\"/_next/static/css/eb3d87f98fe1565f.css\",\"style\",{\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]\n:HL[\"/_next/static/css/1b5e561215938d4d.css\",\"style\",{\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]\n:HL[\"/_next/static/css/0227d069a630d414.css\",\"style\",{\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]\n:HL[\"/_next/static/css/f87fff2ab93d05a7.css\",\"style\",{\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"0:{\"P\":null,\"b\":\"BpM29AX4fmgcUxZ_6t0le\",\"p\":\"\",\"c\":[\"\",\"page\",\"Genome_size\"],\"i\":false,\"f\":[[[\"\",{\"children\":[\"page\",{\"children\":[[\"slug\",\"Genome_size\",\"d\"],{\"children\":[\"__PAGE__\",{}]}]}]},\"$undefined\",\"$undefined\",true],[\"\",[\"$\",\"$1\",\"c\",{\"children\":[[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/62c4caba71dfda84.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}],[\"$\",\"link\",\"1\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/eb3d87f98fe1565f.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}],[\"$\",\"link\",\"2\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/1b5e561215938d4d.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}],[\"$\",\"link\",\"3\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/0227d069a630d414.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]],\"$L2\"]}],{\"children\":[\"page\",[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[[\"slug\",\"Genome_size\",\"d\"],[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L5\",null,{\"children\":[[\"$\",\"$L6\",null,{\"maxWidth\":\"full\",\"mobileOptions\":{\"right\":{\"showFixedIssues\":true,\"showThemeSwitcher\":true,\"showSearch\":true,\"showTableOfContents\":true}}}],[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L7\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}]]}],{\"children\":[\"__PAGE__\",[\"$\",\"$1\",\"c\",{\"children\":[\"$L8\",[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/f87fff2ab93d05a7.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\"}]],[\"$\",\"$L9\",null,{\"children\":[\"$La\",[\"$\",\"$Lb\",null,{\"promise\":\"$@c\"}]]}]]}],{},null,false]},null,false]},null,false]},null,false],[\"$\",\"$1\",\"h\",{\"children\":[null,[[\"$\",\"$Ld\",null,{\"children\":\"$Le\"}],[\"$\",\"meta\",null,{\"name\":\"next-size-adjust\",\"content\":\"\"}]],[\"$\",\"$Lf\",null,{\"children\":[\"$\",\"div\",null,{\"hidden\":true,\"children\":[\"$\",\"$10\",null,{\"fallback\":null,\"children\":\"$L11\"}]}]}]]}],false]],\"m\":\"$undefined\",\"G\":[\"$12\",[]],\"s\":false,\"S\":false}\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"2:[\"$\",\"html\",null,{\"lang\":\"en\",\"className\":\"bg-surface-base antialiased\",\"suppressHydrationWarning\":true,\"children\":[[\"$\",\"head\",null,{\"children\":[[\"$\",\"title\",null,{\"children\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"type\":\"image/x-icon\",\"href\":\"/favicon.ico\",\"sizes\":\"48x48\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-dark.png\",\"media\":\"(prefers-color-scheme: light)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-light.png\",\"media\":\"(prefers-color-scheme: dark)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"apple-touch-icon\",\"href\":\"/icon-192x192.png\"}],[\"$\",\"link\",null,{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\"}],[\"$\",\"meta\",null,{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",null,{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"meta\",null,{\"property\":\"og:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"property\":\"og:url\",\"content\":\"https://grokipedia.com\"}],[\"$\",\"meta\",null,{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:locale\",\"content\":\"en_US\"}],[\"$\",\"meta\",null,{\"property\":\"og:type\",\"content\":\"website\"}],[\"$\",\"meta\",null,{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:alt\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}]]}],[\"$\",\"body\",null,{\"className\":\"flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b\",\"children\":[[\"$\",\"$L13\",null,{\"children\":[\"$\",\"$L14\",null,{\"children\":[\"$\",\"$L15\",null,{\"defaultTheme\":\"dark\",\"nonce\":\"YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz\",\"children\":[\"$\",\"$L16\",null,{\"children\":[\"$\",\"$L17\",null,{\"children\":[\"$\",\"$L18\",null,{\"children\":[\"$\",\"$L19\",null,{\"children\":[\"$\",\"$L1a\",null,{\"children\":[[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$1b\",\"errorStyles\":[],\"errorScripts\":[],\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L1c\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}],[\"$\",\"$L1d\",null,{}]]}]}]}]}]}]}]}]}],[\"$\",\"script\",null,{\"type\":\"application/json\",\"id\":\"server-client-data-experimentation\",\"children\":\"{\\\"status\\\":\\\"uninitialized\\\"}\"}]]}]]}]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"e:[[\"$\",\"meta\",\"0\",{\"charSet\":\"utf-8\"}],[\"$\",\"meta\",\"1\",{\"name\":\"viewport\",\"content\":\"width=device-width, initial-scale=1, interactive-widget=resizes-content\"}]]\na:null\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"1e:I[42712,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"HydrationBoundary\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"21:I[6367,[],\"IconMark\"]\n1f:T4b04,"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"# Genome size\n\n**Genome size** is the total amount of [DNA](DNA \"DNA\") contained within one copy of a single complete [genome](Genome \"Genome\"). It is typically measured in terms of [mass](Mass \"Mass\") in picograms (trillionths or 10−12 of a [gram](Gram \"Gram\"), abbreviated pg) or less frequently in [daltons](Dalton_\\(unit\\) \"Dalton \\(unit\\)\"), or as the total number of [nucleotide](Nucleotide \"Nucleotide\") [base pairs](Base_pair \"Base pair\"), usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In [diploid](Diploid \"Diploid\") [organisms](Organism \"Organism\"), genome size is often used interchangeably with the term [C-value](C-value \"C-value\"). \n\nAn organism's complexity is not directly proportional to its genome size; total DNA content is widely variable between biological taxa. Some single-celled organisms have much more DNA than humans, for reasons that remain unclear (see [Junk DNA](Junk_DNA \"Junk DNA\") and [C-value](C-value \"C-value\")).\n# Variation in genome size and gene content\n\nSince the 1950s, with the emergence of various molecular techniques, the genome sizes of thousands of [eukaryotes](Eukaryote \"Eukaryote\") have been analyzed, and these data are available in online databases for animals, plants, and fungi (see external links), such as the Animal Genome Size Database, Plant DNA C-values Database, and the Protist 10,000 Genomes Project portal. Nuclear genome size is typically measured in eukaryotes using either [densitometric](Densitometry \"Densitometry\") measurements of [Feulgen](Feulgen \"Feulgen\")-stained nuclei (previously using specialized densitometers, now more commonly using computerized [image analysis](Image_analysis \"Image analysis\")[2]) or [flow cytometry](Flow_cytometry \"Flow cytometry\"). In [prokaryotes](Prokaryote \"Prokaryote\"), [pulsed field gel electrophoresis](Pulsed_field_gel_electrophoresis \"Pulsed field gel electrophoresis\") and complete [genome sequencing](Genome_sequencing \"Genome sequencing\") are the predominant methods of genome size determination. \n\nNuclear genome sizes are well known to vary enormously among eukaryotic species. In animals they range more than 6,600-fold, and in land plants they differ by a factor of over 2,300 (with the largest known now ~160 Gbp in the fork fern _Tmesipteris oblanceolata_ as of 2024).[3][4][](https://www.sciencedirect.com/science/article/pii/S0960982219302027)[](https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.16315)[](https://www.cell.com/iscience/fulltext/S2589-0042(24)01111-8) [Protist](Protist \"Protist\") genomes have been reported to vary more than 28,600-fold in size, though some extreme values (e.g., in amoebae) remain debated.[](https://pmc.ncbi.nlm.nih.gov/articles/PMC7596369/)\n\nGenome size in eukaryotes was thought to be proportional to the complexity of an organism but by the mid-20th century it became apparent that closely related species could differ substantially in the size of their genomes. This counter-intuitive observation gave rise to what became known as the \"[C-value paradox](C-value_paradox \"C-value paradox\").\" \n\nBy the end of 1960s this \"paradox\" was resolved by the discovery of [repetitive DNA](Repeated_sequence_\\(DNA\\) \"Repeated sequence \\(DNA\\)\") and the realization that much of the differences in genomes sizes was due to the presence or absence of large amounts of repetitive DNA. Further advances at that time indicated that most of the DNA in large genomes was non-functional [junk DNA](Junk_DNA \"Junk DNA\") and only a small fraction corresponded to functional DNA, including the functional parts of genes. \n\nGenome size correlates with a range of measurable characteristics at the [cell](Cell_\\(biology\\) \"Cell \\(biology\\)\") and organism levels, including cell size, [cell division](Cell_division \"Cell division\") rate, and, depending on the [taxon](Taxon \"Taxon\"), body size, [metabolic rate](Metabolic_rate \"Metabolic rate\"), developmental rate, [organ](Organ_\\(anatomy\\) \"Organ \\(anatomy\\)\") complexity, geographical distribution, or [extinction](Extinction \"Extinction\") risk.[3][4] Early analyses of sequenced genomes (as of 2009) suggested a linear correlation between log-transformed gene number and log-transformed genome size in prokaryotes, viruses, and organelles, and a nonlinear correlation in eukaryotes, potentially due to increasing junk DNA.[5] However, with extensive additional sequencing by 2024, no clear correlation exists between gene number and genome size in eukaryotes, as variation is largely driven by non-coding elements.[](https://elifesciences.org/articles/100574)[](https://www.pnas.org/doi/10.1073/pnas.2404332122)\n\n### Human genome size\n\nIn humans, the total female [diploid](Diploid \"Diploid\") [nuclear genome](Nuclear_genome \"Nuclear genome\") per cell extends for 6.37 Gigabase pairs (Gbp), is 208.23 cm long and weighs 6.51 picograms (pg).[6] Male values are 6.27 Gbp, 205.00 cm, 6.41 pg.[6] Each DNA polymer can contain hundreds of millions of nucleotides, such as in [chromosome 1](Chromosome_1 \"Chromosome 1\"). Chromosome 1 is the largest human [chromosome](Chromosome \"Chromosome\") with approximately 220 million [base pairs](Base_pair \"Base pair\"), and would be 85 mm long if straightened.[7]\n\nIn [eukaryotes](Eukaryote \"Eukaryote\"), in addition to [nuclear DNA](Nuclear_DNA \"Nuclear DNA\"), there is also [mitochondrial DNA](Mitochondrial_DNA \"Mitochondrial DNA\") (mtDNA) which encodes certain proteins used by the mitochondria. The mtDNA is usually relatively small in comparison to the nuclear DNA. For example, the [human mitochondrial DNA](Human_mitochondrial_genetics \"Human mitochondrial genetics\") forms closed circular molecules, each of which contains 16,569[8][9] DNA base pairs,[10] with each such molecule normally containing a full set of the mitochondrial genes. Each human mitochondrion contains, on average, approximately 5 such mtDNA molecules.[10] Each human [cell](Cell_\\(biology\\) \"Cell \\(biology\\)\") contains approximately 100 mitochondria, giving a total number of mtDNA molecules per human cell of approximately 500.[10] However, the amount of mitochondria per cell also varies by cell type, and an [egg cell](Egg_cell \"Egg cell\") can contain 100,000 mitochondria, corresponding to up to 1,500,000 copies of the mitochondrial genome (constituting up to 90% of the DNA of the cell).[11]\n# Genome reduction\n\n**Genome reduction**, also known as **genome degradation**, is the process by which an organism's genome shrinks relative to that of its ancestors. Genomes fluctuate in size regularly, and genome size reduction is most significant in bacteria. \n\nThe most evolutionarily significant cases of genome reduction may be observed in the eukaryotic organelles known to be derived from bacteria: mitochondria and plastids. These organelles are descended from primordial endosymbionts, which were capable of surviving within the host cell and which the host cell likewise needed for survival. Many present-day mitochondria have less than 20 genes in their entire genome, whereas a modern free-living bacterium generally has at least 1,000 genes. Many genes have apparently been transferred to the host nucleus, while others have simply been lost and their function replaced by host processes. \n\nOther bacteria have become endosymbionts or obligate intracellular pathogens and experienced extensive genome reduction as a result. This process seems to be dominated by genetic drift resulting from small population size, low recombination rates, and high mutation rates, as opposed to selection for smaller genomes. Some free-living marine bacterioplanktons also show signs of genome reduction, which are hypothesized to be driven by natural selection.[12][13][14] In contrast, terrestrial prokaryotes appear to have larger genome sizes than both aquatic and host-associated prokaryotes (average of 3.7 Mbp for terrestrial, 3.1 Mbp for aquatic and 3.0 Mbp for host-associated).[15]\n\n### In obligate endosymbiotic species\n\nObligate endosymbiotic species are characterized by a complete inability to survive external to their host environment. While some obligate endosymbionts are pathogenic and pose threats to human health by evading immune systems and manipulating host environments for nutrients, many others form mutualistic relationships, providing essential benefits to their hosts such as nutrient provisioning. Their compact genomes result from massive losses of extraneous DNA, an occurrence strongly associated with the loss of a free-living stage. As much as 90% of the genetic material can be lost when a species makes the evolutionary transition from a free-living to an obligate intracellular lifestyle. During this process, the organism is subjected to a metabolite-rich environment and must persist within the host cell; these factors reduce retention of non-essential genes and increase genetic drift, accelerating the loss of superfluous genes.[16][17][18] Common examples of species with reduced genomes include _Buchnera aphidicola_ (a mutualist), _Rickettsia prowazekii_ (a pathogen), and _Mycobacterium leprae_ (a pathogen). One obligate endosymbiont of leafhoppers, _Nasuia deltocephalinicola_, had one of the smallest known genomes at the time of its discovery (112 kb in 2013).[19] More recent research has identified even smaller genomes, such as approximately 50-52 kb in symbionts of planthoppers (as of 2025).[[](https://www.biorxiv.org/content/10.1101/2025.05.16.654412v1)] While some obligate intracellular species are pathogenic, many provide positive fitness effects to their hosts. \n\nThe **reductive evolution model** has been proposed as an effort to define the genomic commonalities seen in all obligate endosymbionts.[20] This model illustrates four general features of reduced genomes and obligate intracellular species: \n\n1. \"genome streamlining\" resulting from relaxed selection on genes that are superfluous in the intracellular environment;\n2. a bias towards deletions (rather than insertions), which heavily affects genes that have been disrupted by accumulation of mutations (pseudogenes);[21]\n3. very little or no capability for acquiring new DNA; and\n4. considerable reduction of effective population size in endosymbiotic populations, particularly in species that rely on vertical transmission of genetic material.\n\nRecent studies continue to support this model, highlighting deletional bias and genetic drift as key drivers of genome miniaturization, with patterns varying across different symbionts.[22][23][24][40]\n# Conversion from picograms (pg) to base pairs (bp)\n\n\\[\n\\text{number of base pairs} = \\text{mass in pg} \\times 9.78 \\times 10^{8}\n\\]\n\nor simply: \n\n\\[\n1 \\, \\text{pg} = 978 \\, \\text{Mbp}\n\\][1]\n# Drake's rule\n\nIn 1991, [John W. Drake](John_W._Drake \"John W. Drake\") proposed a general rule: that the per-base-pair mutation rate and genome size are inversely correlated, resulting in a roughly constant mutation rate per genome.[25] This rule has been found to be approximately correct for simple genomes such as those in [DNA viruses](DNA_virus \"DNA virus\") and unicellular organisms. Its basis is unknown. \n\nIt has been proposed that the small size of [RNA viruses](RNA_virus \"RNA virus\") is locked into a three-part relation between replication fidelity, genome size, and genetic complexity. The majority of RNA viruses lack an RNA proofreading facility, which limits their replication fidelity and hence their genome size. This has also been described as the [Eigen paradox](Error_threshold_\\(evolution\\)#Eigen's_paradox \"Error threshold \\(evolution\\)\").[26] An exception to the rule of small genome sizes in RNA viruses is found in the [Nidoviruses](Nidovirales \"Nidovirales\"). These viruses appear to have acquired a [3′-to-5′ exoribonuclease](Exoribonuclease \"Exoribonuclease\") (ExoN) which has allowed for an increase in genome size.[27]\n# Genome miniaturization and optimal size\n\nIn 1972, Michael D. Bennett hypothesized a correlation between nuclear DNA content and minimum generation time in herbaceous plants.[28] Van't Hof and Sparrow postulated relationships between DNA content, nuclear volume, cell size, and minimum mitotic cycle time.[29]\n\nCavalier-Smith disputed these and other earlier explanations in his 2005 article,[32] proposing the skeletal theory to explain the relation between genome size and cell volume. According to this theory, nuclear DNA acts as a nucleoskeleton that determines nuclear volume, which scales with cell volume to maintain an optimal karyoplasmic ratio—the ratio of nuclear to cytoplasmic volume that balances nuclear RNA export and cytoplasmic protein synthesis for efficient cell growth.[33][34][32] Larger genomes are more prone to accumulation of non-coding \"skeletal\" DNA, including duplicative transposons.[32] In response to cell size reduction, selection favors deletions over duplications to economize on scarce resources like phosphorus and energy.[32]\n\nFrom an economic perspective, evolutionary pressures favor DNA reduction unless offset by functional benefits, as random deletions are often deleterious but can occasionally enhance fitness. This trade-off between economy and non-coding DNA accumulation helps maintain the karyoplasmic ratio.[32]\n## Mechanisms of genome miniaturization\n\nThe base question behind the process of genome miniaturization is whether it occurs through large steps or due to a constant erosion of the gene content. In order to assess the evolution of this process is necessary to compare an ancestral genome with the one where the shrinkage is supposed to be occurred. Thanks to the similarity among the gene content of _Buchnera aphidicola_ and the enteric bacteria _Escherichia coli_ , 89% identity for the 16S rDNA and 62% for [orthologous](Orthologous \"Orthologous\") genes was possible to shed light on the mechanism of genome miniaturization.[35] The genome of the [endosymbiont](Endosymbiont \"Endosymbiont\") _B. aphidicola_ is characterized by a genome size that is seven times smaller than _E. coli_ (643 kb compared to 4.6 Mb)[36][37] and can be view as a subset of the enteric bacteria gene inventory.[37] From the confrontation of the two genomes emerged that some genes persist as partially degraded.[37] indicating that the function was lost during the process and that consequent events of erosion shortened the length as documented in _Rickettsia_.[38][39][40] This hypothesis is confirmed by the analysis of the [pseudogenes](Pseudogene \"Pseudogene\") of _Buchnera_ where the number of deletions was more than ten times higher compared to the insertion.[40]\n\nIn _Rickettsia prowazekii_ , as with other small genome bacteria, this parasitic endosymbiont has experienced a vast reduction of functional activity. The common effects of the genome shrinking between this endosymbiont and the other parasites are the reduction of the ability to produce phospholipids, repair and recombination and an overall conversion of the composition of the gene to a richer A-T[43] content due to mutation and substitutions.[16][41] Evidence of the deletion of the function of repair and recombination is the loss of the gene _rec_ A, gene involved in the [recombinase](Recombinase \"Recombinase\") pathway. This event happened during the removal of a larger region containing ten genes for a total of almost 10 kb.[37][41] Same faith occurred _uvr_ A, _uvr_ B and _uvr_ C, genes encoding for excision enzymes involved in the repair of damaged DNA due to UV exposure.[35]\n\nOne of the most plausible mechanisms for the explanation of the genome shrinking is the chromosomal rearrangement because insertion/deletion of larger portion of sequence are more easily to be seen in during homologous recombination compared to the illegitimate, therefore the spread of the [transposable elements](Transposable_element \"Transposable element\") will positively affect the rate of deletion.[32] The loss of those genes in the early stages of miniaturization not only this function but must played a role in the evolution of the consequent deletions. Evidences of the fact that larger event of removal occurred before smaller deletion emerged from the comparison of the genome of _Bucknera_ and a reconstructed ancestor, where the gene that have been lost are in fact not randomly dispersed in the ancestor gene but aggregated and the negative relation between number of lost genes and length of the spacers.[35] The event of small local indels plays a marginal role on the genome reduction[44] especially in the early stages where a larger number of genes became superfluous.[45][35]\n\nSingle events instead occurred due to the lack of selection pressure for the retention of genes especially if part of a pathway that lost its function during a previous deletion. An example for this is the deletion of _rec_ F, gene required for the function of _rec_ A, and its flanking genes.[35] One of the consequences of the elimination of such amount of sequences affected even the regulation of the remaining genes. The loss of large section of genomes could in fact lead to a loss in promotor sequences. This could in fact pushed the selection for the evolution of [polycistronic regions](Polycistronic_operon \"Polycistronic operon\") with a positive effect for both size reduction[47] and transcription efficiency.[48]\n## Evidence of genome miniaturization\n\nOne example of the miniaturization of the genome occurred in the microsporidia, an anaerobic intracellular parasite of animals evolved from aerobic fungi. \n\nDuring this process the mitosomes[49] was formed consequent to the reduction of the mitochondria to a relic voided of genomes and metabolic activity except to the production of iron sulfur centers.[50][51] Except for the ribosomes, miniaturized as well, many other organelles have been almost lost during the process of the formation of the smallest known eukaryotic nuclear genome.[32] For example, _Encephalitozoon intestinalis_ has a nuclear genome of 2.3 Mb.[](https://www.cell.com/current-biology/fulltext/S0960-9822(23)00862-X) From their possible ancestor, a zygomycotine fungi, the microsporidia have shrunk their genome eliminating thousands of genes and reduced even the size of protein and protein-coding genes.[52] This extreme process was possible thanks to the advantageous selection for a smaller cell size imposed by the parasitism. \n\nAnother example of miniaturization is represented by the presence of nucleomorphs, enslaved nuclei, inside of the cell of two different algae, cryptophytes and chlorarachniophytes.[53]\n\nNucleomorphs are characterized by one of the smallest genomes known (551 and 380 kb) and as noticed for microsporidia, some genomes are noticeably reduced in length compared to other eukaryotes due to a virtual lack of non-coding DNA.[32] The most interesting factor is represented by the coexistence of those small nuclei inside of a cell that contains another nucleus that never experienced such genome reduction. Moreover, even if the host cells have different volumes from species to species and a consequent variability in genome size, the nucleomorph remain invariant denoting a double effect of selection within the same cell."])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"8:[\"$\",\"$L1e\",null,{\"state\":{\"mutations\":[],\"queries\":[{\"dehydratedAt\":1761887017522,\"state\":{\"data\":{\"page\":{\"citations\":[{\"id\":\"1\",\"title\":\"_**a**_ _**b**_ Dolezel J, Bartoš J, Voglmayr H, Greilhuber J (2003). \\\"Nuclear DNA content and genome size of trout and human\\\". _Cytometry Part A_. **51** (2): 127–128\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1002/cyto.a.10013](https://doi.org/10.1002%2Fcyto.a.10013). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12541287](https://pubmed.ncbi.nlm.nih.gov/12541287). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [221604791](https://api.semanticscholar.org/CorpusID:221604791).\",\"description\":\"1. ^ _**a**_ _**b**_ Dolezel J, Bartoš J, Voglmayr H, Greilhuber J (2003). \\\"Nuclear DNA content and genome size of trout and human\\\". _Cytometry Part A_. **51** (2): 127–128\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1002/cyto.a.10013](https://doi.org/10.1002%2Fcyto.a.10013). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12541287](https://pubmed.ncbi.nlm.nih.gov/12541287). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [221604791](https://api.semanticscholar.org/CorpusID:221604791).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"2\",\"title\":\"** Hardie DC, Gregory TR, Hebert PD (2002). \\\"From pixels to picograms: a beginners' guide to genome quantification by Feulgen image analysis densitometry\\\". _Journal of Histochemistry and Cytochemistry_. **50** (6): 735–749\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1177/002215540205000601](https://doi.org/10.1177%2F002215540205000601). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12019291](https://pubmed.ncbi.nlm.nih.gov/12019291). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [33117040](https://api.semanticscholar.org/CorpusID:33117040).\",\"description\":\"2. **^** Hardie DC, Gregory TR, Hebert PD (2002). \\\"From pixels to picograms: a beginners' guide to genome quantification by Feulgen image analysis densitometry\\\". _Journal of Histochemistry and Cytochemistry_. **50** (6): 735–749\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1177/002215540205000601](https://doi.org/10.1177%2F002215540205000601). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12019291](https://pubmed.ncbi.nlm.nih.gov/12019291). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [33117040](https://api.semanticscholar.org/CorpusID:33117040).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"3\",\"title\":\"_**a**_ _**b**_ Bennett MD, Leitch IJ (2005). \\\"Genome size evolution in plants\\\". In T.R. Gregory (ed.). [_The Evolution of the Genome_](https://archive.org/details/evolutiongenome00greg). San Diego: Elsevier. pp. [89](https://archive.org/details/evolutiongenome00greg/page/n116)–162.\",\"description\":\"3. ^ _**a**_ _**b**_ Bennett MD, Leitch IJ (2005). \\\"Genome size evolution in plants\\\". In T.R. Gregory (ed.). [_The Evolution of the Genome_](https://archive.org/details/evolutiongenome00greg). San Diego: Elsevier. pp. [89](https://archive.org/details/evolutiongenome00greg/page/n116)–162.\",\"url\":\"https://archive.org/details/evolutiongenome00greg\",\"favicon\":\"\"},{\"id\":\"4\",\"title\":\"_**a**_ _**b**_ Gregory TR (2005). \\\"Genome size evolution in animals\\\". In T.R. Gregory (ed.). [_The Evolution of the Genome_](https://archive.org/details/evolutiongenome00greg). San Diego: Elsevier. pp. [3](https://archive.org/details/evolutiongenome00greg/page/n30)–87.\",\"description\":\"4. ^ _**a**_ _**b**_ Gregory TR (2005). \\\"Genome size evolution in animals\\\". In T.R. Gregory (ed.). [_The Evolution of the Genome_](https://archive.org/details/evolutiongenome00greg). San Diego: Elsevier. pp. [3](https://archive.org/details/evolutiongenome00greg/page/n30)–87.\",\"url\":\"https://archive.org/details/evolutiongenome00greg\",\"favicon\":\"\"},{\"id\":\"5\",\"title\":\"** Hou Y, Lin S (2009). [Redfield RJ](Rosemary_Redfield \\\"Rosemary Redfield\\\") (ed.). [\\\"Distinct Gene Number- Genome Size Relationships for Eukaryotes and Non-Eukaryotes: Gene Content Estimation for Dinoflagellate Genomes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2737104). _PLOS ONE_. **4** (9): e6978. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2009PLoSO...4.6978H](https://ui.adsabs.harvard.edu/abs/2009PLoSO...4.6978H). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.pone.0006978](https://doi.org/10.1371%2Fjournal.pone.0006978). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [2737104](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2737104). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [19750009](https://pubmed.ncbi.nlm.nih.gov/19750009).\",\"description\":\"5. **^** Hou Y, Lin S (2009). [Redfield RJ](Rosemary_Redfield \\\"Rosemary Redfield\\\") (ed.). [\\\"Distinct Gene Number- Genome Size Relationships for Eukaryotes and Non-Eukaryotes: Gene Content Estimation for Dinoflagellate Genomes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2737104). _PLOS ONE_. **4** (9): e6978. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2009PLoSO...4.6978H](https://ui.adsabs.harvard.edu/abs/2009PLoSO...4.6978H). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.pone.0006978](https://doi.org/10.1371%2Fjournal.pone.0006978). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [2737104](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2737104). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [19750009](https://pubmed.ncbi.nlm.nih.gov/19750009).\",\"url\":\"Rosemary_Redfield \\\"Rosemary Redfield\\\"\",\"favicon\":\"\"},{\"id\":\"6\",\"title\":\"_**a**_ _**b**_ Piovesan A, Pelleri MC, Antonaros F, Strippoli P, Caracausi M, Vitale L (2019). [\\\"On the length, weight and GC content of the human genome\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780). _BMC Res Notes_. **12** (1): 106. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/s13104-019-4137-z](https://doi.org/10.1186%2Fs13104-019-4137-z). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [6391780](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [30813969](https://pubmed.ncbi.nlm.nih.gov/30813969).\",\"description\":\"6. ^ _**a**_ _**b**_ Piovesan A, Pelleri MC, Antonaros F, Strippoli P, Caracausi M, Vitale L (2019). [\\\"On the length, weight and GC content of the human genome\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780). _BMC Res Notes_. **12** (1): 106. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/s13104-019-4137-z](https://doi.org/10.1186%2Fs13104-019-4137-z). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [6391780](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [30813969](https://pubmed.ncbi.nlm.nih.gov/30813969).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391780\",\"favicon\":\"\"},{\"id\":\"7\",\"title\":\"** Gregory SG, Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, et al. (May 2006). [\\\"The DNA sequence and biological annotation of human chromosome 1\\\"](https://doi.org/10.1038%2Fnature04727). _Nature_. **441** (7091): 315–21\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2006Natur.441..315G](https://ui.adsabs.harvard.edu/abs/2006Natur.441..315G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nature04727](https://doi.org/10.1038%2Fnature04727). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16710414](https://pubmed.ncbi.nlm.nih.gov/16710414).\",\"description\":\"7. **^** Gregory SG, Barlow KF, McLay KE, Kaul R, Swarbreck D, Dunham A, et al. (May 2006). [\\\"The DNA sequence and biological annotation of human chromosome 1\\\"](https://doi.org/10.1038%2Fnature04727). _Nature_. **441** (7091): 315–21\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2006Natur.441..315G](https://ui.adsabs.harvard.edu/abs/2006Natur.441..315G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nature04727](https://doi.org/10.1038%2Fnature04727). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16710414](https://pubmed.ncbi.nlm.nih.gov/16710414).\",\"url\":\"https://doi.org/10.1038%2Fnature04727\",\"favicon\":\"\"},{\"id\":\"8\",\"title\":\"** Anderson, S.; Bankier, A. T.; Barrell, B. G.; de Bruijn, M. H. L.; Coulson, A. R.; Drouin, J.; Eperon, I. C.; Nierlich, D. P.; Roe, B. A.; Sanger, F.; Schreier, P. H.; Smith, A. J. H.; Staden, R.; Young, I. G. (April 1981). \\\"Sequence and organization of the human mitochondrial genome\\\". _Nature_. **290** (5806): 457–465\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1981Natur.290..457A](https://ui.adsabs.harvard.edu/abs/1981Natur.290..457A). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/290457a0](https://doi.org/10.1038%2F290457a0). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [7219534](https://pubmed.ncbi.nlm.nih.gov/7219534). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4355527](https://api.semanticscholar.org/CorpusID:4355527).\",\"description\":\"8. **^** Anderson, S.; Bankier, A. T.; Barrell, B. G.; de Bruijn, M. H. L.; Coulson, A. R.; Drouin, J.; Eperon, I. C.; Nierlich, D. P.; Roe, B. A.; Sanger, F.; Schreier, P. H.; Smith, A. J. H.; Staden, R.; Young, I. G. (April 1981). \\\"Sequence and organization of the human mitochondrial genome\\\". _Nature_. **290** (5806): 457–465\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1981Natur.290..457A](https://ui.adsabs.harvard.edu/abs/1981Natur.290..457A). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/290457a0](https://doi.org/10.1038%2F290457a0). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [7219534](https://pubmed.ncbi.nlm.nih.gov/7219534). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4355527](https://api.semanticscholar.org/CorpusID:4355527).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"9\",\"title\":\"** [\\\"Untitled\\\"](https://web.archive.org/web/20110813123936/http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html). Archived from [the original](http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html) on 2011-08-13. Retrieved 2012-06-13.\",\"description\":\"9. **^** [\\\"Untitled\\\"](https://web.archive.org/web/20110813123936/http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html). Archived from [the original](http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html) on 2011-08-13. Retrieved 2012-06-13.\",\"url\":\"https://web.archive.org/web/20110813123936/http://chemistry.umeche.maine.edu/CHY431/MitoDNA.html\",\"favicon\":\"\"},{\"id\":\"10\",\"title\":\"_**a**_ _**b**_ _**c**_ Satoh, M; Kuroiwa, T (September 1991). \\\"Organization of multiple nucleoids and DNA molecules in mitochondria of a human cell\\\". _Experimental Cell Research_. **196** (1): 137–140\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/0014-4827(91)90467-9](https://doi.org/10.1016%2F0014-4827%2891%2990467-9). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1715276](https://pubmed.ncbi.nlm.nih.gov/1715276).\",\"description\":\"10. ^ _**a**_ _**b**_ _**c**_ Satoh, M; Kuroiwa, T (September 1991). \\\"Organization of multiple nucleoids and DNA molecules in mitochondria of a human cell\\\". _Experimental Cell Research_. **196** (1): 137–140\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/0014-4827(91)90467-9](https://doi.org/10.1016%2F0014-4827%2891%2990467-9). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1715276](https://pubmed.ncbi.nlm.nih.gov/1715276).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"11\",\"title\":\"** Zhang D, Keilty D, Zhang ZF, Chian RC (2017). [\\\"Mitochondria in oocyte aging: current understanding\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767). _Facts Views Vis Obgyn_. **9** (1): 29–38\\\\. [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [5506767](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [28721182](https://pubmed.ncbi.nlm.nih.gov/28721182).\",\"description\":\"11. **^** Zhang D, Keilty D, Zhang ZF, Chian RC (2017). [\\\"Mitochondria in oocyte aging: current understanding\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767). _Facts Views Vis Obgyn_. **9** (1): 29–38\\\\. [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [5506767](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [28721182](https://pubmed.ncbi.nlm.nih.gov/28721182).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506767\",\"favicon\":\"\"},{\"id\":\"12\",\"title\":\"** Dufresne A, Garczarek L, Partensky F (2005). [\\\"Accelerated evolution associated with genome reduction in a free-living prokaryote\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534). _Genome Biol_. **6** (2): R14. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/gb-2005-6-2-r14](https://doi.org/10.1186%2Fgb-2005-6-2-r14). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [551534](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [15693943](https://pubmed.ncbi.nlm.nih.gov/15693943).\",\"description\":\"12. **^** Dufresne A, Garczarek L, Partensky F (2005). [\\\"Accelerated evolution associated with genome reduction in a free-living prokaryote\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534). _Genome Biol_. **6** (2): R14. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/gb-2005-6-2-r14](https://doi.org/10.1186%2Fgb-2005-6-2-r14). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [551534](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [15693943](https://pubmed.ncbi.nlm.nih.gov/15693943).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551534\",\"favicon\":\"\"},{\"id\":\"13\",\"title\":\"** Giovannoni SJ; et al. (2005). \\\"Genome streamlining in a cosmopolitan oceanic bacterium\\\". _Science_. **309** (5738): 1242–1245\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2005Sci...309.1242G](https://ui.adsabs.harvard.edu/abs/2005Sci...309.1242G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.1114057](https://doi.org/10.1126%2Fscience.1114057). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16109880](https://pubmed.ncbi.nlm.nih.gov/16109880). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [16221415](https://api.semanticscholar.org/CorpusID:16221415).\",\"description\":\"13. **^** Giovannoni SJ; et al. (2005). \\\"Genome streamlining in a cosmopolitan oceanic bacterium\\\". _Science_. **309** (5738): 1242–1245\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2005Sci...309.1242G](https://ui.adsabs.harvard.edu/abs/2005Sci...309.1242G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.1114057](https://doi.org/10.1126%2Fscience.1114057). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16109880](https://pubmed.ncbi.nlm.nih.gov/16109880). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [16221415](https://api.semanticscholar.org/CorpusID:16221415).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"14\",\"title\":\"** Giovannoni SJ; et al. (2008). \\\"The small genome of an abundant coastal ocean methylotroph\\\". _Environmental Microbiology_. **10** (7): 1771–1782\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2008EnvMi..10.1771G](https://ui.adsabs.harvard.edu/abs/2008EnvMi..10.1771G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1111/j.1462-2920.2008.01598.x](https://doi.org/10.1111%2Fj.1462-2920.2008.01598.x). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [18393994](https://pubmed.ncbi.nlm.nih.gov/18393994).\",\"description\":\"14. **^** Giovannoni SJ; et al. (2008). \\\"The small genome of an abundant coastal ocean methylotroph\\\". _Environmental Microbiology_. **10** (7): 1771–1782\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2008EnvMi..10.1771G](https://ui.adsabs.harvard.edu/abs/2008EnvMi..10.1771G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1111/j.1462-2920.2008.01598.x](https://doi.org/10.1111%2Fj.1462-2920.2008.01598.x). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [18393994](https://pubmed.ncbi.nlm.nih.gov/18393994).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"15\",\"title\":\"** Rodríguez-Gijón A; et al. (2022). [\\\"A Genomic Perspective Across Earth's Microbiomes Reveals That Genome Size in Archaea and Bacteria Is Linked to Ecosystem Type and Trophic Strategy\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057). _Frontiers in Microbiology_. **12**. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.3389/fmicb.2021.761869](https://doi.org/10.3389%2Ffmicb.2021.761869). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [8767057](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [35069467](https://pubmed.ncbi.nlm.nih.gov/35069467).\",\"description\":\"15. **^** Rodríguez-Gijón A; et al. (2022). [\\\"A Genomic Perspective Across Earth's Microbiomes Reveals That Genome Size in Archaea and Bacteria Is Linked to Ecosystem Type and Trophic Strategy\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057). _Frontiers in Microbiology_. **12**. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.3389/fmicb.2021.761869](https://doi.org/10.3389%2Ffmicb.2021.761869). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [8767057](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [35069467](https://pubmed.ncbi.nlm.nih.gov/35069467).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8767057\",\"favicon\":\"\"},{\"id\":\"16\",\"title\":\"_**a**_ _**b**_ Moran, N. A. (1996-04-02). [\\\"Accelerated evolution and Muller's rachet in endosymbiotic bacteria\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726). _Proceedings of the National Academy of Sciences_. **93** (7): 2873–2878\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1996PNAS...93.2873M](https://ui.adsabs.harvard.edu/abs/1996PNAS...93.2873M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.93.7.2873](https://doi.org/10.1073%2Fpnas.93.7.2873). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [39726](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8610134](https://pubmed.ncbi.nlm.nih.gov/8610134).\",\"description\":\"16. ^ _**a**_ _**b**_ Moran, N. A. (1996-04-02). [\\\"Accelerated evolution and Muller's rachet in endosymbiotic bacteria\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726). _Proceedings of the National Academy of Sciences_. **93** (7): 2873–2878\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1996PNAS...93.2873M](https://ui.adsabs.harvard.edu/abs/1996PNAS...93.2873M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.93.7.2873](https://doi.org/10.1073%2Fpnas.93.7.2873). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [39726](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8610134](https://pubmed.ncbi.nlm.nih.gov/8610134).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39726\",\"favicon\":\"\"},{\"id\":\"17\",\"title\":\"** Wernegreen, J. J.; Moran, N. A. (1999-01-01). [\\\"Evidence for genetic drift in endosymbionts (Buchnera): analyses of protein-coding genes\\\"](https://academic.oup.com/mbe/article/16/1/83/993196). _Molecular Biology and Evolution_. **16** (1): 83–97\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/oxfordjournals.molbev.a026040](https://doi.org/10.1093%2Foxfordjournals.molbev.a026040). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0737-4038](https://search.worldcat.org/issn/0737-4038). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10331254](https://pubmed.ncbi.nlm.nih.gov/10331254).\",\"description\":\"17. **^** Wernegreen, J. J.; Moran, N. A. (1999-01-01). [\\\"Evidence for genetic drift in endosymbionts (Buchnera): analyses of protein-coding genes\\\"](https://academic.oup.com/mbe/article/16/1/83/993196). _Molecular Biology and Evolution_. **16** (1): 83–97\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/oxfordjournals.molbev.a026040](https://doi.org/10.1093%2Foxfordjournals.molbev.a026040). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0737-4038](https://search.worldcat.org/issn/0737-4038). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10331254](https://pubmed.ncbi.nlm.nih.gov/10331254).\",\"url\":\"https://academic.oup.com/mbe/article/16/1/83/993196\",\"favicon\":\"\"},{\"id\":\"18\",\"title\":\"** Spaulding, Allen W.; Dohlen, Carol D. von (2001). \\\"Psyllid endosymbionts exhibit patterns of co-speciation with hosts and destabilizing substitutions in ribosomal RNA\\\". _Insect Molecular Biology_. **10** (1): 57–67\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1046/j.1365-2583.2001.00231.x](https://doi.org/10.1046%2Fj.1365-2583.2001.00231.x). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1365-2583](https://search.worldcat.org/issn/1365-2583). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11240637](https://pubmed.ncbi.nlm.nih.gov/11240637). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [46186732](https://api.semanticscholar.org/CorpusID:46186732).\",\"description\":\"18. **^** Spaulding, Allen W.; Dohlen, Carol D. von (2001). \\\"Psyllid endosymbionts exhibit patterns of co-speciation with hosts and destabilizing substitutions in ribosomal RNA\\\". _Insect Molecular Biology_. **10** (1): 57–67\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1046/j.1365-2583.2001.00231.x](https://doi.org/10.1046%2Fj.1365-2583.2001.00231.x). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1365-2583](https://search.worldcat.org/issn/1365-2583). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11240637](https://pubmed.ncbi.nlm.nih.gov/11240637). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [46186732](https://api.semanticscholar.org/CorpusID:46186732).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"19\",\"title\":\"** [And the Genomes Keep Shrinking…](https://web.archive.org/web/20130823164405/http://phenomena.nationalgeographic.com/2013/08/23/and-the-genomes-keep-shrinking/)\",\"description\":\"19. **^** [And the Genomes Keep Shrinking…](https://web.archive.org/web/20130823164405/http://phenomena.nationalgeographic.com/2013/08/23/and-the-genomes-keep-shrinking/)\",\"url\":\"https://web.archive.org/web/20130823164405/http://phenomena.nationalgeographic.com/2013/08/23/and-the-genomes-keep-shrinking/\",\"favicon\":\"\"},{\"id\":\"20\",\"title\":\"** Wernegreen J (2005). [\\\"For better or worse: Genomic consequences of genomic mutualism and parasitism\\\"](https://web.archive.org/web/20110722013724/http://journals2005.pasteur.ac.ir/COGD/15\\\\(6\\\\).pdf) (PDF). _Current Opinion in Genetics \u0026 Development_. **15** (6): 1–12\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.gde.2005.09.013](https://doi.org/10.1016%2Fj.gde.2005.09.013). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16230003](https://pubmed.ncbi.nlm.nih.gov/16230003). Archived from [the original](http://journals2005.pasteur.ac.ir/COGD/15\\\\(6\\\\).pdf#page=10) (PDF) on 2011-07-22.\",\"description\":\"20. **^** Wernegreen J (2005). [\\\"For better or worse: Genomic consequences of genomic mutualism and parasitism\\\"](https://web.archive.org/web/20110722013724/http://journals2005.pasteur.ac.ir/COGD/15\\\\(6\\\\).pdf) (PDF). _Current Opinion in Genetics \u0026 Development_. **15** (6): 1–12\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.gde.2005.09.013](https://doi.org/10.1016%2Fj.gde.2005.09.013). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16230003](https://pubmed.ncbi.nlm.nih.gov/16230003). Archived from [the original](http://journals2005.pasteur.ac.ir/COGD/15\\\\(6\\\\).pdf#page=10) (PDF) on 2011-07-22.\",\"url\":\"https://web.archive.org/web/20110722013724/http://journals2005.pasteur.ac.ir/COGD/15\\\\(6\\\\\",\"favicon\":\"\"},{\"id\":\"21\",\"title\":\"** Moran NA, Plague GR (2004). \\\"Genomic changes following host restriction in bacteria\\\". _Current Opinion in Genetics \u0026 Development_. **14** (6): 627–633\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.gde.2004.09.003](https://doi.org/10.1016%2Fj.gde.2004.09.003). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [15531157](https://pubmed.ncbi.nlm.nih.gov/15531157).\",\"description\":\"21. **^** Moran NA, Plague GR (2004). \\\"Genomic changes following host restriction in bacteria\\\". _Current Opinion in Genetics \u0026 Development_. **14** (6): 627–633\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/j.gde.2004.09.003](https://doi.org/10.1016%2Fj.gde.2004.09.003). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [15531157](https://pubmed.ncbi.nlm.nih.gov/15531157).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"22\",\"title\":\"** Mushegian, A. R.; Koonin, E. V. (1996-09-17). [\\\"A minimal gene set for cellular life derived by comparison of complete bacterial genomes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373). _Proceedings of the National Academy of Sciences_. **93** (19): 10268–10273\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1996PNAS...9310268M](https://ui.adsabs.harvard.edu/abs/1996PNAS...9310268M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.93.19.10268](https://doi.org/10.1073%2Fpnas.93.19.10268). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [38373](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8816789](https://pubmed.ncbi.nlm.nih.gov/8816789).\",\"description\":\"22. **^** Mushegian, A. R.; Koonin, E. V. (1996-09-17). [\\\"A minimal gene set for cellular life derived by comparison of complete bacterial genomes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373). _Proceedings of the National Academy of Sciences_. **93** (19): 10268–10273\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1996PNAS...9310268M](https://ui.adsabs.harvard.edu/abs/1996PNAS...9310268M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.93.19.10268](https://doi.org/10.1073%2Fpnas.93.19.10268). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [38373](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8816789](https://pubmed.ncbi.nlm.nih.gov/8816789).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38373\",\"favicon\":\"\"},{\"id\":\"23\",\"title\":\"** Huynen, Martijn A.; Bork, Peer (1998-05-26). [\\\"Measuring genome evolution\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486). _Proceedings of the National Academy of Sciences_. **95** (11): 5849–5856\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1998PNAS...95.5849H](https://ui.adsabs.harvard.edu/abs/1998PNAS...95.5849H). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.95.11.5849](https://doi.org/10.1073%2Fpnas.95.11.5849). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [34486](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9600883](https://pubmed.ncbi.nlm.nih.gov/9600883).\",\"description\":\"23. **^** Huynen, Martijn A.; Bork, Peer (1998-05-26). [\\\"Measuring genome evolution\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486). _Proceedings of the National Academy of Sciences_. **95** (11): 5849–5856\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1998PNAS...95.5849H](https://ui.adsabs.harvard.edu/abs/1998PNAS...95.5849H). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.95.11.5849](https://doi.org/10.1073%2Fpnas.95.11.5849). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [34486](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9600883](https://pubmed.ncbi.nlm.nih.gov/9600883).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC34486\",\"favicon\":\"\"},{\"id\":\"24\",\"title\":\"** Maniloff, J (1996-09-17). [\\\"The minimal cell genome: \\\"on being the right size\\\"\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325). _Proceedings of the National Academy of Sciences of the United States of America_. **93** (19): 10004–10006\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1996PNAS...9310004M](https://ui.adsabs.harvard.edu/abs/1996PNAS...9310004M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.93.19.10004](https://doi.org/10.1073%2Fpnas.93.19.10004). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [38325](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8816738](https://pubmed.ncbi.nlm.nih.gov/8816738).\",\"description\":\"24. **^** Maniloff, J (1996-09-17). [\\\"The minimal cell genome: \\\"on being the right size\\\"\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325). _Proceedings of the National Academy of Sciences of the United States of America_. **93** (19): 10004–10006\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1996PNAS...9310004M](https://ui.adsabs.harvard.edu/abs/1996PNAS...9310004M). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.93.19.10004](https://doi.org/10.1073%2Fpnas.93.19.10004). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [38325](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8816738](https://pubmed.ncbi.nlm.nih.gov/8816738).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38325\",\"favicon\":\"\"},{\"id\":\"25\",\"title\":\"** Drake, J W (1991). [\\\"A constant rate of spontaneous mutation in DNA-based microbes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253). _Proc Natl Acad Sci USA_. **88** (16): 7160–7164\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1991PNAS...88.7160D](https://ui.adsabs.harvard.edu/abs/1991PNAS...88.7160D). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.88.16.7160](https://doi.org/10.1073%2Fpnas.88.16.7160). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [52253](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1831267](https://pubmed.ncbi.nlm.nih.gov/1831267).\",\"description\":\"25. **^** Drake, J W (1991). [\\\"A constant rate of spontaneous mutation in DNA-based microbes\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253). _Proc Natl Acad Sci USA_. **88** (16): 7160–7164\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1991PNAS...88.7160D](https://ui.adsabs.harvard.edu/abs/1991PNAS...88.7160D). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.88.16.7160](https://doi.org/10.1073%2Fpnas.88.16.7160). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [52253](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [1831267](https://pubmed.ncbi.nlm.nih.gov/1831267).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52253\",\"favicon\":\"\"},{\"id\":\"26\",\"title\":\"** Kun, A; Santos, M; Szathmary, E (2005). \\\"Real ribozymes suggest a relaxed error threshold\\\". _Nat Genet_. **37** (9): 1008–1011\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/ng1621](https://doi.org/10.1038%2Fng1621). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16127452](https://pubmed.ncbi.nlm.nih.gov/16127452). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [30582475](https://api.semanticscholar.org/CorpusID:30582475).\",\"description\":\"26. **^** Kun, A; Santos, M; Szathmary, E (2005). \\\"Real ribozymes suggest a relaxed error threshold\\\". _Nat Genet_. **37** (9): 1008–1011\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/ng1621](https://doi.org/10.1038%2Fng1621). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [16127452](https://pubmed.ncbi.nlm.nih.gov/16127452). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [30582475](https://api.semanticscholar.org/CorpusID:30582475).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"27\",\"title\":\"** Lauber, C; Goeman, JJ; Parquet Mdel, C; Thi Nga, P; Snijder, EJ; Morita, K; Gorbalenya, AE (Jul 2013). [\\\"The footprint of genome architecture in the largest genome expansion in RNA viruses\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407). _PLOS Pathog_. **9** (7): e1003500. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.ppat.1003500](https://doi.org/10.1371%2Fjournal.ppat.1003500). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3715407](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23874204](https://pubmed.ncbi.nlm.nih.gov/23874204).\",\"description\":\"27. **^** Lauber, C; Goeman, JJ; Parquet Mdel, C; Thi Nga, P; Snijder, EJ; Morita, K; Gorbalenya, AE (Jul 2013). [\\\"The footprint of genome architecture in the largest genome expansion in RNA viruses\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407). _PLOS Pathog_. **9** (7): e1003500. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.ppat.1003500](https://doi.org/10.1371%2Fjournal.ppat.1003500). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3715407](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23874204](https://pubmed.ncbi.nlm.nih.gov/23874204).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715407\",\"favicon\":\"\"},{\"id\":\"28\",\"title\":\"** Bennett, Michael David; Riley, Ralph (1972-06-06). \\\"Nuclear DNA content and minimum generation time in herbaceous plants\\\". _Proceedings of the Royal Society of London. Series B. Biological Sciences_. **181** (1063): 109–135\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1972RSPSB.181..109B](https://ui.adsabs.harvard.edu/abs/1972RSPSB.181..109B). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1098/rspb.1972.0042](https://doi.org/10.1098%2Frspb.1972.0042). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [4403285](https://pubmed.ncbi.nlm.nih.gov/4403285). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [26642634](https://api.semanticscholar.org/CorpusID:26642634).\",\"description\":\"28. **^** Bennett, Michael David; Riley, Ralph (1972-06-06). \\\"Nuclear DNA content and minimum generation time in herbaceous plants\\\". _Proceedings of the Royal Society of London. Series B. Biological Sciences_. **181** (1063): 109–135\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1972RSPSB.181..109B](https://ui.adsabs.harvard.edu/abs/1972RSPSB.181..109B). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1098/rspb.1972.0042](https://doi.org/10.1098%2Frspb.1972.0042). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [4403285](https://pubmed.ncbi.nlm.nih.gov/4403285). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [26642634](https://api.semanticscholar.org/CorpusID:26642634).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"29\",\"title\":\"** Hof, J. Van't; Sparrow, A. H. (June 1963). [\\\"A relationship between DNA content, nuclear volume, and minimum mitotic cycle time\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029). _Proceedings of the National Academy of Sciences of the United States of America_. **49** (6): 897–902\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1963PNAS...49..897V](https://ui.adsabs.harvard.edu/abs/1963PNAS...49..897V). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.49.6.897](https://doi.org/10.1073%2Fpnas.49.6.897). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [300029](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [13996145](https://pubmed.ncbi.nlm.nih.gov/13996145).\",\"description\":\"29. **^** Hof, J. Van't; Sparrow, A. H. (June 1963). [\\\"A relationship between DNA content, nuclear volume, and minimum mitotic cycle time\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029). _Proceedings of the National Academy of Sciences of the United States of America_. **49** (6): 897–902\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1963PNAS...49..897V](https://ui.adsabs.harvard.edu/abs/1963PNAS...49..897V). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.49.6.897](https://doi.org/10.1073%2Fpnas.49.6.897). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [300029](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [13996145](https://pubmed.ncbi.nlm.nih.gov/13996145).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300029\",\"favicon\":\"\"},{\"id\":\"30\",\"title\":\"** Commoner, Barry (June 1964). \\\"Roles Of Deoxyribonucleic Acid in Inheritance\\\". _Nature_. **202** (4936): 960–968\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1964Natur.202..960C](https://ui.adsabs.harvard.edu/abs/1964Natur.202..960C). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/202960a0](https://doi.org/10.1038%2F202960a0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [14197326](https://pubmed.ncbi.nlm.nih.gov/14197326). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4166234](https://api.semanticscholar.org/CorpusID:4166234).\",\"description\":\"30. **^** Commoner, Barry (June 1964). \\\"Roles Of Deoxyribonucleic Acid in Inheritance\\\". _Nature_. **202** (4936): 960–968\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1964Natur.202..960C](https://ui.adsabs.harvard.edu/abs/1964Natur.202..960C). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/202960a0](https://doi.org/10.1038%2F202960a0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [14197326](https://pubmed.ncbi.nlm.nih.gov/14197326). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4166234](https://api.semanticscholar.org/CorpusID:4166234).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"31\",\"title\":\"** Orgel, L. E.; Crick, F. H. C. (April 1980). \\\"Selfish DNA: the ultimate parasite\\\". _Nature_. **284** (5757): 604–607\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1980Natur.284..604O](https://ui.adsabs.harvard.edu/abs/1980Natur.284..604O). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/284604a0](https://doi.org/10.1038%2F284604a0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [7366731](https://pubmed.ncbi.nlm.nih.gov/7366731). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4233826](https://api.semanticscholar.org/CorpusID:4233826).\",\"description\":\"31. **^** Orgel, L. E.; Crick, F. H. C. (April 1980). \\\"Selfish DNA: the ultimate parasite\\\". _Nature_. **284** (5757): 604–607\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1980Natur.284..604O](https://ui.adsabs.harvard.edu/abs/1980Natur.284..604O). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/284604a0](https://doi.org/10.1038%2F284604a0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [7366731](https://pubmed.ncbi.nlm.nih.gov/7366731). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4233826](https://api.semanticscholar.org/CorpusID:4233826).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"32\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ Cavalier-Smith, Thomas (2005-01-01). [\\\"Economy, Speed and Size Matter: Evolutionary Forces Driving Nuclear Genome Miniaturization and Expansion\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715). _Annals of Botany_. **95** (1): 147–175\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/aob/mci010](https://doi.org/10.1093%2Faob%2Fmci010). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0305-7364](https://search.worldcat.org/issn/0305-7364). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [4246715](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [15596464](https://pubmed.ncbi.nlm.nih.gov/15596464).\",\"description\":\"32. ^ _**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ Cavalier-Smith, Thomas (2005-01-01). [\\\"Economy, Speed and Size Matter: Evolutionary Forces Driving Nuclear Genome Miniaturization and Expansion\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715). _Annals of Botany_. **95** (1): 147–175\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/aob/mci010](https://doi.org/10.1093%2Faob%2Fmci010). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0305-7364](https://search.worldcat.org/issn/0305-7364). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [4246715](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [15596464](https://pubmed.ncbi.nlm.nih.gov/15596464).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246715\",\"favicon\":\"\"},{\"id\":\"33\",\"title\":\"** Strasburger, Eduard (1893). _Ueber die wirkungssphäre der Kerne und die Zellgrösse_ (in German). [OCLC](OCLC_\\\\(identifier\\\\) \\\"OCLC \\\\(identifier\\\\)\\\") [80359142](https://search.worldcat.org/oclc/80359142).\",\"description\":\"33. **^** Strasburger, Eduard (1893). _Ueber die wirkungssphäre der Kerne und die Zellgrösse_ (in German). [OCLC](OCLC_\\\\(identifier\\\\) \\\"OCLC \\\\(identifier\\\\)\\\") [80359142](https://search.worldcat.org/oclc/80359142).\",\"url\":\"OCLC_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"34\",\"title\":\"** Huxley, J. S. (May 1925). \\\"The Cell in Development and Heredity\\\". _Nature_. **115** (2897): 669–671\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1925Natur.115..669H](https://ui.adsabs.harvard.edu/abs/1925Natur.115..669H). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/115669a0](https://doi.org/10.1038%2F115669a0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [26264738](https://api.semanticscholar.org/CorpusID:26264738).\",\"description\":\"34. **^** Huxley, J. S. (May 1925). \\\"The Cell in Development and Heredity\\\". _Nature_. **115** (2897): 669–671\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1925Natur.115..669H](https://ui.adsabs.harvard.edu/abs/1925Natur.115..669H). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/115669a0](https://doi.org/10.1038%2F115669a0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [26264738](https://api.semanticscholar.org/CorpusID:26264738).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"35\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ Moran, Nancy A.; Mira, Alex (2001-11-14). [\\\"The process of genome shrinkage in the obligate symbiont Buchnera aphidicola\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839). _Genome Biology_. **2** (12): research0054.1. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/gb-2001-2-12-research0054](https://doi.org/10.1186%2Fgb-2001-2-12-research0054). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1474-760X](https://search.worldcat.org/issn/1474-760X). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [64839](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11790257](https://pubmed.ncbi.nlm.nih.gov/11790257).\",\"description\":\"35. ^ _**a**_ _**b**_ _**c**_ _**d**_ Moran, Nancy A.; Mira, Alex (2001-11-14). [\\\"The process of genome shrinkage in the obligate symbiont Buchnera aphidicola\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839). _Genome Biology_. **2** (12): research0054.1. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1186/gb-2001-2-12-research0054](https://doi.org/10.1186%2Fgb-2001-2-12-research0054). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1474-760X](https://search.worldcat.org/issn/1474-760X). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [64839](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11790257](https://pubmed.ncbi.nlm.nih.gov/11790257).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC64839\",\"favicon\":\"\"},{\"id\":\"36\",\"title\":\"** Blattner, Frederick R.; Plunkett, Guy; Bloch, Craig A.; Perna, Nicole T.; Burland, Valerie; Riley, Monica; Collado-Vides, Julio; Glasner, Jeremy D.; Rode, Christopher K.; Mayhew, George F.; Gregor, Jason (1997-09-05). [\\\"The Complete Genome Sequence of Escherichia coli K-12\\\"](https://doi.org/10.1126%2Fscience.277.5331.1453). _Science_. **277** (5331): 1453–1462\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.277.5331.1453](https://doi.org/10.1126%2Fscience.277.5331.1453). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0036-8075](https://search.worldcat.org/issn/0036-8075). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9278503](https://pubmed.ncbi.nlm.nih.gov/9278503).\",\"description\":\"36. **^** Blattner, Frederick R.; Plunkett, Guy; Bloch, Craig A.; Perna, Nicole T.; Burland, Valerie; Riley, Monica; Collado-Vides, Julio; Glasner, Jeremy D.; Rode, Christopher K.; Mayhew, George F.; Gregor, Jason (1997-09-05). [\\\"The Complete Genome Sequence of Escherichia coli K-12\\\"](https://doi.org/10.1126%2Fscience.277.5331.1453). _Science_. **277** (5331): 1453–1462\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.277.5331.1453](https://doi.org/10.1126%2Fscience.277.5331.1453). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0036-8075](https://search.worldcat.org/issn/0036-8075). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9278503](https://pubmed.ncbi.nlm.nih.gov/9278503).\",\"url\":\"https://doi.org/10.1126%2Fscience.277.5331.1453\",\"favicon\":\"\"},{\"id\":\"37\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ _**e**_ Shigenobu, Shuji; Watanabe, Hidemi; Hattori, Masahira; Sakaki, Yoshiyuki; Ishikawa, Hajime (September 2000). [\\\"Genome sequence of the endocellular bacterial symbiont of aphids Buchnera sp. APS\\\"](https://doi.org/10.1038%2F35024074). _Nature_. **407** (6800): 81–86\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2000Natur.407...81S](https://ui.adsabs.harvard.edu/abs/2000Natur.407...81S). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/35024074](https://doi.org/10.1038%2F35024074). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10993077](https://pubmed.ncbi.nlm.nih.gov/10993077).\",\"description\":\"37. ^ _**a**_ _**b**_ _**c**_ _**d**_ _**e**_ Shigenobu, Shuji; Watanabe, Hidemi; Hattori, Masahira; Sakaki, Yoshiyuki; Ishikawa, Hajime (September 2000). [\\\"Genome sequence of the endocellular bacterial symbiont of aphids Buchnera sp. APS\\\"](https://doi.org/10.1038%2F35024074). _Nature_. **407** (6800): 81–86\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2000Natur.407...81S](https://ui.adsabs.harvard.edu/abs/2000Natur.407...81S). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/35024074](https://doi.org/10.1038%2F35024074). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10993077](https://pubmed.ncbi.nlm.nih.gov/10993077).\",\"url\":\"https://doi.org/10.1038%2F35024074\",\"favicon\":\"\"},{\"id\":\"38\",\"title\":\"** Andersson, J. O.; [Andersson, S. G.](Siv_G._E._Andersson \\\"Siv G. E. Andersson\\\") (1999-09-01). [\\\"Genome degradation is an ongoing process in Rickettsia\\\"](https://academic.oup.com/mbe/article/16/9/1178/2925514). _Molecular Biology and Evolution_. **16** (9): 1178–1191\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/oxfordjournals.molbev.a026208](https://doi.org/10.1093%2Foxfordjournals.molbev.a026208). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0737-4038](https://search.worldcat.org/issn/0737-4038). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10486973](https://pubmed.ncbi.nlm.nih.gov/10486973).\",\"description\":\"38. **^** Andersson, J. O.; [Andersson, S. G.](Siv_G._E._Andersson \\\"Siv G. E. Andersson\\\") (1999-09-01). [\\\"Genome degradation is an ongoing process in Rickettsia\\\"](https://academic.oup.com/mbe/article/16/9/1178/2925514). _Molecular Biology and Evolution_. **16** (9): 1178–1191\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/oxfordjournals.molbev.a026208](https://doi.org/10.1093%2Foxfordjournals.molbev.a026208). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0737-4038](https://search.worldcat.org/issn/0737-4038). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10486973](https://pubmed.ncbi.nlm.nih.gov/10486973).\",\"url\":\"Siv_G._E._Andersson \\\"Siv G. E. Andersson\\\"\",\"favicon\":\"\"},{\"id\":\"39\",\"title\":\"** Andersson, Jan O.; Andersson, Siv G. E. (2001-05-01). [\\\"Pseudogenes, Junk DNA, and the Dynamics of Rickettsia Genomes\\\"](https://doi.org/10.1093%2Foxfordjournals.molbev.a003864). _Molecular Biology and Evolution_. **18** (5): 829–839\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/oxfordjournals.molbev.a003864](https://doi.org/10.1093%2Foxfordjournals.molbev.a003864). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0737-4038](https://search.worldcat.org/issn/0737-4038). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11319266](https://pubmed.ncbi.nlm.nih.gov/11319266).\",\"description\":\"39. **^** Andersson, Jan O.; Andersson, Siv G. E. (2001-05-01). [\\\"Pseudogenes, Junk DNA, and the Dynamics of Rickettsia Genomes\\\"](https://doi.org/10.1093%2Foxfordjournals.molbev.a003864). _Molecular Biology and Evolution_. **18** (5): 829–839\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/oxfordjournals.molbev.a003864](https://doi.org/10.1093%2Foxfordjournals.molbev.a003864). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0737-4038](https://search.worldcat.org/issn/0737-4038). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11319266](https://pubmed.ncbi.nlm.nih.gov/11319266).\",\"url\":\"https://doi.org/10.1093%2Foxfordjournals.molbev.a003864\",\"favicon\":\"\"},{\"id\":\"40\",\"title\":\"_**a**_ _**b**_ Mira, Alex; Ochman, Howard; Moran, Nancy A. (2001-10-01). \\\"Deletional bias and the evolution of bacterial genomes\\\". _Trends in Genetics_. **17** (10): 589–596\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0168-9525(01)02447-7](https://doi.org/10.1016%2FS0168-9525%2801%2902447-7). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0168-9525](https://search.worldcat.org/issn/0168-9525). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11585665](https://pubmed.ncbi.nlm.nih.gov/11585665).\",\"description\":\"40. ^ _**a**_ _**b**_ Mira, Alex; Ochman, Howard; Moran, Nancy A. (2001-10-01). \\\"Deletional bias and the evolution of bacterial genomes\\\". _Trends in Genetics_. **17** (10): 589–596\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0168-9525(01)02447-7](https://doi.org/10.1016%2FS0168-9525%2801%2902447-7). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0168-9525](https://search.worldcat.org/issn/0168-9525). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11585665](https://pubmed.ncbi.nlm.nih.gov/11585665).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"41\",\"title\":\"_**a**_ _**b**_ _**c**_ [Andersson, Siv G. E.](Siv_G._E._Andersson \\\"Siv G. E. Andersson\\\"); Zomorodipour, Alireza; Andersson, Jan O.; Sicheritz-Pontén, Thomas; Alsmark, U. Cecilia M.; Podowski, Raf M.; Näslund, A. Kristina; Eriksson, Ann-Sofie; Winkler, Herbert H.; [Kurland, Charles G.](Charles_Kurland \\\"Charles Kurland\\\") (November 1998). [\\\"The genome sequence of Rickettsia prowazekii and the origin of mitochondria\\\"](https://doi.org/10.1038%2F24094). _Nature_. **396** (6707): 133–140\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1998Natur.396..133A](https://ui.adsabs.harvard.edu/abs/1998Natur.396..133A). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/24094](https://doi.org/10.1038%2F24094). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9823893](https://pubmed.ncbi.nlm.nih.gov/9823893).\",\"description\":\"41. ^ _**a**_ _**b**_ _**c**_ [Andersson, Siv G. E.](Siv_G._E._Andersson \\\"Siv G. E. Andersson\\\"); Zomorodipour, Alireza; Andersson, Jan O.; Sicheritz-Pontén, Thomas; Alsmark, U. Cecilia M.; Podowski, Raf M.; Näslund, A. Kristina; Eriksson, Ann-Sofie; Winkler, Herbert H.; [Kurland, Charles G.](Charles_Kurland \\\"Charles Kurland\\\") (November 1998). [\\\"The genome sequence of Rickettsia prowazekii and the origin of mitochondria\\\"](https://doi.org/10.1038%2F24094). _Nature_. **396** (6707): 133–140\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1998Natur.396..133A](https://ui.adsabs.harvard.edu/abs/1998Natur.396..133A). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/24094](https://doi.org/10.1038%2F24094). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [9823893](https://pubmed.ncbi.nlm.nih.gov/9823893).\",\"url\":\"Siv_G._E._Andersson \\\"Siv G. E. Andersson\\\"\",\"favicon\":\"\"},{\"id\":\"42\",\"title\":\"** Tamas, Ivica; Klasson, Lisa M.; Sandström, Jonas P.; Andersson, Siv G. E. (2001). \\\"Mutualists and parasites: how to paint yourself into a (metabolic) corner\\\". _FEBS Letters_. **498** (2–3): 135–139\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2001FEBSL.498..135T](https://ui.adsabs.harvard.edu/abs/2001FEBSL.498..135T). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0014-5793(01)02459-0](https://doi.org/10.1016%2FS0014-5793%2801%2902459-0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1873-3468](https://search.worldcat.org/issn/1873-3468). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11412844](https://pubmed.ncbi.nlm.nih.gov/11412844). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [40955247](https://api.semanticscholar.org/CorpusID:40955247).\",\"description\":\"42. **^** Tamas, Ivica; Klasson, Lisa M.; Sandström, Jonas P.; Andersson, Siv G. E. (2001). \\\"Mutualists and parasites: how to paint yourself into a (metabolic) corner\\\". _FEBS Letters_. **498** (2–3): 135–139\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2001FEBSL.498..135T](https://ui.adsabs.harvard.edu/abs/2001FEBSL.498..135T). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0014-5793(01)02459-0](https://doi.org/10.1016%2FS0014-5793%2801%2902459-0). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1873-3468](https://search.worldcat.org/issn/1873-3468). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11412844](https://pubmed.ncbi.nlm.nih.gov/11412844). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [40955247](https://api.semanticscholar.org/CorpusID:40955247).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"43\",\"title\":\"** Wernegreen, J. J.; Moran, N. A. (2000-07-22). [\\\"Decay of mutualistic potential in aphid endosymbionts through silencing of biosynthetic loci: Buchnera of Diuraphis\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690). _Proceedings of the Royal Society of London. Series B: Biological Sciences_. **267** (1451): 1423–1431\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1098/rspb.2000.1159](https://doi.org/10.1098%2Frspb.2000.1159). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1690690](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10983826](https://pubmed.ncbi.nlm.nih.gov/10983826).\",\"description\":\"43. **^** Wernegreen, J. J.; Moran, N. A. (2000-07-22). [\\\"Decay of mutualistic potential in aphid endosymbionts through silencing of biosynthetic loci: Buchnera of Diuraphis\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690). _Proceedings of the Royal Society of London. Series B: Biological Sciences_. **267** (1451): 1423–1431\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1098/rspb.2000.1159](https://doi.org/10.1098%2Frspb.2000.1159). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1690690](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [10983826](https://pubmed.ncbi.nlm.nih.gov/10983826).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690690\",\"favicon\":\"\"},{\"id\":\"44\",\"title\":\"** Petrov, Dmitri A. (2002-06-01). \\\"Mutational Equilibrium Model of Genome Size Evolution\\\". _Theoretical Population Biology_. **61** (4): 531–544\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2002TPBio..61..531P](https://ui.adsabs.harvard.edu/abs/2002TPBio..61..531P). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1006/tpbi.2002.1605](https://doi.org/10.1006%2Ftpbi.2002.1605). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0040-5809](https://search.worldcat.org/issn/0040-5809). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12167373](https://pubmed.ncbi.nlm.nih.gov/12167373).\",\"description\":\"44. **^** Petrov, Dmitri A. (2002-06-01). \\\"Mutational Equilibrium Model of Genome Size Evolution\\\". _Theoretical Population Biology_. **61** (4): 531–544\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2002TPBio..61..531P](https://ui.adsabs.harvard.edu/abs/2002TPBio..61..531P). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1006/tpbi.2002.1605](https://doi.org/10.1006%2Ftpbi.2002.1605). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0040-5809](https://search.worldcat.org/issn/0040-5809). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12167373](https://pubmed.ncbi.nlm.nih.gov/12167373).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"45\",\"title\":\"** Gregory, T. Ryan (2003-09-01). \\\"Is small indel bias a determinant of genome size?\\\". _Trends in Genetics_. **19** (9): 485–488\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0168-9525(03)00192-6](https://doi.org/10.1016%2FS0168-9525%2803%2900192-6). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0168-9525](https://search.worldcat.org/issn/0168-9525). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12957541](https://pubmed.ncbi.nlm.nih.gov/12957541).\",\"description\":\"45. **^** Gregory, T. Ryan (2003-09-01). \\\"Is small indel bias a determinant of genome size?\\\". _Trends in Genetics_. **19** (9): 485–488\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S0168-9525(03)00192-6](https://doi.org/10.1016%2FS0168-9525%2803%2900192-6). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0168-9525](https://search.worldcat.org/issn/0168-9525). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12957541](https://pubmed.ncbi.nlm.nih.gov/12957541).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"46\",\"title\":\"** Gasior, Stephen L.; Olivares, Heidi; Ear, Uy; Hari, Danielle M.; Weichselbaum, Ralph; Bishop, Douglas K. (2001-07-17). [\\\"Assembly of RecA-like recombinases: Distinct roles for mediator proteins in mitosis and meiosis\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC37451). _Proceedings of the National Academy of Sciences_. **98** (15): 8411–8418\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2001PNAS...98.8411G](https://ui.adsabs.harvard.edu/abs/2001PNAS...98.8411G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.121046198](https://doi.org/10.1073%2Fpnas.121046198). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [37451](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC37451). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11459983](https://pubmed.ncbi.nlm.nih.gov/11459983).\",\"description\":\"46. **^** Gasior, Stephen L.; Olivares, Heidi; Ear, Uy; Hari, Danielle M.; Weichselbaum, Ralph; Bishop, Douglas K. (2001-07-17). [\\\"Assembly of RecA-like recombinases: Distinct roles for mediator proteins in mitosis and meiosis\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC37451). _Proceedings of the National Academy of Sciences_. **98** (15): 8411–8418\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2001PNAS...98.8411G](https://ui.adsabs.harvard.edu/abs/2001PNAS...98.8411G). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1073/pnas.121046198](https://doi.org/10.1073%2Fpnas.121046198). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0027-8424](https://search.worldcat.org/issn/0027-8424). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [37451](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC37451). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11459983](https://pubmed.ncbi.nlm.nih.gov/11459983).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC37451\",\"favicon\":\"\"},{\"id\":\"47\",\"title\":\"** Selosse, M.-A.; Albert, B.; Godelle, B. (2001-03-01). \\\"Reducing the genome size of organelles favours gene transfer to the nucleus\\\". _Trends in Ecology \u0026 Evolution_. **16** (3): 135–141\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/s0169-5347(00)02084-x](https://doi.org/10.1016%2Fs0169-5347%2800%2902084-x). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1872-8383](https://search.worldcat.org/issn/1872-8383). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11179577](https://pubmed.ncbi.nlm.nih.gov/11179577).\",\"description\":\"47. **^** Selosse, M.-A.; Albert, B.; Godelle, B. (2001-03-01). \\\"Reducing the genome size of organelles favours gene transfer to the nucleus\\\". _Trends in Ecology \u0026 Evolution_. **16** (3): 135–141\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/s0169-5347(00)02084-x](https://doi.org/10.1016%2Fs0169-5347%2800%2902084-x). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1872-8383](https://search.worldcat.org/issn/1872-8383). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11179577](https://pubmed.ncbi.nlm.nih.gov/11179577).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"48\",\"title\":\"** Scherbakov, D. V.; Garber, M. B. (2000-07-01). \\\"Overlapping genes in bacterial and phage genomes\\\". _Molecular Biology_. **34** (4): 485–495\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/BF02759558](https://doi.org/10.1007%2FBF02759558). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1608-3245](https://search.worldcat.org/issn/1608-3245). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [24144602](https://api.semanticscholar.org/CorpusID:24144602).\",\"description\":\"48. **^** Scherbakov, D. V.; Garber, M. B. (2000-07-01). \\\"Overlapping genes in bacterial and phage genomes\\\". _Molecular Biology_. **34** (4): 485–495\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/BF02759558](https://doi.org/10.1007%2FBF02759558). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1608-3245](https://search.worldcat.org/issn/1608-3245). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [24144602](https://api.semanticscholar.org/CorpusID:24144602).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"49\",\"title\":\"** Williams, Bryony A. P.; Hirt, Robert P.; Lucocq, John M.; Embley, T. Martin (August 2002). \\\"A mitochondrial remnant in the microsporidian Trachipleistophora hominis\\\". _Nature_. **418** (6900): 865–869\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2002Natur.418..865W](https://ui.adsabs.harvard.edu/abs/2002Natur.418..865W). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nature00949](https://doi.org/10.1038%2Fnature00949). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12192407](https://pubmed.ncbi.nlm.nih.gov/12192407). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4358253](https://api.semanticscholar.org/CorpusID:4358253).\",\"description\":\"49. **^** Williams, Bryony A. P.; Hirt, Robert P.; Lucocq, John M.; Embley, T. Martin (August 2002). \\\"A mitochondrial remnant in the microsporidian Trachipleistophora hominis\\\". _Nature_. **418** (6900): 865–869\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2002Natur.418..865W](https://ui.adsabs.harvard.edu/abs/2002Natur.418..865W). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nature00949](https://doi.org/10.1038%2Fnature00949). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12192407](https://pubmed.ncbi.nlm.nih.gov/12192407). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [4358253](https://api.semanticscholar.org/CorpusID:4358253).\",\"url\":\"Bibcode_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"50\",\"title\":\"** Keeling, Patrick J.; Fast, Naomi M. (2002). \\\"Microsporidia: Biology and Evolution of Highly Reduced Intracellular Parasites\\\". _Annual Review of Microbiology_. **56** (1): 93–116\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1146/annurev.micro.56.012302.160854](https://doi.org/10.1146%2Fannurev.micro.56.012302.160854). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12142484](https://pubmed.ncbi.nlm.nih.gov/12142484). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [22943269](https://api.semanticscholar.org/CorpusID:22943269).\",\"description\":\"50. **^** Keeling, Patrick J.; Fast, Naomi M. (2002). \\\"Microsporidia: Biology and Evolution of Highly Reduced Intracellular Parasites\\\". _Annual Review of Microbiology_. **56** (1): 93–116\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1146/annurev.micro.56.012302.160854](https://doi.org/10.1146%2Fannurev.micro.56.012302.160854). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12142484](https://pubmed.ncbi.nlm.nih.gov/12142484). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [22943269](https://api.semanticscholar.org/CorpusID:22943269).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"51\",\"title\":\"** Cavalier-Smith, T. (2001). \\\"What are Fungi?\\\". In McLaughlin, David J.; McLaughlin, Esther G.; Lemke, Paul A. (eds.). _Systematics and Evolution_. The Mycota. Springer Berlin Heidelberg. pp. 3–37\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/978-3-662-10376-0_1](https://doi.org/10.1007%2F978-3-662-10376-0_1). [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-3-662-10376-0. `{{cite book}}`: `|work=` ignored (help)\",\"description\":\"51. **^** Cavalier-Smith, T. (2001). \\\"What are Fungi?\\\". In McLaughlin, David J.; McLaughlin, Esther G.; Lemke, Paul A. (eds.). _Systematics and Evolution_. The Mycota. Springer Berlin Heidelberg. pp. 3–37\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1007/978-3-662-10376-0_1](https://doi.org/10.1007%2F978-3-662-10376-0_1). [ISBN](ISBN_\\\\(identifier\\\\) \\\"ISBN \\\\(identifier\\\\)\\\") 978-3-662-10376-0. `{{cite book}}`: `|work=` ignored (help)\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"52\",\"title\":\"** Vivarès, Christian P; Gouy, Manolo; Thomarat, Fabienne; Méténier, Guy (2002-10-01). \\\"Functional and evolutionary analysis of a eukaryotic parasitic genome\\\". _Current Opinion in Microbiology_. **5** (5): 499–505\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S1369-5274(02)00356-9](https://doi.org/10.1016%2FS1369-5274%2802%2900356-9). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1369-5274](https://search.worldcat.org/issn/1369-5274). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12354558](https://pubmed.ncbi.nlm.nih.gov/12354558).\",\"description\":\"52. **^** Vivarès, Christian P; Gouy, Manolo; Thomarat, Fabienne; Méténier, Guy (2002-10-01). \\\"Functional and evolutionary analysis of a eukaryotic parasitic genome\\\". _Current Opinion in Microbiology_. **5** (5): 499–505\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S1369-5274(02)00356-9](https://doi.org/10.1016%2FS1369-5274%2802%2900356-9). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1369-5274](https://search.worldcat.org/issn/1369-5274). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [12354558](https://pubmed.ncbi.nlm.nih.gov/12354558).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"53\",\"title\":\"** Douglas, Susan; Zauner, Stefan; Fraunholz, Martin; Beaton, Margaret; Penny, Susanne; Deng, Lang-Tuo; Wu, Xiaonan; Reith, Michael; Cavalier-Smith, Thomas; Maier, Uwe-G. (April 2001). [\\\"The highly reduced genome of an enslaved algal nucleus\\\"](https://doi.org/10.1038%2F35074092). _Nature_. **410** (6832): 1091–1096\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2001Natur.410.1091D](https://ui.adsabs.harvard.edu/abs/2001Natur.410.1091D). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/35074092](https://doi.org/10.1038%2F35074092). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11323671](https://pubmed.ncbi.nlm.nih.gov/11323671).\",\"description\":\"53. **^** Douglas, Susan; Zauner, Stefan; Fraunholz, Martin; Beaton, Margaret; Penny, Susanne; Deng, Lang-Tuo; Wu, Xiaonan; Reith, Michael; Cavalier-Smith, Thomas; Maier, Uwe-G. (April 2001). [\\\"The highly reduced genome of an enslaved algal nucleus\\\"](https://doi.org/10.1038%2F35074092). _Nature_. **410** (6832): 1091–1096\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2001Natur.410.1091D](https://ui.adsabs.harvard.edu/abs/2001Natur.410.1091D). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/35074092](https://doi.org/10.1038%2F35074092). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1476-4687](https://search.worldcat.org/issn/1476-4687). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11323671](https://pubmed.ncbi.nlm.nih.gov/11323671).\",\"url\":\"https://doi.org/10.1038%2F35074092\",\"favicon\":\"\"}],\"images\":[],\"fixedIssues\":[{\"criticality\":\"Critical\",\"issueType\":\"Quantitative/Units Error\",\"problematicText\":\"In animals they range more than 3,300-fold, and in land plants they differ by a factor of about 1,000.[3][4] Protist genomes have been reported to vary more than 300,000-fold in size, but the high end of this range (Amoeba) has been called into question.\",\"explanation\":\"These variation ranges are based on 2005 data and are now outdated. Current estimates indicate greater variation: over 6,600-fold in animals, over 2,300-fold in land plants, and over 28,600-fold in protists (with some large values still debated). This changes the scale of reported diversity and could mislead on evolutionary patterns.\",\"fixedText\":\"Update the ranges to: over 6,600-fold in animals, over 2,300-fold in land plants, and over 28,600-fold in protists, noting that extreme protist values remain debated. Retain original citations for historical context but add new ones for current estimates.\",\"evidenceSource\":\"[](https://www.sciencedirect.com/science/article/pii/S0960982219302027) for animals; [](https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.16315) for plants; [](https://pmc.ncbi.nlm.nih.gov/articles/PMC7596369/) for protists.\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"Based on currently available completely sequenced genome data (as of April 2009), log-transformed gene number forms a linear correlation with log-transformed genome size in bacteria, archaea, viruses, and organelles combined, whereas a nonlinear (semi-natural logarithm) correlation is seen for eukaryotes.[5] Although the latter contrasts with the previous view that no correlation exists for the eukaryotes, the observed nonlinear correlation for eukaryotes may reflect disproportionately fast-increasing junk DNA in increasingly large eukaryotic genomes. Although sequenced genome data are practically biased toward small genomes, which may compromise the accuracy of the empirically derived correlation, and ultimate proof of the correlation remains to be obtained by sequencing some of the largest eukaryotic genomes, current data do not seem to rule out a possible correlation.\",\"explanation\":\"The analysis relies on data from April 2009, when fewer genomes were sequenced. By 2025, with thousands more eukaryotic genomes sequenced, the consensus is that there is no strong or consistent correlation between gene number and genome size in eukaryotes, primarily due to varying amounts of non-coding (junk) DNA. Presenting the 2009 finding without update misrepresents the current scientific understanding.\",\"fixedText\":\"Revise to state that while a weak correlation was suggested in early analyses (2009), subsequent sequencing of more genomes has confirmed no clear correlation exists between gene number and genome size in eukaryotes, attributing variation mainly to non-coding DNA. Retain ref [5] for the historical claim and add new citations for current view. Remove speculative language about \\\"current data do not rule out.\\\"\",\"evidenceSource\":\"[](https://elifesciences.org/articles/100574) (2024 review stating no relation to number of coding genes in eukaryotes); [](https://www.pnas.org/doi/10.1073/pnas.2404332122) (2025 paper noting lack of correlation with organismal complexity or gene number).\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"No specific mention of recent advancements in genome size estimates or databases.\",\"explanation\":\"The section references \\\"online databases\\\" but lacks specifics or updates. As of 2025, key resources like the Animal Genome Size Database, Plant DNA C-values Database, and protist-focused genomic portals provide more comprehensive data, including recent extremes like the 160 Gbp fern genome. This gap leaves readers without guidance on verifiable current sources, especially given the outdated ranges.\",\"fixedText\":\"Add examples of current databases after mentioning them, e.g., \\\"such as the Animal Genome Size Database, Plant DNA C-values Database, and the Protist 10,000 Genomes Project portal.\\\" Note recent record: e.g., the largest known plant genome is now ~160 Gbp in Tmesipteris oblanceolata (2024).\",\"evidenceSource\":\"[](https://www.genomesize.com/) for animals; [](http://www.kew.org/cval/) for plants (archived but referenced); [](https://academic.oup.com/nar/article/52/D1/D747/7335744) for protist 10K project.\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"One obligate endosymbiont of leafhoppers, _Nasuia deltocephalinicola_, has the smallest genome currently known among cellular organisms at 112 kb.[19]\",\"explanation\":\"The claim is outdated; as of 2025, recent research identifies even smaller bacterial genomes (around 50-52 kb) in planthopper symbionts, making the \\\"smallest currently known\\\" inaccurate without update.\",\"fixedText\":\"Update to note Nasuia as one of the smallest known at the time (2013), and mention recent discoveries of smaller genomes (e.g., ~50 kb in planthopper symbionts as of 2025) with new citation.\",\"evidenceSource\":\"[](https://www.biorxiv.org/content/10.1101/2025.05.16.654412v1)\"},{\"criticality\":\"Critical\",\"issueType\":\"Scope/Applicability Misstatement\",\"problematicText\":\"These species have become a considerable threat to human health, as they are often capable of evading human immune systems and manipulating the host environment to acquire nutrients.\",\"explanation\":\"Obligate endosymbionts include many mutualistic species (e.g., in insects providing nutrients) that are not threats to human health or pathogens; the claim overgeneralizes to human context, misrepresenting the diverse roles of endosymbionts.\",\"fixedText\":\"Rephrase to distinguish between pathogenic and mutualistic endosymbionts, avoiding implication that they are primarily human threats.\",\"evidenceSource\":\"Existing ref [16] (Moran 1996 on endosymbiotic bacteria, including mutualists); web search confirms many are mutualistic.\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"Despite the pathogenicity of most endosymbionts, some obligate intracellular species have positive fitness effects on their hosts.\",\"explanation\":\"No citation supports \\\"pathogenicity of most endosymbionts\\\"; literature shows a significant portion are mutualistic (e.g., Buchnera), not pathogenic, making the claim unsupported and misleading.\",\"fixedText\":\"Remove \\\"pathogenicity of most\\\" and rephrase to acknowledge both pathogenic and mutualistic roles, e.g., \\\"While some are pathogenic, many obligate endosymbionts provide positive fitness effects to their hosts.\\\"\",\"evidenceSource\":\"Web search on proportions; examples in section include mutualists like Buchnera and Nasuia.\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"[No specific text; overall section]\",\"explanation\":\"The section lacks mention of recent advances (post-2020) in understanding genome reduction mechanisms or examples, such as further insights into deletional bias or new minimal genome studies, which are relevant given the article's focus on evolution.\",\"fixedText\":\"Add brief note on ongoing research, e.g., recent studies confirming deletional bias as a key driver, with citation if needed.\",\"evidenceSource\":\"Refs [40] (Mira 2001 on deletional bias), but update with recent reviews.\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"In 1972 Michael David Bennett[28] hypothesized that there was a correlation with the DNA content and the nuclear volume\",\"explanation\":\"Reference 28 (Bennett 1972) discusses correlations between nuclear DNA content and minimum generation time, cell cycle time, and meiotic duration in herbaceous plants, but does not mention or hypothesize a correlation with nuclear volume.\",\"fixedText\":\"Revise to attribute Bennett's hypothesis correctly to correlation between DNA content and minimum generation time or cell cycle duration, retaining citation [28].\",\"evidenceSource\":\"Ref 28\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"Commoner and van’t Hoff and Sparrow before him postulated that even cell size and cell-cycle length were controlled by the amount of DNA.[29][30]\",\"explanation\":\"Reference 30 (Commoner 1964) focuses on the roles of DNA in inheritance and does not postulate that cell size or cell-cycle length are controlled by DNA amount. Reference 29 (van't Hof and Sparrow 1963) does support relationships between DNA content, nuclear volume, cell size, and mitotic cycle time, but the name \\\"van’t Hoff\\\" is a misspelling and misattribution to the unrelated chemist Jacobus Henricus van 't Hoff (1852–1911); the correct author is J. Van't Hof.\",\"fixedText\":\"Remove attribution to Commoner and citation [30] for this claim, as it is unsupported. Correct name to \\\"Van't Hof and Sparrow\\\" and retain [29] for their supported postulates on DNA content influencing nuclear volume, cell size, and cycle time.\",\"evidenceSource\":\"Refs 29 and 30\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"More recent theories have brought us to discuss about the possibility of the presence of a mechanism that constrains physically the development of the genome to an optimal size.[31]\",\"explanation\":\"Reference 31 (Orgel and Crick 1980) discusses \\\"selfish DNA\\\" as parasitic elements spreading with minimal phenotypic effect but does not address physical mechanisms constraining genome size to an optimal level.\",\"fixedText\":\"Remove citation [31] as unsupported; integrate the idea of optimal size constraints into the discussion of Cavalier-Smith's theory [32], which addresses evolutionary forces driving miniaturization and expansion to optimal sizes.\",\"evidenceSource\":\"Ref 31\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"\",\"explanation\":\"The section discusses historical theories but lacks mention of the karyoplasmic (or nucleocytoplasmic) ratio's definition or significance beyond citations, which could clarify its role in genome size regulation. This gap affects understanding of Cavalier-Smith's skeletal theory.\",\"fixedText\":\"Add a brief definition: the karyoplasmic ratio is the ratio of nuclear volume to cytoplasmic volume, maintained for optimal cell function.\",\"evidenceSource\":\"Ref 32\"},{\"criticality\":\"Critical\",\"issueType\":\"Mechanism/Specification Error\",\"problematicText\":\"\\\"In Rickettsia prowazekii , as with other small genome bacteria, this mutualistic endosymbiont\\\"\",\"explanation\":\"Rickettsia prowazekii is an obligate intracellular parasite, not a mutualistic endosymbiont. This misclassifies the organism's relationship with its host, altering the understanding of genome reduction mechanisms in mutualistic vs. parasitic contexts.\",\"fixedText\":\"Change to \\\"parasitic endosymbiont\\\" or rephrase to accurately describe R. prowazekii as a parasite.\",\"evidenceSource\":\"[41] Andersson et al. (1998), which describes R. prowazekii as an obligate intracellular parasite.\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"still retain the bio-synthetic ability of production of amino acid needed by its host.[41][42][37]\\\"\",\"explanation\":\"The citations do not support the claim; R. prowazekii has lost most amino acid biosynthesis pathways and relies on the host, contrary to the statement. [41] details the reduced genome lacking complete AA pathways. This inverts the factual mechanism of genome reduction.\",\"fixedText\":\"Remove or correct the claim: R. prowazekii lacks most amino acid biosynthesis capabilities, consistent with parasitic lifestyle.\",\"evidenceSource\":\"[41] Andersson et al. (1998); web search confirms absence of AA biosynthesis in R. prowazekii.\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"An example for this is the deletion of rec F, gene required for the function of rec A, and its flanking genes.[46]\\\"\",\"explanation\":\"[46] discusses eukaryotic RecA homologs in meiosis/mitosis, unrelated to bacterial genome reduction or recF deletion in endosymbionts. The claim about recF deletion in Buchnera is accurate but cited incorrectly, potentially misleading on the mechanism.\",\"fixedText\":\"Replace citation [46] with [35].\",\"evidenceSource\":\"\"},{\"criticality\":\"Critical\",\"issueType\":\"Mechanism/Specification Error\",\"problematicText\":\"\\\"and the capacity to enter into the host cells.\\\"\",\"explanation\":\"Mitosomes are relic mitochondria within the microsporidian cell and do not enter host cells; this misattributes the parasite's host invasion capability to the organelle, confusing biological mechanisms.\",\"fixedText\":\"Remove the erroneous phrase.\",\"evidenceSource\":\"Web search results confirming mitosome functions (Fe-S cluster assembly, no host entry)\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"eliminating almost 1000 genes\\\"\",\"explanation\":\"The cited source [52] does not mention loss of exactly or almost 1000 genes; microsporidian genomes (~2000 genes) result from loss of thousands of genes compared to fungal ancestors (~6000+ genes).\",\"fixedText\":\"Rephrase to \\\"eliminating thousands of genes\\\" with supporting evidence.\",\"evidenceSource\":\"[52] and web search on gene loss in microsporidia\"},{\"criticality\":\"Critical\",\"issueType\":\"Scope/Applicability Misstatement\",\"problematicText\":\"\\\"chlorarachneans\\\"\",\"explanation\":\"Incorrect terminology; the algal group containing nucleomorphs is chlorarachniophytes, not chlorarachneans.\",\"fixedText\":\"Correct to \\\"chlorarachniophytes\\\".\",\"evidenceSource\":\"Multiple peer-reviewed sources on nucleomorph-bearing algae\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"the smallest genome found in the eukaryotes.\\\"\",\"explanation\":\"The claim is imprecise; while microsporidia have the smallest known eukaryotic nuclear genomes, nucleomorph genomes (discussed in the same section) are smaller but are remnant organelles, not full nuclear genomes. Citation [32] discusses miniaturization but requires clarification.\",\"fixedText\":\"Specify \\\"smallest known eukaryotic nuclear genome\\\".\",\"evidenceSource\":\"[32] and web search confirming smallest nuclear genome in microsporidia\"},{\"criticality\":\"Missing Information\",\"issueType\":\"Missing Information\",\"problematicText\":\"No specific text\",\"explanation\":\"Lacks concrete examples of genome sizes for microsporidia, such as the 2.3 Mb genome of Encephalitozoon intestinalis, which remains the smallest known eukaryotic nuclear genome as of 2023.\",\"fixedText\":\"Add specific genome size example with new citation for precision.\",\"evidenceSource\":\"Web search on smallest eukaryotic genomes\"}],\"slug\":\"Genome_size\",\"title\":\"Genome size\",\"content\":\"$1f\",\"description\":\"Genome size\\n\\nGenome size is the total amount of DNA contained within one copy of a single complete genome.\",\"metadata\":{\"categories\":[\"C-value\"],\"lastModified\":\"1761552900\",\"contentLength\":\"19204\",\"version\":\"1.0\",\"lastEditor\":\"system\",\"language\":\"en\",\"isRedirect\":false,\"redirectTarget\":\"\",\"isWithheld\":false},\"stats\":{\"totalViews\":\"29810\",\"recentViews\":\"29810\",\"dailyAvgViews\":993.6666870117188,\"qualityScore\":1,\"lastViewed\":\"1761887017\"},\"linkedPages\":null},\"found\":true},\"dataUpdateCount\":1,\"dataUpdatedAt\":1761887017520,\"error\":null,\"errorUpdateCount\":0,\"errorUpdatedAt\":0,\"fetchFailureCount\":0,\"fetchFailureReason\":null,\"fetchMeta\":null,\"isInvalidated\":false,\"status\":\"success\",\"fetchStatus\":\"idle\"},\"queryKey\":[\"page\",\"Genome_size\"],\"queryHash\":\"[\\\"page\\\",\\\"Genome_size\\\"]\"}]},\"children\":\"$L20\"}]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"c:{\"metadata\":[[\"$\",\"title\",\"0\",{\"children\":\"Genome size\"}],[\"$\",\"meta\",\"1\",{\"name\":\"description\",\"content\":\"Genome size is the total amount of DNA contained within one copy of a single complete genome. It is typically measured in terms of mass in picograms (trillionths or 10−12 of a gram, abbreviated pg) or less frequently in daltons, or as the total number of nucleotide base pairs, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In diploid organisms, genome size is often used interchangeably with the term C-value. An organism's...\"}],[\"$\",\"meta\",\"2\",{\"name\":\"author\",\"content\":\"system\"}],[\"$\",\"link\",\"3\",{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\",\"crossOrigin\":\"$undefined\"}],[\"$\",\"meta\",\"4\",{\"name\":\"keywords\",\"content\":\"C-value\"}],[\"$\",\"meta\",\"5\",{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",\"6\",{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"link\",\"7\",{\"rel\":\"canonical\",\"href\":\"https://grokipedia.com/page/Genome_size\"}],[\"$\",\"meta\",\"8\",{\"property\":\"og:title\",\"content\":\"Genome size\"}],[\"$\",\"meta\",\"9\",{\"property\":\"og:description\",\"content\":\"Genome size is the total amount of DNA contained within one copy of a single complete genome. It is typically measured in terms of mass in picograms (trillionths or 10−12 of a gram, abbreviated pg) or less frequently in daltons, or as the total number of nucleotide base pairs, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In diploid organisms, genome size is often used interchangeably with the term C-value. An organism's...\"}],[\"$\",\"meta\",\"10\",{\"property\":\"og:url\",\"content\":\"https://grokipedia.com/page/Genome_size\"}],[\"$\",\"meta\",\"11\",{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",\"12\",{\"property\":\"og:locale\",\"content\":\"en\"}],[\"$\",\"meta\",\"13\",{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",\"14\",{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",\"15\",{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",\"16\",{\"property\":\"og:image:alt\",\"content\":\"Genome size\"}],[\"$\",\"meta\",\"17\",{\"property\":\"og:type\",\"content\":\"article\"}],[\"$\",\"meta\",\"18\",{\"property\":\"article:modified_time\",\"content\":\"1970-01-21T09:19:12.900Z\"}],[\"$\",\"meta\",\"19\",{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",\"20\",{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",\"21\",{\"name\":\"twitter:title\",\"content\":\"Genome size\"}],[\"$\",\"meta\",\"22\",{\"name\":\"twitter:description\",\"content\":\"Genome size is the total amount of DNA contained within one copy of a single complete genome. It is typically measured in terms of mass in picograms (trillionths or 10−12 of a gram, abbreviated pg) or less frequently in daltons, or as the total number of nucleotide base pairs, usually in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram is equal to 978 megabases.[1] In diploid organisms, genome size is often used interchangeably with the term C-value. An organism's...\"}],[\"$\",\"meta\",\"23\",{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"link\",\"24\",{\"rel\":\"icon\",\"href\":\"/favicon.ico\",\"type\":\"image/x-icon\",\"sizes\":\"16x16\"}],[\"$\",\"$L21\",\"25\",{}]],\"error\":null,\"digest\":\"$undefined\"}\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"11:\"$c:metadata\"\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"22:I[61172,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"PageEditorProvider\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"23:I[63493,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="YjM4M2JhODgtNjlkMi00NmQ5LWFkYjQtN2Q3ZjcyZWM3ZjQz">self.__next_f.push([1,"20:[\"$\",\"$L22\",null,{\"children\":[\"$\",\"$L23\",null,{\"slug\":\"Genome_size\"}]}]\n"])</script></body></html>