<!DOCTYPE html><html lang="en" class="bg-surface-base antialiased"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1, interactive-widget=resizes-content"/><link rel="stylesheet" href="/_next/static/css/62c4caba71dfda84.css" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/eb3d87f98fe1565f.css" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/1b5e561215938d4d.css" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/0227d069a630d414.css" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/f87fff2ab93d05a7.css" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" data-precedence="next"/><link rel="preload" as="script" fetchPriority="low" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" href="/_next/static/chunks/webpack-e121ed42680f327e.js"/><script src="/_next/static/chunks/78a669d9-e7993486b22c4915.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/40c4a5a7-abdabb07419d1cfc.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/2230-f7c87dc9fa57c408.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/main-app-536d4b8fca62396a.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/63f0ec43-8ee6a76d70472249.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/7670-7326298c9856172b.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/282-691fffb366e24ca5.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/7515-336759ef5dad2b52.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/927-34e2a6da3e15e26b.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/5855-243eeca8f0e4cabd.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/4393-1c8dff25ec904469.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/2225-454cf9f44775e7ce.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/7618-43536a8fb0dd3bfe.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/7720-3a1e7e411adba2d3.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/7086-1f1c4891d766e04f.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/global-error-4d07d20223cd4b4c.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/8208b75a-a48e5a4507a0de91.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/5497-ab85ef3ea59dd353.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/6281-c6e84786dade3614.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/2660-d983a7f287e89f18.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/5002-9735c25beda556ba.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/3655-6717081f7512305a.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/8945-e4e3cf487f5e27c7.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/9214-a10614844a67ba31.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/4789-2080f6497f78b5b6.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/layout-226f2bd71acf9f9e.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/24cf1b50-159cca90b09285e0.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/9ffa21ba-c069766d809bd4c7.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/600-9a9fcafaaa6c1c4c.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/5448-2a3cfd853d899c9a.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/8-8ca70ca619d8e099.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/error-4a29e9399afba038.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><script src="/_next/static/chunks/app/not-found-dd95690acf732f18.js" async="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script><meta name="next-size-adjust" content=""/><title>Grokipedia</title><meta name="description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><link rel="icon" type="image/x-icon" href="/favicon.ico" sizes="48x48"/><link rel="icon" href="/images/icon-dark.png" media="(prefers-color-scheme: light)" type="image/png"/><link rel="icon" href="/images/icon-light.png" media="(prefers-color-scheme: dark)" type="image/png"/><link rel="apple-touch-icon" href="/icon-192x192.png"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><meta property="og:title" content="Grokipedia"/><meta property="og:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta property="og:url" content="https://grokipedia.com"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en_US"/><meta property="og:type" content="website"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Grokipedia"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Grokipedia"/><meta name="twitter:description" content="Grokipedia is an open source, comprehensive collection of all knowledge."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><title>Quantitative trait locus</title><meta name="description" content="A quantitative trait locus (QTL) is a locus (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of organisms.[1] QTLs are mapped by identifying which molecular markers (such as SNPs or AFLPs) correlate with an observed trait. This is often an early step in identifying the actual genes that cause the trait variation. A quantitative trait locus (QTL) is a region of DNA which is associated with a particular phenotypic trait, which varies in..."/><meta name="author" content="system"/><link rel="manifest" href="/manifest.webmanifest"/><meta name="keywords" content="QTL"/><meta name="robots" content="index, follow"/><meta name="googlebot" content="index, follow, max-snippet:-1"/><link rel="canonical" href="https://grokipedia.com/page/Quantitative_trait_locus"/><meta property="og:title" content="Quantitative trait locus"/><meta property="og:description" content="A quantitative trait locus (QTL) is a locus (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of organisms.[1] QTLs are mapped by identifying which molecular markers (such as SNPs or AFLPs) correlate with an observed trait. This is often an early step in identifying the actual genes that cause the trait variation. A quantitative trait locus (QTL) is a region of DNA which is associated with a particular phenotypic trait, which varies in..."/><meta property="og:url" content="https://grokipedia.com/page/Quantitative_trait_locus"/><meta property="og:site_name" content="Grokipedia"/><meta property="og:locale" content="en"/><meta property="og:image" content="https://grokipedia.com/icon-512x512.png"/><meta property="og:image:width" content="512"/><meta property="og:image:height" content="512"/><meta property="og:image:alt" content="Quantitative trait locus"/><meta property="og:type" content="article"/><meta property="article:modified_time" content="1970-01-21T09:19:12.626Z"/><meta name="twitter:card" content="summary"/><meta name="twitter:creator" content="@Grokipedia"/><meta name="twitter:title" content="Quantitative trait locus"/><meta name="twitter:description" content="A quantitative trait locus (QTL) is a locus (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of organisms.[1] QTLs are mapped by identifying which molecular markers (such as SNPs or AFLPs) correlate with an observed trait. This is often an early step in identifying the actual genes that cause the trait variation. A quantitative trait locus (QTL) is a region of DNA which is associated with a particular phenotypic trait, which varies in..."/><meta name="twitter:image" content="https://grokipedia.com/icon-512x512.png"/><link rel="icon" href="/favicon.ico" type="image/x-icon" sizes="16x16"/><meta name="sentry-trace" content="1b66264d7d4fcd2d07237ac2de394948-cce77e216990d318-1"/><meta name="baggage" content="sentry-environment=production,sentry-public_key=5f2258f71198ee26a355127af230c3a6,sentry-trace_id=1b66264d7d4fcd2d07237ac2de394948,sentry-org_id=4508179396558848,sentry-transaction=GET%20%2Fpage%2F%5Bslug%5D,sentry-sampled=true,sentry-sample_rand=0.2925812695116581,sentry-sample_rate=1"/><script src="/_next/static/chunks/polyfills-42372ed130431b0a.js" noModule="" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5"></script></head><body class="flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b"><div hidden=""><!--$--><!--/$--></div><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">((a,b,c,d,e,f,g,h)=>{let i=document.documentElement,j=["light","dark"];function k(b){var c;(Array.isArray(a)?a:[a]).forEach(a=>{let c="class"===a,d=c&&f?e.map(a=>f[a]||a):e;c?(i.classList.remove(...d),i.classList.add(f&&f[b]?f[b]:b)):i.setAttribute(a,b)}),c=b,h&&j.includes(c)&&(i.style.colorScheme=c)}if(d)k(d);else try{let a=localStorage.getItem(b)||c,d=g&&"system"===a?window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light":a;k(d)}catch(a){}})("class","theme","dark",null,["light","dark"],null,true,true)</script><header class="h-16 bg-surface-base fixed top-0 z-50 w-full"><div class="hidden h-full grid-cols-[1fr_3fr_1fr] items-center gap-4 !py-0 md:grid max-w-full py-6 px-4 w-full mx-auto"><div class="-ml-2 justify-self-start"><a class="cursor-pointer" href="/"><svg width="200" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="mx-auto flex w-full"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full px-3 shadow-none sm:w-80 focus-visible:z-1 mx-auto h-full w-full rounded-full bg-surface-l1" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>âŒ˜</span><span class="text-xs">K</span></div></span></div></button></div><div class="flex items-center gap-2 justify-self-end"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0" type="button" id="radix-_R_7apfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button></div></div><div class="flex h-full items-center md:hidden justify-between max-w-full py-6 px-4 w-full mx-auto pt-8"><div class="flex items-center"><div class="-ml-2"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button" aria-label="Toggle table of contents"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-list h-6 w-6"><path d="M3 12h.01"></path><path d="M3 18h.01"></path><path d="M3 6h.01"></path><path d="M8 12h13"></path><path d="M8 18h13"></path><path d="M8 6h13"></path></svg></button></div><a class="cursor-pointer" href="/"><svg width="160" height="auto" viewBox="0 0 955 225" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M99.6937 159.214C90.008 159.214 81.5652 157.414 74.3652 153.814C67.1652 150.214 61.5509 144.986 57.5223 138.129C53.5794 131.271 51.608 123.043 51.608 113.443C51.608 103.843 53.5794 95.4857 57.5223 88.3714C61.5509 81.2572 67.1652 75.7286 74.3652 71.7857C81.6509 67.8429 90.1794 65.8714 99.9509 65.8714C103.208 65.8714 106.679 66.1286 110.365 66.6429C114.051 67.0714 117.222 67.6714 119.879 68.4429C121.508 68.8714 123.008 69.2572 124.379 69.6C125.751 69.9429 127.122 70.1143 128.494 70.1143C129.522 70.1143 130.551 69.9857 131.579 69.7286C131.751 73.5 131.922 77.4 132.094 81.4286C132.265 85.4572 132.522 89.6572 132.865 94.0286C131.065 94.2857 129.565 94.3714 128.365 94.2857C126.651 86.4 123.522 80.5714 118.979 76.8C114.437 72.9429 108.394 71.0143 100.851 71.0143C94.8509 71.0143 89.708 72.1286 85.4223 74.3572C81.2223 76.5 77.7937 79.4572 75.1366 83.2286C72.5652 87 70.6794 91.3714 69.4794 96.3429C68.2794 101.229 67.6794 106.457 67.6794 112.029C67.6794 120.171 68.9223 127.371 71.408 133.629C73.8937 139.886 77.6652 144.814 82.7223 148.414C87.8652 151.929 94.2937 153.686 102.008 153.686C104.408 153.686 107.065 153.471 109.979 153.043C112.894 152.529 115.679 151.757 118.337 150.729V134.529C118.337 131.871 118.208 129.814 117.951 128.357C117.779 126.814 117.222 125.7 116.279 125.014C115.422 124.329 114.008 123.857 112.037 123.6C110.151 123.257 107.537 122.957 104.194 122.7C103.937 121.329 103.937 119.957 104.194 118.586C105.994 118.586 108.179 118.629 110.751 118.714C113.322 118.714 115.894 118.757 118.465 118.843C121.122 118.843 123.308 118.843 125.022 118.843C127.679 118.843 130.722 118.8 134.151 118.714C137.665 118.629 140.665 118.586 143.151 118.586C143.322 119.871 143.322 121.2 143.151 122.571C139.637 123 137.108 123.429 135.565 123.857C134.022 124.286 133.079 125.143 132.737 126.429C132.394 127.714 132.222 129.9 132.222 132.986V137.1C132.222 141.129 132.308 144.514 132.479 147.257C132.737 150 132.951 151.971 133.122 153.171C121.208 157.2 110.065 159.214 99.6937 159.214ZM148.19 157.414C147.933 156.043 147.933 154.714 148.19 153.429C151.105 153.257 153.248 153 154.619 152.657C155.99 152.229 156.89 151.371 157.319 150.086C157.748 148.8 157.962 146.743 157.962 143.914V115.114C157.962 111.771 157.79 109.286 157.448 107.657C157.19 106.029 156.376 104.914 155.005 104.314C153.719 103.629 151.49 103.071 148.319 102.643C148.062 101.443 148.062 100.329 148.319 99.3C151.748 98.6143 155.09 97.8429 158.348 96.9857C161.605 96.0429 165.033 94.8429 168.633 93.3857L170.819 94.1572V105.471C176.819 97.8429 182.948 94.0286 189.205 94.0286C192.205 94.0286 194.433 94.8429 195.89 96.4714C197.348 98.0143 198.076 99.7714 198.076 101.743C198.076 104.057 197.305 105.857 195.762 107.143C194.219 108.343 192.462 108.943 190.49 108.943C188.862 108.943 187.233 108.514 185.605 107.657C184.062 106.8 182.905 105.471 182.133 103.671C179.219 103.671 176.605 104.829 174.29 107.143C172.062 109.371 170.948 111.986 170.948 114.986V142.629C170.948 145.971 171.119 148.371 171.462 149.829C171.89 151.286 172.919 152.229 174.548 152.657C176.262 153 179.005 153.257 182.776 153.429C182.948 154.629 182.948 155.957 182.776 157.414C179.862 157.414 176.862 157.371 173.776 157.286C170.69 157.2 167.519 157.157 164.262 157.157C161.09 157.157 158.262 157.2 155.776 157.286C153.376 157.371 150.848 157.414 148.19 157.414ZM229.285 158.829C223.028 158.829 217.542 157.5 212.828 154.843C208.199 152.1 204.599 148.371 202.028 143.657C199.457 138.857 198.171 133.371 198.171 127.2C198.171 120.514 199.499 114.686 202.157 109.714C204.899 104.743 208.714 100.886 213.599 98.1429C218.485 95.4 224.099 94.0286 230.442 94.0286C236.699 94.0286 242.142 95.4 246.771 98.1429C251.399 100.886 254.999 104.657 257.571 109.457C260.142 114.257 261.428 119.7 261.428 125.786C261.428 132.043 260.057 137.7 257.314 142.757C254.657 147.729 250.928 151.671 246.128 154.586C241.328 157.414 235.714 158.829 229.285 158.829ZM212.699 125.657C212.699 134.4 214.114 141.343 216.942 146.486C219.857 151.543 224.185 154.071 229.928 154.071C235.414 154.071 239.614 151.671 242.528 146.871C245.442 141.986 246.899 135.471 246.899 127.329C246.899 118.671 245.442 111.771 242.528 106.629C239.614 101.486 235.371 98.9143 229.799 98.9143C224.314 98.9143 220.071 101.271 217.071 105.986C214.157 110.7 212.699 117.257 212.699 125.657ZM267.394 157.414C267.308 157.071 267.265 156.729 267.265 156.386C267.265 156.043 267.265 155.7 267.265 155.357C267.265 155.014 267.265 154.714 267.265 154.457C267.265 154.114 267.308 153.771 267.394 153.429C270.308 153.257 272.451 153 273.822 152.657C275.194 152.229 276.094 151.371 276.522 150.086C276.951 148.8 277.165 146.743 277.165 143.914V79.5C277.165 76.8429 276.908 74.8714 276.394 73.5857C275.965 72.2143 274.979 71.2714 273.437 70.7572C271.979 70.1572 269.751 69.7286 266.751 69.4714C266.665 69.1286 266.622 68.8286 266.622 68.5714C266.622 68.3143 266.622 68.0572 266.622 67.8C266.622 67.4572 266.622 67.1572 266.622 66.9C266.622 66.6429 266.665 66.3429 266.751 66C270.179 65.5714 273.694 64.9286 277.294 64.0714C280.894 63.1286 284.579 61.8429 288.351 60.2143L290.537 60.9857C290.451 62.8714 290.365 65.0143 290.279 67.4143C290.279 69.8143 290.279 72.4286 290.279 75.2572V122.957C291.308 122.957 292.165 122.786 292.851 122.443C293.537 122.1 294.094 121.757 294.522 121.414C294.694 121.243 295.037 120.943 295.551 120.514C296.065 120 296.965 119.1 298.251 117.814C299.537 116.529 301.379 114.6 303.779 112.029C306.008 109.543 307.765 107.614 309.051 106.243C310.422 104.786 311.108 103.586 311.108 102.643C311.108 101.957 310.508 101.4 309.308 100.971C308.194 100.457 306.651 100.157 304.679 100.071C304.594 99.7286 304.551 99.3857 304.551 99.0429C304.551 98.7 304.551 98.3572 304.551 98.0143C304.551 97.6714 304.551 97.3286 304.551 96.9857C304.551 96.6429 304.594 96.3 304.679 95.9572C306.394 95.9572 308.537 96 311.108 96.0857C313.765 96.1714 316.122 96.2143 318.179 96.2143C319.208 96.2143 320.665 96.2143 322.551 96.2143C324.522 96.1286 326.494 96.0857 328.465 96.0857C330.437 96.0857 331.894 96.0857 332.837 96.0857C332.922 96.4286 332.965 96.7714 332.965 97.1143C333.051 97.4572 333.094 97.8 333.094 98.1429C333.094 98.4857 333.051 98.8286 332.965 99.1714C332.965 99.4286 332.922 99.7286 332.837 100.071C328.379 100.157 324.608 101.271 321.522 103.414C318.437 105.471 315.479 107.914 312.651 110.743L303.394 120L321.137 143.4C323.022 145.886 324.565 147.857 325.765 149.314C327.051 150.686 328.422 151.671 329.879 152.271C331.337 152.871 333.351 153.257 335.922 153.429C336.094 154.629 336.094 155.957 335.922 157.414C334.808 157.414 333.137 157.371 330.908 157.286C328.765 157.2 325.379 157.157 320.751 157.157C320.065 157.157 319.037 157.157 317.665 157.157C316.294 157.157 315.008 157.243 313.808 157.414C313.808 156.471 313.294 155.186 312.265 153.557C311.237 151.843 310.422 150.6 309.822 149.829C309.565 149.4 308.922 148.5 307.894 147.129C306.865 145.671 305.622 144 304.165 142.114C302.794 140.143 301.379 138.171 299.922 136.2C298.465 134.143 297.137 132.343 295.937 130.8C294.994 129.514 294.094 128.529 293.237 127.843C292.465 127.071 291.479 126.686 290.279 126.686V142.629C290.279 145.971 290.408 148.371 290.665 149.829C291.008 151.286 291.737 152.229 292.851 152.657C294.051 153 295.851 153.257 298.251 153.429C298.422 154.714 298.422 156.043 298.251 157.414C296.022 157.414 293.665 157.371 291.179 157.286C288.779 157.2 286.165 157.157 283.337 157.157C280.594 157.157 277.894 157.2 275.237 157.286C272.579 157.371 269.965 157.414 267.394 157.414ZM369.34 157.414C366.94 157.414 364.454 157.371 361.883 157.286C359.311 157.2 356.526 157.157 353.526 157.157C350.783 157.157 347.997 157.2 345.169 157.286C342.426 157.371 339.769 157.414 337.197 157.414C336.94 156.043 336.94 154.714 337.197 153.429C340.111 153.257 342.254 153 343.626 152.657C344.997 152.229 345.897 151.371 346.326 150.086C346.754 148.8 346.969 146.743 346.969 143.914V115.114C346.969 111.771 346.84 109.286 346.583 107.657C346.326 106.029 345.511 104.914 344.14 104.314C342.854 103.629 340.583 103.071 337.326 102.643C337.069 101.443 337.069 100.329 337.326 99.3C341.097 98.6143 344.697 97.7572 348.126 96.7286C351.554 95.7 354.854 94.5857 358.026 93.3857L360.211 94.1572C360.126 96.8143 360.04 99.3 359.954 101.614C359.954 103.929 359.954 106.114 359.954 108.171V142.629C359.954 145.971 360.126 148.371 360.469 149.829C360.897 151.286 361.754 152.229 363.04 152.657C364.411 153 366.511 153.257 369.34 153.429C369.597 154.714 369.597 156.043 369.34 157.414ZM343.626 71.5286C343.626 69.0429 344.483 66.9429 346.197 65.2286C347.997 63.5143 350.14 62.6572 352.626 62.6572C355.026 62.6572 357.083 63.5143 358.797 65.2286C360.597 66.9429 361.497 69 361.497 71.4C361.497 73.6286 360.64 75.6857 358.926 77.5715C357.211 79.3714 355.026 80.2714 352.369 80.2714C349.969 80.2714 347.911 79.4143 346.197 77.7C344.483 75.9857 343.626 73.9286 343.626 71.5286ZM372.678 188.914C372.592 187.543 372.592 186.214 372.678 184.929C375.678 184.757 377.864 184.457 379.235 184.029C380.607 183.6 381.464 182.743 381.807 181.457C382.235 180.257 382.45 178.286 382.45 175.543V115.114C382.45 111.771 382.278 109.286 381.935 107.657C381.678 106.029 380.864 104.914 379.492 104.314C378.207 103.629 375.978 103.071 372.807 102.643C372.635 101.443 372.635 100.329 372.807 99.3C376.321 98.6143 379.75 97.8 383.092 96.8572C386.435 95.9143 389.907 94.7572 393.507 93.3857L395.307 94.2857C395.221 96.4286 395.178 98.2714 395.178 99.8143C395.178 101.271 395.178 102.686 395.178 104.057C398.521 100.543 401.821 98.0143 405.078 96.4714C408.335 94.8429 411.807 94.0286 415.492 94.0286C420.892 94.0286 425.435 95.4429 429.121 98.2714C432.892 101.1 435.764 104.786 437.735 109.329C439.707 113.871 440.692 118.8 440.692 124.114C440.692 130.371 439.321 136.157 436.578 141.471C433.921 146.7 430.192 150.9 425.392 154.071C420.592 157.243 415.021 158.829 408.678 158.829C406.192 158.829 403.835 158.614 401.607 158.186C399.378 157.843 397.278 157.329 395.307 156.643V174C395.307 177.343 395.564 179.743 396.078 181.2C396.592 182.657 397.707 183.557 399.421 183.9C401.135 184.329 403.835 184.629 407.521 184.8C407.607 186.171 407.607 187.543 407.521 188.914C404.607 188.914 401.65 188.871 398.65 188.786C395.735 188.7 392.521 188.657 389.007 188.657C385.664 188.657 382.792 188.7 380.392 188.786C377.992 188.871 375.421 188.914 372.678 188.914ZM395.178 146.1C398.864 151.414 403.75 154.071 409.835 154.071C414.121 154.071 417.507 152.871 419.992 150.471C422.564 148.071 424.407 144.943 425.521 141.086C426.721 137.229 427.321 133.157 427.321 128.871C427.321 124.586 426.764 120.386 425.65 116.271C424.535 112.157 422.692 108.771 420.121 106.114C417.635 103.371 414.207 102 409.835 102C404.607 102 399.721 104.529 395.178 109.586V146.1ZM477.335 158.829C471.25 158.829 466.107 157.414 461.907 154.586C457.707 151.757 454.493 147.986 452.264 143.271C450.121 138.471 449.05 133.2 449.05 127.457C449.05 121.543 450.164 116.057 452.393 111C454.707 105.943 458.05 101.871 462.421 98.7857C466.793 95.6143 472.107 94.0286 478.364 94.0286C485.907 94.0286 491.778 96.3 495.978 100.843C500.264 105.3 502.407 111.086 502.407 118.2C502.407 119.057 502.364 119.829 502.278 120.514C502.278 121.114 502.235 121.629 502.15 122.057L461.65 121.671C461.65 122.014 461.65 122.314 461.65 122.571C461.65 130.971 463.578 137.571 467.435 142.371C471.293 147.171 476.693 149.571 483.635 149.571C489.464 149.571 495.207 147.686 500.864 143.914C501.721 144.943 502.364 145.929 502.793 146.871C495.507 154.843 487.021 158.829 477.335 158.829ZM462.035 116.914L488.65 116.4C488.735 116.229 488.778 116.014 488.778 115.757C488.778 115.414 488.778 115.2 488.778 115.114C488.778 110.486 487.75 106.629 485.693 103.543C483.721 100.457 480.764 98.9143 476.821 98.9143C473.135 98.9143 469.921 100.414 467.178 103.414C464.435 106.414 462.721 110.914 462.035 116.914ZM559.644 158.957L557.587 158.057L557.073 149.186C554.073 152.443 550.987 154.886 547.816 156.514C544.73 158.057 541.302 158.829 537.53 158.829C532.216 158.829 527.544 157.414 523.516 154.586C519.573 151.671 516.53 147.9 514.387 143.271C512.244 138.643 511.173 133.629 511.173 128.229C511.173 121.971 512.502 116.271 515.159 111.129C517.902 105.986 521.716 101.871 526.602 98.7857C531.573 95.6143 537.359 94.0286 543.959 94.0286C545.93 94.0286 547.987 94.2 550.13 94.5429C552.273 94.8 554.502 95.2714 556.816 95.9572V79.5C556.816 76.8429 556.559 74.8714 556.044 73.5857C555.53 72.2143 554.502 71.2714 552.959 70.7572C551.502 70.1572 549.273 69.7286 546.273 69.4714C546.016 68.3572 546.016 67.2 546.273 66C549.702 65.5714 553.216 64.9286 556.816 64.0714C560.416 63.1286 564.102 61.8429 567.873 60.2143L570.059 60.9857C569.973 62.8714 569.887 65.0143 569.802 67.4143C569.802 69.8143 569.802 72.4286 569.802 75.2572V139.029C569.802 142.286 569.93 144.729 570.187 146.357C570.445 147.986 571.216 149.143 572.502 149.829C573.787 150.429 575.973 150.986 579.059 151.5C579.316 152.614 579.316 153.771 579.059 154.971C575.63 155.486 572.287 156.043 569.03 156.643C565.773 157.243 562.644 158.014 559.644 158.957ZM556.944 106.886C553.259 101.829 548.416 99.3 542.416 99.3C536.502 99.3 532.044 101.486 529.044 105.857C526.044 110.143 524.544 116.1 524.544 123.729C524.544 128.186 525.144 132.471 526.344 136.586C527.544 140.7 529.516 144.086 532.259 146.743C535.002 149.4 538.644 150.729 543.187 150.729C548.93 150.729 553.516 148.243 556.944 143.271V106.886ZM615.699 157.414C613.299 157.414 610.813 157.371 608.242 157.286C605.67 157.2 602.885 157.157 599.885 157.157C597.142 157.157 594.356 157.2 591.527 157.286C588.785 157.371 586.127 157.414 583.556 157.414C583.299 156.043 583.299 154.714 583.556 153.429C586.47 153.257 588.613 153 589.985 152.657C591.356 152.229 592.256 151.371 592.685 150.086C593.113 148.8 593.327 146.743 593.327 143.914V115.114C593.327 111.771 593.199 109.286 592.942 107.657C592.685 106.029 591.87 104.914 590.499 104.314C589.213 103.629 586.942 103.071 583.685 102.643C583.427 101.443 583.427 100.329 583.685 99.3C587.456 98.6143 591.056 97.7572 594.485 96.7286C597.913 95.7 601.213 94.5857 604.385 93.3857L606.57 94.1572C606.485 96.8143 606.399 99.3 606.313 101.614C606.313 103.929 606.313 106.114 606.313 108.171V142.629C606.313 145.971 606.485 148.371 606.827 149.829C607.256 151.286 608.113 152.229 609.399 152.657C610.77 153 612.87 153.257 615.699 153.429C615.956 154.714 615.956 156.043 615.699 157.414ZM589.985 71.5286C589.985 69.0429 590.842 66.9429 592.556 65.2286C594.356 63.5143 596.499 62.6572 598.985 62.6572C601.385 62.6572 603.442 63.5143 605.156 65.2286C606.956 66.9429 607.856 69 607.856 71.4C607.856 73.6286 606.999 75.6857 605.285 77.5715C603.57 79.3714 601.385 80.2714 598.727 80.2714C596.327 80.2714 594.27 79.4143 592.556 77.7C590.842 75.9857 589.985 73.9286 589.985 71.5286ZM665.837 158.571C660.18 158.571 656.623 155.743 655.165 150.086C649.08 155.914 642.865 158.829 636.523 158.829C632.151 158.829 628.423 157.543 625.337 154.971C622.337 152.314 620.837 148.671 620.837 144.043C620.837 132.729 632.365 125.614 655.423 122.7C655.423 120.643 655.423 118.543 655.423 116.4C655.508 114.257 655.551 111.986 655.551 109.586C655.551 106.586 654.437 104.271 652.208 102.643C650.065 100.929 647.451 100.071 644.365 100.071C642.137 100.071 640.251 100.414 638.708 101.1C638.708 103.071 638.28 105.086 637.423 107.143C636.565 109.2 635.365 110.957 633.823 112.414C632.365 113.786 630.565 114.471 628.423 114.471C626.537 114.471 625.037 113.957 623.923 112.929C622.894 111.814 622.38 110.486 622.38 108.943C622.38 106.886 623.323 104.957 625.208 103.157C627.18 101.271 629.665 99.6857 632.665 98.4C635.665 97.0286 638.794 95.9572 642.051 95.1857C645.308 94.4143 648.223 94.0286 650.794 94.0286C653.965 94.0286 656.88 94.6286 659.537 95.8286C662.28 96.9429 664.465 98.6572 666.094 100.971C667.808 103.2 668.665 105.986 668.665 109.329C668.665 111.386 668.623 114.086 668.537 117.429C668.451 120.686 668.365 124.2 668.28 127.971C668.28 131.657 668.237 135.171 668.151 138.514C668.065 141.857 668.023 144.557 668.023 146.614C668.023 149.357 669.223 150.729 671.623 150.729C673.165 150.729 675.523 150.214 678.694 149.186C679.037 149.614 679.251 150.086 679.337 150.6C679.508 151.114 679.637 151.586 679.723 152.014C677.58 154.329 675.308 156 672.908 157.029C670.594 158.057 668.237 158.571 665.837 158.571ZM655.165 146.614L655.423 127.329C647.537 128.357 642.094 130.029 639.094 132.343C636.18 134.657 634.723 137.614 634.723 141.214C634.723 144.214 635.494 146.529 637.037 148.157C638.665 149.7 640.68 150.471 643.08 150.471C644.623 150.471 646.337 150.129 648.223 149.443C650.194 148.757 652.508 147.814 655.165 146.614Z" fill="currentColor"></path><rect x="715" y="65" width="195" height="105" rx="52.5" fill="currentColor" fill-opacity="0.05"></rect><path d="M767.529 142.875L754.049 107.938H762.552L769.309 127.582C769.705 128.812 770.078 130.032 770.43 131.24C770.781 132.449 771.078 133.559 771.32 134.569H771.616C771.858 133.559 772.144 132.449 772.473 131.24C772.825 130.032 773.198 128.812 773.594 127.582L780.186 107.938H788.689L775.472 142.875H767.529ZM810.856 143.633C806.901 143.633 803.528 142.71 800.737 140.865C797.969 138.997 795.871 136.305 794.442 132.79C793.014 129.252 792.3 124.989 792.3 120.001V118.255C792.3 113.245 793.014 108.982 794.442 105.467C795.871 101.929 797.969 99.2373 800.737 97.3916C803.528 95.5239 806.901 94.5901 810.856 94.5901C814.789 94.5901 818.129 95.5129 820.875 97.3586C823.644 99.2043 825.731 101.907 827.138 105.467C828.566 109.004 829.28 113.267 829.28 118.255V120.001C829.28 124.989 828.577 129.252 827.171 132.79C825.764 136.327 823.677 139.019 820.908 140.865C818.162 142.71 814.811 143.633 810.856 143.633ZM810.889 136.382C813.064 136.382 814.877 135.789 816.327 134.602C817.799 133.416 818.898 131.658 819.623 129.329C820.348 126.978 820.711 124.066 820.711 120.595V117.694C820.711 114.223 820.337 111.311 819.59 108.96C818.865 106.609 817.777 104.84 816.327 103.654C814.877 102.445 813.064 101.841 810.889 101.841C808.714 101.841 806.879 102.434 805.385 103.621C803.912 104.785 802.792 106.554 802.023 108.927C801.276 111.278 800.902 114.201 800.902 117.694V120.595C800.902 124.044 801.287 126.945 802.056 129.296C802.825 131.647 803.945 133.416 805.418 134.602C806.89 135.789 808.714 136.382 810.889 136.382ZM838.988 143.205C838 143.205 837.099 142.985 836.286 142.545C835.495 142.106 834.879 141.491 834.44 140.7C834.001 139.909 833.781 139.019 833.781 138.03C833.781 137.019 834.001 136.118 834.44 135.327C834.879 134.536 835.495 133.921 836.286 133.482C837.099 133.042 838 132.823 838.988 132.823C839.977 132.823 840.867 133.042 841.658 133.482C842.449 133.921 843.064 134.536 843.504 135.327C843.943 136.118 844.163 137.019 844.163 138.03C844.163 139.019 843.943 139.909 843.504 140.7C843.064 141.491 842.449 142.106 841.658 142.545C840.867 142.985 839.977 143.205 838.988 143.205ZM857.365 142.875V104.511L846.39 111.663V103.357L858.453 95.3481H865.802V142.875H857.365Z" fill="currentColor"></path></svg></a></div><div class="flex items-center gap-1"><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover px-4 py-2 w-9 h-9 flex-shrink-0 [&amp;_svg]:!w-5" type="button" id="radix-_R_aipfiumdb_" aria-haspopup="menu" aria-expanded="false" data-state="closed"><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M12 18a6 6 0 1 1 0-12 6 6 0 0 1 0 12m0-2a4 4 0 1 0 0-8 4 4 0 0 0 0 8M11 1h2v3h-2zm0 19h2v3h-2zM3.515 4.929l1.414-1.414L7.05 5.636 5.636 7.05zM16.95 18.364l1.414-1.414 2.121 2.121-1.414 1.414zm2.121-14.85 1.414 1.415-2.121 2.121-1.414-1.414zM5.636 16.95l1.414 1.414-2.121 2.121-1.414-1.414zM23 11v2h-3v-2zM4 11v2H1v-2z"></path></svg></span><span class="inline-flex items-center justify-center p-0 m-0 w-6 h-6 absolute rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-6 h-6" focusable="false" style="fill:currentColor"><path d="M10 7a7 7 0 0 0 12 4.9v.1c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2h.1A6.98 6.98 0 0 0 10 7m-6 5a8 8 0 0 0 15.062 3.762A9 9 0 0 1 8.238 4.938 8 8 0 0 0 4 12"></path></svg></span><span class="sr-only">Toggle theme</span></button><button class="focus-visible:ring-ring inline-flex items-center justify-center gap-x-2 whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary hover:bg-overlay-hover h-9 w-9" type="button"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search h-5 w-5"><circle cx="11" cy="11" r="8"></circle><path d="m21 21-4.3-4.3"></path></svg></button></div></div></header><div class="bg-surface-base fixed right-0 top-16 z-50 h-[calc(100vh-4rem)] w-full transform border-l transition-transform duration-300 ease-in-out sm:w-1/2 md:hidden translate-x-full"><div class="flex flex-col gap-4 p-4"><div class="flex flex-row gap-2"><button class="focus-visible:ring-ring inline-flex items-center gap-x-2 whitespace-nowrap text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 border-input text-primary hover:border-primary/15 border hover:bg-overlay-hover py-2 max-w-full shadow-none sm:w-80 h-10 w-full justify-start rounded-full bg-surface-l1 px-4" type="button"><div class="flex items-center justify-between sm:w-full gap-4"><div class="flex items-center justify-start gap-2"><span class="inline-flex items-center justify-center p-0 m-0 w-4 h-4 text-muted" data-namespace="@xai/icons" data-slot="icon"><svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="w-4 h-4" focusable="false" style="fill:currentColor"><path d="m18.031 16.617 4.283 4.282-1.415 1.415-4.282-4.283A8.96 8.96 0 0 1 11 20c-4.968 0-9-4.032-9-9s4.032-9 9-9 9 4.032 9 9a8.96 8.96 0 0 1-1.969 5.617m-2.006-.742A6.98 6.98 0 0 0 18 11c0-3.867-3.133-7-7-7s-7 3.133-7 7 3.133 7 7 7a6.98 6.98 0 0 0 4.875-1.975z"></path></svg></span><p class="text-sm text-muted hidden sm:block">Search</p></div><span class="hidden sm:block"><div class="bg-foreground/10 text-muted inline-flex items-center rounded-md px-1 py-0.5 text-center font-sans text-xs font-normal tracking-widest rtl:space-x-reverse"><span>âŒ˜</span><span class="text-xs">K</span></div></span></div></button></div></div></div><div><div class="min-[1350px]:grid min-[1350px]:grid-cols-[1fr_3fr_1fr]"><nav class="bg-surface-base hidden h-[calc(100vh-4rem)] overflow-y-auto px-6 pb-32 min-[1350px]:sticky min-[1350px]:top-16 min-[1350px]:block scrollbar-none [-ms-overflow-style:none] [scrollbar-width:none] [&amp;::-webkit-scrollbar]:hidden [overscroll-behavior:contain] pt-8"><ul class="space-y-2 text-sm"><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#quantitative-trait-locus" class="transition-opacity hover:opacity-100 opacity-50">Quantitative trait locus</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#history-of-quantitative-trait-loci" class="transition-opacity hover:opacity-100 opacity-50">History of Quantitative Trait Loci</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#history" class="transition-opacity hover:opacity-100 opacity-50">History</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#quantitative-traits" class="transition-opacity hover:opacity-100 opacity-50">Quantitative traits</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#multifactorial-traits-in-general" class="transition-opacity hover:opacity-100 opacity-50">Multifactorial traits in general</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#heritable-disease-and-multifactorial-inheritance" class="transition-opacity hover:opacity-100 opacity-50">Heritable disease and multifactorial inheritance</a></li><li style="padding-left:1.5rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#examples" class="transition-opacity hover:opacity-100 opacity-50">Examples</a></li><li style="padding-left:0rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#qtl-mapping" class="transition-opacity hover:opacity-100 opacity-50">QTL mapping</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#analysis-of-variance" class="transition-opacity hover:opacity-100 opacity-50">Analysis of variance</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#interval-mapping" class="transition-opacity hover:opacity-100 opacity-50">Interval mapping</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#composite-interval-mapping-cim" class="transition-opacity hover:opacity-100 opacity-50">Composite interval mapping (CIM)</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#family-pedigree-based-mapping" class="transition-opacity hover:opacity-100 opacity-50">Family-pedigree based mapping</a></li><li style="padding-left:0.75rem" class="flex items-start gap-2"><div class="bg-foreground mt-[0.5rem] h-1 w-1 flex-shrink-0 transition-opacity opacity-0"></div><a href="#references" class="transition-opacity hover:opacity-100 opacity-50">References</a></li></ul></nav><div class="relative top-16 pb-32 pt-8 px-4 md:px-8"><div class="mx-auto max-w-[850px]"><button data-state="closed" type="button" class="flex items-center"><div class="text-fg-tertiary mb-2 flex cursor-help items-center gap-2 text-sm"><span class="inline-flex items-center justify-center p-0 m-0" data-namespace="@xai/icons" data-slot="icon" style="height:16px;width:16px"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 33 33" aria-hidden="true" class="" focusable="false" style="fill:currentColor;height:16px;width:16px"><path fill="currentColor" d="m13.237 21.04 11.082-8.19c.543-.4 1.32-.244 1.578.38 1.363 3.288.754 7.241-1.957 9.955-2.71 2.714-6.482 3.31-9.93 1.954l-3.765 1.745c5.401 3.697 11.96 2.782 16.059-1.324 3.251-3.255 4.258-7.692 3.317-11.693l.008.009c-1.365-5.878.336-8.227 3.82-13.031q.123-.17.247-.345l-4.585 4.59v-.014L13.234 21.044M10.95 23.031c-3.877-3.707-3.208-9.446.1-12.755 2.446-2.449 6.454-3.448 9.952-1.979L24.76 6.56c-.677-.49-1.545-1.017-2.54-1.387A12.465 12.465 0 0 0 8.675 7.901c-3.519 3.523-4.625 8.94-2.725 13.561 1.42 3.454-.907 5.898-3.251 8.364-.83.874-1.664 1.749-2.335 2.674l10.583-9.466"></path></svg></span><span>Fact-checked by Grok<!-- --> <!-- -->4 days ago</span></div></button><article class="text-[16px]"><h1 id="quantitative-trait-locus" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Quantitative trait locus<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">A <strong>quantitative trait locus</strong> (<strong>QTL</strong>) is a <span class="inline text-[1em] leading-7">locus</span> (section of <span class="inline text-[1em] leading-7">DNA</span>) that correlates with variation of a <span class="inline text-[1em] leading-7">quantitative trait</span> in the <span class="inline text-[1em] leading-7">phenotype</span> of a <span class="inline text-[1em] leading-7">population</span> of <span class="inline text-[1em] leading-7">organisms</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_iqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup> QTLs are mapped by identifying which molecular markers (such as <span class="inline text-[1em] leading-7">SNPs</span> or <span class="inline text-[1em] leading-7">AFLPs</span>) correlate with an observed trait. This is often an early step in identifying the actual <span class="inline text-[1em] leading-7">genes</span> that cause the trait variation.</span>
<span class="mb-4 block break-words text-[1em] leading-7">A <strong>quantitative trait locus</strong> (<strong>QTL</strong>) is a region of <span class="inline text-[1em] leading-7">DNA</span> which is associated with a particular <span class="inline text-[1em] leading-7">phenotypic</span> <span class="inline text-[1em] leading-7">trait</span>, which varies in degree and which can be attributed to <span class="inline text-[1em] leading-7">polygenic</span> effects, i.e., the product of two or more <span class="inline text-[1em] leading-7">genes</span>, and their environment.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_g1abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[2]</sup> These QTLs are often found on different <span class="inline text-[1em] leading-7">chromosomes</span>. The number of QTLs which explain variation in the phenotypic trait indicates the <span class="inline text-[1em] leading-7">genetic architecture</span> of a trait. It may indicate that plant height is controlled by many genes of small effect, or by a few genes of large effect.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Typically, QTLs underlie continuous <span class="inline text-[1em] leading-7">traits</span> (those traits which vary continuously, e.g. height) as opposed to discrete traits (traits that have two or several character values, e.g. red hair in humans, a recessive trait, or smooth vs. wrinkled peas used by <span class="inline text-[1em] leading-7">Mendel</span> in his experiments).</span>
<span class="mb-4 block break-words text-[1em] leading-7">Moreover, a single <span class="inline text-[1em] leading-7">phenotypic</span> trait is usually determined by many genes. Consequently, many QTLs are associated with a single trait. Another use of QTLs is to identify <span class="inline text-[1em] leading-7">candidate genes</span> underlying a trait. The DNA sequence of any genes in this region can then be compared to a database of DNA for genes whose function is already known, this task being fundamental for marker-assisted crop improvement.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_62abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[3]</sup></span>
<h1 id="history-of-quantitative-trait-loci" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">History of Quantitative Trait Loci<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<h2 id="history" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">History<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Mendelian inheritance</span> was rediscovered at the beginning of the 20th century. As <span class="inline text-[1em] leading-7">Mendel</span>&#x27;s ideas spread, geneticists began to connect Mendel&#x27;s rules of inheritance of single factors to <span class="inline text-[1em] leading-7">Darwinian evolution</span>. For early geneticists, it was not immediately clear that the smooth variation in traits like body size (i.e., <span class="inline text-[1em] leading-7">incomplete dominance</span>) was caused by the inheritance of single genetic factors. Although <span class="inline text-[1em] leading-7">Darwin</span> himself observed that inbred features of fancy pigeons were inherited in accordance with Mendel&#x27;s laws (although Darwin did not actually know about Mendel&#x27;s ideas when he made the observation), it was not obvious that these features selected by fancy pigeon breeders can similarly explain quantitative variation in nature.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_b3qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">An early attempt by <span class="inline text-[1em] leading-7">William Ernest Castle</span> to unify the laws of Mendelian inheritance with Darwin&#x27;s theory of speciation invoked the idea that species become distinct from one another as one species or the other acquires a novel Mendelian factor.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_44abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[5]</sup> Castle&#x27;s conclusion was based on the observation that novel traits that could be studied in the lab and that show Mendelian inheritance patterns reflect a large deviation from the wild type, and Castle believed that acquisition of such features is the basis of &quot;discontinuous variation&quot; that characterizes speciation.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_64abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[5]</sup> Darwin discussed the inheritance of similar mutant features but did not invoke them as a requirement of speciation.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_84abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup> Instead Darwin used the emergence of such features in breeding populations as evidence that mutation can occur at random within breeding populations, which is a central premise of his model of selection in nature.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_a4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[4]</sup> Later in his career, Castle would refine his model for speciation to allow for small variation to contribute to speciation over time. He also was able to demonstrate this point by selectively breeding laboratory populations of rats to obtain a hooded phenotype over several generations.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_c4abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[6]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Castle&#x27;s was perhaps the first attempt made in the scientific literature to direct evolution by artificial selection of a trait with continuous underlying variation, however the practice had previously been widely employed in the development of <span class="inline text-[1em] leading-7">agriculture</span> to obtain livestock or plants with favorable features from populations that show quantitative variation in traits like body size or grain yield.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Castle&#x27;s work was among the first to attempt to unify the recently rediscovered laws of Mendelian inheritance with Darwin&#x27;s theory of evolution. Still, it would be almost thirty years until the theoretical framework for evolution of <span class="inline text-[1em] leading-7">complex traits</span> would be widely formalized.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_45abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup> In an early summary of the theory of evolution of continuous variation, <span class="inline text-[1em] leading-7">Sewall Wright</span>, a graduate student who trained under Castle, summarized contemporary thinking about the genetic basis of quantitative natural variation: &quot;As genetic studies continued, ever smaller differences were found to mendelize, and any character, sufficiently investigated, turned out to be affected by many factors.&quot;<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_85abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[7]</sup> Wright and others formalized population genetics theory that had been worked out over the preceding 30 years explaining how such traits can be inherited and create stably breeding populations with unique characteristics. Quantitative trait genetics today leverages Wright&#x27;s observations about the statistical relationship between genotype and phenotype in families and populations to understand how certain genetic features can affect variation in natural and derived populations.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The specific concept of a quantitative trait locus (QTL)â€”a chromosomal region associated with variation in a quantitative traitâ€”developed in the mid-20th century. In 1923, Karl Sax published the first study demonstrating linkage between a Mendelian marker (seed coat pattern) and a quantitative trait (seed weight) in common bean (<em>Phaseolus vulgaris</em>), providing early evidence for QTLs.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_45qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[8]</sup> In 1961, J.M. Thoday advanced the field by developing methods to locate &quot;polygenes&quot; (now QTLs) using selected lines and phenotypic markers in fruit flies (<em>Drosophila melanogaster</em>).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_85qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[9]</sup> The term &quot;quantitative trait locus&quot; was coined by Hubert Geldermann in 1975 to describe genomic regions influencing complex traits.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_a5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[10]</sup> The modern era of QTL mapping began in the late 1980s with the use of molecular markers; Lander and Botstein (1989) introduced a statistical framework using restriction fragment length polymorphism (RFLP) linkage maps to detect QTLs in experimental crosses.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_c5qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[11]</sup></span>
<h1 id="quantitative-traits" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">Quantitative traits<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7"><strong>Polygenic inheritance</strong> refers to inheritance of a <span class="inline text-[1em] leading-7">phenotypic</span> characteristic (trait) that is attributable to two or more <span class="inline text-[1em] leading-7">genes</span> and can be measured quantitatively. <strong>Multifactorial inheritance</strong> refers to polygenic inheritance that also includes interactions with the environment. Unlike <span class="inline text-[1em] leading-7">monogenic traits</span>, polygenic traits do not follow patterns of <span class="inline text-[1em] leading-7">Mendelian inheritance</span> (discrete categories). Instead, their phenotypes typically vary along a continuous gradient depicted by a <span class="inline text-[1em] leading-7">bell curve</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_f6qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[12]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">An example of a polygenic trait is <span class="inline text-[1em] leading-7">human skin color</span> variation. Several genes factor into determining a person&#x27;s natural skin color, so modifying only one of those genes can change skin color slightly or in some cases, such as for <span class="inline text-[1em] leading-7">SLC24A5</span>, moderately. Many disorders with <span class="inline text-[1em] leading-7">genetic components</span> are polygenic, including <span class="inline text-[1em] leading-7">autism</span>, <span class="inline text-[1em] leading-7">cancer</span>, <span class="inline text-[1em] leading-7">diabetes</span> and numerous others. Most phenotypic characteristics are the result of the interaction of multiple genes.</span>
<span class="mb-4 block break-words text-[1em] leading-7">Multifactorially inherited diseases are said to constitute the majority of genetic disorders affecting humans which will result in hospitalization or special care of some kind.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_27qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup><sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_37qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[14]</sup></span>
<h3 id="multifactorial-traits-in-general" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Multifactorial traits in general<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">Traits controlled both by the environment and by genetic factors are called multifactorial. Usually, multifactorial traits outside of illness result in what we see as <strong>continuous characteristics</strong> in organisms, especially human organisms such as: height,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_48qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup> skin color, and body mass.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_68qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[15]</sup> All of these phenotypes are complicated by a great deal of give-and-take between genes and environmental effects.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_88qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup> The continuous distribution of traits such as height and skin color described above, reflects the action of genes that do not manifest typical patterns of dominance and recessiveness. Instead the contributions of each involved locus are thought to be additive. Writers have distinguished this kind of inheritance as <em>polygenic</em> , or <em>quantitative inheritance</em>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_e8qbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Thus, due to the nature of polygenic traits, inheritance will not follow the same pattern as a simple <span class="inline text-[1em] leading-7">monohybrid</span> or <span class="inline text-[1em] leading-7">dihybrid cross</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_69abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[14]</sup> Polygenic inheritance can be explained as Mendelian inheritance at many loci,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_89abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup> resulting in a trait which is <span class="inline text-[1em] leading-7">normally-distributed</span>. If <em>n</em> is the number of involved loci, then the coefficients of the <span class="inline text-[1em] leading-7">binomial expansion</span> of <span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mo stretchy="false">(</mo><mi>a</mi><mo>+</mo><mi>b</mi><msup><mo stretchy="false">)</mo><mrow><mn>2</mn><mi>n</mi></mrow></msup></mrow><annotation encoding="application/x-tex">(a + b)^{2n}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em"></span><span class="mopen">(</span><span class="mord mathnormal">a</span><span class="mspace" style="margin-right:0.2222em"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222em"></span></span><span class="base"><span class="strut" style="height:1.0641em;vertical-align:-0.25em"></span><span class="mord mathnormal">b</span><span class="mclose"><span class="mclose">)</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.8141em"><span style="top:-3.063em;margin-right:0.05em"><span class="pstrut" style="height:2.7em"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">2</span><span class="mord mathnormal mtight">n</span></span></span></span></span></span></span></span></span></span></span></span> will give the frequency of distribution of all <em>n</em> allele <span class="inline text-[1em] leading-7">combinations</span>. For sufficiently high values of <em>n</em> , this binomial distribution will begin to resemble a normal distribution. From this viewpoint, a disease state will become apparent at one of the tails of the distribution, past some threshold value. Disease states of increasing severity will be expected the further one goes past the threshold and away from the <span class="inline text-[1em] leading-7">mean</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_q9abav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup></span>
<h3 id="heritable-disease-and-multifactorial-inheritance" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Heritable disease and multifactorial inheritance<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">When more than one gene is involved, with or without the presence of environmental triggers, we say that the disease is the result of multifactorial inheritance. Disease risk is often modeled using a liability threshold model, in which the cumulative effects of multiple genetic variants additively contribute to an underlying liability, and disease manifests when this liability, modulated by environmental factors, exceeds a certain threshold.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2aabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[17]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">The more genes involved in the cross, the more the distribution of the <span class="inline text-[1em] leading-7">genotypes</span> will resemble a <span class="inline text-[1em] leading-7">normal, or Gaussian</span> distribution.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6aqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup> This shows that multifactorial inheritance is polygenic, and genetic frequencies can be predicted by way of a polyhybrid <span class="inline text-[1em] leading-7">Mendelian</span> cross. Phenotypic frequencies are a different matter, especially if they are complicated by environmental factors.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The paradigm of polygenic inheritance as being used to define multifactorial disease has encountered much disagreement. Turnpenny (2004) discusses how simple polygenic inheritance cannot explain some diseases such as the onset of Type I diabetes mellitus, and that in cases such as these, not all genes are thought to make an equal contribution.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2babav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[16]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">The assumption of polygenic inheritance is that all involved loci make an equal contribution to the symptoms of the disease. This should result in a normal (Gaussian) distribution of genotypes. When it does not, the idea of polygenetic inheritance cannot be supported for that illness.</span>
<h3 id="examples" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.428571em] font-semibold" node="[object Object]">Examples<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h3>
<span class="mb-4 block break-words text-[1em] leading-7">The above are well-known examples of diseases having both genetic and environmental components. Other examples involve atopic diseases such as <span class="inline text-[1em] leading-7">eczema</span> or <span class="inline text-[1em] leading-7">dermatitis</span>,<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6cqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[13]</sup> <span class="inline text-[1em] leading-7">spina bifida</span> (open spine), and <span class="inline text-[1em] leading-7">anencephaly</span> (open skull).<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_ccqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[18]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Schizophrenia is a highly polygenic disorder with a strong genetic basis (heritability estimates of 60-80%), where over 200 risk loci have been identified through genome-wide association studies as of 2023, though these interact with environmental factors and no single variant is causative.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2dabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[19]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">If close relatives such as siblings are affected, this increases the likelihood of shared polygenic risk factors, elevating the individual&#x27;s disease risk compared to the general population; the risk decreases as affected relatives become more distant, such as cousins, due to reduced shared genetic factors.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2dqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[18]</sup> Inheritance in multifactorially-inherited diseases does not follow the same pattern as a simple <span class="inline text-[1em] leading-7">monohybrid</span> or <span class="inline text-[1em] leading-7">dihybrid cross</span>.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_8dqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[14]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">If a genetic cause is suspected and little else is known about the illness, then it remains to be seen exactly how many genes are involved in the phenotypic expression of the disease. Once that is determined, the question must be answered: if two people share similar genetic risk profiles, why are there differences in expression between them? Generally, what makes the two individuals different are likely to be environmental factors. Due to the involved nature of genetic investigations needed to determine such inheritance patterns, this is not usually the first avenue of investigation one would choose to determine etiology. Quantitative trait locus (QTL) mapping is a key method used to identify the genetic loci contributing to such complex traits.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2eabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[1]</sup></span>
<h1 id="qtl-mapping" class="group relative mb-2 scroll-mt-24 font-serif text-[2.125em] font-semibold tracking-[-1px] [&amp;:not(:first-child)]:mt-14" node="[object Object]">QTL mapping<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h1>
<span class="mb-4 block break-words text-[1em] leading-7">For organisms whose genomes are known, one might now try to exclude genes in the identified region whose function is known with some certainty not to be connected with the trait in question. If the genome is not available, it may be an option to sequence the identified region and determine the putative functions of genes by their similarity to genes with known function, usually in other genomes. This can be done using <span class="inline text-[1em] leading-7">BLAST</span>, an online tool that allows users to enter a primary sequence and search for similar sequences within the BLAST database of genes from various organisms. It is often not the actual gene underlying the phenotypic trait, but rather a region of DNA that is closely linked with the gene<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4fabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[20]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Another interest of statistical geneticists using QTL mapping is to determine the complexity of the genetic architecture underlying a phenotypic trait. For example, they may be interested in knowing whether a phenotype is shaped by many independent loci, or by a few loci, and do those loci interact. This can provide information on how the phenotype may be evolving.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2fqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[21]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Since the mid-2000s, classical QTL analyses have been combined with gene expression profiling, initially using DNA microarrays and now commonly RNA-seq, to identify expression QTLs (eQTLs) that describe <span class="inline text-[1em] leading-7">cis</span>- and <span class="inline text-[1em] leading-7">trans</span>-controlling elements for the expression of often disease-associated genes.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_6gabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[22]</sup> Such expression QTLs (eQTLs) describe <span class="inline text-[1em] leading-7">cis</span>- and <span class="inline text-[1em] leading-7">trans</span>-controlling elements for the expression of often disease-associated genes.</span>
<span class="mb-4 block break-words text-[1em] leading-7">In recent years (as of 2025), QTL mapping has been enhanced by integration with genome-wide association studies (GWAS) and high-throughput sequencing, enabling finer resolution in diverse organisms, such as ultra-high-density bin maps for horticultural traits in pepper.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2gqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[23]</sup></span>
<h2 id="analysis-of-variance" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Analysis of variance<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">The simplest method for QTL mapping is analysis of variance (<span class="inline text-[1em] leading-7">ANOVA</span>, sometimes called &quot;marker regression&quot;) at the marker loci. In this method, in a backcross, one may calculate a <span class="inline text-[1em] leading-7">t-statistic</span> to compare the averages of the two marker <span class="inline text-[1em] leading-7">genotype</span> groups. For other types of crosses (such as the intercross), where there are more than two possible genotypes, one uses a more general form of ANOVA, which provides a so-called <span class="inline text-[1em] leading-7">F-statistic</span>. The ANOVA approach for QTL mapping has three important weaknesses. First, we do not receive separate estimates of QTL location and QTL effect. QTL location is indicated only by looking at which markers give the greatest differences between genotype group averages, and the apparent QTL effect at a marker will be smaller than the true QTL effect as a result of <span class="inline text-[1em] leading-7">recombination</span> between the marker and the QTL. Second, we must discard individuals whose genotypes are missing at the marker. Third, when the markers are widely spaced, the QTL may be quite far from all markers, and so the power for QTL detection will decrease.</span>
<h2 id="interval-mapping" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Interval mapping<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">Lander and Botstein developed interval mapping, which overcomes the three disadvantages of analysis of variance at marker loci.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2iqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[11]</sup> Interval mapping is currently the most popular approach for QTL mapping in experimental crosses. The method makes use of a <span class="inline text-[1em] leading-7">genetic map</span> of the typed markers, and, like analysis of variance, assumes the presence of a single QTL. In interval mapping, each locus is considered one at a time and the logarithm of the <span class="inline text-[1em] leading-7">odds ratio</span> (<span class="inline text-[1em] leading-7">LOD score</span>) is calculated for the model that the given locus is a true QTL. The odds ratio is related to the <span class="inline text-[1em] leading-7">Pearson correlation coefficient</span> between the phenotype and the marker genotype for each individual in the experimental cross.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The term &#x27;interval mapping&#x27; is used for estimating the position of a QTL within two markers (often indicated as &#x27;marker-bracket&#x27;). Interval mapping is originally based on the maximum likelihood but there are also very good approximations possible with simple regression.</span>
<span class="mb-4 block break-words text-[1em] leading-7">The principle for QTL mapping is: 1) The likelihood can be calculated for a given set of parameters (particularly QTL effect and QTL position) given the observed data on phenotypes and marker genotypes. 2) The estimates for the parameters are those where the likelihood is highest. 3) A significance threshold can be established by permutation testing.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_2jqbav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[25]</sup></span>
<span class="mb-4 block break-words text-[1em] leading-7">Conventional methods for the detection of quantitative trait loci (QTLs) are based on a comparison of single QTL models with a model assuming no QTL. For instance in the &quot;interval mapping&quot; method<!-- --> the likelihood for a single putative QTL is assessed at each location on the genome. However, QTLs located elsewhere on the genome can have an interfering effect. As a consequence, the power of detection may be compromised, and the estimates of locations and effects of QTLs may be biased (Lander and Botstein 1989; Knapp 1991). Even nonexisting so-called &quot;ghost&quot; QTLs may appear (Haley and Knott 1992; Martinez and Curnow 1992). Therefore, multiple QTLs could be mapped more efficiently and more accurately by using multiple QTL models.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_4kabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[27]</sup></span>
<h2 id="composite-interval-mapping-cim" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Composite interval mapping (CIM)<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7">One popular approach to handle QTL mapping where multiple QTL contribute to a trait is to iteratively scan the genome and add known QTL to the regression model as QTLs are identified. This method, termed composite interval mapping, determines both the location and effect size of QTL more accurately than single-QTL approaches, especially in small mapping populations where the effect of correlation between genotypes in the mapping population may be problematic.</span>
<span class="mb-4 block break-words text-[1em] leading-7">In this method, one performs interval mapping using a subset of marker loci as covariates. These markers serve as proxies for other QTLs to increase the resolution of interval mapping, by accounting for linked QTLs and reducing the residual variation. The key problem with CIM concerns the choice of suitable marker loci to serve as covariates; once these have been chosen, CIM turns the model selection problem into a single-dimensional scan. The choice of marker covariates has not been solved, however. Not surprisingly, the appropriate markers are those closest to the true QTLs, and so if one could find these, the QTL mapping problem would be complete anyway.</span>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Inclusive composite interval mapping</span> (ICIM) has also been proposed as a potential method for QTL mapping.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_3mabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[28]</sup></span>
<h2 id="family-pedigree-based-mapping" class="group relative mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden" node="[object Object]">Family-pedigree based mapping<button class="focus-visible:ring-ring gap-x-2 whitespace-nowrap rounded-full font-medium focus-visible:outline-none focus-visible:ring-1 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-50 [&amp;&gt;[data-slot=icon]]:-mx-0.5 [&amp;&gt;[data-slot=icon]]:my-0.5 [&amp;&gt;[data-slot=icon]]:size-5 [&amp;&gt;[data-slot=icon]]:shrink-0 [&amp;&gt;[data-slot=icon]]:text-[--btn-icon] [&amp;&gt;[data-slot=icon]]:sm:my-1 [&amp;&gt;[data-slot=icon]]:sm:size-4 text-primary text-xs ml-2 inline-flex h-6 w-6 items-center justify-center p-0 align-middle opacity-0 transition-opacity hover:bg-gray-100 dark:hover:bg-gray-800" type="button" aria-label="Copy link to heading" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-link h-4 w-4"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></button></h2>
<span class="mb-4 block break-words text-[1em] leading-7"><span class="inline text-[1em] leading-7">Family-based QTL mapping</span>, or Family-pedigree based mapping (Linkage and <span class="inline text-[1em] leading-7">association mapping</span>), involves multiple families instead of a single family. Family-based QTL mapping has been the only way for mapping of genes where experimental crosses are difficult to make. However, due to some advantages, now plant geneticists are attempting to incorporate some of the methods pioneered in human genetics.<sup class="text-fg-secondary ml-[2px] cursor-pointer text-xs hover:underline" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-_R_5nabav9fiumdb_" data-state="closed" tabindex="-1" style="outline:none">[29]</sup> Using family-pedigree based approach has been discussed (Bink et al. 2008). Family-based linkage and association has been successfully implemented (Rosyara et al. 2007)</span></article><div id="references" class="min-w-0 scroll-mt-8 overflow-hidden"><div class="text-[16px]"><h2 id="references" node="[object Object]" class="mb-2 mt-7 scroll-mt-24 font-serif text-[1.714286em] font-semibold border-border-l1 pb-1 border-b overflow-hidden">References</h2></div><ol class="columns-1 gap-x-12 [counter-reset:item] md:columns-2"><li id="1" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Miles, C; Wayne, M (2008). <a href="http://www.nature.com/scitable/topicpage/quantitative-trait-locus-qtl-analysis-53904" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Quantitative trait locus (QTL) analysis&quot;</a>. <em>Nature Education</em>. <strong>1</strong> (1).</span></div></li><li id="2" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Oduola Abiola; et al. (2003). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;The nature and identification of quantitative trait loci: a community&#x27;s view&quot;</a>. <em><span class="inline text-[1em] leading-7">Nature Reviews Genetics</span></em>. <strong>4</strong> (11). <span class="inline text-[1em] leading-7">Nature Portfolio</span>: 911â€“916. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2Fnrg1206" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/nrg1206</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2063446</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/14634638" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">14634638</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:27285742" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">27285742</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:195367115" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">195367115</a>.</span></div></li><li id="3" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Watanabe, Satoshi; Hideshima, Rumiko; Xia, Zhengjun; et al. (2009). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Map-Based Cloning of the Gene Associated With the Soybean Maturity Locus E3&quot;</a>. <em>Genetics</em>. <strong>182</strong> (4): 1251â€“1262. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1534%2Fgenetics.108.098772" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1534/genetics.108.098772</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2728863</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/19474204" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">19474204</a>.</span></div></li><li id="4" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://web.archive.org/web/20131003145126/http://www.literature.org/authors/darwin-charles/the-origin-of-species/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Origin of Species&quot;</a>. Archived from <a href="http://www.literature.org/authors/darwin-charles/the-origin-of-species/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on 3 October 2013. Retrieved 24 September 2013.</span></div></li><li id="5" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Castle WE (1903). <a href="https://zenodo.org/record/1447900" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Mendel&#x27;s Law of Heredity&quot;</a>. <em>Science</em>. <strong>18</strong> (456): 396â€“406. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/1903Sci....18..396C" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1903Sci....18..396C</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1126%2Fscience.18.456.396" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1126/science.18.456.396</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/17752783" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">17752783</a>. <span class="inline text-[1em] leading-7">S2CID</span> <a href="https://api.semanticscholar.org/CorpusID:11670642" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11670642</a>.</span></div></li><li id="https://www.nap.edu/read/579/chapter/1" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.nap.edu/read/579/chapter/1" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.nap.edu/read/579/chapter/1</a></span></div></li><li id="7" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Wright, Sewall (1 March 1931). <a href="http://www.genetics.org/content/16/2/97" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Evolution in Mendelian Populations&quot;</a>. <em>Genetics</em>. <strong>16</strong> (2): 97â€“159. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fgenetics%2F16.2.97" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/genetics/16.2.97</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1201091" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1201091</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/17246615" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">17246615</a> â€“ via <a href="http://www.genetics.org" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">www.genetics.org</a>.</span></div></li><li id="https://www.genetics.org/content/8/6/552" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.genetics.org/content/8/6/552" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.genetics.org/content/8/6/552</a></span></div></li><li id="https://www.nature.com/articles/191368a0" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.nature.com/articles/191368a0" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.nature.com/articles/191368a0</a></span></div></li><li id="https://link.springer.com/article/10.1007/BF02861451" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://link.springer.com/article/10.1007/BF02861451" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://link.springer.com/article/10.1007/BF02861451</a></span></div></li><li id="17" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Lander, E.S.; Botstein, D. (1989). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Mapping mendelian factors underlying quantitative traits using RFLP linkage maps&quot;</a>. <em>Genetics</em>. <strong>121</strong> (1): 185â€“199. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fgenetics%2F121.1.185" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/genetics/121.1.185</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1203601</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/2563713" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2563713</a>.</span></div></li><li id="8" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Ricki Lewis (2003), <a href="http://highered.mcgraw-hill.com/sites/007246268x/student_view0/chapter7/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300"><em>Multifactorial Traits</em></a>, McGraw-Hill Higher Education.</span></div></li><li id="9" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Tissot, Robert. <a href="http://www.uic.edu/classes/bms/bms655/lesson11.html" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Human Genetics for 1st Year Students: Multifactorial Inheritance&quot;</a>. Retrieved 6 January 2007.</span></div></li><li id="10" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Birth Defects Genetics Centre, University of South Dakota School of Medicine. <a href="https://web.archive.org/web/20061230084542/http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Multifactorial Inheritance&quot;</a>. <em>Clinical Genetics: A Self-Study Guide for Health Care Providers</em>. University of South Dakota School of Medicine. Archived from <a href="http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on 30 December 2006. Retrieved 6 January 2007.</span></div></li><li id="11" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://web.archive.org/web/20131217193339/http://www.medterms.com/script/main/art.asp?articlekey=4453" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Definition of Multifactorial inheritance&quot;</a>. <em>MedicineNet.com MedTerms Dictionary</em>. MedicineNet.com. Archived from <a href="http://www.medterms.com/script/main/art.asp?articlekey=4453" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on 17 December 2013. Retrieved 6 January 2007.</span></div></li><li id="12" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Turnpenny, Peter (2004). <a href="http://www.fleshandbones.com/readingroom/viewchapter.cfm?ID=1041" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Chapter 9&quot;</a> (PDF). <em>Emery&#x27;s Elements of Medical Genetics</em> (12th ed.). Elsevier. Retrieved 6 January 2007.</span></div></li><li id="https://basicmedicalkey.com/multifactorial-inheritance-and-common-diseases/" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://basicmedicalkey.com/multifactorial-inheritance-and-common-diseases/" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://basicmedicalkey.com/multifactorial-inheritance-and-common-diseases/</a></span></div></li><li id="13" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Proud, Virginia &amp; Roberts, Helen (31 December 2005). <a href="https://web.archive.org/web/20061015185017/http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Medical Genetics: Multifactorial Inheritance&quot;</a>. Children&#x27;s Hospital of the King&#x27;s Daughters. Archived from <a href="http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">the original</a> on 15 October 2006. Retrieved 6 January 2007.</span></div></li><li id="https://www.nature.com/articles/s41380-023-02293-8" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.nature.com/articles/s41380-023-02293-8" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.nature.com/articles/s41380-023-02293-8</a></span></div></li><li id="14" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;BLAST: Basic Local Alignment Search Tool&quot;</a>. <em>blast.ncbi.nlm.nih.gov</em>. Retrieved 18 February 2018.</span></div></li><li id="15" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Grisel, Judith E.; Crabbe, John C. (1995). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Quantitative Trait Loci Mapping&quot;</a>. <em>Alcohol Health and Research World</em>. <strong>19</strong> (3): 220â€“227. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/0090-838X" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">0090-838X</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">6875759</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/31798043" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">31798043</a>.</span></div></li><li id="16" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Westra HJ, et al. (2013). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Systematic identification of trans eQTLs as putative drivers of known disease associations&quot;</a>. <em>Nat Genet</em>. <strong>45</strong> (10): 1238â€“1243. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2Fng.2756" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/ng.2756</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">3991562</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/24013639" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">24013639</a>.</span></div></li><li id="https://www.pnas.org/doi/10.1073/pnas.2413764121" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7"><a href="https://www.pnas.org/doi/10.1073/pnas.2413764121" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">https://www.pnas.org/doi/10.1073/pnas.2413764121</a></span></div></li><li id="18" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Lynch, M. &amp; Walsh, B. Genetics and Analysis of Quantitative Traits edn 1 (Sinauer Associates, 1998).</span></div></li><li id="19" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Bloom J. S.; Ehrenreich I. M.; Loo W. T.; Lite T.-L. V.; Kruglyak L. (2013). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Finding the sources of missing heritability in a yeast cross&quot;</a>. <em>Nature</em>. <strong>494</strong> (7436): 234â€“237. <span class="inline text-[1em] leading-7">arXiv</span>:<a href="https://arxiv.org/abs/1208.2865" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1208.2865</a>. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2013Natur.494..234B" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2013Natur.494..234B</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1038%2Fnature11867" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1038/nature11867</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4001867</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/23376951" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">23376951</a>.</span></div></li><li id="20" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps. ES Lander and D Botstein. Genetics. 1989</span></div></li><li id="21" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Jansen, R C (1 September 1993). <a href="http://www.genetics.org/content/135/1/205.full.pdf" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Interval mapping of multiple quantitative trait loci&quot;</a> (PDF). <em>Genetics</em>. <strong>135</strong> (1): 205â€“211. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1093%2Fgenetics%2F135.1.205" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1093/genetics/135.1.205</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205619" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1205619</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/8224820" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">8224820</a>. Retrieved 1 March 2023.</span></div></li><li id="22" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Li, Shanshan; Wang, Jiankang; Zhang, Luyan (10 July 2015). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">&quot;Inclusive Composite Interval Mapping of QTL by Environment Interactions in Biparental Populations&quot;</a>. <em>PLOS ONE</em>. <strong>10</strong> (7): e0132414. <span class="inline text-[1em] leading-7">Bibcode</span>:<a href="https://ui.adsabs.harvard.edu/abs/2015PLoSO..1032414L" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">2015PLoSO..1032414L</a>. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1371%2Fjournal.pone.0132414" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1371/journal.pone.0132414</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1932-6203" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1932-6203</a>. <span class="inline text-[1em] leading-7">PMC</span> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">4498613</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/26161656" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">26161656</a>.</span></div></li><li id="23" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Jannink, J; Bink, Mc; Jansen, Rc (August 2001). &quot;Using complex plant pedigrees to map valuable genes&quot;. <em>Trends in Plant Science</em>. <strong>6</strong> (8): 337â€“42. <span class="inline text-[1em] leading-7">doi</span>:<a href="https://doi.org/10.1016%2FS1360-1385%2801%2902017-9" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">10.1016/S1360-1385(01)02017-9</a>. <span class="inline text-[1em] leading-7">ISSN</span> <a href="https://search.worldcat.org/issn/1360-1385" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">1360-1385</a>. <span class="inline text-[1em] leading-7">PMID</span> <a href="https://pubmed.ncbi.nlm.nih.gov/11495765" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">11495765</a>.</span></div></li><li id="24" class="mb-2 grid break-inside-avoid grid-cols-[auto_1fr] gap-2 text-sm leading-relaxed [counter-increment:item] before:font-semibold before:content-[counter(item)&#x27;.&#x27;]"><div class="relative -top-[3px] overflow-hidden break-words text-[16px] [&amp;_*]:mb-0"><span class="mb-4 block break-words text-[1em] leading-7">Rosyara, U. R.; Maxson-stein, K.L.; Glover, K.D.; Stein, J.M.; Gonzalez-hernandez, J.L. (2007). &quot;Family-based mapping of FHB resistance QTLs in hexaploid wheat&quot;. <em>Proceedings of National Fusarium Head Blight Forum</em>.</span></div></li></ol></div><div class="text-fg-secondary text-xs [&amp;_*]:mb-2"><hr class="border-border-l1 my-8 rounded-sm border-t"/>
<span class="mb-4 block break-words text-[1em] leading-7">The content is adapted from <a href="https://en.wikipedia.org/wiki/Quantitative_trait_locus" target="_blank" rel="noopener noreferrer" class="break-all text-[1em] text-blue-500 hover:underline dark:text-blue-300">Wikipedia</a>, licensed under Creative Commons Attribution-ShareAlike 4.0 License.</span></div></div></div><div class="hidden min-[1400px]:block"></div></div></div><!--$--><!--/$--><div role="region" aria-label="Notifications (F8)" tabindex="-1" style="pointer-events:none"><ol tabindex="-1" class="pointer-events-none fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 max-sm:left-0 sm:bottom-0 sm:right-0 sm:top-auto sm:max-w-[420px] sm:flex-col pl-16 pr-2 pt-16 sm:px-4"></ol></div><script type="application/json" id="server-client-data-experimentation">{"status":"uninitialized"}</script><script src="/_next/static/chunks/webpack-e121ed42680f327e.js" nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5" id="_R_" async=""></script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">(self.__next_f=self.__next_f||[]).push([0])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"1:\"$Sreact.fragment\"\n3:I[91363,[],\"\"]\n4:I[23775,[],\"\"]\n5:I[57654,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"PageHeaderProvider\"]\n6:I[17618,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"3699\",\"static/chunks/app/page/%5Bslug%5D/layout-55a344716dd57c44.js\"],\"HeaderContent\"]\n7:I[25529,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"6751\",\"static/chunks/app/page/%5Bslug%5D/not-found-69a3edc7db5749d6.js\"],\"default\"]\n9:I[6666,[],\"OutletBoundary\"]\nb:I[80415,[],\"AsyncMetadataOutlet\"]\nd:I[6666,[],\"ViewportBoundary\"]\nf:I[6666,[],\"MetadataBoundary\"]\n10:\"$Sreact.suspense\"\n12:I[95909,[\"4219\",\"static/chunks/app/global-error-4d07d20223cd4b4c.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"13:I[51498,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"ConstantsProvider\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"14:I[91073,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"15:I[38642,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"16:I[99648,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"17:I[78825,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"18:I[8550,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MixpanelProvider\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"19:I[12290,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"MobileTocProvider\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"1a:I[53947,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"NuqsAdapter\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"1b:I[91873,[\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8039\",\"static/chunks/app/error-4a29e9399afba038.js\"],\"default\"]\n1c:I[5091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"7618\",\"static/chunks/7618-43536a8fb0dd3bfe.js\",\"4345\",\"static/chunks/app/not-found-dd95690acf732f18.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"1d:I[16091,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"927\",\"static/chunks/927-34e2a6da3e15e26b.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"2660\",\"static/chunks/2660-d983a7f287e89f18.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"3655\",\"static/chunks/3655-6717081f7512305a.js\",\"8945\",\"static/chunks/8945-e4e3cf487f5e27c7.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"2225\",\"static/chunks/2225-454cf9f44775e7ce.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"7177\",\"static/chunks/app/layout-226f2bd71acf9f9e.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,":HL[\"/_next/static/media/1f2316909698f815.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/3d4419af2cf8609b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/4dec29efcaeb336c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/74452ea3ef0f9101.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/904ef0a86fe32a00.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/d886a03bcda7ad8f.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/e1447589d6f59c4b.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/f5a90156f8995c8c.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/css/62c4caba71dfda84.css\",\"style\",{\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]\n:HL[\"/_next/static/css/eb3d87f98fe1565f.css\",\"style\",{\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]\n:HL[\"/_next/static/css/1b5e561215938d4d.css\",\"style\",{\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]\n:HL[\"/_next/static/css/0227d069a630d414.css\",\"style\",{\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]\n:HL[\"/_next/static/css/f87fff2ab93d05a7.css\",\"style\",{\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"0:{\"P\":null,\"b\":\"BpM29AX4fmgcUxZ_6t0le\",\"p\":\"\",\"c\":[\"\",\"page\",\"Quantitative_trait_locus\"],\"i\":false,\"f\":[[[\"\",{\"children\":[\"page\",{\"children\":[[\"slug\",\"Quantitative_trait_locus\",\"d\"],{\"children\":[\"__PAGE__\",{}]}]}]},\"$undefined\",\"$undefined\",true],[\"\",[\"$\",\"$1\",\"c\",{\"children\":[[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/62c4caba71dfda84.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}],[\"$\",\"link\",\"1\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/eb3d87f98fe1565f.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}],[\"$\",\"link\",\"2\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/1b5e561215938d4d.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}],[\"$\",\"link\",\"3\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/0227d069a630d414.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]],\"$L2\"]}],{\"children\":[\"page\",[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[[\"slug\",\"Quantitative_trait_locus\",\"d\"],[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L5\",null,{\"children\":[[\"$\",\"$L6\",null,{\"maxWidth\":\"full\",\"mobileOptions\":{\"right\":{\"showFixedIssues\":true,\"showThemeSwitcher\":true,\"showSearch\":true,\"showTableOfContents\":true}}}],[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L7\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}]]}],{\"children\":[\"__PAGE__\",[\"$\",\"$1\",\"c\",{\"children\":[\"$L8\",[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/f87fff2ab93d05a7.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\"}]],[\"$\",\"$L9\",null,{\"children\":[\"$La\",[\"$\",\"$Lb\",null,{\"promise\":\"$@c\"}]]}]]}],{},null,false]},null,false]},null,false]},null,false],[\"$\",\"$1\",\"h\",{\"children\":[null,[[\"$\",\"$Ld\",null,{\"children\":\"$Le\"}],[\"$\",\"meta\",null,{\"name\":\"next-size-adjust\",\"content\":\"\"}]],[\"$\",\"$Lf\",null,{\"children\":[\"$\",\"div\",null,{\"hidden\":true,\"children\":[\"$\",\"$10\",null,{\"fallback\":null,\"children\":\"$L11\"}]}]}]]}],false]],\"m\":\"$undefined\",\"G\":[\"$12\",[]],\"s\":false,\"S\":false}\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"2:[\"$\",\"html\",null,{\"lang\":\"en\",\"className\":\"bg-surface-base antialiased\",\"suppressHydrationWarning\":true,\"children\":[[\"$\",\"head\",null,{\"children\":[[\"$\",\"title\",null,{\"children\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"type\":\"image/x-icon\",\"href\":\"/favicon.ico\",\"sizes\":\"48x48\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-dark.png\",\"media\":\"(prefers-color-scheme: light)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"href\":\"/images/icon-light.png\",\"media\":\"(prefers-color-scheme: dark)\",\"type\":\"image/png\"}],[\"$\",\"link\",null,{\"rel\":\"apple-touch-icon\",\"href\":\"/icon-192x192.png\"}],[\"$\",\"link\",null,{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\"}],[\"$\",\"meta\",null,{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",null,{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"meta\",null,{\"property\":\"og:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"property\":\"og:url\",\"content\":\"https://grokipedia.com\"}],[\"$\",\"meta\",null,{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"property\":\"og:locale\",\"content\":\"en_US\"}],[\"$\",\"meta\",null,{\"property\":\"og:type\",\"content\":\"website\"}],[\"$\",\"meta\",null,{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",null,{\"property\":\"og:image:alt\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:title\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:description\",\"content\":\"Grokipedia is an open source, comprehensive collection of all knowledge.\"}],[\"$\",\"meta\",null,{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}]]}],[\"$\",\"body\",null,{\"className\":\"flex h-[100svh] w-full flex-col __variable_13c637 __variable_8a0ba0 __variable_779740 __variable_2484fd __variable_525cc3 __variable_13486b\",\"children\":[[\"$\",\"$L13\",null,{\"children\":[\"$\",\"$L14\",null,{\"children\":[\"$\",\"$L15\",null,{\"defaultTheme\":\"dark\",\"nonce\":\"MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5\",\"children\":[\"$\",\"$L16\",null,{\"children\":[\"$\",\"$L17\",null,{\"children\":[\"$\",\"$L18\",null,{\"children\":[\"$\",\"$L19\",null,{\"children\":[\"$\",\"$L1a\",null,{\"children\":[[\"$\",\"$L3\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$1b\",\"errorStyles\":[],\"errorScripts\":[],\"template\":[\"$\",\"$L4\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[\"$\",\"$L1c\",null,{}],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}],[\"$\",\"$L1d\",null,{}]]}]}]}]}]}]}]}]}],[\"$\",\"script\",null,{\"type\":\"application/json\",\"id\":\"server-client-data-experimentation\",\"children\":\"{\\\"status\\\":\\\"uninitialized\\\"}\"}]]}]]}]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"e:[[\"$\",\"meta\",\"0\",{\"charSet\":\"utf-8\"}],[\"$\",\"meta\",\"1\",{\"name\":\"viewport\",\"content\":\"width=device-width, initial-scale=1, interactive-widget=resizes-content\"}]]\na:null\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"1e:I[42712,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"HydrationBoundary\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"21:I[6367,[],\"IconMark\"]\n1f:T562f,"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"# Quantitative trait locus\n\nA **quantitative trait locus** (**QTL**) is a [locus](Locus_\\(genetics\\) \"Locus \\(genetics\\)\") (section of [DNA](DNA \"DNA\")) that correlates with variation of a [quantitative trait](Quantitative_trait \"Quantitative trait\") in the [phenotype](Phenotype \"Phenotype\") of a [population](Population_genetics \"Population genetics\") of [organisms](Organism \"Organism\").[1] QTLs are mapped by identifying which molecular markers (such as [SNPs](Single-nucleotide_polymorphism \"Single-nucleotide polymorphism\") or [AFLPs](Amplified_fragment_length_polymorphism \"Amplified fragment length polymorphism\")) correlate with an observed trait. This is often an early step in identifying the actual [genes](Gene \"Gene\") that cause the trait variation.\n# Quantitative trait locus\n\nA **quantitative trait locus** (**QTL**) is a region of [DNA](DNA \"DNA\") which is associated with a particular [phenotypic](Phenotype \"Phenotype\") [trait](Trait_\\(biological\\) \"Trait \\(biological\\)\"), which varies in degree and which can be attributed to [polygenic](Polygene \"Polygene\") effects, i.e., the product of two or more [genes](Gene \"Gene\"), and their environment.[2] These QTLs are often found on different [chromosomes](Chromosomes \"Chromosomes\"). The number of QTLs which explain variation in the phenotypic trait indicates the [genetic architecture](Genetic_architecture \"Genetic architecture\") of a trait. It may indicate that plant height is controlled by many genes of small effect, or by a few genes of large effect. \n\nTypically, QTLs underlie continuous [traits](Trait_\\(biological\\) \"Trait \\(biological\\)\") (those traits which vary continuously, e.g. height) as opposed to discrete traits (traits that have two or several character values, e.g. red hair in humans, a recessive trait, or smooth vs. wrinkled peas used by [Mendel](Gregor_Mendel \"Gregor Mendel\") in his experiments). \n\nMoreover, a single [phenotypic](Phenotype \"Phenotype\") trait is usually determined by many genes. Consequently, many QTLs are associated with a single trait. Another use of QTLs is to identify [candidate genes](Candidate_gene \"Candidate gene\") underlying a trait. The DNA sequence of any genes in this region can then be compared to a database of DNA for genes whose function is already known, this task being fundamental for marker-assisted crop improvement.[3]\n# History of Quantitative Trait Loci\n\n## History\n\n[Mendelian inheritance](Mendelian_inheritance \"Mendelian inheritance\") was rediscovered at the beginning of the 20th century. As [Mendel](Gregor_Mendel \"Gregor Mendel\")'s ideas spread, geneticists began to connect Mendel's rules of inheritance of single factors to [Darwinian evolution](Darwinian_evolution \"Darwinian evolution\"). For early geneticists, it was not immediately clear that the smooth variation in traits like body size (i.e., [incomplete dominance](Dominance_\\(genetics\\)#Incomplete_dominance \"Dominance \\(genetics\\)\")) was caused by the inheritance of single genetic factors. Although [Darwin](Charles_Darwin \"Charles Darwin\") himself observed that inbred features of fancy pigeons were inherited in accordance with Mendel's laws (although Darwin did not actually know about Mendel's ideas when he made the observation), it was not obvious that these features selected by fancy pigeon breeders can similarly explain quantitative variation in nature.[4]\n\nAn early attempt by [William Ernest Castle](William_Ernest_Castle \"William Ernest Castle\") to unify the laws of Mendelian inheritance with Darwin's theory of speciation invoked the idea that species become distinct from one another as one species or the other acquires a novel Mendelian factor.[5] Castle's conclusion was based on the observation that novel traits that could be studied in the lab and that show Mendelian inheritance patterns reflect a large deviation from the wild type, and Castle believed that acquisition of such features is the basis of \"discontinuous variation\" that characterizes speciation.[5] Darwin discussed the inheritance of similar mutant features but did not invoke them as a requirement of speciation.[4] Instead Darwin used the emergence of such features in breeding populations as evidence that mutation can occur at random within breeding populations, which is a central premise of his model of selection in nature.[4] Later in his career, Castle would refine his model for speciation to allow for small variation to contribute to speciation over time. He also was able to demonstrate this point by selectively breeding laboratory populations of rats to obtain a hooded phenotype over several generations.[](https://www.nap.edu/read/579/chapter/1)\n\nCastle's was perhaps the first attempt made in the scientific literature to direct evolution by artificial selection of a trait with continuous underlying variation, however the practice had previously been widely employed in the development of [agriculture](Agriculture \"Agriculture\") to obtain livestock or plants with favorable features from populations that show quantitative variation in traits like body size or grain yield. \n\nCastle's work was among the first to attempt to unify the recently rediscovered laws of Mendelian inheritance with Darwin's theory of evolution. Still, it would be almost thirty years until the theoretical framework for evolution of [complex traits](Complex_traits \"Complex traits\") would be widely formalized.[7] In an early summary of the theory of evolution of continuous variation, [Sewall Wright](Sewall_Wright \"Sewall Wright\"), a graduate student who trained under Castle, summarized contemporary thinking about the genetic basis of quantitative natural variation: \"As genetic studies continued, ever smaller differences were found to mendelize, and any character, sufficiently investigated, turned out to be affected by many factors.\"[7] Wright and others formalized population genetics theory that had been worked out over the preceding 30 years explaining how such traits can be inherited and create stably breeding populations with unique characteristics. Quantitative trait genetics today leverages Wright's observations about the statistical relationship between genotype and phenotype in families and populations to understand how certain genetic features can affect variation in natural and derived populations.\n\nThe specific concept of a quantitative trait locus (QTL)â€”a chromosomal region associated with variation in a quantitative traitâ€”developed in the mid-20th century. In 1923, Karl Sax published the first study demonstrating linkage between a Mendelian marker (seed coat pattern) and a quantitative trait (seed weight) in common bean (_Phaseolus vulgaris_), providing early evidence for QTLs.[](https://www.genetics.org/content/8/6/552) In 1961, J.M. Thoday advanced the field by developing methods to locate \"polygenes\" (now QTLs) using selected lines and phenotypic markers in fruit flies (_Drosophila melanogaster_).[](https://www.nature.com/articles/191368a0) The term \"quantitative trait locus\" was coined by Hubert Geldermann in 1975 to describe genomic regions influencing complex traits.[](https://link.springer.com/article/10.1007/BF02861451) The modern era of QTL mapping began in the late 1980s with the use of molecular markers; Lander and Botstein (1989) introduced a statistical framework using restriction fragment length polymorphism (RFLP) linkage maps to detect QTLs in experimental crosses.[17]\n# Quantitative traits\n\n**Polygenic inheritance** refers to inheritance of a [phenotypic](Phenotype \"Phenotype\") characteristic (trait) that is attributable to two or more [genes](Gene \"Gene\") and can be measured quantitatively. **Multifactorial inheritance** refers to polygenic inheritance that also includes interactions with the environment. Unlike [monogenic traits](Monogenic_trait \"Monogenic trait\"), polygenic traits do not follow patterns of [Mendelian inheritance](Mendelian_inheritance \"Mendelian inheritance\") (discrete categories). Instead, their phenotypes typically vary along a continuous gradient depicted by a [bell curve](Normal_distribution \"Normal distribution\").[8]\n\nAn example of a polygenic trait is [human skin color](Human_skin_color \"Human skin color\") variation. Several genes factor into determining a person's natural skin color, so modifying only one of those genes can change skin color slightly or in some cases, such as for [SLC24A5](SLC24A5 \"SLC24A5\"), moderately. Many disorders with [genetic components](Genetic_disorder \"Genetic disorder\") are polygenic, including [autism](Autism \"Autism\"), [cancer](Cancer \"Cancer\"), [diabetes](Diabetes \"Diabetes\") and numerous others. Most phenotypic characteristics are the result of the interaction of multiple genes. \n\n  \nMultifactorially inherited diseases are said to constitute the majority of genetic disorders affecting humans which will result in hospitalization or special care of some kind.[9][10]\n\n### Multifactorial traits in general\n\nTraits controlled both by the environment and by genetic factors are called multifactorial. Usually, multifactorial traits outside of illness result in what we see as **continuous characteristics** in organisms, especially human organisms such as: height,[9] skin color, and body mass.[11] All of these phenotypes are complicated by a great deal of give-and-take between genes and environmental effects.[9] The continuous distribution of traits such as height and skin color described above, reflects the action of genes that do not manifest typical patterns of dominance and recessiveness. Instead the contributions of each involved locus are thought to be additive. Writers have distinguished this kind of inheritance as _polygenic_ , or _quantitative inheritance_.[12]\n\nThus, due to the nature of polygenic traits, inheritance will not follow the same pattern as a simple [monohybrid](Monohybrid_cross \"Monohybrid cross\") or [dihybrid cross](Dihybrid_cross \"Dihybrid cross\").[10] Polygenic inheritance can be explained as Mendelian inheritance at many loci,[9] resulting in a trait which is [normally-distributed](Normal_distribution \"Normal distribution\"). If _n_ is the number of involved loci, then the coefficients of the [binomial expansion](Binomial_distribution \"Binomial distribution\") of \\( (a + b)^{2n} \\) will give the frequency of distribution of all _n_ allele [combinations](Combination \"Combination\"). For sufficiently high values of _n_ , this binomial distribution will begin to resemble a normal distribution. From this viewpoint, a disease state will become apparent at one of the tails of the distribution, past some threshold value. Disease states of increasing severity will be expected the further one goes past the threshold and away from the [mean](Statistical_mean \"Statistical mean\").[12]\n\n### Heritable disease and multifactorial inheritance\n\nWhen more than one gene is involved, with or without the presence of environmental triggers, we say that the disease is the result of multifactorial inheritance. Disease risk is often modeled using a liability threshold model, in which the cumulative effects of multiple genetic variants additively contribute to an underlying liability, and disease manifests when this liability, modulated by environmental factors, exceeds a certain threshold.[](https://basicmedicalkey.com/multifactorial-inheritance-and-common-diseases/)\n\nThe more genes involved in the cross, the more the distribution of the [genotypes](Genotype \"Genotype\") will resemble a [normal, or Gaussian](Normal_distribution \"Normal distribution\") distribution.[9] This shows that multifactorial inheritance is polygenic, and genetic frequencies can be predicted by way of a polyhybrid [Mendelian](Mendelian_inheritance \"Mendelian inheritance\") cross. Phenotypic frequencies are a different matter, especially if they are complicated by environmental factors. \n\nThe paradigm of polygenic inheritance as being used to define multifactorial disease has encountered much disagreement. Turnpenny (2004) discusses how simple polygenic inheritance cannot explain some diseases such as the onset of Type I diabetes mellitus, and that in cases such as these, not all genes are thought to make an equal contribution.[12]\n\nThe assumption of polygenic inheritance is that all involved loci make an equal contribution to the symptoms of the disease. This should result in a normal (Gaussian) distribution of genotypes. When it does not, the idea of polygenetic inheritance cannot be supported for that illness. \n\n### Examples\n\nThe above are well-known examples of diseases having both genetic and environmental components. Other examples involve atopic diseases such as [eczema](Atopic_eczema \"Atopic eczema\") or [dermatitis](Atopic_dermatitis \"Atopic dermatitis\"),[9] [spina bifida](Spina_bifida \"Spina bifida\") (open spine), and [anencephaly](Anencephaly \"Anencephaly\") (open skull).[13]\n\nSchizophrenia is a highly polygenic disorder with a strong genetic basis (heritability estimates of 60-80%), where over 200 risk loci have been identified through genome-wide association studies as of 2023, though these interact with environmental factors and no single variant is causative.[](https://www.nature.com/articles/s41380-023-02293-8)\n\nIf close relatives such as siblings are affected, this increases the likelihood of shared polygenic risk factors, elevating the individual's disease risk compared to the general population; the risk decreases as affected relatives become more distant, such as cousins, due to reduced shared genetic factors.[13] Inheritance in multifactorially-inherited diseases does not follow the same pattern as a simple [monohybrid](Monohybrid_cross \"Monohybrid cross\") or [dihybrid cross](Dihybrid_cross \"Dihybrid cross\").[10]\n\nIf a genetic cause is suspected and little else is known about the illness, then it remains to be seen exactly how many genes are involved in the phenotypic expression of the disease. Once that is determined, the question must be answered: if two people share similar genetic risk profiles, why are there differences in expression between them? Generally, what makes the two individuals different are likely to be environmental factors. Due to the involved nature of genetic investigations needed to determine such inheritance patterns, this is not usually the first avenue of investigation one would choose to determine etiology. Quantitative trait locus (QTL) mapping is a key method used to identify the genetic loci contributing to such complex traits.[1]\n# QTL mapping\n\nFor organisms whose genomes are known, one might now try to exclude genes in the identified region whose function is known with some certainty not to be connected with the trait in question. If the genome is not available, it may be an option to sequence the identified region and determine the putative functions of genes by their similarity to genes with known function, usually in other genomes. This can be done using [BLAST](BLAST_\\(biotechnology\\) \"BLAST \\(biotechnology\\)\"), an online tool that allows users to enter a primary sequence and search for similar sequences within the BLAST database of genes from various organisms. It is often not the actual gene underlying the phenotypic trait, but rather a region of DNA that is closely linked with the gene[14]\n\nAnother interest of statistical geneticists using QTL mapping is to determine the complexity of the genetic architecture underlying a phenotypic trait. For example, they may be interested in knowing whether a phenotype is shaped by many independent loci, or by a few loci, and do those loci interact. This can provide information on how the phenotype may be evolving.[15]\n\nSince the mid-2000s, classical QTL analyses have been combined with gene expression profiling, initially using DNA microarrays and now commonly RNA-seq, to identify expression QTLs (eQTLs) that describe [cis](Cis-acting \"Cis-acting\")- and [trans](Trans-acting \"Trans-acting\")-controlling elements for the expression of often disease-associated genes.[16] Such expression QTLs (eQTLs) describe [cis](Cis-acting \"Cis-acting\")\\- and [trans](Trans-acting \"Trans-acting\")-controlling elements for the expression of often disease-associated genes.\n\nIn recent years (as of 2025), QTL mapping has been enhanced by integration with genome-wide association studies (GWAS) and high-throughput sequencing, enabling finer resolution in diverse organisms, such as ultra-high-density bin maps for horticultural traits in pepper.[](https://www.pnas.org/doi/10.1073/pnas.2413764121)\n\n## Analysis of variance\n\nThe simplest method for QTL mapping is analysis of variance ([ANOVA](ANOVA \"ANOVA\"), sometimes called \"marker regression\") at the marker loci. In this method, in a backcross, one may calculate a [t-statistic](T-statistic \"T-statistic\") to compare the averages of the two marker [genotype](Genotype \"Genotype\") groups. For other types of crosses (such as the intercross), where there are more than two possible genotypes, one uses a more general form of ANOVA, which provides a so-called [F-statistic](F-statistics \"F-statistics\"). The ANOVA approach for QTL mapping has three important weaknesses. First, we do not receive separate estimates of QTL location and QTL effect. QTL location is indicated only by looking at which markers give the greatest differences between genotype group averages, and the apparent QTL effect at a marker will be smaller than the true QTL effect as a result of [recombination](Genetic_recombination \"Genetic recombination\") between the marker and the QTL. Second, we must discard individuals whose genotypes are missing at the marker. Third, when the markers are widely spaced, the QTL may be quite far from all markers, and so the power for QTL detection will decrease. \n\n## Interval mapping\n\nLander and Botstein developed interval mapping, which overcomes the three disadvantages of analysis of variance at marker loci.[17] Interval mapping is currently the most popular approach for QTL mapping in experimental crosses. The method makes use of a [genetic map](Genetic_map \"Genetic map\") of the typed markers, and, like analysis of variance, assumes the presence of a single QTL. In interval mapping, each locus is considered one at a time and the logarithm of the [odds ratio](Odds_ratio \"Odds ratio\") ([LOD score](LOD_score \"LOD score\")) is calculated for the model that the given locus is a true QTL. The odds ratio is related to the [Pearson correlation coefficient](Pearson_correlation_coefficient \"Pearson correlation coefficient\") between the phenotype and the marker genotype for each individual in the experimental cross.[18]\n\nThe term 'interval mapping' is used for estimating the position of a QTL within two markers (often indicated as 'marker-bracket'). Interval mapping is originally based on the maximum likelihood but there are also very good approximations possible with simple regression. \n\nThe principle for QTL mapping is: 1) The likelihood can be calculated for a given set of parameters (particularly QTL effect and QTL position) given the observed data on phenotypes and marker genotypes. 2) The estimates for the parameters are those where the likelihood is highest. 3) A significance threshold can be established by permutation testing.[19]\n\nConventional methods for the detection of quantitative trait loci (QTLs) are based on a comparison of single QTL models with a model assuming no QTL. For instance in the \"interval mapping\" method[20] the likelihood for a single putative QTL is assessed at each location on the genome. However, QTLs located elsewhere on the genome can have an interfering effect. As a consequence, the power of detection may be compromised, and the estimates of locations and effects of QTLs may be biased (Lander and Botstein 1989; Knapp 1991). Even nonexisting so-called \"ghost\" QTLs may appear (Haley and Knott 1992; Martinez and Curnow 1992). Therefore, multiple QTLs could be mapped more efficiently and more accurately by using multiple QTL models.[21]\n\n## Composite interval mapping (CIM)\n\nOne popular approach to handle QTL mapping where multiple QTL contribute to a trait is to iteratively scan the genome and add known QTL to the regression model as QTLs are identified. This method, termed composite interval mapping, determines both the location and effect size of QTL more accurately than single-QTL approaches, especially in small mapping populations where the effect of correlation between genotypes in the mapping population may be problematic. \n\nIn this method, one performs interval mapping using a subset of marker loci as covariates. These markers serve as proxies for other QTLs to increase the resolution of interval mapping, by accounting for linked QTLs and reducing the residual variation. The key problem with CIM concerns the choice of suitable marker loci to serve as covariates; once these have been chosen, CIM turns the model selection problem into a single-dimensional scan. The choice of marker covariates has not been solved, however. Not surprisingly, the appropriate markers are those closest to the true QTLs, and so if one could find these, the QTL mapping problem would be complete anyway. \n\n[Inclusive composite interval mapping](Inclusive_composite_interval_mapping \"Inclusive composite interval mapping\") (ICIM) has also been proposed as a potential method for QTL mapping.[22]\n\n## Family-pedigree based mapping\n\n[Family-based QTL mapping](Family-based_QTL_mapping \"Family-based QTL mapping\"), or Family-pedigree based mapping (Linkage and [association mapping](Association_mapping \"Association mapping\")), involves multiple families instead of a single family. Family-based QTL mapping has been the only way for mapping of genes where experimental crosses are difficult to make. However, due to some advantages, now plant geneticists are attempting to incorporate some of the methods pioneered in human genetics.[23] Using family-pedigree based approach has been discussed (Bink et al. 2008). Family-based linkage and association has been successfully implemented (Rosyara et al. 2007)[24]"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"8:[\"$\",\"$L1e\",null,{\"state\":{\"mutations\":[],\"queries\":[{\"dehydratedAt\":1761887212474,\"state\":{\"data\":{\"page\":{\"citations\":[{\"id\":\"1\",\"title\":\"** Miles, C; Wayne, M (2008). [\\\"Quantitative trait locus (QTL) analysis\\\"](http://www.nature.com/scitable/topicpage/quantitative-trait-locus-qtl-analysis-53904). _Nature Education_. **1** (1).\",\"description\":\"1. **^** Miles, C; Wayne, M (2008). [\\\"Quantitative trait locus (QTL) analysis\\\"](http://www.nature.com/scitable/topicpage/quantitative-trait-locus-qtl-analysis-53904). _Nature Education_. **1** (1).\",\"url\":\"http://www.nature.com/scitable/topicpage/quantitative-trait-locus-qtl-analysis-53904\",\"favicon\":\"\"},{\"id\":\"2\",\"title\":\"** Oduola Abiola; et al. (2003). [\\\"The nature and identification of quantitative trait loci: a community's view\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446). _[Nature Reviews Genetics](Nature_Reviews_Genetics \\\"Nature Reviews Genetics\\\")_. **4** (11). [Nature Portfolio](Nature_Portfolio \\\"Nature Portfolio\\\"): 911â€“916\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nrg1206](https://doi.org/10.1038%2Fnrg1206). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [2063446](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [14634638](https://pubmed.ncbi.nlm.nih.gov/14634638). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [27285742](https://api.semanticscholar.org/CorpusID:27285742). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [195367115](https://api.semanticscholar.org/CorpusID:195367115).\",\"description\":\"2. **^** Oduola Abiola; et al. (2003). [\\\"The nature and identification of quantitative trait loci: a community's view\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446). _[Nature Reviews Genetics](Nature_Reviews_Genetics \\\"Nature Reviews Genetics\\\")_. **4** (11). [Nature Portfolio](Nature_Portfolio \\\"Nature Portfolio\\\"): 911â€“916\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nrg1206](https://doi.org/10.1038%2Fnrg1206). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [2063446](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [14634638](https://pubmed.ncbi.nlm.nih.gov/14634638). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [27285742](https://api.semanticscholar.org/CorpusID:27285742). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [195367115](https://api.semanticscholar.org/CorpusID:195367115).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063446\",\"favicon\":\"\"},{\"id\":\"3\",\"title\":\"** Watanabe, Satoshi; Hideshima, Rumiko; Xia, Zhengjun; et al. (2009). [\\\"Map-Based Cloning of the Gene Associated With the Soybean Maturity Locus E3\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863). _Genetics_. **182** (4): 1251â€“1262\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1534/genetics.108.098772](https://doi.org/10.1534%2Fgenetics.108.098772). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [2728863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [19474204](https://pubmed.ncbi.nlm.nih.gov/19474204).\",\"description\":\"3. **^** Watanabe, Satoshi; Hideshima, Rumiko; Xia, Zhengjun; et al. (2009). [\\\"Map-Based Cloning of the Gene Associated With the Soybean Maturity Locus E3\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863). _Genetics_. **182** (4): 1251â€“1262\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1534/genetics.108.098772](https://doi.org/10.1534%2Fgenetics.108.098772). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [2728863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [19474204](https://pubmed.ncbi.nlm.nih.gov/19474204).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728863\",\"favicon\":\"\"},{\"id\":\"4\",\"title\":\"_**a**_ _**b**_ _**c**_ [\\\"Origin of Species\\\"](https://web.archive.org/web/20131003145126/http://www.literature.org/authors/darwin-charles/the-origin-of-species/). Archived from [the original](http://www.literature.org/authors/darwin-charles/the-origin-of-species/) on 3 October 2013. Retrieved 24 September 2013.\",\"description\":\"4. ^ _**a**_ _**b**_ _**c**_ [\\\"Origin of Species\\\"](https://web.archive.org/web/20131003145126/http://www.literature.org/authors/darwin-charles/the-origin-of-species/). Archived from [the original](http://www.literature.org/authors/darwin-charles/the-origin-of-species/) on 3 October 2013. Retrieved 24 September 2013.\",\"url\":\"https://web.archive.org/web/20131003145126/http://www.literature.org/authors/darwin-charles/the-origin-of-species/\",\"favicon\":\"\"},{\"id\":\"5\",\"title\":\"_**a**_ _**b**_ Castle WE (1903). [\\\"Mendel's Law of Heredity\\\"](https://zenodo.org/record/1447900). _Science_. **18** (456): 396â€“406\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1903Sci....18..396C](https://ui.adsabs.harvard.edu/abs/1903Sci....18..396C). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.18.456.396](https://doi.org/10.1126%2Fscience.18.456.396). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [17752783](https://pubmed.ncbi.nlm.nih.gov/17752783). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [11670642](https://api.semanticscholar.org/CorpusID:11670642).\",\"description\":\"5. ^ _**a**_ _**b**_ Castle WE (1903). [\\\"Mendel's Law of Heredity\\\"](https://zenodo.org/record/1447900). _Science_. **18** (456): 396â€“406\\\\. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[1903Sci....18..396C](https://ui.adsabs.harvard.edu/abs/1903Sci....18..396C). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1126/science.18.456.396](https://doi.org/10.1126%2Fscience.18.456.396). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [17752783](https://pubmed.ncbi.nlm.nih.gov/17752783). [S2CID](S2CID_\\\\(identifier\\\\) \\\"S2CID \\\\(identifier\\\\)\\\") [11670642](https://api.semanticscholar.org/CorpusID:11670642).\",\"url\":\"https://zenodo.org/record/1447900\",\"favicon\":\"\"},{\"id\":\"6\",\"title\":\"** Castle, W. E. (1 May 1951). [\\\"Variation in the Hooded Pattern of Rats, and a New Allele of Hooded\\\"](http://www.genetics.org/content/36/3/254). _Genetics_. **36** (3): 254â€“266\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/36.3.254](https://doi.org/10.1093%2Fgenetics%2F36.3.254). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1209518](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209518). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [14840647](https://pubmed.ncbi.nlm.nih.gov/14840647) â€“ via www.genetics.org.\",\"description\":\"6. **^** Castle, W. E. (1 May 1951). [\\\"Variation in the Hooded Pattern of Rats, and a New Allele of Hooded\\\"](http://www.genetics.org/content/36/3/254). _Genetics_. **36** (3): 254â€“266\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/36.3.254](https://doi.org/10.1093%2Fgenetics%2F36.3.254). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1209518](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209518). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [14840647](https://pubmed.ncbi.nlm.nih.gov/14840647) â€“ via www.genetics.org.\",\"url\":\"http://www.genetics.org/content/36/3/254\",\"favicon\":\"\"},{\"id\":\"7\",\"title\":\"_**a**_ _**b**_ Wright, Sewall (1 March 1931). [\\\"Evolution in Mendelian Populations\\\"](http://www.genetics.org/content/16/2/97). _Genetics_. **16** (2): 97â€“159\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/16.2.97](https://doi.org/10.1093%2Fgenetics%2F16.2.97). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1201091](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1201091). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [17246615](https://pubmed.ncbi.nlm.nih.gov/17246615) â€“ via www.genetics.org.\",\"description\":\"7. ^ _**a**_ _**b**_ Wright, Sewall (1 March 1931). [\\\"Evolution in Mendelian Populations\\\"](http://www.genetics.org/content/16/2/97). _Genetics_. **16** (2): 97â€“159\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/16.2.97](https://doi.org/10.1093%2Fgenetics%2F16.2.97). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1201091](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1201091). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [17246615](https://pubmed.ncbi.nlm.nih.gov/17246615) â€“ via www.genetics.org.\",\"url\":\"http://www.genetics.org/content/16/2/97\",\"favicon\":\"\"},{\"id\":\"8\",\"title\":\"** Ricki Lewis (2003), [_Multifactorial Traits_](http://highered.mcgraw-hill.com/sites/007246268x/student_view0/chapter7/), McGraw-Hill Higher Education.\",\"description\":\"8. **^** Ricki Lewis (2003), [_Multifactorial Traits_](http://highered.mcgraw-hill.com/sites/007246268x/student_view0/chapter7/), McGraw-Hill Higher Education.\",\"url\":\"http://highered.mcgraw-hill.com/sites/007246268x/student_view0/chapter7/\",\"favicon\":\"\"},{\"id\":\"9\",\"title\":\"_**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ Tissot, Robert. [\\\"Human Genetics for 1st Year Students: Multifactorial Inheritance\\\"](http://www.uic.edu/classes/bms/bms655/lesson11.html). Retrieved 6 January 2007.\",\"description\":\"9. ^ _**a**_ _**b**_ _**c**_ _**d**_ _**e**_ _**f**_ Tissot, Robert. [\\\"Human Genetics for 1st Year Students: Multifactorial Inheritance\\\"](http://www.uic.edu/classes/bms/bms655/lesson11.html). Retrieved 6 January 2007.\",\"url\":\"http://www.uic.edu/classes/bms/bms655/lesson11.html\",\"favicon\":\"\"},{\"id\":\"10\",\"title\":\"_**a**_ _**b**_ _**c**_ Birth Defects Genetics Centre, University of South Dakota School of Medicine. [\\\"Multifactorial Inheritance\\\"](https://web.archive.org/web/20061230084542/http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm). _Clinical Genetics: A Self-Study Guide for Health Care Providers_. University of South Dakota School of Medicine. Archived from [the original](http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm) on 30 December 2006. Retrieved 6 January 2007.\",\"description\":\"10. ^ _**a**_ _**b**_ _**c**_ Birth Defects Genetics Centre, University of South Dakota School of Medicine. [\\\"Multifactorial Inheritance\\\"](https://web.archive.org/web/20061230084542/http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm). _Clinical Genetics: A Self-Study Guide for Health Care Providers_. University of South Dakota School of Medicine. Archived from [the original](http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm) on 30 December 2006. Retrieved 6 January 2007.\",\"url\":\"https://web.archive.org/web/20061230084542/http://www.usd.edu/med/som/genetics/curriculum/1GMULTI5.htm\",\"favicon\":\"\"},{\"id\":\"11\",\"title\":\"** [\\\"Definition of Multifactorial inheritance\\\"](https://web.archive.org/web/20131217193339/http://www.medterms.com/script/main/art.asp?articlekey=4453). _MedicineNet.com MedTerms Dictionary_. MedicineNet.com. Archived from [the original](http://www.medterms.com/script/main/art.asp?articlekey=4453) on 17 December 2013. Retrieved 6 January 2007.\",\"description\":\"11. **^** [\\\"Definition of Multifactorial inheritance\\\"](https://web.archive.org/web/20131217193339/http://www.medterms.com/script/main/art.asp?articlekey=4453). _MedicineNet.com MedTerms Dictionary_. MedicineNet.com. Archived from [the original](http://www.medterms.com/script/main/art.asp?articlekey=4453) on 17 December 2013. Retrieved 6 January 2007.\",\"url\":\"https://web.archive.org/web/20131217193339/http://www.medterms.com/script/main/art.asp?articlekey=4453\",\"favicon\":\"\"},{\"id\":\"12\",\"title\":\"_**a**_ _**b**_ _**c**_ Turnpenny, Peter (2004). [\\\"Chapter 9\\\"](http://www.fleshandbones.com/readingroom/viewchapter.cfm?ID=1041) (PDF). _Emery's Elements of Medical Genetics_ (12th ed.). Elsevier. Retrieved 6 January 2007.\",\"description\":\"12. ^ _**a**_ _**b**_ _**c**_ Turnpenny, Peter (2004). [\\\"Chapter 9\\\"](http://www.fleshandbones.com/readingroom/viewchapter.cfm?ID=1041) (PDF). _Emery's Elements of Medical Genetics_ (12th ed.). Elsevier. Retrieved 6 January 2007.\",\"url\":\"http://www.fleshandbones.com/readingroom/viewchapter.cfm?ID=1041\",\"favicon\":\"\"},{\"id\":\"13\",\"title\":\"_**a**_ _**b**_ Proud, Virginia \u0026 Roberts, Helen (31 December 2005). [\\\"Medical Genetics: Multifactorial Inheritance\\\"](https://web.archive.org/web/20061015185017/http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134). Children's Hospital of the King's Daughters. Archived from [the original](http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134) on 15 October 2006. Retrieved 6 January 2007.\",\"description\":\"13. ^ _**a**_ _**b**_ Proud, Virginia \u0026 Roberts, Helen (31 December 2005). [\\\"Medical Genetics: Multifactorial Inheritance\\\"](https://web.archive.org/web/20061015185017/http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134). Children's Hospital of the King's Daughters. Archived from [the original](http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134) on 15 October 2006. Retrieved 6 January 2007.\",\"url\":\"https://web.archive.org/web/20061015185017/http://www.chkd.org/HealthLibrary/Content.aspx?pageid=P02134\",\"favicon\":\"\"},{\"id\":\"14\",\"title\":\"** [\\\"BLAST: Basic Local Alignment Search Tool\\\"](https://blast.ncbi.nlm.nih.gov/Blast.cgi). _blast.ncbi.nlm.nih.gov_. Retrieved 18 February 2018.\",\"description\":\"14. **^** [\\\"BLAST: Basic Local Alignment Search Tool\\\"](https://blast.ncbi.nlm.nih.gov/Blast.cgi). _blast.ncbi.nlm.nih.gov_. Retrieved 18 February 2018.\",\"url\":\"https://blast.ncbi.nlm.nih.gov/Blast.cgi\",\"favicon\":\"\"},{\"id\":\"15\",\"title\":\"** Grisel, Judith E.; Crabbe, John C. (1995). [\\\"Quantitative Trait Loci Mapping\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759). _Alcohol Health and Research World_. **19** (3): 220â€“227\\\\. [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0090-838X](https://search.worldcat.org/issn/0090-838X). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [6875759](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [31798043](https://pubmed.ncbi.nlm.nih.gov/31798043).\",\"description\":\"15. **^** Grisel, Judith E.; Crabbe, John C. (1995). [\\\"Quantitative Trait Loci Mapping\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759). _Alcohol Health and Research World_. **19** (3): 220â€“227\\\\. [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [0090-838X](https://search.worldcat.org/issn/0090-838X). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [6875759](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [31798043](https://pubmed.ncbi.nlm.nih.gov/31798043).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6875759\",\"favicon\":\"\"},{\"id\":\"16\",\"title\":\"** Westra HJ, et al. (2013). [\\\"Systematic identification of trans eQTLs as putative drivers of known disease associations\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562). _Nat Genet_. **45** (10): 1238â€“1243\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/ng.2756](https://doi.org/10.1038%2Fng.2756). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3991562](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [24013639](https://pubmed.ncbi.nlm.nih.gov/24013639).\",\"description\":\"16. **^** Westra HJ, et al. (2013). [\\\"Systematic identification of trans eQTLs as putative drivers of known disease associations\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562). _Nat Genet_. **45** (10): 1238â€“1243\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/ng.2756](https://doi.org/10.1038%2Fng.2756). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [3991562](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [24013639](https://pubmed.ncbi.nlm.nih.gov/24013639).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3991562\",\"favicon\":\"\"},{\"id\":\"17\",\"title\":\"** Lander, E.S.; Botstein, D. (1989). [\\\"Mapping mendelian factors underlying quantitative traits using RFLP linkage maps\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601). _Genetics_. **121** (1): 185â€“199\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/121.1.185](https://doi.org/10.1093%2Fgenetics%2F121.1.185). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1203601](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [2563713](https://pubmed.ncbi.nlm.nih.gov/2563713).\",\"description\":\"17. **^** Lander, E.S.; Botstein, D. (1989). [\\\"Mapping mendelian factors underlying quantitative traits using RFLP linkage maps\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601). _Genetics_. **121** (1): 185â€“199\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/121.1.185](https://doi.org/10.1093%2Fgenetics%2F121.1.185). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1203601](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [2563713](https://pubmed.ncbi.nlm.nih.gov/2563713).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203601\",\"favicon\":\"\"},{\"id\":\"18\",\"title\":\"** Lynch, M. \u0026 Walsh, B. Genetics and Analysis of Quantitative Traits edn 1 (Sinauer Associates, 1998).\",\"description\":\"18. **^** Lynch, M. \u0026 Walsh, B. Genetics and Analysis of Quantitative Traits edn 1 (Sinauer Associates, 1998).\",\"url\":\"\",\"favicon\":\"\"},{\"id\":\"19\",\"title\":\"** Bloom J. S.; Ehrenreich I. M.; Loo W. T.; Lite T.-L. V.; Kruglyak L. (2013). [\\\"Finding the sources of missing heritability in a yeast cross\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867). _Nature_. **494** (7436): 234â€“237\\\\. [arXiv](ArXiv_\\\\(identifier\\\\) \\\"ArXiv \\\\(identifier\\\\)\\\"):[1208.2865](https://arxiv.org/abs/1208.2865). [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2013Natur.494..234B](https://ui.adsabs.harvard.edu/abs/2013Natur.494..234B). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nature11867](https://doi.org/10.1038%2Fnature11867). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [4001867](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23376951](https://pubmed.ncbi.nlm.nih.gov/23376951).\",\"description\":\"19. **^** Bloom J. S.; Ehrenreich I. M.; Loo W. T.; Lite T.-L. V.; Kruglyak L. (2013). [\\\"Finding the sources of missing heritability in a yeast cross\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867). _Nature_. **494** (7436): 234â€“237\\\\. [arXiv](ArXiv_\\\\(identifier\\\\) \\\"ArXiv \\\\(identifier\\\\)\\\"):[1208.2865](https://arxiv.org/abs/1208.2865). [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2013Natur.494..234B](https://ui.adsabs.harvard.edu/abs/2013Natur.494..234B). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1038/nature11867](https://doi.org/10.1038%2Fnature11867). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [4001867](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [23376951](https://pubmed.ncbi.nlm.nih.gov/23376951).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001867\",\"favicon\":\"\"},{\"id\":\"20\",\"title\":\"** Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps. ES Lander and D Botstein. Genetics. 1989\",\"description\":\"20. **^** Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps. ES Lander and D Botstein. Genetics. 1989\",\"url\":\"\",\"favicon\":\"\"},{\"id\":\"21\",\"title\":\"** Jansen, R C (1 September 1993). [\\\"Interval mapping of multiple quantitative trait loci\\\"](http://www.genetics.org/content/135/1/205.full.pdf) (PDF). _Genetics_. **135** (1): 205â€“211\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/135.1.205](https://doi.org/10.1093%2Fgenetics%2F135.1.205). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1205619](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205619). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8224820](https://pubmed.ncbi.nlm.nih.gov/8224820). Retrieved 1 March 2023.\",\"description\":\"21. **^** Jansen, R C (1 September 1993). [\\\"Interval mapping of multiple quantitative trait loci\\\"](http://www.genetics.org/content/135/1/205.full.pdf) (PDF). _Genetics_. **135** (1): 205â€“211\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1093/genetics/135.1.205](https://doi.org/10.1093%2Fgenetics%2F135.1.205). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [1205619](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205619). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [8224820](https://pubmed.ncbi.nlm.nih.gov/8224820). Retrieved 1 March 2023.\",\"url\":\"http://www.genetics.org/content/135/1/205.full.pdf\",\"favicon\":\"\"},{\"id\":\"22\",\"title\":\"** Li, Shanshan; Wang, Jiankang; Zhang, Luyan (10 July 2015). [\\\"Inclusive Composite Interval Mapping of QTL by Environment Interactions in Biparental Populations\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613). _PLOS ONE_. **10** (7): e0132414. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2015PLoSO..1032414L](https://ui.adsabs.harvard.edu/abs/2015PLoSO..1032414L). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.pone.0132414](https://doi.org/10.1371%2Fjournal.pone.0132414). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1932-6203](https://search.worldcat.org/issn/1932-6203). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [4498613](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [26161656](https://pubmed.ncbi.nlm.nih.gov/26161656).\",\"description\":\"22. **^** Li, Shanshan; Wang, Jiankang; Zhang, Luyan (10 July 2015). [\\\"Inclusive Composite Interval Mapping of QTL by Environment Interactions in Biparental Populations\\\"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613). _PLOS ONE_. **10** (7): e0132414. [Bibcode](Bibcode_\\\\(identifier\\\\) \\\"Bibcode \\\\(identifier\\\\)\\\"):[2015PLoSO..1032414L](https://ui.adsabs.harvard.edu/abs/2015PLoSO..1032414L). [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1371/journal.pone.0132414](https://doi.org/10.1371%2Fjournal.pone.0132414). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1932-6203](https://search.worldcat.org/issn/1932-6203). [PMC](PMC_\\\\(identifier\\\\) \\\"PMC \\\\(identifier\\\\)\\\") [4498613](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [26161656](https://pubmed.ncbi.nlm.nih.gov/26161656).\",\"url\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498613\",\"favicon\":\"\"},{\"id\":\"23\",\"title\":\"** Jannink, J; Bink, Mc; Jansen, Rc (August 2001). \\\"Using complex plant pedigrees to map valuable genes\\\". _Trends in Plant Science_. **6** (8): 337â€“42\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S1360-1385(01)02017-9](https://doi.org/10.1016%2FS1360-1385%2801%2902017-9). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1360-1385](https://search.worldcat.org/issn/1360-1385). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11495765](https://pubmed.ncbi.nlm.nih.gov/11495765).\",\"description\":\"23. **^** Jannink, J; Bink, Mc; Jansen, Rc (August 2001). \\\"Using complex plant pedigrees to map valuable genes\\\". _Trends in Plant Science_. **6** (8): 337â€“42\\\\. [doi](Doi_\\\\(identifier\\\\) \\\"Doi \\\\(identifier\\\\)\\\"):[10.1016/S1360-1385(01)02017-9](https://doi.org/10.1016%2FS1360-1385%2801%2902017-9). [ISSN](ISSN_\\\\(identifier\\\\) \\\"ISSN \\\\(identifier\\\\)\\\") [1360-1385](https://search.worldcat.org/issn/1360-1385). [PMID](PMID_\\\\(identifier\\\\) \\\"PMID \\\\(identifier\\\\)\\\") [11495765](https://pubmed.ncbi.nlm.nih.gov/11495765).\",\"url\":\"Doi_\\\\(identifier\\\\\",\"favicon\":\"\"},{\"id\":\"24\",\"title\":\"** Rosyara, U. R.; Maxson-stein, K.L.; Glover, K.D.; Stein, J.M.; Gonzalez-hernandez, J.L. (2007). \\\"Family-based mapping of FHB resistance QTLs in hexaploid wheat\\\". _Proceedings of National Fusarium Head Blight Forum_.\",\"description\":\"24. **^** Rosyara, U. R.; Maxson-stein, K.L.; Glover, K.D.; Stein, J.M.; Gonzalez-hernandez, J.L. (2007). \\\"Family-based mapping of FHB resistance QTLs in hexaploid wheat\\\". _Proceedings of National Fusarium Head Blight Forum_.\",\"url\":\"\",\"favicon\":\"\"}],\"images\":[],\"fixedIssues\":[{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"He also was able to demonstrate this point by selectively breeding laboratory populations of rats to obtain a hooded phenotype over several generations.[6]\",\"explanation\":\"The cited reference [6] is a 1951 paper by Castle on variation in hooded patterns and a new allele, but it does not describe the selective breeding experiments. These experiments, demonstrating the role of small variations in evolution, were conducted in the early 1910s with collaborator J.C. Phillips and published in papers from 1911â€“1915.\",\"fixedText\":\"Remove [6] and replace with a citation to a source describing Castle's early rat selection experiments, such as his biographical memoir or a 1911 paper.\",\"evidenceSource\":\"[](https://www.nap.edu/read/579/chapter/1) [National Academy of Sciences Biographical Memoir of William Ernest Castle]\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"[The section concludes with:] Quantitative trait genetics today leverages Wright's observations about the statistical relationship between genotype and phenotype in families and populations to understand how certain genetic features can affect variation in natural and derived populations.\",\"explanation\":\"The section provides historical context for quantitative genetics up to 1931 but omits key milestones specific to the QTL concept and mapping methods, creating a significant gap in the historical overview for an article on quantitative trait loci. Critical developments include Sax's 1923 association study, Thoday's 1961 polygene location method, Geldermann's 1975 introduction of the QTL term, and Lander \u0026 Botstein's 1989 molecular mapping framework.\",\"fixedText\":\"Add 1-2 paragraphs at the end detailing these QTL-specific historical milestones with citations to authoritative sources.\",\"evidenceSource\":\"[17]; [](https://www.nature.com/scitable/topicpage/quantitative-trait-locus-qtl-analysis-53904/)\"},{\"criticality\":\"Critical\",\"issueType\":\"Mechanism/Specification Error\",\"problematicText\":\"\\\"A mutation resulting in a disease state is often recessive, so both alleles must be mutant in order for the disease to be expressed phenotypically. A disease or syndrome may also be the result of the expression of mutant alleles at more than one locus.\\\"\",\"explanation\":\"In the context of multifactorial inheritance, disease expression typically follows a polygenic liability threshold model involving additive effects from many common genetic variants (often SNPs) across multiple loci, combined with environmental factors, rather than rare recessive mutations requiring homozygosity or expression at specific loci.\",\"fixedText\":\"Rephrase to explain the liability threshold model with additive polygenic and environmental contributions exceeding a threshold for disease manifestation.\",\"evidenceSource\":\"[12]; [](https://basicmedicalkey.com/multifactorial-inheritance-and-common-diseases/)\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"If multifactorial inheritance is indeed the case, then the chance of the patient contracting the disease is reduced only if cousins and more distant relatives have the disease.[13] While multifactorially-inherited diseases tend to run in families, inheritance will not follow the same pattern as a simple [monohybrid](Monohybrid_cross \\\"Monohybrid cross\\\") or [dihybrid cross](Dihybrid_cross \\\"Dihybrid cross\\\").[10] If a genetic cause is suspected and little else is known about the illness, then it remains to be seen exactly how many genes are involved in the phenotypic expression of the disease. Once that is determined, the question must be answered: if two people have the required genes, why are there differences in expression between them? Generally, what makes the two individuals different are likely to be environmental factors.\\\"\",\"explanation\":\"The cited source [13] describes higher risk for closer relatives and decreasing risk with distance, but the text misphrases the relationship, incorrectly using \\\"only if\\\" to suggest causation rather than correlation with relatedness, and conflates polygenic risk with Mendelian \\\"carrier\\\" status, which does not apply.\",\"fixedText\":\"Clarify that risk decreases with the degree of relatedness (higher for siblings, lower for cousins); remove \\\"genetic carrier\\\" language as it implies Mendelian inheritance; emphasize environmental modulation of shared polygenic risk.\",\"evidenceSource\":\"[13]; [](https://www.urmc.rochester.edu/encyclopedia/content?contenttypeid=90\u0026contentid=p02134)\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"While [schizophrenia](Schizophrenia \\\"Schizophrenia\\\") is widely believed to be multifactorially genetic by [biopsychiatrists](Biopsychiatry \\\"Biopsychiatry\\\"), no characteristic genetic markers have been determined with any certainty.\\\"\",\"explanation\":\"This claim is outdated; as of 2023, genome-wide association studies (GWAS) have identified over 200 genetic loci associated with schizophrenia risk, confirming its highly polygenic nature with heritability of 60-80%, though no single variant is deterministic.\",\"fixedText\":\"Update to: Schizophrenia is a highly polygenic disorder with strong genetic contributions (heritability 60-80%), where hundreds of risk loci have been identified through GWAS as of 2023, interacting with environmental factors.\",\"evidenceSource\":\"[](https://www.nature.com/articles/s41380-023-02293-8)\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"\",\"explanation\":\"The section discusses polygenic/multifactorial traits but lacks mention of how QTL mapping identifies contributing loci, which is central to the article's topic; this creates a knowledge gap in connecting traits to QTL analysis covered later.\",\"fixedText\":\"Add a brief neutral statement linking quantitative traits to QTL detection without duplicating the QTL mapping section.\",\"evidenceSource\":\"[1]; [](https://pubmed.ncbi.nlm.nih.gov/40348420/)\"},{\"criticality\":\"Critical\",\"issueType\":\"Timeliness/Currency\",\"problematicText\":\"\\\"In a recent development, classical QTL analyses were combined with gene expression profiling i.e. by DNA microarrays.\\\"\",\"explanation\":\"The concept of expression QTLs (eQTLs) was first published around 2006 (e.g., Rockman and Kruglyak, 2006), making \\\"recent development\\\" inaccurate as of 2025. Additionally, DNA microarrays are largely superseded by RNA-seq for gene expression profiling in modern studies.\",\"fixedText\":\"Rephrase to \\\"Since the mid-2000s, classical QTL analyses have been combined with gene expression profiling, initially using DNA microarrays and now commonly RNA-seq, to identify expression QTLs (eQTLs) that describe cis- and trans-acting elements regulating the expression of often disease-associated genes.\\\"\",\"evidenceSource\":\"[](https://pubmed.ncbi.nlm.nih.gov/17038575/) (Rockman MV, Kruglyak L. Genetics of global gene expression. Nat Rev Genet. 2006)\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"Observed epistatic effects have been found beneficial to identify the gene responsible by a cross-validation of genes within the interacting loci with metabolic pathway- and scientific literature databases.\\\"\",\"explanation\":\"This claim about the benefits of observed epistatic effects in gene identification lacks any citation and appears unsupported in the provided references. Unsourced factual claims violate verifiability.\",\"fixedText\":\"Remove the unsourced sentence to ensure verifiability.\",\"evidenceSource\":\"No supporting ref\"},{\"criticality\":\"Critical\",\"issueType\":\"Source/Citation Mismatch\",\"problematicText\":\"\\\"Family-based linkage and association has been successfully implemented (Rosyara et al. 2009)[24]\\\"\",\"explanation\":\"Citation [24] refers to Rosyara et al. (2007), not 2009. The 2009 paper is mentioned in additional notes but not formally cited as [24], creating a mismatch between text and reference.\",\"fixedText\":\"Correct to \\\"Rosyara et al. (2007)[24]\\\" or add proper citation for the 2009 paper if intended.\",\"evidenceSource\":\"[24] (Rosyara et al. 2007)\"},{\"criticality\":\"Critical\",\"issueType\":\"Internal Contradiction\",\"problematicText\":\"(Under Interval mapping subsection) \\\"Conventional methods for the detection of quantitative trait loci (QTLs) are based on a comparison of single QTL models with a model assuming no QTL. ... This method, termed composite interval mapping determine both the location and effects size of QTL more accurately than single-QTL approaches...\\\"\",\"explanation\":\"This paragraph discusses limitations of single-QTL methods and introduces composite interval mapping (CIM), but it is placed under the \\\"Interval mapping\\\" subsection, leading to duplication with the later \\\"Composite interval mapping (CIM)\\\" subsection. This creates structural confusion and redundant content.\",\"fixedText\":\"Move the misplaced paragraph to the start of the \\\"Composite interval mapping (CIM)\\\" subsection and remove duplication. Correct grammar: \\\"determines\\\" instead of \\\"determine\\\".\",\"evidenceSource\":\"Article structure and section subsections\"},{\"criticality\":\"Critical\",\"issueType\":\"Missing Information\",\"problematicText\":\"N/A\",\"explanation\":\"The section focuses on classical QTL mapping methods but lacks mention of modern integrations, such as with genome-wide association studies (GWAS), high-density genotyping, or multi-omics approaches (e.g., integrating eQTL with proteomics), which have advanced QTL detection since 2020. Recent examples include ultra-high-density bin maps for precise QTL mapping in crops (as of 2024).\",\"fixedText\":\"Add a brief paragraph or sentence in the introduction or a new subsection on modern developments, e.g., \\\"In recent years (as of 2025), QTL mapping has been enhanced by integration with GWAS and high-throughput sequencing, enabling finer resolution in diverse organisms.\\\"\",\"evidenceSource\":\"[](https://www.pnas.org/doi/10.1073/pnas.2413764121) (Han et al., 2024, PNAS)\"}],\"slug\":\"Quantitative_trait_locus\",\"title\":\"Quantitative trait locus\",\"content\":\"$1f\",\"description\":\"Quantitative trait locus\\n\\nA quantitative trait locus (QTL) is a locus \\\"Locus \\\\(genetics\\\\)\\\") (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of...\",\"metadata\":{\"categories\":[\"QTL\"],\"lastModified\":\"1761552626\",\"contentLength\":\"22063\",\"version\":\"1.0\",\"lastEditor\":\"system\",\"language\":\"en\",\"isRedirect\":false,\"redirectTarget\":\"\",\"isWithheld\":false},\"stats\":{\"totalViews\":\"44333\",\"recentViews\":\"44333\",\"dailyAvgViews\":1477.7667236328125,\"qualityScore\":1,\"lastViewed\":\"1761887212\"},\"linkedPages\":null},\"found\":true},\"dataUpdateCount\":1,\"dataUpdatedAt\":1761887212472,\"error\":null,\"errorUpdateCount\":0,\"errorUpdatedAt\":0,\"fetchFailureCount\":0,\"fetchFailureReason\":null,\"fetchMeta\":null,\"isInvalidated\":false,\"status\":\"success\",\"fetchStatus\":\"idle\"},\"queryKey\":[\"page\",\"Quantitative_trait_locus\"],\"queryHash\":\"[\\\"page\\\",\\\"Quantitative_trait_locus\\\"]\"}]},\"children\":\"$L20\"}]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"c:{\"metadata\":[[\"$\",\"title\",\"0\",{\"children\":\"Quantitative trait locus\"}],[\"$\",\"meta\",\"1\",{\"name\":\"description\",\"content\":\"A quantitative trait locus (QTL) is a locus (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of organisms.[1] QTLs are mapped by identifying which molecular markers (such as SNPs or AFLPs) correlate with an observed trait. This is often an early step in identifying the actual genes that cause the trait variation. A quantitative trait locus (QTL) is a region of DNA which is associated with a particular phenotypic trait, which varies in...\"}],[\"$\",\"meta\",\"2\",{\"name\":\"author\",\"content\":\"system\"}],[\"$\",\"link\",\"3\",{\"rel\":\"manifest\",\"href\":\"/manifest.webmanifest\",\"crossOrigin\":\"$undefined\"}],[\"$\",\"meta\",\"4\",{\"name\":\"keywords\",\"content\":\"QTL\"}],[\"$\",\"meta\",\"5\",{\"name\":\"robots\",\"content\":\"index, follow\"}],[\"$\",\"meta\",\"6\",{\"name\":\"googlebot\",\"content\":\"index, follow, max-snippet:-1\"}],[\"$\",\"link\",\"7\",{\"rel\":\"canonical\",\"href\":\"https://grokipedia.com/page/Quantitative_trait_locus\"}],[\"$\",\"meta\",\"8\",{\"property\":\"og:title\",\"content\":\"Quantitative trait locus\"}],[\"$\",\"meta\",\"9\",{\"property\":\"og:description\",\"content\":\"A quantitative trait locus (QTL) is a locus (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of organisms.[1] QTLs are mapped by identifying which molecular markers (such as SNPs or AFLPs) correlate with an observed trait. This is often an early step in identifying the actual genes that cause the trait variation. A quantitative trait locus (QTL) is a region of DNA which is associated with a particular phenotypic trait, which varies in...\"}],[\"$\",\"meta\",\"10\",{\"property\":\"og:url\",\"content\":\"https://grokipedia.com/page/Quantitative_trait_locus\"}],[\"$\",\"meta\",\"11\",{\"property\":\"og:site_name\",\"content\":\"Grokipedia\"}],[\"$\",\"meta\",\"12\",{\"property\":\"og:locale\",\"content\":\"en\"}],[\"$\",\"meta\",\"13\",{\"property\":\"og:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"meta\",\"14\",{\"property\":\"og:image:width\",\"content\":\"512\"}],[\"$\",\"meta\",\"15\",{\"property\":\"og:image:height\",\"content\":\"512\"}],[\"$\",\"meta\",\"16\",{\"property\":\"og:image:alt\",\"content\":\"Quantitative trait locus\"}],[\"$\",\"meta\",\"17\",{\"property\":\"og:type\",\"content\":\"article\"}],[\"$\",\"meta\",\"18\",{\"property\":\"article:modified_time\",\"content\":\"1970-01-21T09:19:12.626Z\"}],[\"$\",\"meta\",\"19\",{\"name\":\"twitter:card\",\"content\":\"summary\"}],[\"$\",\"meta\",\"20\",{\"name\":\"twitter:creator\",\"content\":\"@Grokipedia\"}],[\"$\",\"meta\",\"21\",{\"name\":\"twitter:title\",\"content\":\"Quantitative trait locus\"}],[\"$\",\"meta\",\"22\",{\"name\":\"twitter:description\",\"content\":\"A quantitative trait locus (QTL) is a locus (section of DNA) that correlates with variation of a quantitative trait in the phenotype of a population of organisms.[1] QTLs are mapped by identifying which molecular markers (such as SNPs or AFLPs) correlate with an observed trait. This is often an early step in identifying the actual genes that cause the trait variation. A quantitative trait locus (QTL) is a region of DNA which is associated with a particular phenotypic trait, which varies in...\"}],[\"$\",\"meta\",\"23\",{\"name\":\"twitter:image\",\"content\":\"https://grokipedia.com/icon-512x512.png\"}],[\"$\",\"link\",\"24\",{\"rel\":\"icon\",\"href\":\"/favicon.ico\",\"type\":\"image/x-icon\",\"sizes\":\"16x16\"}],[\"$\",\"$L21\",\"25\",{}]],\"error\":null,\"digest\":\"$undefined\"}\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"11:\"$c:metadata\"\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"22:I[61172,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"PageEditorProvider\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"23:I[63493,[\"8739\",\"static/chunks/63f0ec43-8ee6a76d70472249.js\",\"2254\",\"static/chunks/8208b75a-a48e5a4507a0de91.js\",\"4684\",\"static/chunks/24cf1b50-159cca90b09285e0.js\",\"1829\",\"static/chunks/9ffa21ba-c069766d809bd4c7.js\",\"7670\",\"static/chunks/7670-7326298c9856172b.js\",\"282\",\"static/chunks/282-691fffb366e24ca5.js\",\"7515\",\"static/chunks/7515-336759ef5dad2b52.js\",\"5497\",\"static/chunks/5497-ab85ef3ea59dd353.js\",\"7720\",\"static/chunks/7720-3a1e7e411adba2d3.js\",\"6281\",\"static/chunks/6281-c6e84786dade3614.js\",\"7086\",\"static/chunks/7086-1f1c4891d766e04f.js\",\"600\",\"static/chunks/600-9a9fcafaaa6c1c4c.js\",\"5002\",\"static/chunks/5002-9735c25beda556ba.js\",\"5448\",\"static/chunks/5448-2a3cfd853d899c9a.js\",\"5855\",\"static/chunks/5855-243eeca8f0e4cabd.js\",\"4393\",\"static/chunks/4393-1c8dff25ec904469.js\",\"9214\",\"static/chunks/9214-a10614844a67ba31.js\",\"8\",\"static/chunks/8-8ca70ca619d8e099.js\",\"4789\",\"static/chunks/4789-2080f6497f78b5b6.js\",\"6304\",\"static/chunks/app/page/%5Bslug%5D/page-540a27b6839afde5.js\"],\"default\"]\n"])</script><script nonce="MWJlOGMzYjMtODk3YS00ZGViLThhNGUtNDBhMDMyZmZlNzM5">self.__next_f.push([1,"20:[\"$\",\"$L22\",null,{\"children\":[\"$\",\"$L23\",null,{\"slug\":\"Quantitative_trait_locus\"}]}]\n"])</script></body></html>